-- MySQL dump 10.13  Distrib 8.0.33, for Win64 (x86_64)
--
-- Host: localhost    Database: dev
-- ------------------------------------------------------
-- Server version	8.0.36

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!50503 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `idc_collections_attribute`
--

DROP TABLE IF EXISTS `idc_collections_attribute`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_attribute` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(64) NOT NULL,
  `display_name` varchar(100) NOT NULL,
  `description` longtext,
  `data_type` varchar(1) NOT NULL,
  `active` tinyint(1) NOT NULL,
  `is_cross_collex` tinyint(1) NOT NULL,
  `preformatted_values` tinyint(1) NOT NULL,
  `units` varchar(256) DEFAULT NULL,
  `default_ui_display` tinyint(1) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=180 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_attribute`
--

LOCK TABLES `idc_collections_attribute` WRITE;
/*!40000 ALTER TABLE `idc_collections_attribute` DISABLE KEYS */;
INSERT INTO `idc_collections_attribute` VALUES (1,'program_name','Program',NULL,'C',1,1,0,NULL,0),(2,'case_barcode','Case Barcode',NULL,'S',1,1,0,NULL,0),(3,'case_gdc_id','Case GDC ID',NULL,'S',1,1,0,NULL,0),(4,'program_dbgap_accession_number','Program Dbgap Accession Number',NULL,'C',1,1,0,NULL,0),(5,'project_short_name','Project Short Name',NULL,'C',1,1,1,NULL,1),(6,'project_name','Project Name',NULL,'C',1,1,0,NULL,1),(7,'disease_code','Disease Code',NULL,'C',1,1,1,NULL,1),(8,'gender','Gender',NULL,'C',1,1,0,NULL,1),(9,'vital_status','Vital Status',NULL,'C',1,1,0,NULL,1),(10,'race','Race',NULL,'C',1,1,0,NULL,1),(11,'ethnicity','Ethnicity',NULL,'C',1,1,0,NULL,1),(12,'age_at_diagnosis','Age At Diagnosis',NULL,'N',1,1,0,NULL,1),(13,'days_to_birth','Days To Birth',NULL,'N',1,1,0,NULL,0),(14,'days_to_death','Days To Death',NULL,'N',1,1,0,NULL,0),(15,'days_to_initial_pathologic_diagnosis','Days To Initial Pathologic Diagnosis',NULL,'N',1,1,0,NULL,0),(16,'days_to_last_followup','Days To Last Followup',NULL,'N',1,1,0,NULL,0),(17,'days_to_last_known_alive','Days To Last Known Alive',NULL,'N',1,1,0,NULL,0),(18,'days_to_submitted_specimen_dx','Days To Submitted Specimen Dx',NULL,'N',1,1,0,NULL,0),(19,'clinical_stage','Clinical Stage',NULL,'C',1,1,0,NULL,0),(20,'clinical_T','Clinical T',NULL,'S',1,1,0,NULL,0),(21,'clinical_N','Clinical N',NULL,'S',1,1,0,NULL,0),(22,'clinical_M','Clinical M',NULL,'S',1,1,0,NULL,0),(23,'pathologic_stage','Pathologic Stage',NULL,'C',1,1,1,NULL,1),(24,'pathologic_T','Pathologic T',NULL,'C',1,1,0,NULL,0),(25,'pathologic_N','Pathologic N',NULL,'C',1,1,0,NULL,0),(26,'pathologic_M','Pathologic M',NULL,'C',1,1,0,NULL,0),(27,'year_of_initial_pathologic_diagnosis','Year Of Initial Pathologic Diagnosis',NULL,'N',1,1,0,NULL,0),(28,'tumor_tissue_site','Tumor Tissue Site',NULL,'C',1,1,0,NULL,1),(29,'primary_neoplasm_melanoma_dx','Primary Neoplasm Melanoma Dx',NULL,'S',1,1,0,NULL,0),(30,'anatomic_neoplasm_subdivision','Anatomic Neoplasm Subdivision',NULL,'C',1,1,0,NULL,0),(31,'country','Country',NULL,'C',1,1,0,NULL,1),(32,'other_dx','Other Dx',NULL,'S',1,1,0,NULL,0),(33,'other_malignancy_anatomic_site','Other Malignancy Anatomic Site',NULL,'C',1,1,0,NULL,0),(34,'other_malignancy_type','Other Malignancy Type',NULL,'C',1,1,0,NULL,0),(35,'other_malignancy_histological_type','Other Malignancy Histological Type',NULL,'C',1,1,0,NULL,0),(36,'history_of_neoadjuvant_treatment','History Of Neoadjuvant Treatment',NULL,'C',1,1,0,NULL,0),(37,'primary_therapy_outcome_success','Primary Therapy Outcome Success',NULL,'C',1,1,0,NULL,0),(38,'histological_type','Histological Type',NULL,'C',1,1,0,NULL,1),(39,'neoplasm_histologic_grade','Neoplasm Histologic Grade',NULL,'C',1,1,0,NULL,0),(40,'icd_10','ICD-10',NULL,'C',1,1,0,NULL,0),(41,'icd_o_3_site','ICD-O-3 Site',NULL,'C',1,1,0,NULL,0),(42,'icd_o_3_histology','ICD-O-3 Histology',NULL,'C',1,1,0,NULL,0),(43,'person_neoplasm_cancer_status','Person Neoplasm Cancer Status',NULL,'C',1,1,0,NULL,0),(44,'residual_tumor','Residual Tumor',NULL,'C',1,1,0,NULL,0),(45,'tumor_type','Tumor Type',NULL,'C',1,1,0,NULL,0),(46,'new_tumor_event_after_initial_treatment','New Tumor Event After Initial Treatment',NULL,'C',1,1,0,NULL,0),(47,'lymphnodes_examined','Lymphnodes Examined',NULL,'C',1,1,0,NULL,0),(48,'number_of_lymphnodes_examined','Number Of Lymphnodes Examined',NULL,'N',1,1,0,NULL,0),(49,'number_of_lymphnodes_positive_by_he','Number Of Lymphnodes Positive By He',NULL,'N',1,1,0,NULL,0),(50,'lymphatic_invasion','Lymphatic Invasion',NULL,'C',1,1,0,NULL,0),(51,'venous_invasion','Venous Invasion',NULL,'C',1,1,0,NULL,0),(52,'lymphovascular_invasion_present','Lymphovascular Invasion Present',NULL,'C',1,1,0,NULL,0),(53,'bcr','BCR',NULL,'C',1,1,0,NULL,0),(54,'batch_number','Batch Number',NULL,'N',1,1,0,NULL,0),(55,'tss_code','TSS Code',NULL,'C',1,1,0,NULL,0),(56,'age_began_smoking_in_years','Age Began Smoking In Years',NULL,'N',1,1,0,NULL,0),(57,'year_of_tobacco_smoking_onset','Year Of Tobacco Smoking Onset',NULL,'N',1,1,0,NULL,0),(58,'stopped_smoking_year','Stopped Smoking Year',NULL,'N',1,1,0,NULL,0),(59,'tobacco_smoking_history','Tobacco Smoking History',NULL,'N',1,1,0,NULL,0),(60,'number_pack_years_smoked','Number Pack Years Smoked',NULL,'N',1,1,0,NULL,0),(61,'height','Height',NULL,'N',1,1,0,NULL,0),(62,'weight','Weight',NULL,'N',1,1,0,NULL,0),(63,'bmi','BMI',NULL,'N',1,1,0,NULL,1),(64,'mononucleotide_and_dinucleotide_marker_panel_analysis_status','Mononucleotide And Dinucleotide Marker Panel Analysis Status',NULL,'C',1,1,0,NULL,0),(65,'menopause_status','Menopause Status',NULL,'C',1,1,0,NULL,0),(66,'pregnancies','pregnancies',NULL,'C',1,1,0,NULL,0),(67,'hpv_status','HPV Status',NULL,'C',1,1,0,NULL,0),(68,'hpv_calls','HPV Calls',NULL,'S',1,1,0,NULL,0),(69,'h_pylori_infection','H. pylori infection',NULL,'C',1,1,0,NULL,0),(70,'gleason_score_combined','Gleason Score Combined',NULL,'N',1,1,0,NULL,0),(71,'psa_value','Psa Value',NULL,'N',1,1,0,NULL,0),(72,'colorectal_cancer','Colorectal Cancer',NULL,'C',1,1,0,NULL,0),(73,'history_of_colon_polyps','History Of Colon Polyps',NULL,'C',1,1,0,NULL,0),(74,'sample_barcode','Sample Barcode',NULL,'S',1,1,0,NULL,0),(75,'sample_gdc_id','Sample GDC ID',NULL,'S',1,1,0,NULL,0),(76,'sample_type','Sample Type',NULL,'C',1,1,0,NULL,1),(77,'sample_type_name','Sample Type Name',NULL,'C',1,1,0,NULL,0),(78,'days_to_collection','Days To Collection',NULL,'N',1,1,0,NULL,0),(79,'days_to_sample_procurement','Days To Sample Procurement',NULL,'N',1,1,0,NULL,0),(80,'is_ffpe','is FFPE',NULL,'C',1,1,0,NULL,0),(81,'num_portions','Num Portions',NULL,'N',1,1,0,NULL,0),(82,'num_slides','Num Slides',NULL,'N',1,1,0,NULL,0),(83,'avg_percent_lymphocyte_infiltration','Avg Percent Lymphocyte Infiltration',NULL,'N',1,1,0,NULL,0),(84,'avg_percent_monocyte_infiltration','Avg Percent Monocyte Infiltration',NULL,'N',1,1,0,NULL,0),(85,'avg_percent_necrosis','Avg Percent Necrosis',NULL,'N',1,1,0,NULL,0),(86,'avg_percent_neutrophil_infiltration','Avg Percent Neutrophil Infiltration',NULL,'N',1,1,0,NULL,0),(87,'avg_percent_normal_cells','Avg Percent Normal Cells',NULL,'N',1,1,0,NULL,0),(88,'avg_percent_stromal_cells','Avg Percent Stromal Cells',NULL,'N',1,1,0,NULL,0),(89,'avg_percent_tumor_cells','Avg Percent Tumor Cells',NULL,'N',1,1,0,NULL,0),(90,'avg_percent_tumor_nuclei','Avg Percent Tumor Nuclei',NULL,'N',1,1,0,NULL,0),(91,'max_percent_lymphocyte_infiltration','Max Percent Lymphocyte Infiltration',NULL,'N',1,1,0,NULL,0),(92,'max_percent_monocyte_infiltration','Max Percent Monocyte Infiltration',NULL,'N',1,1,0,NULL,0),(93,'max_percent_necrosis','Max Percent Necrosis',NULL,'N',1,1,0,NULL,0),(94,'max_percent_neutrophil_infiltration','Max Percent Neutrophil Infiltration',NULL,'N',1,1,0,NULL,0),(95,'max_percent_normal_cells','Max Percent Normal Cells',NULL,'N',1,1,0,NULL,0),(96,'max_percent_stromal_cells','Max Percent Stromal Cells',NULL,'N',1,1,0,NULL,0),(97,'max_percent_tumor_cells','Max Percent Tumor Cells',NULL,'N',1,1,0,NULL,0),(98,'max_percent_tumor_nuclei','Max Percent Tumor Nuclei',NULL,'N',1,1,0,NULL,0),(99,'min_percent_lymphocyte_infiltration','Min Percent Lymphocyte Infiltration',NULL,'N',1,1,0,NULL,0),(100,'min_percent_monocyte_infiltration','Min Percent Monocyte Infiltration',NULL,'N',1,1,0,NULL,0),(101,'min_percent_necrosis','Min Percent Necrosis',NULL,'N',1,1,0,NULL,0),(102,'min_percent_neutrophil_infiltration','Min Percent Neutrophil Infiltration',NULL,'N',1,1,0,NULL,0),(103,'min_percent_normal_cells','Min Percent Normal Cells',NULL,'N',1,1,0,NULL,0),(104,'min_percent_stromal_cells','Min Percent Stromal Cells',NULL,'N',1,1,0,NULL,0),(105,'min_percent_tumor_cells','Min Percent Tumor Cells',NULL,'N',1,1,0,NULL,0),(106,'min_percent_tumor_nuclei','Min Percent Tumor Nuclei',NULL,'N',1,1,0,NULL,0),(107,'Modality','Modality',NULL,'C',1,1,0,NULL,1),(108,'BodyPartExamined','Body Part Examined',NULL,'C',1,1,1,NULL,1),(109,'StudyDescription','Study Description',NULL,'S',1,1,0,NULL,1),(110,'StudyInstanceUID','Study Instance UID',NULL,'S',1,1,0,NULL,1),(111,'PatientID','PatientID',NULL,'S',1,1,0,NULL,1),(112,'Program','Program',NULL,'C',1,1,0,NULL,0),(113,'SeriesInstanceUID','Series Instance UID',NULL,'S',1,1,0,NULL,1),(114,'SOPInstanceUID','SOP Instance UID',NULL,'S',1,1,0,NULL,1),(115,'SeriesDescription','Series Description',NULL,'S',1,1,0,NULL,1),(116,'SliceThickness','Slice Thickness',NULL,'N',1,1,0,'millimeter',1),(117,'SeriesNumber','Series Number',NULL,'S',1,1,1,NULL,1),(118,'StudyDate','Study Date',NULL,'S',1,1,0,NULL,1),(119,'SOPClassUID','Object Class',NULL,'C',1,1,0,NULL,1),(120,'collection_id','Collection',NULL,'C',1,1,0,NULL,1),(121,'gcs_url','GCS Path',NULL,'S',1,1,1,NULL,0),(122,'source_DOI','Source DOI',NULL,'S',1,1,0,NULL,0),(123,'gcs_bucket','GCS Bucket',NULL,'S',1,1,0,NULL,0),(124,'crdc_study_uuid','GDC StudyInstanceUID',NULL,'S',1,1,0,NULL,0),(125,'crdc_series_uuid','GDC SeriesInstanceUID',NULL,'S',1,1,0,NULL,0),(126,'crdc_instance_uuid','GDC SOPInstanceUID',NULL,'S',1,1,0,NULL,0),(127,'gcs_generation','GCS Generation',NULL,'S',0,1,0,NULL,0),(128,'tcia_tumorLocation','Primary Site Location',NULL,'C',1,1,0,NULL,1),(129,'AnatomicRegionSequence','Anatomic Region',NULL,'C',1,1,0,NULL,1),(130,'SegmentedPropertyCategoryCodeSequence','Segmentation Category',NULL,'C',1,1,0,NULL,1),(131,'SegmentedPropertyTypeCodeSequence','Segmentation Type',NULL,'C',1,1,0,NULL,1),(132,'FrameOfReferenceUID','FrameOfReferenceUID',NULL,'S',1,1,0,NULL,0),(133,'SegmentNumber','SegmentNumber',NULL,'N',1,1,0,NULL,0),(134,'SegmentAlgorithmType','Segment Algorithm Type',NULL,'C',1,1,1,NULL,1),(135,'SUVbw','SUVbw (Mean)',NULL,'N',0,1,0,'Standardized Uptake Value body weight',1),(136,'Volume','Volume',NULL,'N',1,1,0,'cubic millimeter',1),(137,'Diameter','Diameter',NULL,'N',1,1,0,'millimeter',1),(138,'Surface_area_of_mesh','Surface area of mesh',NULL,'N',1,1,0,'square millimeter',1),(139,'Total_Lesion_Glycolysis','Total Lesion Glycolysis',NULL,'N',0,1,0,'Gram',1),(140,'Standardized_Added_Metabolic_Activity','Standardized Added Metabolic Activity',NULL,'N',0,1,0,'Gram',1),(141,'Percent_Within_First_Quarter_of_Intensity_Range','Percent Within First Quarter of Intensity Range',NULL,'N',0,1,0,'Percent',1),(142,'Percent_Within_Third_Quarter_of_Intensity_Range','Percent Within Third Quarter of Intensity Range',NULL,'N',0,1,0,'Percent',1),(143,'Percent_Within_Fourth_Quarter_of_Intensity_Range','Percent Within Fourth Quarter of Intensity Range',NULL,'N',0,1,0,'Percent',1),(144,'Percent_Within_Second_Quarter_of_Intensity_Range','Percent Within Second Quarter of Intensity Range',NULL,'N',0,1,0,'Percent',1),(145,'Standardized_Added_Metabolic_Activity_Background','Standardized Added Metabolic Activity Background',NULL,'N',0,1,0,'Standardized Uptake Value body weight',1),(146,'Glycolysis_Within_First_Quarter_of_Intensity_Range','Glycolysis Within First Quarter of Intensity Range',NULL,'N',0,1,0,'Gram',1),(147,'Glycolysis_Within_Third_Quarter_of_Intensity_Range','Glycolysis Within Third Quarter of Intensity Range',NULL,'N',0,1,0,'Gram',1),(148,'Glycolysis_Within_Fourth_Quarter_of_Intensity_Range','Glycolysis Within Fourth Quarter of Intensity Range',NULL,'N',0,1,0,'Gram',1),(149,'Glycolysis_Within_Second_Quarter_of_Intensity_Range','Glycolysis Within Second Quarter of Intensity Range',NULL,'N',0,1,0,'Gram',1),(150,'Internal_structure','Internal structure',NULL,'C',1,1,0,NULL,1),(151,'Sphericity','Sphericity',NULL,'C',1,1,0,NULL,1),(152,'Calcification','Calcification',NULL,'C',1,1,0,NULL,1),(153,'Lobular_Pattern','Lobular Pattern',NULL,'C',1,1,0,NULL,1),(154,'Spiculation','Spiculation',NULL,'C',1,1,0,NULL,1),(155,'Margin','Margin',NULL,'C',1,1,0,NULL,1),(156,'Texture','Texture',NULL,'C',1,1,0,NULL,1),(157,'Subtlety_score','Subtlety score',NULL,'C',1,1,0,NULL,1),(158,'Malignancy','Malignancy',NULL,'C',1,1,0,NULL,1),(159,'Apparent_Diffusion_Coefficient','Apparent Diffusion Coefficient',NULL,'N',1,1,0,'um2/s',1),(160,'tcia_species','Species',NULL,'C',1,1,0,NULL,1),(161,'Manufacturer','Manufacturer',NULL,'C',1,1,1,NULL,1),(162,'ManufacturerModelName','Manufacturer Model Name',NULL,'C',1,1,1,NULL,1),(163,'license_short_name','License',NULL,'C',1,1,1,NULL,1),(164,'access','Access',NULL,'C',0,1,0,NULL,1),(165,'analysis_results_id','Analysis Results',NULL,'C',1,1,1,NULL,1),(166,'SamplesPerPixel','Samples Per Pixel',NULL,'C',1,1,1,NULL,1),(167,'Volume_of_Mesh','Volume of Mesh',NULL,'N',1,1,0,NULL,1),(168,'Sphericity_quant','Sphericity (Quant)',NULL,'N',1,1,0,NULL,1),(169,'illuminationType','Illumination Type',NULL,'C',1,1,0,NULL,1),(170,'primaryAnatomicStructure','Primary Anatomic Structure',NULL,'C',1,1,0,NULL,1),(171,'ObjectiveLensPower','Objective Lens Power',NULL,'M',1,1,0,NULL,1),(172,'min_PixelSpacing','Min Pixel Spacing',NULL,'N',1,1,0,NULL,1),(173,'max_TotalPixelMatrixColumns','Max Total Pixel Matrix Columns',NULL,'N',1,1,0,NULL,1),(174,'max_TotalPixelMatrixRows','Max Total Pixel Matrix Rows',NULL,'N',1,1,0,NULL,1),(175,'aws_url','AWS Storage Path',NULL,'S',1,1,1,NULL,0),(176,'instance_size','Total Instance Size',NULL,'N',1,1,0,NULL,0),(177,'CancerType','Cancer Type',NULL,'C',1,1,1,NULL,1),(178,'aws_bucket','AWS Bucket',NULL,'S',1,1,0,NULL,0),(179,'SegmentAlgorithmName','Algorithm Name',NULL,'C',1,1,1,NULL,1);
/*!40000 ALTER TABLE `idc_collections_attribute` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_attribute_data_sources`
--

DROP TABLE IF EXISTS `idc_collections_attribute_data_sources`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_attribute_data_sources` (
  `id` int NOT NULL AUTO_INCREMENT,
  `attribute_id` int NOT NULL,
  `datasource_id` int NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `idc_collections_attribut_attribute_id_datasource__ca23f352_uniq` (`attribute_id`,`datasource_id`),
  KEY `idc_collections_attr_datasource_id_d0010b6f_fk_idc_colle` (`datasource_id`),
  CONSTRAINT `idc_collections_attr_attribute_id_e98baf11_fk_idc_colle` FOREIGN KEY (`attribute_id`) REFERENCES `idc_collections_attribute` (`id`),
  CONSTRAINT `idc_collections_attr_datasource_id_d0010b6f_fk_idc_colle` FOREIGN KEY (`datasource_id`) REFERENCES `idc_collections_datasource` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=3299 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_attribute_data_sources`
--

LOCK TABLES `idc_collections_attribute_data_sources` WRITE;
/*!40000 ALTER TABLE `idc_collections_attribute_data_sources` DISABLE KEYS */;
INSERT INTO `idc_collections_attribute_data_sources` VALUES (2,1,2),(1,1,3),(3,1,5),(4,1,6),(6,2,2),(5,2,3),(7,2,5),(8,2,6),(10,3,2),(9,3,3),(11,3,5),(12,3,6),(13,4,2),(14,4,6),(16,5,2),(15,5,3),(17,5,5),(18,5,6),(19,6,2),(20,6,6),(21,7,2),(22,7,6),(23,8,2),(24,8,6),(25,9,2),(26,9,6),(27,10,2),(28,10,6),(29,11,2),(30,11,6),(31,12,2),(32,12,6),(33,13,2),(34,13,6),(35,14,2),(36,14,6),(37,15,2),(38,15,6),(39,16,2),(40,16,6),(41,17,2),(42,17,6),(43,18,2),(44,18,6),(45,19,2),(46,19,6),(47,20,2),(48,20,6),(49,21,2),(50,21,6),(51,22,2),(52,22,6),(53,23,2),(54,23,6),(55,24,2),(56,24,6),(57,25,2),(58,25,6),(59,26,2),(60,26,6),(61,27,2),(62,27,6),(63,28,2),(64,28,6),(65,29,2),(66,29,6),(67,30,2),(68,30,6),(69,31,2),(70,31,6),(71,32,2),(72,32,6),(73,33,2),(74,33,6),(75,34,2),(76,34,6),(77,35,2),(78,35,6),(79,36,2),(80,36,6),(81,37,2),(82,37,6),(83,38,2),(84,38,6),(85,39,2),(86,39,6),(87,40,2),(88,40,6),(89,41,2),(90,41,6),(91,42,2),(92,42,6),(93,43,2),(94,43,6),(95,44,2),(96,44,6),(97,45,2),(98,45,6),(99,46,2),(100,46,6),(101,47,2),(102,47,6),(103,48,2),(104,48,6),(105,49,2),(106,49,6),(107,50,2),(108,50,6),(109,51,2),(110,51,6),(111,52,2),(112,52,6),(114,53,2),(113,53,3),(115,53,5),(116,53,6),(118,54,2),(117,54,3),(119,54,5),(120,54,6),(121,55,2),(122,55,6),(123,56,2),(124,56,6),(125,57,2),(126,57,6),(127,58,2),(128,58,6),(129,59,2),(130,59,6),(131,60,2),(132,60,6),(133,61,2),(134,61,6),(135,62,2),(136,62,6),(137,63,2),(138,63,6),(139,64,2),(140,64,6),(141,65,2),(142,65,6),(143,66,2),(144,66,6),(145,67,2),(146,67,6),(147,68,2),(148,68,6),(149,69,2),(150,69,6),(151,70,2),(152,70,6),(153,71,2),(154,71,6),(155,72,2),(156,72,6),(157,73,2),(158,73,6),(159,74,3),(160,74,5),(161,75,3),(162,75,5),(163,76,3),(164,76,5),(165,77,3),(166,77,5),(167,78,3),(168,78,5),(169,79,3),(170,79,5),(171,80,3),(172,80,5),(173,81,3),(174,81,5),(175,82,3),(176,82,5),(177,83,3),(178,83,5),(179,84,3),(180,84,5),(181,85,3),(182,85,5),(183,86,3),(184,86,5),(185,87,3),(186,87,5),(187,88,3),(188,88,5),(189,89,3),(190,89,5),(191,90,3),(192,90,5),(193,91,3),(194,91,5),(195,92,3),(196,92,5),(197,93,3),(198,93,5),(199,94,3),(200,94,5),(201,95,3),(202,95,5),(203,96,3),(204,96,5),(205,97,3),(206,97,5),(207,98,3),(208,98,5),(209,99,3),(210,99,5),(211,100,3),(212,100,5),(213,101,3),(214,101,5),(215,102,3),(216,102,5),(217,103,3),(218,103,5),(219,104,3),(220,104,5),(221,105,3),(222,105,5),(223,106,3),(224,106,5),(225,107,1),(226,107,4),(358,107,10),(468,107,11),(469,107,12),(416,107,13),(628,107,14),(629,107,15),(572,107,16),(788,107,17),(789,107,18),(732,107,19),(960,107,20),(961,107,21),(901,107,22),(1133,107,23),(1134,107,24),(1073,107,25),(1305,107,26),(1306,107,27),(1245,107,28),(1417,107,29),(1474,107,30),(1531,107,31),(1589,107,32),(1646,107,33),(1703,107,34),(1764,107,35),(1822,107,36),(1880,107,37),(1942,107,38),(2001,107,39),(2060,107,40),(2120,107,41),(2167,107,42),(2214,107,43),(2262,107,44),(2309,107,45),(2356,107,46),(2428,107,47),(2483,107,48),(2538,107,49),(2597,107,50),(2652,107,51),(2707,107,52),(2770,107,53),(2827,107,54),(2884,107,55),(2946,107,56),(3004,107,57),(3062,107,58),(3122,107,59),(3180,107,60),(3238,107,61),(227,108,1),(228,108,4),(389,108,10),(470,108,11),(471,108,12),(447,108,13),(630,108,14),(631,108,15),(603,108,16),(790,108,17),(791,108,18),(763,108,19),(962,108,20),(963,108,21),(932,108,22),(1135,108,23),(1136,108,24),(1104,108,25),(1307,108,26),(1308,108,27),(1276,108,28),(1418,108,29),(1475,108,30),(1562,108,31),(1590,108,32),(1647,108,33),(1734,108,34),(1765,108,35),(1823,108,36),(1911,108,37),(1943,108,38),(2002,108,39),(2091,108,40),(2121,108,41),(2168,108,42),(2233,108,43),(2263,108,44),(2310,108,45),(2375,108,46),(2429,108,47),(2484,108,48),(2557,108,49),(2598,108,50),(2653,108,51),(2726,108,52),(2771,108,53),(2828,108,54),(2903,108,55),(2947,108,56),(3005,108,57),(3081,108,58),(3123,108,59),(3181,108,60),(3257,108,61),(229,109,1),(230,109,4),(394,109,10),(472,109,11),(473,109,12),(452,109,13),(632,109,14),(633,109,15),(608,109,16),(792,109,17),(793,109,18),(768,109,19),(964,109,20),(965,109,21),(937,109,22),(1137,109,23),(1138,109,24),(1109,109,25),(1309,109,26),(1310,109,27),(1281,109,28),(1419,109,29),(1476,109,30),(1567,109,31),(1591,109,32),(1648,109,33),(1739,109,34),(1766,109,35),(1824,109,36),(1916,109,37),(1944,109,38),(2003,109,39),(2096,109,40),(2122,109,41),(2169,109,42),(2238,109,43),(2264,109,44),(2311,109,45),(2380,109,46),(2430,109,47),(2485,109,48),(2562,109,49),(2599,109,50),(2654,109,51),(2731,109,52),(2772,109,53),(2829,109,54),(2908,109,55),(2948,109,56),(3006,109,57),(3086,109,58),(3124,109,59),(3182,109,60),(3262,109,61),(231,110,1),(232,110,4),(395,110,10),(474,110,11),(475,110,12),(453,110,13),(634,110,14),(635,110,15),(609,110,16),(794,110,17),(795,110,18),(769,110,19),(966,110,20),(967,110,21),(938,110,22),(1139,110,23),(1140,110,24),(1110,110,25),(1311,110,26),(1312,110,27),(1282,110,28),(1420,110,29),(1477,110,30),(1568,110,31),(1592,110,32),(1649,110,33),(1740,110,34),(1767,110,35),(1825,110,36),(1917,110,37),(1945,110,38),(2004,110,39),(2097,110,40),(2123,110,41),(2170,110,42),(2239,110,43),(2265,110,44),(2312,110,45),(2381,110,46),(2431,110,47),(2486,110,48),(2563,110,49),(2600,110,50),(2655,110,51),(2732,110,52),(2773,110,53),(2830,110,54),(2909,110,55),(2949,110,56),(3007,110,57),(3087,110,58),(3125,110,59),(3183,110,60),(3263,110,61),(233,111,1),(234,111,4),(396,111,10),(476,111,11),(477,111,12),(454,111,13),(636,111,14),(637,111,15),(610,111,16),(796,111,17),(797,111,18),(770,111,19),(968,111,20),(969,111,21),(939,111,22),(1141,111,23),(1142,111,24),(1111,111,25),(1313,111,26),(1314,111,27),(1283,111,28),(1421,111,29),(1478,111,30),(1569,111,31),(1593,111,32),(1650,111,33),(1741,111,34),(1768,111,35),(1826,111,36),(1918,111,37),(1946,111,38),(2005,111,39),(2098,111,40),(2124,111,41),(2171,111,42),(2240,111,43),(2266,111,44),(2313,111,45),(2382,111,46),(2432,111,47),(2487,111,48),(2564,111,49),(2601,111,50),(2656,111,51),(2733,111,52),(2774,111,53),(2831,111,54),(2910,111,55),(2950,111,56),(3008,111,57),(3088,111,58),(3126,111,59),(3184,111,60),(3264,111,61),(235,112,1),(236,112,4),(390,112,10),(478,112,11),(479,112,12),(448,112,13),(638,112,14),(639,112,15),(604,112,16),(798,112,17),(799,112,18),(764,112,19),(970,112,20),(971,112,21),(933,112,22),(1143,112,23),(1144,112,24),(1105,112,25),(1315,112,26),(1316,112,27),(1277,112,28),(1422,112,29),(1479,112,30),(1563,112,31),(1594,112,32),(1651,112,33),(1735,112,34),(1769,112,35),(1827,112,36),(1912,112,37),(1947,112,38),(2006,112,39),(2092,112,40),(2125,112,41),(2172,112,42),(2234,112,43),(2267,112,44),(2314,112,45),(2376,112,46),(2433,112,47),(2488,112,48),(2558,112,49),(2602,112,50),(2657,112,51),(2727,112,52),(2775,112,53),(2832,112,54),(2904,112,55),(2951,112,56),(3009,112,57),(3082,112,58),(3127,112,59),(3185,112,60),(3258,112,61),(237,113,1),(238,113,4),(397,113,10),(480,113,11),(481,113,12),(455,113,13),(640,113,14),(641,113,15),(611,113,16),(800,113,17),(801,113,18),(771,113,19),(972,113,20),(973,113,21),(940,113,22),(1145,113,23),(1146,113,24),(1112,113,25),(1317,113,26),(1318,113,27),(1284,113,28),(1423,113,29),(1480,113,30),(1570,113,31),(1595,113,32),(1652,113,33),(1742,113,34),(1770,113,35),(1828,113,36),(1919,113,37),(1948,113,38),(2007,113,39),(2099,113,40),(2126,113,41),(2173,113,42),(2241,113,43),(2268,113,44),(2315,113,45),(2383,113,46),(2434,113,47),(2489,113,48),(2565,113,49),(2603,113,50),(2658,113,51),(2734,113,52),(2776,113,53),(2833,113,54),(2911,113,55),(2952,113,56),(3010,113,57),(3089,113,58),(3128,113,59),(3186,113,60),(3265,113,61),(239,114,1),(240,114,4),(398,114,10),(482,114,11),(483,114,12),(456,114,13),(642,114,14),(643,114,15),(612,114,16),(802,114,17),(803,114,18),(772,114,19),(974,114,20),(975,114,21),(941,114,22),(1147,114,23),(1148,114,24),(1113,114,25),(1319,114,26),(1320,114,27),(1285,114,28),(1424,114,29),(1481,114,30),(1571,114,31),(1596,114,32),(1653,114,33),(1743,114,34),(1771,114,35),(1829,114,36),(1920,114,37),(1949,114,38),(2008,114,39),(2100,114,40),(2127,114,41),(2174,114,42),(2242,114,43),(2269,114,44),(2316,114,45),(2384,114,46),(2435,114,47),(2490,114,48),(2566,114,49),(2604,114,50),(2659,114,51),(2735,114,52),(2777,114,53),(2834,114,54),(2912,114,55),(2953,114,56),(3011,114,57),(3090,114,58),(3129,114,59),(3187,114,60),(3266,114,61),(241,115,1),(242,115,4),(399,115,10),(484,115,11),(485,115,12),(457,115,13),(644,115,14),(645,115,15),(613,115,16),(804,115,17),(805,115,18),(773,115,19),(976,115,20),(977,115,21),(942,115,22),(1149,115,23),(1150,115,24),(1114,115,25),(1321,115,26),(1322,115,27),(1286,115,28),(1425,115,29),(1482,115,30),(1572,115,31),(1597,115,32),(1654,115,33),(1744,115,34),(1772,115,35),(1830,115,36),(1921,115,37),(1950,115,38),(2009,115,39),(2101,115,40),(2128,115,41),(2175,115,42),(2243,115,43),(2270,115,44),(2317,115,45),(2385,115,46),(2436,115,47),(2491,115,48),(2567,115,49),(2605,115,50),(2660,115,51),(2736,115,52),(2778,115,53),(2835,115,54),(2913,115,55),(2954,115,56),(3012,115,57),(3091,115,58),(3130,115,59),(3188,115,60),(3267,115,61),(243,116,1),(244,116,4),(400,116,10),(486,116,11),(487,116,12),(458,116,13),(646,116,14),(647,116,15),(614,116,16),(806,116,17),(807,116,18),(774,116,19),(978,116,20),(979,116,21),(943,116,22),(1151,116,23),(1152,116,24),(1115,116,25),(1323,116,26),(1324,116,27),(1287,116,28),(1426,116,29),(1483,116,30),(1573,116,31),(1598,116,32),(1655,116,33),(1745,116,34),(1773,116,35),(1831,116,36),(1922,116,37),(1951,116,38),(2010,116,39),(2102,116,40),(2129,116,41),(2176,116,42),(2244,116,43),(2271,116,44),(2318,116,45),(2386,116,46),(2437,116,47),(2492,116,48),(2568,116,49),(2606,116,50),(2661,116,51),(2737,116,52),(2779,116,53),(2836,116,54),(2914,116,55),(2955,116,56),(3013,116,57),(3092,116,58),(3131,116,59),(3189,116,60),(3268,116,61),(245,117,1),(246,117,4),(401,117,10),(488,117,11),(489,117,12),(459,117,13),(648,117,14),(649,117,15),(615,117,16),(808,117,17),(809,117,18),(775,117,19),(980,117,20),(981,117,21),(944,117,22),(1153,117,23),(1154,117,24),(1116,117,25),(1325,117,26),(1326,117,27),(1288,117,28),(1427,117,29),(1484,117,30),(1574,117,31),(1599,117,32),(1656,117,33),(1746,117,34),(1774,117,35),(1832,117,36),(1923,117,37),(1952,117,38),(2011,117,39),(2103,117,40),(2130,117,41),(2177,117,42),(2245,117,43),(2272,117,44),(2319,117,45),(2387,117,46),(2438,117,47),(2493,117,48),(2569,117,49),(2607,117,50),(2662,117,51),(2738,117,52),(2780,117,53),(2837,117,54),(2915,117,55),(2956,117,56),(3014,117,57),(3093,117,58),(3132,117,59),(3190,117,60),(3269,117,61),(247,118,1),(248,118,4),(402,118,10),(490,118,11),(491,118,12),(460,118,13),(650,118,14),(651,118,15),(616,118,16),(810,118,17),(811,118,18),(776,118,19),(982,118,20),(983,118,21),(945,118,22),(1155,118,23),(1156,118,24),(1117,118,25),(1327,118,26),(1328,118,27),(1289,118,28),(1428,118,29),(1485,118,30),(1575,118,31),(1600,118,32),(1657,118,33),(1747,118,34),(1775,118,35),(1833,118,36),(1924,118,37),(1953,118,38),(2012,118,39),(2104,118,40),(2131,118,41),(2178,118,42),(2246,118,43),(2273,118,44),(2320,118,45),(2388,118,46),(2439,118,47),(2494,118,48),(2570,118,49),(2608,118,50),(2663,118,51),(2739,118,52),(2781,118,53),(2838,118,54),(2916,118,55),(2957,118,56),(3015,118,57),(3094,118,58),(3133,118,59),(3191,118,60),(3270,118,61),(249,119,1),(250,119,4),(391,119,10),(492,119,11),(493,119,12),(449,119,13),(652,119,14),(653,119,15),(605,119,16),(812,119,17),(813,119,18),(765,119,19),(984,119,20),(985,119,21),(934,119,22),(1157,119,23),(1158,119,24),(1106,119,25),(1329,119,26),(1330,119,27),(1278,119,28),(1429,119,29),(1486,119,30),(1564,119,31),(1601,119,32),(1658,119,33),(1736,119,34),(1776,119,35),(1834,119,36),(1913,119,37),(1954,119,38),(2013,119,39),(2093,119,40),(2132,119,41),(2179,119,42),(2235,119,43),(2274,119,44),(2321,119,45),(2377,119,46),(2440,119,47),(2495,119,48),(2559,119,49),(2609,119,50),(2664,119,51),(2728,119,52),(2782,119,53),(2839,119,54),(2905,119,55),(2958,119,56),(3016,119,57),(3083,119,58),(3134,119,59),(3192,119,60),(3259,119,61),(251,120,1),(252,120,4),(392,120,10),(494,120,11),(495,120,12),(450,120,13),(654,120,14),(655,120,15),(606,120,16),(814,120,17),(815,120,18),(766,120,19),(986,120,20),(987,120,21),(935,120,22),(1159,120,23),(1160,120,24),(1107,120,25),(1331,120,26),(1332,120,27),(1279,120,28),(1430,120,29),(1487,120,30),(1565,120,31),(1602,120,32),(1659,120,33),(1737,120,34),(1777,120,35),(1835,120,36),(1914,120,37),(1955,120,38),(2014,120,39),(2094,120,40),(2133,120,41),(2180,120,42),(2236,120,43),(2275,120,44),(2322,120,45),(2378,120,46),(2441,120,47),(2496,120,48),(2560,120,49),(2610,120,50),(2665,120,51),(2729,120,52),(2783,120,53),(2840,120,54),(2906,120,55),(2959,120,56),(3017,120,57),(3084,120,58),(3135,120,59),(3193,120,60),(3260,120,61),(253,121,4),(403,121,10),(461,121,13),(617,121,16),(777,121,19),(946,121,22),(1118,121,25),(1290,121,28),(1576,121,31),(1748,121,34),(1925,121,37),(2105,121,40),(2247,121,43),(2389,121,46),(2571,121,49),(2740,121,52),(2917,121,55),(3095,121,58),(3271,121,61),(254,122,1),(255,122,4),(404,122,10),(496,122,11),(497,122,12),(462,122,13),(656,122,14),(657,122,15),(618,122,16),(816,122,17),(817,122,18),(778,122,19),(988,122,20),(989,122,21),(947,122,22),(1161,122,23),(1162,122,24),(1119,122,25),(1333,122,26),(1334,122,27),(1291,122,28),(1431,122,29),(1488,122,30),(1577,122,31),(1603,122,32),(1660,122,33),(1749,122,34),(1778,122,35),(1836,122,36),(1926,122,37),(1956,122,38),(2015,122,39),(2106,122,40),(2134,122,41),(2181,122,42),(2248,122,43),(2276,122,44),(2323,122,45),(2390,122,46),(2442,122,47),(2497,122,48),(2572,122,49),(2611,122,50),(2666,122,51),(2741,122,52),(2784,122,53),(2841,122,54),(2918,122,55),(2960,122,56),(3018,122,57),(3096,122,58),(3136,122,59),(3194,122,60),(3272,122,61),(256,123,1),(257,123,4),(405,123,10),(498,123,11),(499,123,12),(463,123,13),(658,123,14),(659,123,15),(619,123,16),(818,123,17),(819,123,18),(779,123,19),(990,123,20),(991,123,21),(948,123,22),(1163,123,23),(1164,123,24),(1120,123,25),(1335,123,26),(1336,123,27),(1292,123,28),(1432,123,29),(1489,123,30),(1578,123,31),(1604,123,32),(1661,123,33),(1750,123,34),(1779,123,35),(1837,123,36),(1927,123,37),(1957,123,38),(2016,123,39),(2107,123,40),(2135,123,41),(2182,123,42),(2249,123,43),(2277,123,44),(2324,123,45),(2391,123,46),(2443,123,47),(2498,123,48),(2573,123,49),(2612,123,50),(2667,123,51),(2742,123,52),(2785,123,53),(2842,123,54),(2919,123,55),(2961,123,56),(3019,123,57),(3097,123,58),(3137,123,59),(3195,123,60),(3273,123,61),(258,124,1),(259,124,4),(406,124,10),(500,124,11),(501,124,12),(464,124,13),(660,124,14),(661,124,15),(620,124,16),(820,124,17),(821,124,18),(780,124,19),(992,124,20),(993,124,21),(949,124,22),(1165,124,23),(1166,124,24),(1121,124,25),(1337,124,26),(1338,124,27),(1293,124,28),(1433,124,29),(1490,124,30),(1579,124,31),(1605,124,32),(1662,124,33),(1751,124,34),(1780,124,35),(1838,124,36),(1928,124,37),(1958,124,38),(2017,124,39),(2108,124,40),(2136,124,41),(2183,124,42),(2250,124,43),(2278,124,44),(2325,124,45),(2392,124,46),(2444,124,47),(2499,124,48),(2574,124,49),(2613,124,50),(2668,124,51),(2743,124,52),(2786,124,53),(2843,124,54),(2920,124,55),(2962,124,56),(3020,124,57),(3098,124,58),(3138,124,59),(3196,124,60),(3274,124,61),(260,125,1),(261,125,4),(407,125,10),(502,125,11),(503,125,12),(465,125,13),(662,125,14),(663,125,15),(621,125,16),(822,125,17),(823,125,18),(781,125,19),(994,125,20),(995,125,21),(950,125,22),(1167,125,23),(1168,125,24),(1122,125,25),(1339,125,26),(1340,125,27),(1294,125,28),(1434,125,29),(1491,125,30),(1580,125,31),(1606,125,32),(1663,125,33),(1752,125,34),(1781,125,35),(1839,125,36),(1929,125,37),(1959,125,38),(2018,125,39),(2109,125,40),(2137,125,41),(2184,125,42),(2251,125,43),(2279,125,44),(2326,125,45),(2393,125,46),(2445,125,47),(2500,125,48),(2575,125,49),(2614,125,50),(2669,125,51),(2744,125,52),(2787,125,53),(2844,125,54),(2921,125,55),(2963,125,56),(3021,125,57),(3099,125,58),(3139,125,59),(3197,125,60),(3275,125,61),(262,126,1),(263,126,4),(408,126,10),(504,126,11),(505,126,12),(466,126,13),(664,126,14),(665,126,15),(622,126,16),(824,126,17),(825,126,18),(782,126,19),(996,126,20),(997,126,21),(951,126,22),(1169,126,23),(1170,126,24),(1123,126,25),(1341,126,26),(1342,126,27),(1295,126,28),(1435,126,29),(1492,126,30),(1581,126,31),(1607,126,32),(1664,126,33),(1753,126,34),(1782,126,35),(1840,126,36),(1930,126,37),(1960,126,38),(2019,126,39),(2110,126,40),(2138,126,41),(2185,126,42),(2252,126,43),(2280,126,44),(2327,126,45),(2394,126,46),(2446,126,47),(2501,126,48),(2576,126,49),(2615,126,50),(2670,126,51),(2745,126,52),(2788,126,53),(2845,126,54),(2922,126,55),(2964,126,56),(3022,126,57),(3100,126,58),(3140,126,59),(3198,126,60),(3276,126,61),(264,127,1),(265,127,4),(409,127,10),(506,127,11),(507,127,12),(467,127,13),(666,127,14),(667,127,15),(623,127,16),(826,127,17),(827,127,18),(783,127,19),(998,127,20),(999,127,21),(952,127,22),(1171,127,23),(1172,127,24),(1124,127,25),(1343,127,26),(1344,127,27),(1296,127,28),(1436,127,29),(1493,127,30),(1582,127,31),(1608,127,32),(1665,127,33),(1754,127,34),(1783,127,35),(1841,127,36),(1931,127,37),(1961,127,38),(2020,127,39),(2111,127,40),(2139,127,41),(2186,127,42),(2253,127,43),(2281,127,44),(2328,127,45),(2395,127,46),(2447,127,47),(2502,127,48),(2577,127,49),(2616,127,50),(2671,127,51),(2746,127,52),(2789,127,53),(2846,127,54),(2923,127,55),(2965,127,56),(3023,127,57),(3101,127,58),(3141,127,59),(3199,127,60),(3277,127,61),(266,128,1),(267,128,4),(359,128,10),(508,128,11),(509,128,12),(417,128,13),(668,128,14),(669,128,15),(573,128,16),(828,128,17),(829,128,18),(733,128,19),(1000,128,20),(1001,128,21),(902,128,22),(1173,128,23),(1174,128,24),(1074,128,25),(1345,128,26),(1346,128,27),(1246,128,28),(1437,128,29),(1494,128,30),(1532,128,31),(1609,128,32),(1666,128,33),(1704,128,34),(1784,128,35),(1842,128,36),(1881,128,37),(1962,128,38),(2021,128,39),(2061,128,40),(2140,128,41),(2187,128,42),(2215,128,43),(2282,128,44),(2329,128,45),(2357,128,46),(2448,128,47),(2503,128,48),(2539,128,49),(2617,128,50),(2672,128,51),(2708,128,52),(2790,128,53),(2847,128,54),(2885,128,55),(2966,128,56),(3024,128,57),(3063,128,58),(3142,128,59),(3200,128,60),(3239,128,61),(268,129,1),(328,129,4),(360,129,10),(510,129,11),(511,129,12),(418,129,13),(670,129,14),(671,129,15),(574,129,16),(830,129,17),(831,129,18),(734,129,19),(1002,129,20),(1003,129,21),(903,129,22),(1175,129,23),(1176,129,24),(1075,129,25),(1347,129,26),(1348,129,27),(1247,129,28),(1438,129,29),(1495,129,30),(1533,129,31),(1610,129,32),(1667,129,33),(1705,129,34),(1785,129,35),(1843,129,36),(1882,129,37),(1963,129,38),(2022,129,39),(2062,129,40),(2141,129,41),(2188,129,42),(2216,129,43),(2283,129,44),(2330,129,45),(2358,129,46),(2449,129,47),(2504,129,48),(2540,129,49),(2618,129,50),(2673,129,51),(2709,129,52),(2791,129,53),(2848,129,54),(2886,129,55),(2967,129,56),(3025,129,57),(3064,129,58),(3143,129,59),(3201,129,60),(3240,129,61),(270,130,1),(329,130,4),(361,130,10),(512,130,11),(513,130,12),(419,130,13),(672,130,14),(673,130,15),(575,130,16),(832,130,17),(833,130,18),(735,130,19),(1004,130,20),(1005,130,21),(904,130,22),(1177,130,23),(1178,130,24),(1076,130,25),(1349,130,26),(1350,130,27),(1248,130,28),(1439,130,29),(1496,130,30),(1534,130,31),(1611,130,32),(1668,130,33),(1706,130,34),(1786,130,35),(1844,130,36),(1883,130,37),(1964,130,38),(2023,130,39),(2063,130,40),(2142,130,41),(2189,130,42),(2217,130,43),(2284,130,44),(2331,130,45),(2359,130,46),(2450,130,47),(2505,130,48),(2541,130,49),(2619,130,50),(2674,130,51),(2710,130,52),(2792,130,53),(2849,130,54),(2887,130,55),(2968,130,56),(3026,130,57),(3065,130,58),(3144,130,59),(3202,130,60),(3241,130,61),(272,131,1),(330,131,4),(362,131,10),(514,131,11),(515,131,12),(420,131,13),(674,131,14),(675,131,15),(576,131,16),(834,131,17),(835,131,18),(736,131,19),(1006,131,20),(1007,131,21),(905,131,22),(1179,131,23),(1180,131,24),(1077,131,25),(1351,131,26),(1352,131,27),(1249,131,28),(1440,131,29),(1497,131,30),(1535,131,31),(1612,131,32),(1669,131,33),(1707,131,34),(1787,131,35),(1845,131,36),(1884,131,37),(1965,131,38),(2024,131,39),(2064,131,40),(2143,131,41),(2190,131,42),(2218,131,43),(2285,131,44),(2332,131,45),(2360,131,46),(2451,131,47),(2506,131,48),(2542,131,49),(2620,131,50),(2675,131,51),(2711,131,52),(2793,131,53),(2850,131,54),(2888,131,55),(2969,131,56),(3027,131,57),(3066,131,58),(3145,131,59),(3203,131,60),(3242,131,61),(274,132,1),(331,132,4),(393,132,10),(516,132,11),(517,132,12),(451,132,13),(676,132,14),(677,132,15),(607,132,16),(836,132,17),(837,132,18),(767,132,19),(1008,132,20),(1009,132,21),(936,132,22),(1181,132,23),(1182,132,24),(1108,132,25),(1353,132,26),(1354,132,27),(1280,132,28),(1441,132,29),(1498,132,30),(1566,132,31),(1613,132,32),(1670,132,33),(1738,132,34),(1788,132,35),(1846,132,36),(1915,132,37),(1966,132,38),(2025,132,39),(2095,132,40),(2144,132,41),(2191,132,42),(2237,132,43),(2286,132,44),(2333,132,45),(2379,132,46),(2452,132,47),(2507,132,48),(2561,132,49),(2621,132,50),(2676,132,51),(2730,132,52),(2794,132,53),(2851,132,54),(2907,132,55),(2970,132,56),(3028,132,57),(3085,132,58),(3146,132,59),(3204,132,60),(3261,132,61),(276,133,1),(332,133,4),(363,133,10),(518,133,11),(519,133,12),(421,133,13),(678,133,14),(679,133,15),(577,133,16),(838,133,17),(839,133,18),(737,133,19),(1010,133,20),(1011,133,21),(906,133,22),(1183,133,23),(1184,133,24),(1078,133,25),(1355,133,26),(1356,133,27),(1250,133,28),(1442,133,29),(1499,133,30),(1536,133,31),(1614,133,32),(1671,133,33),(1708,133,34),(1789,133,35),(1847,133,36),(1885,133,37),(1967,133,38),(2026,133,39),(2065,133,40),(2145,133,41),(2192,133,42),(2219,133,43),(2287,133,44),(2334,133,45),(2361,133,46),(2453,133,47),(2508,133,48),(2543,133,49),(2622,133,50),(2677,133,51),(2712,133,52),(2795,133,53),(2852,133,54),(2889,133,55),(2971,133,56),(3029,133,57),(3067,133,58),(3147,133,59),(3205,133,60),(3243,133,61),(278,134,1),(333,134,4),(364,134,10),(520,134,11),(521,134,12),(422,134,13),(680,134,14),(681,134,15),(578,134,16),(840,134,17),(841,134,18),(738,134,19),(1012,134,20),(1013,134,21),(907,134,22),(1185,134,23),(1186,134,24),(1079,134,25),(1357,134,26),(1358,134,27),(1251,134,28),(1443,134,29),(1500,134,30),(1537,134,31),(1615,134,32),(1672,134,33),(1709,134,34),(1790,134,35),(1848,134,36),(1886,134,37),(1968,134,38),(2027,134,39),(2066,134,40),(2146,134,41),(2193,134,42),(2220,134,43),(2288,134,44),(2335,134,45),(2362,134,46),(2454,134,47),(2509,134,48),(2544,134,49),(2623,134,50),(2678,134,51),(2713,134,52),(2796,134,53),(2853,134,54),(2890,134,55),(2972,134,56),(3030,134,57),(3068,134,58),(3148,134,59),(3206,134,60),(3244,134,61),(280,135,1),(343,135,4),(365,135,10),(522,135,11),(523,135,12),(423,135,13),(682,135,14),(683,135,15),(579,135,16),(842,135,17),(843,135,18),(739,135,19),(1014,135,20),(1015,135,21),(908,135,22),(1187,135,23),(1188,135,24),(1080,135,25),(1359,135,26),(1360,135,27),(1252,135,28),(1444,135,29),(1501,135,30),(1538,135,31),(1616,135,32),(1673,135,33),(1710,135,34),(1791,135,35),(1849,135,36),(1887,135,37),(1969,135,38),(2028,135,39),(2067,135,40),(282,136,1),(344,136,4),(366,136,10),(524,136,11),(525,136,12),(424,136,13),(684,136,14),(685,136,15),(580,136,16),(844,136,17),(845,136,18),(740,136,19),(1016,136,20),(1017,136,21),(909,136,22),(1189,136,23),(1190,136,24),(1081,136,25),(1361,136,26),(1362,136,27),(1253,136,28),(1445,136,29),(1502,136,30),(1539,136,31),(1617,136,32),(1674,136,33),(1711,136,34),(1792,136,35),(1850,136,36),(1888,136,37),(1970,136,38),(2029,136,39),(2068,136,40),(2147,136,41),(2194,136,42),(2221,136,43),(2289,136,44),(2336,136,45),(2363,136,46),(2455,136,47),(2510,136,48),(2545,136,49),(2624,136,50),(2679,136,51),(2714,136,52),(2797,136,53),(2854,136,54),(2891,136,55),(2973,136,56),(3031,136,57),(3069,136,58),(3149,136,59),(3207,136,60),(3245,136,61),(284,137,1),(345,137,4),(367,137,10),(526,137,11),(527,137,12),(425,137,13),(686,137,14),(687,137,15),(581,137,16),(846,137,17),(847,137,18),(741,137,19),(1018,137,20),(1019,137,21),(910,137,22),(1191,137,23),(1192,137,24),(1082,137,25),(1363,137,26),(1364,137,27),(1254,137,28),(1446,137,29),(1503,137,30),(1540,137,31),(1618,137,32),(1675,137,33),(1712,137,34),(1793,137,35),(1851,137,36),(1889,137,37),(1971,137,38),(2030,137,39),(2069,137,40),(2148,137,41),(2195,137,42),(2222,137,43),(2290,137,44),(2337,137,45),(2364,137,46),(2456,137,47),(2511,137,48),(2546,137,49),(2625,137,50),(2680,137,51),(2715,137,52),(2798,137,53),(2855,137,54),(2892,137,55),(2974,137,56),(3032,137,57),(3070,137,58),(3150,137,59),(3208,137,60),(3246,137,61),(286,138,1),(346,138,4),(368,138,10),(528,138,11),(529,138,12),(426,138,13),(688,138,14),(689,138,15),(582,138,16),(848,138,17),(849,138,18),(742,138,19),(1020,138,20),(1021,138,21),(911,138,22),(1193,138,23),(1194,138,24),(1083,138,25),(1365,138,26),(1366,138,27),(1255,138,28),(1447,138,29),(1504,138,30),(1541,138,31),(1619,138,32),(1676,138,33),(1713,138,34),(1794,138,35),(1852,138,36),(1890,138,37),(1972,138,38),(2031,138,39),(2070,138,40),(2149,138,41),(2196,138,42),(2223,138,43),(2291,138,44),(2338,138,45),(2365,138,46),(2457,138,47),(2512,138,48),(2547,138,49),(2626,138,50),(2681,138,51),(2716,138,52),(2799,138,53),(2856,138,54),(2893,138,55),(2975,138,56),(3033,138,57),(3071,138,58),(3151,138,59),(3209,138,60),(3247,138,61),(288,139,1),(347,139,4),(369,139,10),(530,139,11),(531,139,12),(427,139,13),(690,139,14),(691,139,15),(583,139,16),(850,139,17),(851,139,18),(743,139,19),(1022,139,20),(1023,139,21),(912,139,22),(1195,139,23),(1196,139,24),(1084,139,25),(1367,139,26),(1368,139,27),(1256,139,28),(1448,139,29),(1505,139,30),(1542,139,31),(1620,139,32),(1677,139,33),(1714,139,34),(1795,139,35),(1853,139,36),(1891,139,37),(1973,139,38),(2032,139,39),(2071,139,40),(290,140,1),(348,140,4),(370,140,10),(532,140,11),(533,140,12),(428,140,13),(692,140,14),(693,140,15),(584,140,16),(852,140,17),(853,140,18),(744,140,19),(1024,140,20),(1025,140,21),(913,140,22),(1197,140,23),(1198,140,24),(1085,140,25),(1369,140,26),(1370,140,27),(1257,140,28),(1449,140,29),(1506,140,30),(1543,140,31),(1621,140,32),(1678,140,33),(1715,140,34),(1796,140,35),(1854,140,36),(1892,140,37),(1974,140,38),(2033,140,39),(2072,140,40),(292,141,1),(349,141,4),(371,141,10),(534,141,11),(535,141,12),(429,141,13),(694,141,14),(695,141,15),(585,141,16),(854,141,17),(855,141,18),(745,141,19),(1026,141,20),(1027,141,21),(914,141,22),(1199,141,23),(1200,141,24),(1086,141,25),(1371,141,26),(1372,141,27),(1258,141,28),(1450,141,29),(1507,141,30),(1544,141,31),(1622,141,32),(1679,141,33),(1716,141,34),(1797,141,35),(1855,141,36),(1893,141,37),(1975,141,38),(2034,141,39),(2073,141,40),(294,142,1),(350,142,4),(372,142,10),(536,142,11),(537,142,12),(430,142,13),(696,142,14),(697,142,15),(586,142,16),(856,142,17),(857,142,18),(746,142,19),(1028,142,20),(1029,142,21),(915,142,22),(1201,142,23),(1202,142,24),(1087,142,25),(1373,142,26),(1374,142,27),(1259,142,28),(1451,142,29),(1508,142,30),(1545,142,31),(1623,142,32),(1680,142,33),(1717,142,34),(1798,142,35),(1856,142,36),(1894,142,37),(1976,142,38),(2035,142,39),(2074,142,40),(296,143,1),(351,143,4),(373,143,10),(538,143,11),(539,143,12),(431,143,13),(698,143,14),(699,143,15),(587,143,16),(858,143,17),(859,143,18),(747,143,19),(1030,143,20),(1031,143,21),(916,143,22),(1203,143,23),(1204,143,24),(1088,143,25),(1375,143,26),(1376,143,27),(1260,143,28),(1452,143,29),(1509,143,30),(1546,143,31),(1624,143,32),(1681,143,33),(1718,143,34),(1799,143,35),(1857,143,36),(1895,143,37),(1977,143,38),(2036,143,39),(2075,143,40),(298,144,1),(352,144,4),(374,144,10),(540,144,11),(541,144,12),(432,144,13),(700,144,14),(701,144,15),(588,144,16),(860,144,17),(861,144,18),(748,144,19),(1032,144,20),(1033,144,21),(917,144,22),(1205,144,23),(1206,144,24),(1089,144,25),(1377,144,26),(1378,144,27),(1261,144,28),(1453,144,29),(1510,144,30),(1547,144,31),(1625,144,32),(1682,144,33),(1719,144,34),(1800,144,35),(1858,144,36),(1896,144,37),(1978,144,38),(2037,144,39),(2076,144,40),(300,145,1),(353,145,4),(375,145,10),(542,145,11),(543,145,12),(433,145,13),(702,145,14),(703,145,15),(589,145,16),(862,145,17),(863,145,18),(749,145,19),(1034,145,20),(1035,145,21),(918,145,22),(1207,145,23),(1208,145,24),(1090,145,25),(1379,145,26),(1380,145,27),(1262,145,28),(1454,145,29),(1511,145,30),(1548,145,31),(1626,145,32),(1683,145,33),(1720,145,34),(1801,145,35),(1859,145,36),(1897,145,37),(1979,145,38),(2038,145,39),(2077,145,40),(302,146,1),(354,146,4),(376,146,10),(544,146,11),(545,146,12),(434,146,13),(704,146,14),(705,146,15),(590,146,16),(864,146,17),(865,146,18),(750,146,19),(1036,146,20),(1037,146,21),(919,146,22),(1209,146,23),(1210,146,24),(1091,146,25),(1381,146,26),(1382,146,27),(1263,146,28),(1455,146,29),(1512,146,30),(1549,146,31),(1627,146,32),(1684,146,33),(1721,146,34),(1802,146,35),(1860,146,36),(1898,146,37),(1980,146,38),(2039,146,39),(2078,146,40),(304,147,1),(355,147,4),(377,147,10),(546,147,11),(547,147,12),(435,147,13),(706,147,14),(707,147,15),(591,147,16),(866,147,17),(867,147,18),(751,147,19),(1038,147,20),(1039,147,21),(920,147,22),(1211,147,23),(1212,147,24),(1092,147,25),(1383,147,26),(1384,147,27),(1264,147,28),(1456,147,29),(1513,147,30),(1550,147,31),(1628,147,32),(1685,147,33),(1722,147,34),(1803,147,35),(1861,147,36),(1899,147,37),(1981,147,38),(2040,147,39),(2079,147,40),(306,148,1),(356,148,4),(378,148,10),(548,148,11),(549,148,12),(436,148,13),(708,148,14),(709,148,15),(592,148,16),(868,148,17),(869,148,18),(752,148,19),(1040,148,20),(1041,148,21),(921,148,22),(1213,148,23),(1214,148,24),(1093,148,25),(1385,148,26),(1386,148,27),(1265,148,28),(1457,148,29),(1514,148,30),(1551,148,31),(1629,148,32),(1686,148,33),(1723,148,34),(1804,148,35),(1862,148,36),(1900,148,37),(1982,148,38),(2041,148,39),(2080,148,40),(308,149,1),(357,149,4),(379,149,10),(550,149,11),(551,149,12),(437,149,13),(710,149,14),(711,149,15),(593,149,16),(870,149,17),(871,149,18),(753,149,19),(1042,149,20),(1043,149,21),(922,149,22),(1215,149,23),(1216,149,24),(1094,149,25),(1387,149,26),(1388,149,27),(1266,149,28),(1458,149,29),(1515,149,30),(1552,149,31),(1630,149,32),(1687,149,33),(1724,149,34),(1805,149,35),(1863,149,36),(1901,149,37),(1983,149,38),(2042,149,39),(2081,149,40),(310,150,1),(334,150,4),(380,150,10),(552,150,11),(553,150,12),(438,150,13),(712,150,14),(713,150,15),(594,150,16),(872,150,17),(873,150,18),(754,150,19),(1044,150,20),(1045,150,21),(923,150,22),(1217,150,23),(1218,150,24),(1095,150,25),(1389,150,26),(1390,150,27),(1267,150,28),(1459,150,29),(1516,150,30),(1553,150,31),(1631,150,32),(1688,150,33),(1725,150,34),(1806,150,35),(1864,150,36),(1902,150,37),(1984,150,38),(2043,150,39),(2082,150,40),(2150,150,41),(2197,150,42),(2224,150,43),(2292,150,44),(2339,150,45),(2366,150,46),(2458,150,47),(2513,150,48),(2548,150,49),(2627,150,50),(2682,150,51),(2717,150,52),(2800,150,53),(2857,150,54),(2894,150,55),(2976,150,56),(3034,150,57),(3072,150,58),(3152,150,59),(3210,150,60),(3248,150,61),(312,151,1),(335,151,4),(381,151,10),(554,151,11),(555,151,12),(439,151,13),(714,151,14),(715,151,15),(595,151,16),(874,151,17),(875,151,18),(755,151,19),(1046,151,20),(1047,151,21),(924,151,22),(1219,151,23),(1220,151,24),(1096,151,25),(1391,151,26),(1392,151,27),(1268,151,28),(1460,151,29),(1517,151,30),(1554,151,31),(1632,151,32),(1689,151,33),(1726,151,34),(1807,151,35),(1865,151,36),(1903,151,37),(1985,151,38),(2044,151,39),(2083,151,40),(2151,151,41),(2198,151,42),(2225,151,43),(2293,151,44),(2340,151,45),(2367,151,46),(2459,151,47),(2514,151,48),(2549,151,49),(2628,151,50),(2683,151,51),(2718,151,52),(2801,151,53),(2858,151,54),(2895,151,55),(2977,151,56),(3035,151,57),(3073,151,58),(3153,151,59),(3211,151,60),(3249,151,61),(314,152,1),(336,152,4),(382,152,10),(556,152,11),(557,152,12),(440,152,13),(716,152,14),(717,152,15),(596,152,16),(876,152,17),(877,152,18),(756,152,19),(1048,152,20),(1049,152,21),(925,152,22),(1221,152,23),(1222,152,24),(1097,152,25),(1393,152,26),(1394,152,27),(1269,152,28),(1461,152,29),(1518,152,30),(1555,152,31),(1633,152,32),(1690,152,33),(1727,152,34),(1808,152,35),(1866,152,36),(1904,152,37),(1986,152,38),(2045,152,39),(2084,152,40),(2152,152,41),(2199,152,42),(2226,152,43),(2294,152,44),(2341,152,45),(2368,152,46),(2460,152,47),(2515,152,48),(2550,152,49),(2629,152,50),(2684,152,51),(2719,152,52),(2802,152,53),(2859,152,54),(2896,152,55),(2978,152,56),(3036,152,57),(3074,152,58),(3154,152,59),(3212,152,60),(3250,152,61),(316,153,1),(337,153,4),(383,153,10),(558,153,11),(559,153,12),(441,153,13),(718,153,14),(719,153,15),(597,153,16),(878,153,17),(879,153,18),(757,153,19),(1050,153,20),(1051,153,21),(926,153,22),(1223,153,23),(1224,153,24),(1098,153,25),(1395,153,26),(1396,153,27),(1270,153,28),(1462,153,29),(1519,153,30),(1556,153,31),(1634,153,32),(1691,153,33),(1728,153,34),(1809,153,35),(1867,153,36),(1905,153,37),(1987,153,38),(2046,153,39),(2085,153,40),(2153,153,41),(2200,153,42),(2227,153,43),(2295,153,44),(2342,153,45),(2369,153,46),(2461,153,47),(2516,153,48),(2551,153,49),(2630,153,50),(2685,153,51),(2720,153,52),(2803,153,53),(2860,153,54),(2897,153,55),(2979,153,56),(3037,153,57),(3075,153,58),(3155,153,59),(3213,153,60),(3251,153,61),(318,154,1),(338,154,4),(384,154,10),(560,154,11),(561,154,12),(442,154,13),(720,154,14),(721,154,15),(598,154,16),(880,154,17),(881,154,18),(758,154,19),(1052,154,20),(1053,154,21),(927,154,22),(1225,154,23),(1226,154,24),(1099,154,25),(1397,154,26),(1398,154,27),(1271,154,28),(1463,154,29),(1520,154,30),(1557,154,31),(1635,154,32),(1692,154,33),(1729,154,34),(1810,154,35),(1868,154,36),(1906,154,37),(1988,154,38),(2047,154,39),(2086,154,40),(2154,154,41),(2201,154,42),(2228,154,43),(2296,154,44),(2343,154,45),(2370,154,46),(2462,154,47),(2517,154,48),(2552,154,49),(2631,154,50),(2686,154,51),(2721,154,52),(2804,154,53),(2861,154,54),(2898,154,55),(2980,154,56),(3038,154,57),(3076,154,58),(3156,154,59),(3214,154,60),(3252,154,61),(320,155,1),(339,155,4),(385,155,10),(562,155,11),(563,155,12),(443,155,13),(722,155,14),(723,155,15),(599,155,16),(882,155,17),(883,155,18),(759,155,19),(1054,155,20),(1055,155,21),(928,155,22),(1227,155,23),(1228,155,24),(1100,155,25),(1399,155,26),(1400,155,27),(1272,155,28),(1464,155,29),(1521,155,30),(1558,155,31),(1636,155,32),(1693,155,33),(1730,155,34),(1811,155,35),(1869,155,36),(1907,155,37),(1989,155,38),(2048,155,39),(2087,155,40),(2155,155,41),(2202,155,42),(2229,155,43),(2297,155,44),(2344,155,45),(2371,155,46),(2463,155,47),(2518,155,48),(2553,155,49),(2632,155,50),(2687,155,51),(2722,155,52),(2805,155,53),(2862,155,54),(2899,155,55),(2981,155,56),(3039,155,57),(3077,155,58),(3157,155,59),(3215,155,60),(3253,155,61),(322,156,1),(340,156,4),(386,156,10),(564,156,11),(565,156,12),(444,156,13),(724,156,14),(725,156,15),(600,156,16),(884,156,17),(885,156,18),(760,156,19),(1056,156,20),(1057,156,21),(929,156,22),(1229,156,23),(1230,156,24),(1101,156,25),(1401,156,26),(1402,156,27),(1273,156,28),(1465,156,29),(1522,156,30),(1559,156,31),(1637,156,32),(1694,156,33),(1731,156,34),(1812,156,35),(1870,156,36),(1908,156,37),(1990,156,38),(2049,156,39),(2088,156,40),(2156,156,41),(2203,156,42),(2230,156,43),(2298,156,44),(2345,156,45),(2372,156,46),(2464,156,47),(2519,156,48),(2554,156,49),(2633,156,50),(2688,156,51),(2723,156,52),(2806,156,53),(2863,156,54),(2900,156,55),(2982,156,56),(3040,156,57),(3078,156,58),(3158,156,59),(3216,156,60),(3254,156,61),(324,157,1),(341,157,4),(387,157,10),(566,157,11),(567,157,12),(445,157,13),(726,157,14),(727,157,15),(601,157,16),(886,157,17),(887,157,18),(761,157,19),(1058,157,20),(1059,157,21),(930,157,22),(1231,157,23),(1232,157,24),(1102,157,25),(1403,157,26),(1404,157,27),(1274,157,28),(1466,157,29),(1523,157,30),(1560,157,31),(1638,157,32),(1695,157,33),(1732,157,34),(1813,157,35),(1871,157,36),(1909,157,37),(1991,157,38),(2050,157,39),(2089,157,40),(2157,157,41),(2204,157,42),(2231,157,43),(2299,157,44),(2346,157,45),(2373,157,46),(2465,157,47),(2520,157,48),(2555,157,49),(2634,157,50),(2689,157,51),(2724,157,52),(2807,157,53),(2864,157,54),(2901,157,55),(2983,157,56),(3041,157,57),(3079,157,58),(3159,157,59),(3217,157,60),(3255,157,61),(326,158,1),(342,158,4),(388,158,10),(568,158,11),(569,158,12),(446,158,13),(728,158,14),(729,158,15),(602,158,16),(888,158,17),(889,158,18),(762,158,19),(1060,158,20),(1061,158,21),(931,158,22),(1233,158,23),(1234,158,24),(1103,158,25),(1405,158,26),(1406,158,27),(1275,158,28),(1467,158,29),(1524,158,30),(1561,158,31),(1639,158,32),(1696,158,33),(1733,158,34),(1814,158,35),(1872,158,36),(1910,158,37),(1992,158,38),(2051,158,39),(2090,158,40),(2158,158,41),(2205,158,42),(2232,158,43),(2300,158,44),(2347,158,45),(2374,158,46),(2466,158,47),(2521,158,48),(2556,158,49),(2635,158,50),(2690,158,51),(2725,158,52),(2808,158,53),(2865,158,54),(2902,158,55),(2984,158,56),(3042,158,57),(3080,158,58),(3160,158,59),(3218,158,60),(3256,158,61),(410,159,11),(411,159,12),(412,159,13),(624,159,14),(625,159,15),(570,159,16),(784,159,17),(785,159,18),(730,159,19),(956,159,20),(957,159,21),(899,159,22),(1129,159,23),(1130,159,24),(1071,159,25),(1301,159,26),(1302,159,27),(1243,159,28),(1415,159,29),(1472,159,30),(1529,159,31),(1587,159,32),(1644,159,33),(1701,159,34),(1762,159,35),(1820,159,36),(1878,159,37),(1940,159,38),(1999,159,39),(2058,159,40),(2118,159,41),(2165,159,42),(2212,159,43),(2260,159,44),(2307,159,45),(2354,159,46),(2426,159,47),(2481,159,48),(2536,159,49),(2595,159,50),(2650,159,51),(2705,159,52),(2768,159,53),(2825,159,54),(2882,159,55),(2944,159,56),(3002,159,57),(3060,159,58),(3120,159,59),(3178,159,60),(3236,159,61),(413,160,11),(414,160,12),(415,160,13),(626,160,14),(627,160,15),(571,160,16),(786,160,17),(787,160,18),(731,160,19),(958,160,20),(959,160,21),(900,160,22),(1131,160,23),(1132,160,24),(1072,160,25),(1303,160,26),(1304,160,27),(1244,160,28),(1416,160,29),(1473,160,30),(1530,160,31),(1588,160,32),(1645,160,33),(1702,160,34),(1763,160,35),(1821,160,36),(1879,160,37),(1941,160,38),(2000,160,39),(2059,160,40),(2119,160,41),(2166,160,42),(2213,160,43),(2261,160,44),(2308,160,45),(2355,160,46),(2427,160,47),(2482,160,48),(2537,160,49),(2596,160,50),(2651,160,51),(2706,160,52),(2769,160,53),(2826,160,54),(2883,160,55),(2945,160,56),(3003,160,57),(3061,160,58),(3121,160,59),(3179,160,60),(3237,160,61),(890,161,17),(891,161,18),(892,161,19),(1062,161,20),(1063,161,21),(953,161,22),(1235,161,23),(1236,161,24),(1125,161,25),(1407,161,26),(1408,161,27),(1297,161,28),(1468,161,29),(1525,161,30),(1583,161,31),(1640,161,32),(1697,161,33),(1755,161,34),(1815,161,35),(1873,161,36),(1932,161,37),(1993,161,38),(2052,161,39),(2112,161,40),(2159,161,41),(2206,161,42),(2254,161,43),(2301,161,44),(2348,161,45),(2396,161,46),(2467,161,47),(2522,161,48),(2578,161,49),(2636,161,50),(2691,161,51),(2747,161,52),(2809,161,53),(2866,161,54),(2924,161,55),(2985,161,56),(3043,161,57),(3102,161,58),(3161,161,59),(3219,161,60),(3278,161,61),(893,162,17),(894,162,18),(895,162,19),(1064,162,20),(1065,162,21),(954,162,22),(1237,162,23),(1238,162,24),(1126,162,25),(1409,162,26),(1410,162,27),(1298,162,28),(1469,162,29),(1526,162,30),(1584,162,31),(1641,162,32),(1698,162,33),(1756,162,34),(1816,162,35),(1874,162,36),(1933,162,37),(1994,162,38),(2053,162,39),(2113,162,40),(2160,162,41),(2207,162,42),(2255,162,43),(2302,162,44),(2349,162,45),(2397,162,46),(2468,162,47),(2523,162,48),(2579,162,49),(2637,162,50),(2692,162,51),(2748,162,52),(2810,162,53),(2867,162,54),(2925,162,55),(2986,162,56),(3044,162,57),(3103,162,58),(3162,162,59),(3220,162,60),(3279,162,61),(896,163,17),(897,163,18),(898,163,19),(1066,163,20),(1067,163,21),(955,163,22),(1239,163,23),(1240,163,24),(1127,163,25),(1411,163,26),(1412,163,27),(1299,163,28),(1470,163,29),(1527,163,30),(1585,163,31),(1642,163,32),(1699,163,33),(1757,163,34),(1817,163,35),(1875,163,36),(1934,163,37),(1995,163,38),(2054,163,39),(2114,163,40),(2161,163,41),(2208,163,42),(2256,163,43),(2303,163,44),(2350,163,45),(2398,163,46),(2469,163,47),(2524,163,48),(2580,163,49),(2638,163,50),(2693,163,51),(2749,163,52),(2811,163,53),(2868,163,54),(2926,163,55),(2987,163,56),(3045,163,57),(3104,163,58),(3163,163,59),(3221,163,60),(3280,163,61),(1068,164,20),(1069,164,21),(1070,164,22),(1241,164,23),(1242,164,24),(1128,164,25),(1413,164,26),(1414,164,27),(1300,164,28),(1471,164,29),(1528,164,30),(1586,164,31),(1643,164,32),(1700,164,33),(1758,164,34),(1818,164,35),(1876,164,36),(1935,164,37),(1996,164,38),(2055,164,39),(2115,164,40),(2162,164,41),(2209,164,42),(2257,164,43),(2304,164,44),(2351,164,45),(2399,164,46),(2470,164,47),(2525,164,48),(2581,164,49),(2639,164,50),(2694,164,51),(2750,164,52),(2812,164,53),(2869,164,54),(2927,164,55),(2988,164,56),(3046,164,57),(3105,164,58),(3164,164,59),(3222,164,60),(3281,164,61),(1759,165,32),(1760,165,33),(1761,165,34),(1819,165,35),(1877,165,36),(1936,165,37),(1997,165,38),(2056,165,39),(2116,165,40),(2163,165,41),(2210,165,42),(2258,165,43),(2305,165,44),(2352,165,45),(2400,165,46),(2471,165,47),(2526,165,48),(2582,165,49),(2640,165,50),(2695,165,51),(2751,165,52),(2813,165,53),(2870,165,54),(2928,165,55),(2989,165,56),(3047,165,57),(3106,165,58),(3165,165,59),(3223,165,60),(3282,165,61),(1937,166,35),(1938,166,36),(1939,166,37),(1998,166,38),(2057,166,39),(2117,166,40),(2164,166,41),(2211,166,42),(2259,166,43),(2306,166,44),(2353,166,45),(2401,166,46),(2472,166,47),(2527,166,48),(2583,166,49),(2641,166,50),(2696,166,51),(2752,166,52),(2814,166,53),(2871,166,54),(2929,166,55),(2990,166,56),(3048,166,57),(3107,166,58),(3166,166,59),(3224,166,60),(3283,166,61),(2402,167,44),(2403,167,45),(2404,167,46),(2473,167,47),(2528,167,48),(2584,167,49),(2642,167,50),(2697,167,51),(2753,167,52),(2815,167,53),(2872,167,54),(2930,167,55),(2991,167,56),(3049,167,57),(3108,167,58),(3167,167,59),(3225,167,60),(3284,167,61),(2405,168,44),(2406,168,45),(2407,168,46),(2474,168,47),(2529,168,48),(2585,168,49),(2643,168,50),(2698,168,51),(2754,168,52),(2816,168,53),(2873,168,54),(2931,168,55),(2992,168,56),(3050,168,57),(3109,168,58),(3168,168,59),(3226,168,60),(3285,168,61),(2408,169,44),(2409,169,45),(2410,169,46),(2475,169,47),(2530,169,48),(2586,169,49),(2644,169,50),(2699,169,51),(2755,169,52),(2817,169,53),(2874,169,54),(2932,169,55),(2993,169,56),(3051,169,57),(3110,169,58),(3169,169,59),(3227,169,60),(3286,169,61),(2411,170,44),(2412,170,45),(2413,170,46),(2476,170,47),(2531,170,48),(2587,170,49),(2645,170,50),(2700,170,51),(2756,170,52),(2818,170,53),(2875,170,54),(2933,170,55),(2994,170,56),(3052,170,57),(3111,170,58),(3170,170,59),(3228,170,60),(3287,170,61),(2414,171,44),(2415,171,45),(2416,171,46),(2477,171,47),(2532,171,48),(2588,171,49),(2646,171,50),(2701,171,51),(2757,171,52),(2819,171,53),(2876,171,54),(2934,171,55),(2995,171,56),(3053,171,57),(3112,171,58),(3171,171,59),(3229,171,60),(3288,171,61),(2417,172,44),(2418,172,45),(2419,172,46),(2478,172,47),(2533,172,48),(2589,172,49),(2647,172,50),(2702,172,51),(2758,172,52),(2820,172,53),(2877,172,54),(2935,172,55),(2996,172,56),(3054,172,57),(3113,172,58),(3172,172,59),(3230,172,60),(3289,172,61),(2420,173,44),(2421,173,45),(2422,173,46),(2479,173,47),(2534,173,48),(2590,173,49),(2648,173,50),(2703,173,51),(2759,173,52),(2821,173,53),(2878,173,54),(2936,173,55),(2997,173,56),(3055,173,57),(3114,173,58),(3173,173,59),(3231,173,60),(3290,173,61),(2423,174,44),(2424,174,45),(2425,174,46),(2480,174,47),(2535,174,48),(2591,174,49),(2649,174,50),(2704,174,51),(2760,174,52),(2822,174,53),(2879,174,54),(2937,174,55),(2998,174,56),(3056,174,57),(3115,174,58),(3174,174,59),(3232,174,60),(3291,174,61),(2592,175,49),(2761,175,52),(2938,175,55),(3116,175,58),(3292,175,61),(2762,176,50),(2763,176,51),(2764,176,52),(2823,176,53),(2880,176,54),(2939,176,55),(2999,176,56),(3057,176,57),(3117,176,58),(3175,176,59),(3233,176,60),(3293,176,61),(2765,177,50),(2766,177,51),(2767,177,52),(2824,177,53),(2881,177,54),(2940,177,55),(3000,177,56),(3058,177,57),(3118,177,58),(3176,177,59),(3234,177,60),(3294,177,61),(2941,178,53),(2942,178,54),(2943,178,55),(3001,178,56),(3059,178,57),(3119,178,58),(3177,178,59),(3235,178,60),(3295,178,61),(3296,179,59),(3297,179,60),(3298,179,61);
/*!40000 ALTER TABLE `idc_collections_attribute_data_sources` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_attribute_display_category`
--

DROP TABLE IF EXISTS `idc_collections_attribute_display_category`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_attribute_display_category` (
  `id` int NOT NULL AUTO_INCREMENT,
  `category` varchar(256) NOT NULL,
  `category_display_name` varchar(256) NOT NULL,
  `attribute_id` int NOT NULL,
  PRIMARY KEY (`id`),
  KEY `idc_collections_attr_attribute_id_66bf05b8_fk_idc_colle` (`attribute_id`),
  CONSTRAINT `idc_collections_attr_attribute_id_66bf05b8_fk_idc_colle` FOREIGN KEY (`attribute_id`) REFERENCES `idc_collections_attribute` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=34 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_attribute_display_category`
--

LOCK TABLES `idc_collections_attribute_display_category` WRITE;
/*!40000 ALTER TABLE `idc_collections_attribute_display_category` DISABLE KEYS */;
INSERT INTO `idc_collections_attribute_display_category` VALUES (1,'segmentation','Segmentation',129),(2,'segmentation','Segmentation',130),(3,'segmentation','Segmentation',131),(4,'segmentation','Segmentation',132),(5,'segmentation','Segmentation',133),(6,'segmentation','Segmentation',134),(7,'quantitative','Quantitative Analysis',135),(8,'quantitative','Quantitative Analysis',136),(9,'quantitative','Quantitative Analysis',137),(10,'quantitative','Quantitative Analysis',138),(11,'quantitative','Quantitative Analysis',139),(12,'quantitative','Quantitative Analysis',140),(13,'quantitative','Quantitative Analysis',141),(14,'quantitative','Quantitative Analysis',142),(15,'quantitative','Quantitative Analysis',143),(16,'quantitative','Quantitative Analysis',144),(17,'quantitative','Quantitative Analysis',145),(18,'quantitative','Quantitative Analysis',146),(19,'quantitative','Quantitative Analysis',147),(20,'quantitative','Quantitative Analysis',148),(21,'quantitative','Quantitative Analysis',149),(22,'qualitative','Qualitative Analysis',150),(23,'qualitative','Qualitative Analysis',151),(24,'qualitative','Qualitative Analysis',152),(25,'qualitative','Qualitative Analysis',153),(26,'qualitative','Qualitative Analysis',154),(27,'qualitative','Qualitative Analysis',155),(28,'qualitative','Qualitative Analysis',156),(29,'qualitative','Qualitative Analysis',157),(30,'qualitative','Qualitative Analysis',158),(31,'quantitative','Quantitative Analysis',167),(32,'quantitative','Quantitative Analysis',168),(33,'segmentation','Segmentation',179);
/*!40000 ALTER TABLE `idc_collections_attribute_display_category` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_attribute_display_values`
--

DROP TABLE IF EXISTS `idc_collections_attribute_display_values`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_attribute_display_values` (
  `id` int NOT NULL AUTO_INCREMENT,
  `raw_value` varchar(256) NOT NULL,
  `display_value` varchar(256) NOT NULL,
  `attribute_id` int NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `idc_collections_attribut_raw_value_attribute_id_70463571_uniq` (`raw_value`,`attribute_id`),
  KEY `idc_collections_attr_attribute_id_0318b732_fk_idc_colle` (`attribute_id`),
  CONSTRAINT `idc_collections_attr_attribute_id_0318b732_fk_idc_colle` FOREIGN KEY (`attribute_id`) REFERENCES `idc_collections_attribute` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=761 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_attribute_display_values`
--

LOCK TABLES `idc_collections_attribute_display_values` WRITE;
/*!40000 ALTER TABLE `idc_collections_attribute_display_values` DISABLE KEYS */;
INSERT INTO `idc_collections_attribute_display_values` VALUES (1,'1','Lifelong Non-smoker',59),(2,'3','Current Reformed Smoker for > 15 years',59),(3,'2','Current Smoker',59),(4,'5','Current Reformed Smoker',59),(5,'4','Current Reformed Smoker for <= 15 years',59),(6,'7','Smoking History Not Documented',59),(7,'6','Smoker at Diagnosis',59),(8,'None','NA',59),(9,'underweight','Underweight: BMI less than 18.5',63),(10,'normal weight','Normal weight: BMI is 18.5 - 24.9',63),(11,'overweight','Overweight: BMI is 25 - 29.9',63),(12,'obese','Obese: BMI is 30 or more',63),(13,'02','Recurrent Solid Tumor',76),(14,'03','Primary Blood Derived Cancer - Peripheral Blood',76),(15,'13','EBV Immortalized Normal',76),(16,'01','Primary Solid Tumor',76),(17,'06','Metastatic',76),(18,'07','Additional Metastatic',76),(19,'04','Recurrent Blood Derived Cancer - Bone Marrow',76),(20,'05','Additional - New Primary',76),(21,'08','Human Tumor Original Cells',76),(22,'09','Primary Blood Derived Cancer - Bone Marrow',76),(23,'50','Cell Lines',76),(24,'40','Recurrent Blood Derived Cancer - Peripheral Blood',76),(25,'60','Primary Xenograft Tissue',76),(26,'61','Cell Line Derived Xenograft Tissue',76),(27,'12','Buccal Cell Normal',76),(28,'14','Bone Marrow Normal',76),(29,'11','Solid Tissue Normal',76),(30,'20','Control Analyte',76),(31,'10','Blood Derived Normal',76),(32,'15','Fibroblasts from Bone Marrow Normal',76),(33,'41','Blood Derived Cancer - Bone Marrow; Post-treatment',76),(34,'42','Blood Derived Cancer - Peripheral Blood; Post-treatment',76),(35,'CTPT','CTPT',107),(36,'AR','Autorefraction',107),(37,'ASMT','Content Assessment Results',107),(38,'AU','Audio',107),(39,'BDUS','Bone Densitometry (ultrasound)',107),(40,'BI','Biomagnetic imaging',107),(41,'BMD','Bone Densitometry (X-Ray)',107),(42,'CR','Computed Radiography',107),(43,'CT','Computed Tomography',107),(44,'CTPROTOCOL','CT Protocol (Performed)',107),(45,'DG','Diaphanography',107),(46,'DOC','Document',107),(47,'DX','Digital Radiography',107),(48,'ECG','Electrocardiography',107),(49,'EPS','Cardiac Electrophysiology',107),(50,'ES','Endoscopy',107),(51,'FID','Fiducials',107),(52,'GM','General Microscopy',107),(53,'HC','Hard Copy',107),(54,'HD','Hemodynamic Waveform',107),(55,'IO','Intra-Oral Radiography',107),(56,'IOL','Intraocular Lens Data',107),(57,'IVOCT','Intravascular Optical Coherence Tomography',107),(58,'IVUS','Intravascular Ultrasound',107),(59,'KER','Keratometry',107),(60,'KO','Key Object Selection',107),(61,'LEN','Lensometry',107),(62,'LS','Laser surface scan',107),(63,'MG','Mammography',107),(64,'MR','Magnetic Resonance',107),(65,'M3D','Model for 3D Manufacturing',107),(66,'NM','Nuclear Medicine',107),(67,'OAM','Ophthalmic Axial Measurements',107),(68,'OCT','Optical Coherence Tomography (non-Ophthalmic)',107),(69,'OP','Ophthalmic Photography',107),(70,'OPM','Ophthalmic Mapping',107),(71,'OPT','Ophthalmic Tomography',107),(72,'OPTBSV','Ophthalmic Tomography B-scan Volume Analysis',107),(73,'OPTENF','Ophthalmic Tomography En Face',107),(74,'OPV','Ophthalmic Visual Field',107),(75,'OSS','Optical Surface Scan',107),(76,'OT','Other',107),(77,'PLAN','Plan',107),(78,'PR','Presentation State',107),(79,'PT','Positron emission tomography (PET)',107),(80,'PX','Panoramic X-Ray',107),(81,'REG','Registration',107),(82,'RESP','Respiratory Waveform',107),(83,'RF','Radio Fluoroscopy',107),(84,'RG','Radiographic imaging (conventional film/screen)',107),(85,'RTDOSE','Radiotherapy Dose',107),(86,'RTIMAGE','Radiotherapy Image',107),(87,'RTINTENT','Radiotherapy Intent',107),(88,'RTPLAN','Radiotherapy Plan',107),(89,'RTRAD','RT Radiation',107),(90,'RTRECORD','RT Treatment Record',107),(91,'RTSEGANN','Radiotherapy Segment Annotation',107),(92,'RTSTRUCT','Radiotherapy Structure Set',107),(93,'RWV','Real World Value Map',107),(94,'SEG','Segmentation',107),(95,'SM','Slide Microscopy',107),(96,'SMR','Stereometric Relationship',107),(97,'SR','SR Document',107),(98,'SRF','Subjective Refraction',107),(99,'STAIN','Automated Slide Stainer',107),(100,'TEXTUREMAP','Texture Map',107),(101,'TG','Thermography',107),(102,'US','Ultrasound',107),(103,'VA','Visual Acuity',107),(104,'XA','X-Ray Angiography',107),(105,'XC','External-camera Photography',107),(106,'1.2.840.10008.5.1.4.1.1.9.5.1','Arterial Pulse Waveform Storage',119),(107,'1.2.840.10008.5.1.4.1.1.77.1.5.8','Ophthalmic Optical Coherence Tomography B-scan Volume Analysis Storage',119),(108,'1.2.840.10008.5.1.4.1.1.200.3','Protocol Approval',119),(109,'1.2.840.10008.5.1.4.1.1.481.10','RT Physician Intent Storage',119),(110,'1.2.840.10008.5.1.4.1.1.481.11','RT Segment Annotation Storage',119),(111,'1.2.840.10008.5.1.4.1.1.481.12','RT Radiation Set Storage',119),(112,'1.2.840.10008.5.1.4.1.1.481.13','C-Arm Photon-Electron Radiation Storage',119),(113,'1.2.840.10008.5.1.4.1.1.13.1.3','Breast Tomosynthesis Image Storage',119),(114,'1.2.840.10008.5.1.4.1.1.481.1','RT Image Storage',119),(115,'1.2.840.10008.5.1.4.1.1.77.1.6','VL Whole Slide Microscopy Image Storage',119),(116,'1.2.840.10008.5.1.4.1.1.481.3','RT Structure Set Storage',119),(117,'1.2.840.10008.5.1.4.1.1.12.1','X-Ray Angiographic Image Storage',119),(118,'1.2.840.10008.5.1.4.38.1','Hanging Protocol',119),(119,'1.2.840.10008.5.1.4.1.1.131','Basic Structured Display Storage',119),(120,'1.2.840.10008.5.1.4.1.1.88.75','Performed Imaging Agent Administration SR Storage',119),(121,'1.2.840.10008.5.1.4.1.1.88.11','Basic Text SR Storage',119),(122,'1.2.840.10008.5.1.4.1.1.81.1','Ophthalmic Thickness Map Storage',119),(123,'1.2.840.10008.5.1.4.1.1.77.1.2.1','Video Microscopic Image Storage',119),(124,'1.2.840.10008.5.1.4.1.1.20','Nuclear Medicine Image Storage',119),(125,'1.2.840.10008.5.1.4.1.1.77.1.2','VL Microscopic Image Storage',119),(126,'1.2.840.10008.5.1.4.1.1.66.5','Surface Segmentation Storage',119),(127,'1.2.840.10008.5.1.4.34.7','RT Beams Delivery Instruction Storage',119),(128,'1.2.840.10008.5.1.4.1.1.481.2','RT Dose Storage',119),(129,'1.2.840.10008.5.1.4.1.1.66','Raw Data Storage',119),(130,'1.2.840.10008.5.1.4.1.1.67','Real World Value Mapping Storage',119),(131,'1.2.840.10008.5.1.4.1.1.66.4','Segmentation Storage',119),(132,'1.2.840.10008.5.1.4.44.1','Implant Assembly Template',119),(133,'1.2.840.10008.5.1.4.1.1.481.8','RT Ion Plan Storage',119),(134,'1.2.840.10008.5.1.4.1.1.66.1','Spatial Registration Storage',119),(135,'1.2.840.10008.5.1.4.1.1.88.70','Implantation Plan SR Storage',119),(136,'1.2.840.10008.5.1.4.1.1.77.1.1','VL Endoscopic Image Storage',119),(137,'1.2.840.10008.5.1.4.1.1.88.65','Chest CAD SR Storage',119),(138,'1.2.840.10008.5.1.4.1.1.88.69','Colon CAD SR Storage',119),(139,'1.2.840.10008.5.1.4.1.1.77.1.4','VL Photographic Image Storage',119),(140,'1.2.840.10008.5.1.4.1.1.77.1.1.1','Video Endoscopic Image Storage',119),(141,'1.2.840.10008.5.1.4.1.1.77.1.4.1','Video Photographic Image Storage',119),(142,'1.2.840.10008.5.1.4.1.1.9.4.2','General Audio Waveform Storage',119),(143,'1.2.840.10008.5.1.4.1.1.82.1','Corneal Topography Map Storage',119),(144,'1.2.840.10008.5.1.4.1.1.78.6','Spectacle Prescription Report Storage',119),(145,'1.2.840.10008.5.1.4.1.1.481.9','RT Ion Beams Treatment Record Storage',119),(146,'1.2.840.10008.5.1.4.1.1.88.50','Mammography CAD SR Storage',119),(147,'1.2.840.10008.5.1.4.1.1.66.6','Tractography Results Storage',119),(148,'1.2.840.10008.5.1.4.1.1.77.1.5.6','Wide Field Ophthalmic Photography 3D Coordinates Image Storage',119),(149,'1.2.840.10008.5.1.4.1.1.481.5','RT Plan Storage',119),(150,'1.2.840.10008.5.1.4.1.1.77.1.5.3','Stereometric Relationship Storage',119),(151,'1.2.840.10008.5.1.4.1.1.78.1','Lensometry Measurements Storage',119),(152,'1.2.840.10008.5.1.4.1.1.66.3','Deformable Spatial Registration Storage',119),(153,'1.2.840.10008.5.1.4.39.1','Color Palette',119),(154,'1.2.840.10008.5.1.4.1.1.66.2','Spatial Fiducials Storage',119),(155,'1.2.840.10008.5.1.4.45.1','Implant Template Group',119),(156,'1.2.840.10008.5.1.4.1.1.481.4','RT Beams Treatment Record Storage',119),(157,'1.2.840.10008.5.1.4.1.1.481.6','RT Brachy Treatment Record Storage',119),(158,'1.2.840.10008.5.1.4.1.1.78.7','Ophthalmic Axial Measurements Storage',119),(159,'1.2.840.10008.5.1.4.1.1.200.1','CT Defined Procedure Protocol',119),(160,'1.2.840.10008.5.1.4.1.1.77.1.3','VL Slide-Coordinates Microscopic Image Storage',119),(161,'1.2.840.10008.5.1.4.1.1.88.67','X-Ray Radiation Dose SR Storage',119),(162,'1.2.840.10008.5.1.4.1.1.88.72','Simplified Adult Echo SR Storage',119),(163,'1.2.840.10008.5.1.4.1.1.88.34','Comprehensive 3D SR Storage',119),(164,'1.2.840.10008.5.1.4.1.1.80.1','Ophthalmic Visual Field Static Perimetry Measurements Storage',119),(165,'1.2.840.10008.5.1.4.1.1.88.59','Key Object Selection Document Storage',119),(166,'1.2.840.10008.5.1.4.1.1.88.33','Comprehensive SR Storage',119),(167,'1.2.840.10008.5.1.4.1.1.88.35','Extensible SR Storage',119),(168,'1.2.840.10008.5.1.4.1.1.104.1','Encapsulated PDF Storage',119),(169,'1.2.840.10008.5.1.4.1.1.79.1','Macular Grid Thickness and Volume Report',119),(170,'1.2.840.10008.5.1.4.1.1.1','Computed Radiography Image Storage',119),(171,'1.2.840.10008.5.1.4.1.1.4','MR Image Storage',119),(172,'1.2.840.10008.5.1.4.1.1.88.68','Radiopharmaceutical Radiation Dose SR Storage',119),(173,'1.2.840.10008.5.1.4.1.1.1.1','Digital X-Ray Image Storage - For Presentation',119),(174,'1.2.840.10008.5.1.4.1.1.1.1.1','Digital X-Ray Image Storage - For Processing',119),(175,'1.2.840.10008.5.1.4.1.1.2','CT Image Storage',119),(176,'1.2.840.10008.5.1.4.1.1.6.2','Enhanced US Volume Storage',119),(177,'1.2.840.10008.5.1.4.1.1.4.1','Enhanced MR Image Storage',119),(178,'1.2.840.10008.5.1.4.1.1.128','Positron Emission Tomography Image Storage',119),(179,'1.2.840.10008.5.1.4.1.1.4.3','Enhanced MR Color Image Storage',119),(180,'1.2.840.10008.5.1.4.1.1.1.3','Digital Intra-Oral X-Ray Image Storage - For Presentation',119),(181,'1.2.840.10008.5.1.4.1.1.1.3.1','Digital Intra-Oral X-Ray Image Storage - For Processing',119),(182,'1.2.840.10008.5.1.4.1.1.2.1','Enhanced CT Image Storage',119),(183,'1.2.840.10008.5.1.4.1.1.4.2','MR Spectroscopy Storage',119),(184,'1.2.840.10008.5.1.4.1.1.90.1','Content Assessment Results Storage',119),(185,'1.2.840.10008.5.1.4.1.1.1.2','Digital Mammography X-Ray Image Storage - For Presentation',119),(186,'1.2.840.10008.5.1.4.1.1.1.2.1','Digital Mammography X-Ray Image Storage - For Processing',119),(187,'1.2.840.10008.5.1.4.1.1.13.1.4','Breast Projection X-Ray Image Storage - For Presentation',119),(188,'1.2.840.10008.5.1.4.1.1.13.1.5','Breast Projection X-Ray Image Storage - For Processing',119),(189,'1.2.840.10008.5.1.4.1.1.104.3','Encapsulated STL Storage',119),(190,'1.2.840.10008.5.1.4.1.1.88.71','Acquisition Context SR Storage',119),(191,'1.2.840.10008.5.1.4.1.1.88.22','Enhanced SR Storage',119),(192,'1.2.840.10008.5.1.4.1.1.78.3','Keratometry Measurements Storage',119),(193,'1.2.840.10008.5.1.4.1.1.104.2','Encapsulated CDA Storage',119),(194,'1.2.840.10008.5.1.4.1.1.30','Parametric Map Storage',119),(195,'1.2.840.10008.5.1.4.1.1.11.1','Grayscale Softcopy Presentation State Storage',119),(196,'1.2.840.10008.5.1.4.1.1.11.2','Color Softcopy Presentation State Storage',119),(197,'1.2.840.10008.5.1.4.1.1.78.5','Visual Acuity Measurements Storage',119),(198,'1.2.840.10008.5.1.4.1.1.11.3','Pseudo-Color Softcopy Presentation State Storage',119),(199,'1.2.840.10008.5.1.4.1.1.78.4','Subjective Refraction Measurements Storage',119),(200,'1.2.840.10008.5.1.4.1.1.130','Enhanced PET Image Storage',119),(201,'1.2.840.10008.5.1.4.1.1.77.1.5.7','Ophthalmic Optical Coherence Tomography En Face Image Storage',119),(202,'1.2.840.10008.5.1.4.1.1.78.2','Autorefraction Measurements Storage',119),(203,'1.2.840.10008.5.1.4.1.1.68.1','Surface Scan Mesh Storage',119),(204,'1.2.840.10008.5.1.4.1.1.11.5','XA/XRF Grayscale Softcopy Presentation State Storage',119),(205,'1.2.840.10008.5.1.4.1.1.77.1.5.4','Ophthalmic Tomography Image Storage',119),(206,'1.2.840.10008.5.1.4.1.1.4.4','Legacy Converted Enhanced MR Image Storage',119),(207,'1.2.840.10008.5.1.4.1.1.2.2','Legacy Converted Enhanced CT Image Storage',119),(208,'1.2.840.10008.5.1.4.1.1.88.40','Procedure Log Storage',119),(209,'1.2.840.10008.5.1.4.1.1.11.9','Volume Rendering Volumetric Presentation State Storage',119),(210,'1.2.840.10008.5.1.4.1.1.77.1.5.1','Ophthalmic Photography 8 Bit Image Storage',119),(211,'1.2.840.10008.5.1.4.1.1.11.4','Blending Softcopy Presentation State Storage',119),(212,'1.2.840.10008.5.1.4.1.1.7','Secondary Capture Image Storage',119),(213,'1.2.840.10008.5.1.4.34.10','RT Brachy Application Setup Delivery Instruction Storage',119),(214,'1.2.840.10008.5.1.4.1.1.78.8','Intraocular Lens Calculations Storage',119),(215,'1.2.840.10008.5.1.4.1.1.11.8','Advanced Blending Presentation State Storage',119),(216,'1.2.840.10008.5.1.4.1.1.13.1.1','X-Ray 3D Angiographic Image Storage',119),(217,'1.2.840.10008.5.1.4.1.1.13.1.2','X-Ray 3D Craniofacial Image Storage',119),(218,'1.2.840.10008.5.1.4.1.1.77.1.5.5','Wide Field Ophthalmic Photography Stereographic Projection Image Storage',119),(219,'1.2.840.10008.5.1.4.1.1.481.7','RT Treatment Summary Record Storage',119),(220,'1.2.840.10008.5.1.4.1.1.9.6.1','Respiratory Waveform Storage',119),(221,'1.2.840.10008.5.1.4.43.1','Generic Implant Template',119),(222,'1.2.840.10008.5.1.4.1.1.128.1','Legacy Converted Enhanced PET Image Storage',119),(223,'1.2.840.10008.5.1.4.1.1.68.2','Surface Scan Point Cloud Storage',119),(224,'1.2.840.10008.5.1.4.1.1.77.1.5.2','Ophthalmic Photography 16 Bit Image Storage',119),(225,'tcga_ucec','TCGA-UCEC',120),(226,'tcga_thca','TCGA-THCA',120),(227,'tcga_stad','TCGA-STAD',120),(228,'tcga_sarc','TCGA-SARC',120),(229,'tcga_read','TCGA-READ',120),(230,'tcga_prad','TCGA-PRAD',120),(231,'tcga_ov','TCGA-OV',120),(232,'tcga_lusc','TCGA-LUSC',120),(233,'tcga_luad','TCGA-LUAD',120),(234,'tcga_lihc','TCGA-LIHC',120),(235,'tcga_lgg','TCGA-LGG',120),(236,'tcga_kirp','TCGA-KIRP',120),(237,'tcga_kirc','TCGA-KIRC',120),(238,'tcga_kich','TCGA-KICH',120),(239,'tcga_hnsc','TCGA-HNSC',120),(240,'tcga_gbm','TCGA-GBM',120),(241,'tcga_esca','TCGA-ESCA',120),(242,'tcga_coad','TCGA-COAD',120),(243,'tcga_cesc','TCGA-CESC',120),(244,'tcga_brca','TCGA-BRCA',120),(245,'tcga_blca','TCGA-BLCA',120),(246,'T-04000:SRT','Breast (SRT)',129),(247,'T-28000:SRT','Lung (SRT)',129),(248,'T-C4004:SRT','lymph node of head and neck (SRT)',129),(249,'T-62000:SRT','Liver (SRT)',129),(250,'T-C5300:SRT','pharyngeal tonsil (adenoid) (SRT)',129),(251,'T-53131:SRT','base of tongue (SRT)',129),(252,'T-55320:SRT','pyriform sinus (SRT)',129),(253,'T-42300:SRT','Aortic arch (SRT)',129),(254,'T-A6000:SRT','Cerebellum (SRT)',129),(255,'T-55300:SRT','hypopharynx (SRT)',129),(256,'T-55200:SRT','oropharynx (SRT)',129),(257,'T-23000:SRT','nasopharynx (SRT)',129),(258,'T-D0050:SRT','Tissue (SRT)',130),(259,'M-01000:SRT','Morphologically Altered Structure (SRT)',130),(260,'R-42018:SRT','Spatial and Relational Concept (SRT)',130),(261,'B-1000:MRSC','Breast (MRSC)',131),(262,'M-03010:SRT','Nodule (SRT)',131),(263,'M-80006:SRT','Neoplasm, Secondary (SRT)',131),(264,'C94970:NCIt','Reference Region (NCIt)',131),(265,'M-80003:SRT','Neoplasm, Primary (SRT)',131),(266,'M-03000:SRT','Mass (SRT)',131),(267,'39607008:SCT','Lung (SCT)',129),(268,'123037004:SCT','Anatomical Structure (SCT)',130),(269,'49755003:SCT','Morphologically abnormal structure (SCT)',130),(270,'32849002:SCT','Esophagus (SCT)',131),(271,'2748008:SCT','Spinal cord (SCT)',131),(272,'80891009:SCT','Heart (SCT)',131),(273,'39607008:SCT','Lung (SCT)',131),(274,'86049000:SCT','Neoplasm, Primary (SCT)',131),(275,'mouse_astrocytoma','Mouse-Astrocytoma',120),(276,'anti_pd_1_lung','Anti-PD-1_Lung',120),(277,'apollo','APOLLO-1-VA',120),(278,'rider_lung_pet_ct','RIDER Lung PET-CT',120),(279,'anti_pd_1_melanoma','Anti-PD-1_MELANOMA',120),(280,'qin_gbm_treatment_response','QIN GBM Treatment Response',120),(281,'qin_brain_dsc_mri','QIN-BRAIN-DSC-MRI',120),(282,'rider_neuro_mri','RIDER NEURO MRI',120),(283,'victre','VICTRE',120),(284,'rider_breast_mri','RIDER Breast MRI',120),(285,'c4kc_kits','C4KC-KiTS',120),(286,'mouse_mammary','Mouse-Mammary',120),(287,'dro_toolkit','DRO-Toolkit',120),(288,'cc_radiomics_phantom_2','CC-Radiomics-Phantom-2',120),(289,'qin_pet_phantom','QIN PET Phantom',120),(290,'qiba_ct_1c','QIBA CT-1C',120),(291,'rider_phantom_mri','RIDER Phantom MRI',120),(292,'pseudo_phi_dicom_data','Pseudo-PHI-DICOM-Data',120),(293,'naf_prostate','NaF PROSTATE',120),(294,'prostate_mri','PROSTATE-MRI',120),(295,'aapm_rt_mac','AAPM-RT-MAC',120),(296,'phantom_fda','Phantom FDA',120),(297,'midrc_ricord_1b','MIDRC-RICORD-1B',120),(298,'midrc_ricord_1c','MIDRC-RICORD-1C',120),(299,'midrc_ricord_1a','MIDRC-RICORD-1A',120),(300,'nsclc_radiomics_interobserver1','NSCLC-Radiomics-Interobserver1',120),(301,'acrin_nsclc_fdg_pet','ACRIN-NSCLC-FDG-PET',120),(302,'cptac_cm','CPTAC-CM',120),(303,'acrin_dsc_mr_brain','ACRIN-DSC-MR-Brain',120),(304,'acrin_fmiso_brain','ACRIN-FMISO-Brain',120),(305,'pdmr_997537_175_t','PDMR-997537-175-T',120),(306,'breast_cancer_screening_dbt','Breast-Cancer-Screening-DBT',120),(307,'acrin_flt_breast','ACRIN-FLT-Breast',120),(308,'qin_breast','QIN-BREAST',120),(309,'pdmr_833975_119_r','PDMR-833975-119-R',120),(310,'pdmr_292921_168_r','PDMR-292921-168-R',120),(311,'pdmr_425362_245_t','PDMR-425362-245-T',120),(312,'pdmr_bl0293_f563','PDMR-BL0293-F563',120),(313,'cptac_pda','CPTAC-PDA',120),(314,'head_neck_radiomics_hn1','HEAD-NECK-RADIOMICS-HN1',120),(315,'opc_radiomics','OPC-Radiomics',120),(316,'cptac_sar','CPTAC-SAR',120),(317,'icdc_glioma','ICDC-Glioma',120),(318,'lung_pet_ct_dx','Lung-PET-CT-Dx',120),(319,'lung_fused_ct_pathology','Lung-Fused-CT-Pathology',120),(320,'lctsc','LCTSC',120),(321,'qin_lung_ct','QIN LUNG CT',120),(322,'4d_lung','4D-Lung',120),(323,'brain_tumor_progression','Brain-Tumor-Progression',120),(324,'rider_lung_ct','RIDER Lung CT',120),(325,'ct_colonography','CT COLONOGRAPHY',120),(326,'cbis_ddsm','CBIS-DDSM',120),(327,'rider_phantom_pet_ct','RIDER Phantom PET-CT',120),(328,'pancreas_ct','Pancreas-CT',120),(329,'qin_prostate_repeatability','QIN-PROSTATE-Repeatability',120),(330,'prostate_mri_us_biopsy','Prostate-MRI-US-Biopsy',120),(331,'prostatex','PROSTATEx',120),(332,'prostate_fused_mri_pathology','Prostate Fused-MRI-Pathology',120),(333,'prostate_3t','Prostate-3T',120),(334,'hnscc_3dct_rt','HNSCC-3DCT-RT',120),(335,'head_neck_cetuximab','Head-Neck Cetuximab',120),(336,'cc_radiomics_phantom','CC-Radiomics-Phantom',120),(337,'lung_phantom','Lung Phantom',120),(338,'ct_lymph_nodes','CT Lymph Nodes',120),(339,'cc_radiomics_phantom_3','CC-Radiomics-Phantom-3',120),(340,'pelvic_reference_data','Pelvic-Reference-Data',120),(341,'vestibular_schwannoma_seg','Vestibular-Schwannoma-SEG',120),(342,'covid_19_ar','COVID-19-AR',120),(343,'nsclc_radiomics_genomics','NSCLC-Radiomics-Genomics',120),(344,'ivygap','IvyGAP',120),(345,'mri_dir','MRI-DIR',120),(346,'nsclc_radiomics','NSCLC-Radiomics',120),(347,'lungct_diagnosis','LungCT-Diagnosis',120),(348,'spie_aapm_lung_ct_challenge','SPIE-AAPM Lung CT Challenge',120),(349,'ispy1','ISPY1',120),(350,'breast_mri_nact_pilot','Breast-MRI-NACT-Pilot',120),(351,'qin_breast_dce_mri','QIN Breast DCE-MRI',120),(352,'breast_diagnosis','BREAST-DIAGNOSIS',120),(353,'qin_headneck','QIN-HEADNECK',120),(354,'hnscc','HNSCC',120),(355,'lidc_idri','LIDC-IDRI',120),(356,'prostate_diagnosis','PROSTATE-DIAGNOSIS',120),(357,'lgg_1p19qdeletion','LGG-1p19qDeletion',120),(358,'cptac_lscc','CPTAC-LSCC',120),(359,'cptac_gbm','CPTAC-GBM',120),(360,'cptac_hnscc','CPTAC-HNSCC',120),(361,'cptac_luad','CPTAC-LUAD',120),(362,'cptac_ccrcc','CPTAC-CCRCC',120),(363,'cptac_ucec','CPTAC-UCEC',120),(364,'rembrandt','REMBRANDT',120),(365,'nsclc_radiogenomics','NSCLC Radiogenomics',120),(366,'head_neck_pet_ct','Head-Neck-PET-CT',120),(367,'soft_tissue_sarcoma','Soft-tissue-Sarcoma',120),(368,'T-D0010:SRT','Entire body (SRT)',129),(369,'T-D1000:SRT','Head and Neck (SRT)',129),(370,'T-D05E4:SRT','Peripheral zone of the prostate (SRT)',129),(371,'A-00004:SRT','Physical Object (SRT)',129),(372,'T-D0050:SRT','Tissue (SRT)',129),(373,'T-D000A:SRT','Anatomical Structure (SRT)',130),(374,'A-00004:SRT','Physical Object (SRT)',130),(375,'717025007:SCT','Anterior fibromuscular stroma of prostate (SCT)',131),(376,'T-32000:SRT','Heart (SRT)',131),(377,'T-71000:SRT','Kidney (SRT)',131),(378,'M-01100:SRT','Lesion (SRT)',131),(379,'108369006:SCT','Neoplasm (SCT)',131),(380,'M-8FFFF:SRT','Neoplasm (SRT)',131),(381,'G-A460:SRT','Normal (SRT)',131),(382,'279706003:SCT','Peripheral zone of prostate (SCT)',131),(383,'T-D05E4:SRT','Peripheral zone of the prostate (SRT)',131),(384,'41216001:SCT','Prostate (SCT)',131),(385,'T-9200B:SRT','Prostate (SRT)',131),(386,'71553001:SCT','Prostatic Urethra (SCT)',131),(387,'T-D0050:SRT','Tissue (SRT)',131),(388,'399384005:SCT','Transition zone of prostate (SCT)',131),(389,'T-D820B:SRT','upper arm (SRT)',131),(390,'1.2.840.10008.5.1.4.1.1.12.2','X-Ray Radiofluoroscopic Image Storage',119),(391,'1.2.840.10008.5.1.4.1.1.3.1','Ultrasound Multi-frame Image Storage',119),(392,'1.2.840.10008.5.1.4.1.1.6.1','Ultrasound Image Storage',119),(393,'SC','Secondary Capture',107),(394,'acrin_hnscc_fdg_pet_ct','ACRIN-HNSCC-FDG-PET-CT',120),(395,'apollo_5_lscc','APOLLO-5-LSCC',120),(396,'b_mode_and_ceus_liver','B-mode-and-CEUS-Liver',120),(397,'cmmd','CMMD',120),(398,'covid_19_ny_sbu','COVID-19-NY-SBU',120),(399,'nlst','NLST',120),(400,'duke_breast_cancer_mri','Duke-Breast-Cancer-MRI',120),(401,'T-A0100:SRT','Brain (SRT)',129),(402,'M-54000:SRT','Necrosis (SRT)',131),(403,'M-36300:SRT','Edema (SRT)',131),(404,'C113842:NCIt','Enhancing Lesion (NCIt)',131),(405,'apollo_5_thym','APOLLO-5-THYM',120),(406,'apollo_5_luad','APOLLO-5-LUAD',120),(407,'pancreatic_ct_cbct_seg','Pancreatic-CT-CBCT-SEG',120),(408,'apollo_5_paad','APOLLO-5-PAAD',120),(409,'cptac_aml','CPTAC-AML',120),(410,'pediatric_ct_seg','Pediatric-CT-SEG',120),(411,'cptac_brca','CPTAC-BRCA',120),(412,'apollo_5_esca','APOLLO-5-ESCA',120),(413,'cptac_coad','CPTAC-COAD',120),(414,'cptac_ov','CPTAC-OV',120),(415,'tcga_chol','TCGA-CHOL',120),(416,'tcga_uvm','TCGA-UVM',120),(417,'tcga_thym','TCGA-THYM',120),(418,'tcga_ucs','TCGA-UCS',120),(419,'tcga_acc','TCGA-ACC',120),(420,'tcga_skcm','TCGA-SKCM',120),(421,'tcga_paad','TCGA-PAAD',120),(422,'tcga_pcpg','TCGA-PCPG',120),(423,'tcga_dlbc','TCGA-DLBC',120),(424,'tcga_meso','TCGA-MESO',120),(425,'tcga_tgct','TCGA-TGCT',120),(426,'acrin_contralateral_breast_mr','ACRIN-Contralateral-Breast-MR',120),(427,'stageii_colorectal_ct','StageII-Colorectal-CT',120),(428,'76752008:SCT','Breast (SCT)',129),(429,'85756007:SCT','Tissue (SCT)',130),(430,'89749009:SCT','Fatty tissue of breast (SCT)',131),(431,'B-6000:MRSC','DWI Tumor (MRSC)',131),(432,'C114467:NCIt','Mammary Fibroglandular Tissue (NCIt)',131),(433,'htan_wustl','HTAN-WUSTL',120),(434,'htan_hms','HTAN-HMS',120),(435,'ispy2','ISPY2',120),(436,'acrin_6698','ACRIN-6698',120),(437,'htan_ohsu','HTAN-OHSU',120),(438,'upenn_gbm','UPENN-GBM',120),(439,'cmb_lca','CMB-LCA',120),(440,'pdmr_521955_158_r4','PDMR-521955-158-R4',120),(441,'cmb_crc','CMB-CRC',120),(442,'hcc_tace_seg','HCC-TACE-Seg',120),(443,'cmb_mel','CMB-MEL',120),(444,'gbm_dsc_mri_dro','GBM-DSC-MRI-DRO',120),(445,'cmb_mml','CMB-MML',120),(446,'cmb_gec','CMB-GEC',120),(447,'cmb_pca','CMB-PCA',120),(448,'7832008:SCT','Abdominal aorta (SCT)',131),(449,'4147007:SCT','Mass (SCT)',131),(450,'32764006:SCT','Portal vein (SCT)',131),(451,'10200004:SCT','Liver (SCT)',131),(452,'6574001:SCT','Necrosis (SCT)',131),(453,'38266002','Entire body',129),(454,'279706003','Peripheral zone of the prostate',129),(455,'260787004','Physical Object',129),(456,'76752008','Breast',129),(457,'39607008','Lung',129),(458,'85756007','Tissue',129),(459,'49755003','Morphologically Abnormal Structure',130),(460,'260787004','Physical Object',130),(461,'123037004','Anatomical Structure',130),(462,'85756007','Tissue',130),(463,'86049000','Neoplasm, Primary',131),(464,'17621005','Normal',131),(465,'32764006','Portal vein',131),(466,'10200004','Liver',131),(467,'27925004','Nodule',131),(468,'279706003','Peripheral zone of prostate',131),(469,'2748008','Spinal cord',131),(470,'80891009','Heart',131),(471,'85756007','Tissue',131),(472,'64033007','Kidney',131),(473,'717025007','Anterior fibromuscular stroma of prostate',131),(474,'108369006','Neoplasm',131),(475,'52988006','Lesion',131),(476,'6574001','Necrosis',131),(477,'32849002','Esophagus',131),(478,'399384005','Transition zone of prostate',131),(479,'39607008','Lung',131),(480,'71553001','Prostatic Urethra',131),(481,'41216001','Prostate',131),(482,'4147007','Mass',131),(483,'7832008','Abdominal aorta',131),(484,'ct_vs_pet_ventilation_imaging','CT-vs-PET-Ventilation-Imaging',120),(485,'ctpred_sunitinib_pannet','CTpred-Sunitinib-panNET',120),(486,'15825003','Aorta',131),(487,'44567001','Trachea',131),(488,'DCM:111743','Epifluorescence illumination',169),(489,'DCM:111744','Brightfield illumination',169),(490,'SCT:39937001','Skin',170),(491,'SCT:71854001','Colon',170),(492,'SCT:71400007','Mesothelium',170),(493,'SCT:28273000','Bile duct',170),(494,'SCT:64163001','Head of pancreas',170),(495,'SCT:12738006','Brain',170),(496,'SCT:10200004','Liver',170),(497,'SCT:74862005','Uvea',170),(498,'SCT:71252005','Cervix',170),(499,'SCT:32713005','Cecum',170),(500,'SCT:35039007','Uterus',170),(501,'SCT:34402009','Rectum',170),(502,'SCT:89837001','Bladder',170),(503,'SCT:14016003','Bone marrow',170),(504,'SCT:68594002','Adrenal cortex',170),(505,'SCT:40689003','Testis',170),(506,'SCT:774007','Head and Neck',170),(507,'SCT:73239005','Tail of pancreas',170),(508,'SCT:6969002','Lymphoid tissue',170),(509,'SCT:64033007','Kidney',170),(510,'SCT:49832006','Rectosigmoid junction',170),(511,'SCT:40133006','Body of pancreas',170),(512,'SCT:9875009','Thymus',170),(513,'SCT:76752008','Breast',170),(514,'SCT:69695003','Stomach',170),(515,'SCT:15776009','Pancreas',170),(516,'SCT:69748006','Thyroid',170),(517,'SCT:485005','Transverse colon',170),(518,'SCT:41216001','Prostate',170),(519,'SCT:15497006','Ovary',170),(520,'SCT:32849002','Esophagus',170),(521,'SCT:66754008','Appendix',170),(522,'SCT:39607008','Lung',170),(523,'SCT:60184004','Sigmoid colon',170),(524,'SCT:272673000','Bone',170),(525,'29092000','Vein',131),(526,'62683002','mediastinal lymph node',131),(527,'SCT:9040008','Ascending colon',170),(528,'SCT:48338005','Hepatic flexure of colon',170),(529,'SCT:32622004','Descending colon',170),(530,'htan_vanderbilt','HTAN-VANDERBILT',120),(531,'rider_pilot','RIDER Pilot',120),(532,'cc_tumor_heterogeneity','CC-Tumor-Heterogeneity',120),(533,'adrenal_acc_ki67_seg','Adrenal-ACC-Ki67-Seg',120),(534,'colorectal_liver_metastases','Colorectal-Liver-Metastases',120),(535,'prostate_anatomical_edge_cases','Prostate-Anatomical-Edge-Cases',120),(536,'nlm_visible_human_project','NLM-Visible-Human-Project',120),(537,'10200004','Liver',129),(538,'pdmr_texture_analysis','PDMR-Texture-Analysis',120),(539,'rms_mutation_prediction','RMS-Mutation-Prediction',120),(540,'SCT:15425007','Peritoneum',170),(541,'SCT:15924003','Maxillary sinus',170),(542,'SCT:72410000','Mediastinum',170),(543,'SCT:80243003','Eyelid',170),(544,'SCT:20233005','Scrotum',170),(545,'SCT:45048000','Neck',170),(546,'SCT:279549004','Nasal cavity',170),(547,'SCT:26140008','Uvula',170),(548,'SCT:45289007','Parotid gland',170),(549,'SCT:113327001','Pinna',170),(550,'SCT:789564000','Retropharyngeal space',170),(551,'SCT:261148001','Paraspinal',170),(552,'SCT:818981001','Abdomen',170),(553,'SCT:34707002','Bilary tract',170),(554,'SCT:16982005','Shoulder',170),(555,'SCT:56459004','Foot',170),(556,'SCT:13170006','Perirectal region',170),(557,'SCT:363654007','Orbit',170),(558,'SCT:38864007','Perineum',170),(559,'SCT:53840002','Calf',170),(560,'SCT:31389004','Oropharynx',170),(561,'SCT:71341001','Femur',170),(562,'SCT:49460000','Soft palate',170),(563,'SCT:46862004','Buttock',170),(564,'SCT:59441001','Lymph node',170),(565,'SCT:117590005','Ear',170),(566,'SCT:45292006','Vulva',170),(567,'SCT:123851003','Mouth',170),(568,'SCT:397158004','Perianal region',170),(569,'SCT:91470000','Axilla',170),(570,'SCT:37446007','Periprostatic tissue',170),(571,'SCT:421060004','Spine',170),(572,'SCT:77568009','Back',170),(573,'SCT:40983000','Upper arm',170),(574,'SCT:84765007','Pterygoid region',170),(575,'SCT:31467002','Base of skull',170),(576,'SCT:87644002','Epididymis',170),(577,'SCT:76784001','Vagina',170),(578,'SCT:68367000','Thigh',170),(579,'SCT:16953009','Elbow joint',170),(580,'SCT:25342003','Middle ear',170),(581,'SCT:30021000','Lower leg',170),(582,'SCT:74670003','Wrist joint',170),(583,'SCT:43799004','Chest',170),(584,'SCT:14975008','Forearm',170),(585,'SCT:69541002','Infratemporal fossa',170),(586,'SCT:60819002','Cheek',170),(587,'SCT:84301002','External auditory canal',170),(588,'SCT:41695006','Scalp',170),(589,'SCT:82849001','Retroperitoneum',170),(590,'SCT:818982008','Abdomen and pelvis',170),(591,'SCT:26893007','Inguinal region',170),(592,'SCT:360955006','Nasopharynx',170),(593,'SCT:450721000','Temporal region',170),(594,'SCT:822992007','Abdominal wall',170),(595,'SCT:91609006','Mandible',170),(596,'SCT:76738006','Pterygoid muscle',170),(597,'SCT:54066008','Pharynx',170),(598,'SCT:12611008','Tibia',170),(599,'SCT:69536005','Head',170),(600,'SCT:49957000','Spermatic cord',170),(601,'SCT:816092008','Pelvis',170),(602,'SCT:17861009','Oropharyngeal tonsil',170),(603,'SCT:30291003','Chin',170),(604,'SCT:85562004','Hand',170),(605,'SCT:45206002','Nose',170),(606,'64033007','Kidney',129),(607,'12738006','Brain',129),(608,'35764002','Brain ventricle',131),(609,'367643001','Cyst',131),(610,'12738006','Brain',131),(611,'vestibular_schwannoma_mc_rc','Vestibular-Schwannoma-MC-RC',120),(612,'cmb_aml','CMB-AML',120),(613,'ea1141','EA1141',120),(614,'ct_phantom4radiomics','CT-Phantom4Radiomics',120),(615,'remind','ReMIND',120),(616,'309825002','Spatial and Relational Concept',130),(617,'SCT:39607008','Lung',129),(618,'SCT:76752008','Breast',129),(619,'SCT:85756007','Tissue',129),(620,'SCT:12738006','Brain',129),(621,'SCT:64033007','Kidney',129),(622,'SCT:260787004','Physical Object',129),(623,'SCT:38266002','Entire body',129),(624,'SCT:10200004','Liver',129),(625,'SCT:279706003','Peripheral zone of the prostate',129),(626,'SCT:89545001','Face',131),(627,'SCT:90572001','Lower lobe of lung',131),(628,'SCT:49668003','L5 vertebra',131),(629,'SCT:73829009','Right atrium',131),(630,'SCT:7832008','Abdominal aorta',131),(631,'SCT:717025007','Anterior fibromuscular stroma of prostate',131),(632,'SCT:11994002','L4 vertebra',131),(633,'SCT:32764006','Portal vein',131),(634,'SCT:113205007','C3 vertebra',131),(635,'SCT:15825003','Aorta',131),(636,'SCT:4147007','Mass',131),(637,'SCT:86049000','Neoplasm, Primary',131),(638,'SCT:73634005','Common iliac artery',131),(639,'SCT:279706003','Peripheral zone of the prostate',131),(640,'SCT:72481006','Middle lobe of right lung',131),(641,'SCT:82471001','Left atrium',131),(642,'SCT:64033007','Kidney',131),(643,'SCT:85756007','Tissue',131),(644,'SCT:82687006','T9 vertebra',131),(645,'SCT:12989004','T11 vertebra',131),(646,'SCT:53085002','Right ventricle of heart',131),(647,'SCT:89837001','Urinary bladder',131),(648,'SCT:1626008','T3 vertebra',131),(649,'SCT:367643001','Cyst',131),(650,'SCT:80891009','Heart',131),(651,'SCT:39607008','Lung',131),(652,'SCT:71854001','Colon',131),(653,'SCT:54735007','Sacrum',131),(654,'SCT:35764002','Brain ventricle',131),(655,'SCT:39976000','C2 vertebra',131),(656,'SCT:73071006','T4 vertebra',131),(657,'SCT:17621005','Normal',131),(658,'SCT:2748008','Spinal cord',131),(659,'SCT:15776009','Pancreas',131),(660,'SCT:52988006','Lesion',131),(661,'SCT:7610001','T10 vertebra',131),(662,'SCT:56401006','T5 vertebra',131),(663,'SCT:29836001','Hip',131),(664,'SCT:45653009','Upper lobe of lung',131),(665,'SCT:87391001','C7 vertebra',131),(666,'SCT:62683002','mediastinal lymph node',131),(667,'SCT:87878005','Left ventricle of heart',131),(668,'SCT:45296009','T6 vertebra',131),(669,'SCT:66794005','L1 vertebra',131),(670,'SCT:44567001','Trachea',131),(671,'SCT:38848004','Duodenum',131),(672,'SCT:36470004','L3 vertebra',131),(673,'SCT:64131007','Inferior vena cava',131),(674,'SCT:41216001','Prostate',131),(675,'SCT:14293000','L2 vertebra',131),(676,'SCT:36054005','C6 vertebra',131),(677,'SCT:64864005','T1 vertebra',131),(678,'SCT:108369006','Neoplasm',131),(679,'SCT:81040000','Pulmonary artery',131),(680,'SCT:5329002','C4 vertebra',131),(681,'SCT:29092000','Vein',131),(682,'SCT:79601000','Scapula',131),(683,'SCT:30315005','Small Intestine',131),(684,'SCT:32849002','Esophagus',131),(685,'SCT:14806007','C1 vertebra',131),(686,'SCT:62487009','T7 vertebra',131),(687,'SCT:12738006','Brain',131),(688,'SCT:11068009','T8 vertebra',131),(689,'SCT:27925004','Nodule',131),(690,'SCT:23215003','T12 vertebra',131),(691,'SCT:28231008','Gallbladder',131),(692,'SCT:85050009','Humerus',131),(693,'SCT:71341001','Femur',131),(694,'SCT:78961009','Spleen',131),(695,'SCT:10200004','Liver',131),(696,'SCT:6574001','Necrosis',131),(697,'SCT:36978003','C5 vertebra',131),(698,'SCT:71553001','Prostatic Urethra',131),(699,'SCT:46027005','Common iliac vein',131),(700,'SCT:399384005','Transition zone of prostate',131),(701,'SCT:51299004','Clavicle',131),(702,'SCT:69695003','Stomach',131),(703,'SCT:53733008','T2 vertebra',131),(704,'SCT:23451007','Adrenal gland',131),(705,'advanced_mri_breast_lesions','Advanced-MRI-Breast-Lesions',120),(706,'SCT:309825002','Spatial and Relational Concept',130),(707,'SCT:85756007','Tissue',130),(708,'SCT:49755003','Morphologically Abnormal Structure',130),(709,'SCT:123037004','Anatomical Structure',130),(710,'SCT:260787004','Physical Object',130),(711,'1.2.840.10008.5.1.4.1.1.91.1','Microscopy Bulk Simple Annotations Storage',119),(712,'1.2.840.10008.5.1.4.1.1.481.16','RT Radiation Record Set Storage',119),(713,'1.2.840.10008.5.1.4.1.1.9.3.1','Cardiac Electrophysiology Waveform Storage',119),(714,'1.2.840.10008.5.1.4.1.1.481.25','RT Patient Position Acquisition Instruction Storage',119),(715,'1.2.840.10008.5.1.4.1.1.77.1.9','Confocal Microscopy Tiled Pyramidal Image Storage',119),(716,'1.2.840.10008.5.1.4.1.1.9.7.2','Electromyogram Waveform Storage',119),(717,'1.2.840.10008.5.1.4.1.1.200.2','CT Performed Procedure Protocol Storage',119),(718,'1.2.840.10008.5.1.4.1.1.104.4','Encapsulated OBJ Storage',119),(719,'1.2.840.10008.5.1.4.1.1.9.7.3','Electrooculogram Waveform Storage',119),(720,'1.2.840.10008.5.1.4.1.1.481.17','RT Radiation Salvage Record Storage',119),(721,'1.2.840.10008.5.1.4.1.1.11.7','Compositing Planar MPR Volumetric Presentation State Storage',119),(722,'1.2.840.10008.5.1.4.1.1.7.2','Multi-frame Grayscale Byte Secondary Capture Image Storage',119),(723,'1.2.840.10008.5.1.4.1.1.104.5','Encapsulated MTL Storage',119),(724,'1.2.840.10008.5.1.4.1.1.481.19','C-Arm Photon-Electron Radiation Record Storage',119),(725,'1.2.840.10008.5.1.4.1.1.481.24','Enhanced Continuous RT Image Storage',119),(726,'1.2.840.10008.5.1.4.1.1.7.4','Multi-frame True Color Secondary Capture Image Storage',119),(727,'1.2.840.10008.5.1.4.1.1.481.20','Robotic Radiation Record Storage',119),(728,'1.2.840.10008.5.1.4.1.1.12.2.1','Enhanced XRF Image Storage',119),(729,'1.2.840.10008.5.1.4.1.1.9.8.1','Body Position Waveform Storage',119),(730,'1.2.840.10008.5.1.4.1.1.88.76','Enhanced X-Ray Radiation Dose SR Storage',119),(731,'1.2.840.10008.5.1.4.1.1.9.1.3','Ambulatory ECG Waveform Storage',119),(732,'1.2.840.10008.5.1.4.1.1.481.18','Tomotherapeutic Radiation Record Storage',119),(733,'1.2.840.10008.5.1.4.1.1.481.14','Tomotherapeutic Radiation Storage',119),(734,'1.2.840.10008.5.1.4.1.1.9.2.1','Hemodynamic Waveform Storage',119),(735,'1.2.840.10008.5.1.4.1.1.7.3','Multi-frame Grayscale Word Secondary Capture Image Storage',119),(736,'1.2.840.10008.5.1.4.1.1.77.1.7','Dermoscopic Photography Image Storage',119),(737,'1.2.840.10008.5.1.4.1.1.9.1.1','12-lead ECG Waveform Storage',119),(738,'1.2.840.10008.5.1.4.1.1.9.7.1','Routine Scalp Electroencephalogram Waveform Storage',119),(739,'1.2.840.10008.5.1.4.1.1.14.2','Intravascular Optical Coherence Tomography Image Storage - For Processing',119),(740,'1.2.840.10008.5.1.4.1.1.481.23','Enhanced RT Image Storage',119),(741,'1.2.840.10008.5.1.4.1.1.200.8','XA Performed Procedure Protocol Storage',119),(742,'1.2.840.10008.5.1.4.1.1.481.15','Robotic-Arm Radiation Storage',119),(743,'1.2.840.10008.5.1.4.1.1.9.4.1','Basic Voice Audio Waveform Storage',119),(744,'1.2.840.10008.5.1.4.1.1.11.10','Segmented Volume Rendering Volumetric Presentation State Storage',119),(745,'1.2.840.10008.5.1.4.1.1.77.1.8','Confocal Microscopy Image Storage',119),(746,'1.2.840.10008.5.1.4.1.1.11.11','Multiple Volume Rendering Volumetric Presentation State Storage',119),(747,'1.2.840.10008.5.1.4.1.1.11.12','Variable Modality LUT Softcopy Presentation State Storage',119),(748,'1.2.840.10008.5.1.4.1.1.14.1','Intravascular Optical Coherence Tomography Image Storage - For Presentation',119),(749,'1.2.840.10008.5.1.4.1.1.9.1.2','General ECG Waveform Storage',119),(750,'1.2.840.10008.5.1.4.1.1.7.1','Multi-frame Single Bit Secondary Capture Image Storage',119),(751,'1.2.840.10008.5.1.4.1.1.88.74','Planned Imaging Agent Administration SR Storage',119),(752,'1.2.840.10008.5.1.4.1.1.6.3','Photoacoustic Image Storage',119),(753,'1.2.840.10008.5.1.4.1.1.9.6.2','Multi-channel Respiratory Waveform Storage',119),(754,'1.2.840.10008.5.1.4.1.1.9.7.4','Sleep Electroencephalogram Waveform Storage',119),(755,'1.2.840.10008.5.1.4.1.1.481.21','RT Radiation Set Delivery Instruction Storage',119),(756,'1.2.840.10008.5.1.4.1.1.9.1.4','General 32-bit ECG Waveform Storage',119),(757,'1.2.840.10008.5.1.4.1.1.12.1.1','Enhanced XA Image Storage',119),(758,'1.2.840.10008.5.1.4.1.1.11.6','Grayscale Planar MPR Volumetric Presentation State Storage',119),(759,'1.2.840.10008.5.1.4.1.1.481.22','RT Treatment Preparation Storage',119),(760,'1.2.840.10008.5.1.4.1.1.88.73','Patient Radiation Dose SR Storage',119);
/*!40000 ALTER TABLE `idc_collections_attribute_display_values` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_attribute_ranges`
--

DROP TABLE IF EXISTS `idc_collections_attribute_ranges`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_attribute_ranges` (
  `id` int NOT NULL AUTO_INCREMENT,
  `type` varchar(1) NOT NULL,
  `include_lower` tinyint(1) NOT NULL,
  `include_upper` tinyint(1) NOT NULL,
  `unbounded` tinyint(1) NOT NULL,
  `first` varchar(128) NOT NULL,
  `last` varchar(128) NOT NULL,
  `gap` varchar(128) NOT NULL,
  `label` varchar(256) DEFAULT NULL,
  `attribute_id` int NOT NULL,
  PRIMARY KEY (`id`),
  KEY `idc_collections_attr_attribute_id_8f39a215_fk_idc_colle` (`attribute_id`),
  CONSTRAINT `idc_collections_attr_attribute_id_8f39a215_fk_idc_colle` FOREIGN KEY (`attribute_id`) REFERENCES `idc_collections_attribute` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=132 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_attribute_ranges`
--

LOCK TABLES `idc_collections_attribute_ranges` WRITE;
/*!40000 ALTER TABLE `idc_collections_attribute_ranges` DISABLE KEYS */;
INSERT INTO `idc_collections_attribute_ranges` VALUES (1,'I',1,0,1,'10','80','10',NULL,12),(2,'I',1,0,1,'-15000','-5000','3000',NULL,13),(3,'I',0,1,1,'500','6000','500',NULL,14),(4,'I',1,0,1,'10','80','10',NULL,15),(5,'I',0,1,1,'500','6000','500',NULL,16),(6,'I',0,1,1,'500','6000','500',NULL,17),(7,'I',1,0,1,'10','80','10',NULL,18),(8,'I',1,0,0,'1976','2015','5',NULL,27),(9,'I',1,0,1,'10','80','10',NULL,48),(10,'I',1,0,1,'10','80','10',NULL,49),(11,'I',1,0,1,'10','80','10',NULL,54),(12,'I',1,0,1,'10','80','10',NULL,56),(13,'I',1,0,1,'10','80','10',NULL,57),(14,'I',1,0,1,'10','80','10',NULL,58),(15,'I',1,0,1,'10','80','10',NULL,59),(16,'I',1,0,1,'10','80','10',NULL,60),(17,'I',1,0,1,'10','80','10',NULL,61),(18,'I',1,0,1,'10','80','10',NULL,62),(19,'F',1,0,1,'*','18.5','0','underweight',63),(20,'F',1,1,1,'30','*','0','obese',63),(21,'F',1,0,1,'18.5','25','0','normal weight',63),(22,'F',1,0,1,'25','30','0','overweight',63),(23,'I',1,0,1,'10','80','10',NULL,70),(24,'I',1,0,1,'10','80','10',NULL,71),(25,'I',1,0,1,'10','80','10',NULL,78),(26,'I',1,0,1,'10','80','10',NULL,79),(27,'I',1,0,1,'10','80','10',NULL,81),(28,'I',1,0,1,'10','80','10',NULL,82),(29,'I',1,0,1,'10','80','10',NULL,83),(30,'I',1,0,1,'10','80','10',NULL,84),(31,'I',1,0,1,'10','80','10',NULL,85),(32,'I',1,0,1,'10','80','10',NULL,86),(33,'I',1,0,1,'10','80','10',NULL,87),(34,'I',1,0,1,'10','80','10',NULL,88),(35,'I',1,0,1,'10','80','10',NULL,89),(36,'I',1,0,1,'10','80','10',NULL,90),(37,'I',1,0,1,'10','80','10',NULL,91),(38,'I',1,0,1,'10','80','10',NULL,92),(39,'I',1,0,1,'10','80','10',NULL,93),(40,'I',1,0,1,'10','80','10',NULL,94),(41,'I',1,0,1,'10','80','10',NULL,95),(42,'I',1,0,1,'10','80','10',NULL,96),(43,'I',1,0,1,'10','80','10',NULL,97),(44,'I',1,0,1,'10','80','10',NULL,98),(45,'I',1,0,1,'10','80','10',NULL,99),(46,'I',1,0,1,'10','80','10',NULL,100),(47,'I',1,0,1,'10','80','10',NULL,101),(48,'I',1,0,1,'10','80','10',NULL,102),(49,'I',1,0,1,'10','80','10',NULL,103),(50,'I',1,0,1,'10','80','10',NULL,104),(51,'I',1,0,1,'10','80','10',NULL,105),(52,'I',1,0,1,'10','80','10',NULL,106),(53,'I',1,0,1,'10','80','10',NULL,133),(54,'I',1,0,1,'3','12','1',NULL,135),(55,'I',1,0,1,'10','28000','2800',NULL,136),(56,'I',1,0,1,'6','55','6',NULL,137),(57,'I',1,0,1,'150','4500','435',NULL,138),(58,'I',1,0,1,'78','698','78',NULL,139),(59,'I',1,0,1,'56','502','56',NULL,140),(60,'I',1,0,1,'10','90','10',NULL,141),(61,'I',1,0,1,'10','90','10',NULL,142),(62,'I',1,0,1,'10','90','10',NULL,143),(63,'I',1,0,1,'10','90','10',NULL,144),(64,'F',1,0,1,'0.5','5','0.5',NULL,145),(65,'I',1,0,1,'28','251','28',NULL,146),(66,'I',1,0,1,'25','225','25',NULL,147),(67,'I',1,0,1,'19','170','19',NULL,148),(68,'I',1,0,1,'25','227','25',NULL,149),(69,'F',1,0,1,'0','1900','200',NULL,116),(70,'F',1,0,1,'2000','1610000','150000',NULL,167),(71,'F',1,0,1,'0.3','0.9','0.05',NULL,168),(75,'F',1,0,1,'0','200','0',NULL,173),(76,'F',1,0,1,'200','400','0',NULL,173),(77,'F',1,0,1,'400','500','0',NULL,173),(78,'F',1,0,1,'500','600','0',NULL,173),(79,'F',1,0,1,'600','800','0',NULL,173),(80,'F',1,0,1,'800','1000','0',NULL,173),(81,'F',1,0,1,'1000','2000','0',NULL,173),(82,'F',1,0,1,'2000','4000','0',NULL,173),(83,'F',1,0,1,'4000','10000','0',NULL,173),(84,'F',1,0,1,'10000','12000','0',NULL,173),(85,'F',1,0,1,'12000','14000','0',NULL,173),(86,'F',1,0,1,'14000','16000','0',NULL,173),(87,'F',1,0,1,'16000','18000','0',NULL,173),(88,'F',1,0,1,'18000','20000','0',NULL,173),(89,'F',1,0,1,'20000','23000','0',NULL,173),(90,'F',1,0,1,'0','250','0',NULL,174),(91,'F',1,0,1,'250','500','0',NULL,174),(92,'F',1,0,1,'500','750','0',NULL,174),(93,'F',1,0,1,'750','1000','0',NULL,174),(94,'F',1,0,1,'1000','2500','0',NULL,174),(95,'F',1,0,1,'2500','5000','0',NULL,174),(96,'F',1,0,1,'5000','7500','0',NULL,174),(97,'F',1,0,1,'7500','10000','0',NULL,174),(98,'F',1,0,1,'10000','25000','0',NULL,174),(99,'F',1,0,1,'25000','50000','0',NULL,174),(100,'F',1,0,1,'50000','75000','0',NULL,174),(101,'F',1,0,1,'75000','100000','0',NULL,174),(102,'F',1,0,1,'100000','125000','0',NULL,174),(103,'F',1,0,1,'125000','150000','0',NULL,174),(104,'F',1,0,1,'150000','175000','0',NULL,174),(105,'F',1,0,1,'200000','225000','0',NULL,174),(106,'F',1,0,1,'225000','260000','0',NULL,174),(107,'F',1,0,1,'0','0.0002','0',NULL,172),(108,'F',1,0,1,'0.0002','0.0004','0',NULL,172),(109,'F',1,0,1,'0.0004','0.0006','0',NULL,172),(110,'F',1,0,1,'0.0006','0.0008','0',NULL,172),(111,'F',1,0,1,'0.0008','0.001','0',NULL,172),(112,'F',1,0,1,'0.001','0.02','0',NULL,172),(113,'F',1,0,1,'0.02','0.04','0',NULL,172),(114,'F',1,0,1,'0.04','0.06','0',NULL,172),(115,'F',1,0,1,'0.06','0.08','0',NULL,172),(116,'F',1,0,1,'0.08','0.1','0',NULL,172),(117,'F',1,0,1,'0.1','0.2','0',NULL,172),(118,'F',1,0,1,'0.2','0.4','0',NULL,172),(119,'F',1,0,1,'0.4','0.6','0',NULL,172),(120,'F',1,0,1,'0.6','0.8','0',NULL,172),(121,'F',1,0,1,'0.8','1','0',NULL,172),(122,'F',1,0,1,'1','3','0',NULL,172),(123,'F',1,0,1,'3','5','0',NULL,172),(124,'F',1,0,1,'5','7','0',NULL,172),(125,'F',1,1,1,'7','9','0',NULL,172),(126,'I',1,0,1,'23000','40000','0',NULL,173),(127,'I',1,0,1,'40000','80000','0',NULL,173),(128,'I',1,0,1,'80000','100000','0',NULL,173),(129,'I',1,0,1,'150000','180000','0',NULL,173),(130,'I',1,0,1,'180000','230000','0',NULL,173),(131,'I',1,0,1,'100000','150000','0',NULL,173);
/*!40000 ALTER TABLE `idc_collections_attribute_ranges` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_attribute_set_type`
--

DROP TABLE IF EXISTS `idc_collections_attribute_set_type`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_attribute_set_type` (
  `id` int NOT NULL AUTO_INCREMENT,
  `child_record_search` varchar(256) DEFAULT NULL,
  `attribute_id` int NOT NULL,
  `datasettype_id` int NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `idc_collections_attribut_datasettype_id_attribute_79d3f0be_uniq` (`datasettype_id`,`attribute_id`),
  KEY `idc_collections_attr_attribute_id_b4a83520_fk_idc_colle` (`attribute_id`),
  CONSTRAINT `idc_collections_attr_attribute_id_b4a83520_fk_idc_colle` FOREIGN KEY (`attribute_id`) REFERENCES `idc_collections_attribute` (`id`),
  CONSTRAINT `idc_collections_attr_datasettype_id_fbc0dd7e_fk_idc_colle` FOREIGN KEY (`datasettype_id`) REFERENCES `idc_collections_datasettype` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=180 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_attribute_set_type`
--

LOCK TABLES `idc_collections_attribute_set_type` WRITE;
/*!40000 ALTER TABLE `idc_collections_attribute_set_type` DISABLE KEYS */;
INSERT INTO `idc_collections_attribute_set_type` VALUES (1,NULL,1,2),(2,NULL,2,2),(3,NULL,3,2),(4,NULL,4,2),(5,NULL,5,2),(6,NULL,6,2),(7,NULL,7,2),(8,NULL,8,2),(9,NULL,9,2),(10,NULL,10,2),(11,NULL,11,2),(12,NULL,12,2),(13,NULL,13,2),(14,NULL,14,2),(15,NULL,15,2),(16,NULL,16,2),(17,NULL,17,2),(18,NULL,18,2),(19,NULL,19,2),(20,NULL,20,2),(21,NULL,21,2),(22,NULL,22,2),(23,NULL,23,2),(24,NULL,24,2),(25,NULL,25,2),(26,NULL,26,2),(27,NULL,27,2),(28,NULL,28,2),(29,NULL,29,2),(30,NULL,30,2),(31,NULL,31,2),(32,NULL,32,2),(33,NULL,33,2),(34,NULL,34,2),(35,NULL,35,2),(36,NULL,36,2),(37,NULL,37,2),(38,NULL,38,2),(39,NULL,39,2),(40,NULL,40,2),(41,NULL,41,2),(42,NULL,42,2),(43,NULL,43,2),(44,NULL,44,2),(45,NULL,45,2),(46,NULL,46,2),(47,NULL,47,2),(48,NULL,48,2),(49,NULL,49,2),(50,NULL,50,2),(51,NULL,51,2),(52,NULL,52,2),(53,NULL,53,2),(54,NULL,54,2),(55,NULL,55,2),(56,NULL,56,2),(57,NULL,57,2),(58,NULL,58,2),(59,NULL,59,2),(60,NULL,60,2),(61,NULL,61,2),(62,NULL,62,2),(63,NULL,63,2),(64,NULL,64,2),(65,NULL,65,2),(66,NULL,66,2),(67,NULL,67,2),(68,NULL,68,2),(69,NULL,69,2),(70,NULL,70,2),(71,NULL,71,2),(72,NULL,72,2),(73,NULL,73,2),(74,NULL,74,2),(75,NULL,75,2),(76,NULL,76,2),(77,NULL,77,2),(78,NULL,78,2),(79,NULL,79,2),(80,NULL,80,2),(81,NULL,81,2),(82,NULL,82,2),(83,NULL,83,2),(84,NULL,84,2),(85,NULL,85,2),(86,NULL,86,2),(87,NULL,87,2),(88,NULL,88,2),(89,NULL,89,2),(90,NULL,90,2),(91,NULL,91,2),(92,NULL,92,2),(93,NULL,93,2),(94,NULL,94,2),(95,NULL,95,2),(96,NULL,96,2),(97,NULL,97,2),(98,NULL,98,2),(99,NULL,99,2),(100,NULL,100,2),(101,NULL,101,2),(102,NULL,102,2),(103,NULL,103,2),(104,NULL,104,2),(105,NULL,105,2),(106,NULL,106,2),(107,'StudyInstanceUID',107,1),(108,'StudyInstanceUID',108,1),(109,'StudyInstanceUID',109,1),(110,'StudyInstanceUID',110,1),(111,'StudyInstanceUID',111,1),(112,'StudyInstanceUID',112,1),(113,'StudyInstanceUID',113,1),(114,'StudyInstanceUID',114,1),(115,'StudyInstanceUID',115,1),(116,'StudyInstanceUID',116,1),(117,'StudyInstanceUID',117,1),(118,'StudyInstanceUID',118,1),(119,'StudyInstanceUID',119,1),(120,'StudyInstanceUID',120,1),(121,'StudyInstanceUID',121,1),(122,'StudyInstanceUID',122,1),(123,'StudyInstanceUID',123,1),(124,'StudyInstanceUID',124,1),(125,'StudyInstanceUID',125,1),(126,'StudyInstanceUID',126,1),(127,'StudyInstanceUID',127,1),(128,'StudyInstanceUID',128,1),(129,'StudyInstanceUID',129,3),(130,'StudyInstanceUID',130,3),(131,'StudyInstanceUID',131,3),(132,'StudyInstanceUID',132,3),(133,'StudyInstanceUID',133,3),(134,'StudyInstanceUID',134,3),(135,'StudyInstanceUID',135,3),(136,'StudyInstanceUID',136,3),(137,'StudyInstanceUID',137,3),(138,'StudyInstanceUID',138,3),(139,'StudyInstanceUID',139,3),(140,'StudyInstanceUID',140,3),(141,'StudyInstanceUID',141,3),(142,'StudyInstanceUID',142,3),(143,'StudyInstanceUID',143,3),(144,'StudyInstanceUID',144,3),(145,'StudyInstanceUID',145,3),(146,'StudyInstanceUID',146,3),(147,'StudyInstanceUID',147,3),(148,'StudyInstanceUID',148,3),(149,'StudyInstanceUID',149,3),(150,'StudyInstanceUID',150,3),(151,'StudyInstanceUID',151,3),(152,'StudyInstanceUID',152,3),(153,'StudyInstanceUID',153,3),(154,'StudyInstanceUID',154,3),(155,'StudyInstanceUID',155,3),(156,'StudyInstanceUID',156,3),(157,'StudyInstanceUID',157,3),(158,'StudyInstanceUID',158,3),(159,NULL,159,3),(160,NULL,160,1),(161,'StudyInstanceUID',161,1),(162,'StudyInstanceUID',162,1),(163,'StudyInstanceUID',163,1),(164,'StudyInstanceUID',164,1),(165,'StudyInstanceUID',165,1),(166,'StudyInstanceUID',166,1),(167,'StudyInstanceUID',167,3),(168,'StudyInstanceUID',168,3),(169,'StudyInstanceUID',169,1),(170,'StudyInstanceUID',170,1),(171,'StudyInstanceUID',171,1),(172,'StudyInstanceUID',172,1),(173,'StudyInstanceUID',173,1),(174,'StudyInstanceUID',174,1),(175,'StudyInstanceUID',175,1),(176,'StudyInstanceUID',176,1),(177,'StudyInstanceUID',177,1),(178,'StudyInstanceUID',178,1),(179,'StudyInstanceUID',179,3);
/*!40000 ALTER TABLE `idc_collections_attribute_set_type` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_attribute_tooltips`
--

DROP TABLE IF EXISTS `idc_collections_attribute_tooltips`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_attribute_tooltips` (
  `id` int NOT NULL AUTO_INCREMENT,
  `tooltip_id` varchar(256) NOT NULL,
  `tooltip` varchar(4096) NOT NULL,
  `attribute_id` int NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `idc_collections_attribut_tooltip_id_attribute_id_593c74af_uniq` (`tooltip_id`,`attribute_id`),
  KEY `idc_collections_attr_attribute_id_c34e21a2_fk_idc_colle` (`attribute_id`),
  CONSTRAINT `idc_collections_attr_attribute_id_c34e21a2_fk_idc_colle` FOREIGN KEY (`attribute_id`) REFERENCES `idc_collections_attribute` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=193 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_attribute_tooltips`
--

LOCK TABLES `idc_collections_attribute_tooltips` WRITE;
/*!40000 ALTER TABLE `idc_collections_attribute_tooltips` DISABLE KEYS */;
INSERT INTO `idc_collections_attribute_tooltips` VALUES (1,'tcga_prad','<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>.&nbsp;Currently this image collection of prostate adenocarcinoma (PRAD) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.<br /><br /></p>\n <p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.YXOGLM4Y\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-PRAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(2,'tcga_blca','<p>\n	The Cancer Genome Atlas-Bladder Endothelial Carcinoma (TCGA-BLCA) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP), with the cooperation of several of the TCGA tissue-contributing institutions, has archived a large portion of the radiological images of the genetically-analyzed BLCA cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.8LNG8XDR\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-BLCA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(3,'tcga_ucec','<p>\n	The Cancer Genome Atlas-Uterine Corpus Endometrial Carcinoma (TCGA-UCEC) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed UCEC cases.</p><br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.GKJ0ZWAC\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-UCEC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(4,'tcga_hnsc','\n<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>. Currently this large PET/CT multi-sequence image collection of  head and neck squamous cell carcinoma (HNSC) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.</p>\n</br>\n<p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.LXKQ47MS\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-HNSC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(5,'tcga_lusc','<p>\n	The Cancer Genome Atlas-Lung Squamous Cell Carcinoma (TCGA-LUSC) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the LUSC cases.</p>\n<br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.TYGKKFMQ\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-LUSC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(6,'tcga_kirp','\n<p> The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>.&nbsp;Currently this large CT and MR multi-sequence image collection of  kidney renal papillary cell carcinoma (KIRP) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.&nbsp;<br />\n</br><p><a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.ACWOGBEF\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-KIRP <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(7,'tcga_thca','<p>\n	The Cancer Genome Atlas-Thyroid Cancer (TCGA-THCA) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed THCA cases.</p><br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.9ZFRVF1B\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-THCA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(8,'tcga_sarc','<p>\n	The Cancer Genome Atlas-Sarcoma Cancer (TCGA-SARC) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed SARC cases.</p><br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.CX6YLSUX\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-SARC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(9,'tcga_esca','<p>\n	The Cancer Genome Atlas-Esophageal Carcinoma (TCGA-ESCA) data collection is part of a larger effort to enhance the <a href=\"http://cancergenome.nih.gov/\">TCGA&nbsp;</a> data set with characterized radiological images. The <a href=\"https://imaging.cancer.gov/\">Cancer Imaging Program (CIP)</a> with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed ESCA cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.VPTNRGFY\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-ESCA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(10,'tcga_cesc','<p>\n	The Cancer Genome Atlas-Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma (TCGA-CESC) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed CESC cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.SQ4M8YP4\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-CESC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(11,'tcga_stad','<p>\n	The Cancer Genome Atlas-Stomach Adenoarcinoma (TCGA-STAD) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed STAD cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.GDHL9KIM\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-STAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(12,'tcga_coad','<p>\n	The Cancer Genome Atlas-Colon Adenocarcinoma (TCGA-COAD) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP), with the cooperation of several of the TCGA tissue-contributing institutions, has archived a large portion of the radiological images of the COAD cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.HJJHBOXZ\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-COAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(13,'tcga_kich','<p>\n	The Cancer Genome Atlas-Kidney Chromophobe (TCGA-KICH) data collection is part of a larger effort to enhance the The Cancer Genome Atlas (TCGA) <a href=\"https://cancergenome.nih.gov/\">https://cancergenome.nih.gov/</a> data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions has archived a large portion of the radiological images of the genetically-analyzed KICH cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.YU3RBCZN\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-KICH <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(14,'tcga_read','<p>\n	The Cancer Genome Atlas-Rectum Adenocarcinoma (TCGA-READ) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP), with the cooperation of several TCGA tissue-contributing institutions, has archived a large portion of the radiological images of the genetically-analyzed READ cases.</p>\n<br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.F7PPNPNU\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-READ <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(15,'ispy1','<p>\n	ISPY1/ACRIN 6657 was designed as a prospective study to test MRI for ability to predict response to treatment and risk-of-recurrence in patients with stage 2 or 3 breast cancer receiving neoadjuvant chemotherapy (NACT). ACRIN 6657 was conducted as a companion study to CALGB 150007, a correlative science study evaluating tissue-based biomarkers in the setting of neoadjuvant treatment of breast cancer. Collectively, CALGB 150007 and ACRIN 6657 formed the basis of the multicenter Investigation of Serial Studies to Predict Your Therapeutic Response with Imaging and moLecular Analysis (I-SPY TRIAL) breast cancer trial, a study of imaging and tissue-based biomarkers for predicting pathologic complete response (pCR) and recurrence-free survival (RFS).<br />\n	<br />\n	The collection consists of 847 MR studies from 222 subjects.<br />\n	<br />\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2016.HdHpgJLK\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ISPY1 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(16,'qin_headneck','<p>\n	This collection is a set of head and neck cancer patients each of whom has had multiple PET/CT FDG scans &ndash; before and after therapy &ndash; and with follow up scans where clinically indicated.&nbsp; The data were provided to help facilitate research activities of <a href=\"https://wiki.cancerimagingarchive.net/display/Public/Quantitative+Imaging+Network+Collections\" target=\"_blank\">NCI&#39;s Quantitative Imaging Network</a>.</p>\n<p>\n	Please see the <a href=\"https://wiki.cancerimagingarchive.net/display/Public/QIN-HEADNECK\" target=\"_blank\">QIN-HeadNeck</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(17,'lidc_idri','<div><span>The Lung Image Database Consortium (LIDC-IDRI) image collection consists of diagnostic and lung cancer screening thoracic CT scans with marked-up annotated lesions. It is a web-accessible international resource for development, training, and evaluation of computer-assisted diagnostic (CAD) methods for lung cancer detection and diagnosis. </span><span><br>&#10;<br>&#10;For more information about the LIDC-IDRI images please visit the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.LO9QL9SX\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><u>LIDC-IDRI</u> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page.&#160; For comprehensive information about utilizing the associated metadata please see the </span><span>Research Project page for the</span><span> <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/XwA2\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Lung Image Database Consortium <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>.</span></p>',120),(18,'tcga_luad','<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a>  is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a> Currently this large CT multi-sequence image collection of lung adenocarcinoma (LUAD) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.</p><br />\n\n<p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.JGNIHEP5\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-LUAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(19,'tcga_lihc','<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>.&nbsp;Currently this CT and MR multi-sequence image collection of liver hepatocellular carcinoma (LIHC) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.</p><br />\n<p><a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.IMMQW8UQ\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-LIHC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>   page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(20,'tcga_brca','The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a></span> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>.&nbsp;Currently this MR multi-sequence image collection of breast invasive carcinoma patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.&nbsp;<br />\n\n<p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.AB2NAZRP\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-BRCA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(21,'tcga_ov','<p>\n	The Cancer Genome Atlas-Ovarian Cancer (TCGA-OV) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed OV cases.</p>\n<br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.NDO1MDFQ\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-OV <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(22,'tcga_kirc','<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>.&nbsp;Currently this large CT and MR multi-sequence image collection of kidney renal clear cell carcinoma (KIRC) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.</p><br />\n</br><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.V6PBVTDR\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA -KIRC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>  page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(23,'tcga_lgg','\n<div>\n	<strong>Note:&nbsp;This collection has special restrictions on its usage. See <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/c4hF\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Data Usage Policies and Restrictions <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>.</strong></p>\n</br><p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>. Currently this large MR multi-sequence image collection of low grade glioma patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.</p><br />\n<p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.L4LTD3TK\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-LGG <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(24,'tcga_gbm','\n<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a> Currently this large MR multi-sequence image collection of glioblastoma patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome. </p><br />\n<a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.RNYFUYE9\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-GBM <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a></span> page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(25,'nsclc_radiomics','<p>\n	This collection contains images from 422 non-small cell lung cancer (NSCLC) patients. For these patients pretreatment CT scans and clinical outcome data are available. This dataset refers to the Lung1 dataset of the study published in Nature Communications</p><br />\n<p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.PF0M9REI\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">NSCLC-Radiomics <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(26,'mouse_astrocytoma','<p>\n	This collection consists of magnetic resonance images (MRI) of genetically engineered mouse models (GEMMs) of high grade astrocytoma, including glioblastoma multiforme (GBM).</p>\n<p>\n	The MRI data contained herein includes anatomic T2 weighted images and dynamic contrast enhanced MRI.</p><br />\n<p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/K9TCIA.2017.SGW7CAQW\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Mouse-Astrocytoma <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>  page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(27,'anti_pd_1_lung','<p>\n	This collection includes 46 lung cases treated with&nbsp;anti-PD1 immunotherapy in 2016, each with pre-treatment imaging (CT,PT,SC) and most with 1 imaging follow-up timepoint.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/tcia.2019.zjjwb9ip\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Anti-PD-1_Lung <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(28,'apollo','<p>	This data collection consists of images and associated data acquired from the <a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/apollo-network\"> APOLLO Network</a> and may be subject to usage restrictions.</p>\n<p>	The <strong><em>A</em></strong>pplied <strong><em>P</em></strong>roteogenomics <strong><em>O</em></strong>rganizationa<strong><em>L</em></strong> <strong><em>L</em></strong>earning and <strong><em>O</em></strong>utcomes (APOLLO) network is a collaboration between NCI, the Department of Defense (DoD), and the Department of Veterans Affairs (VA) to incorporate proteogenomics into patient care as a way of looking beyond the genome, to the activity and expression of the proteins that the genome encodes. The emerging field of proteogenomics aims to better predict how patients will respond to therapy by screening their tumors for both genetic abnormalities and protein information, an approach that has been made possible in recent years due to advances in proteomic technology.</p>\n<p>	Please see the <a href=\"https://wiki.cancerimagingarchive.net/display/Public/APOLLO-1-VA\">APOLLO</a> wiki page to learn more.</p>\n',120),(29,'rider_lung_pet_ct','<p><a href = \"https://doi.org/10.7937/K9/TCIA.2015.OFIP7TVM\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Lung PET-CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> is a subsection of the broader Reference Image   Database to  Evaluate Therapy Response (RIDER) collection.&nbsp; RIDER is a   targeted  data collection for the purpose of generating an initial   consensus on  how to harmonize data collection and analysis for   quantitative imaging  methods as applied to measure the response to drug   or radiation  therapy. The long term goal is to provide a resource to   permit  harmonized methods for data collection and analysis across   different  commercial imaging platforms, as required to support   multi-site  clinical trials, using imaging as a biomarker for therapy   response.</p>\n<p>More information about the <a href = \"https://doi.org/10.7937/K9/TCIA.2015.OFIP7TVM\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Lung PET-CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> sub-collection can be  found in the corresponding section of the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/EAA9\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;wiki  page.</p>',120),(30,'anti_pd_1_melanoma','<p>\n	This collection includes 47 melanoma cases treated with anti-PD1 immunotherapy in 2016, each with pre-treatment and 1 or more imaging (CT, PET, MRI) follow-up timepoints.</p>\n<p>\n	<span _ngcontent-cmj-c43=\"\" class=\"toolTip step show\"><span _ngcontent-cmj-c43=\"\">Please see the <a href=\"https://doi.org/10.7937/tcia.2019.1ae0qtcu\">Anti-PD-1_MELANOMA</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n',120),(31,'qin_gbm_treatment_response','<div>\n	This collection contains &ldquo;double baseline&rdquo; multi-parametric MRI images collected on participants with newly diagnosed glioblastoma,&nbsp; scanned after surgery but prior to the start of therapy, typically 2-5 days apart. The structural images include T1-weighted pre- and post-contrast , T2-weighted , FLAIR and MEMPRAGE images.&nbsp; Advanced MRI sequences included Diffusion Weighted (DW) Imaging, Dynamic Contrast-Enhanced MRI (DCE-MRI), and Dynamic Susceptibility Contrast MRI (DSC-MRI).</p>\n<div>\n	The value of this collection is to provide clinical image data to establish the test-retest characteristics of parameters calculated from DW-MRI, DCE-MRI, and DSC-MRI such as ADC, Ktrans and rCBV.&nbsp;</p>\n<div>\n	&nbsp;</p>\n<div>\n	Please see the&nbsp;<a href=\"https://doi.org/10.7937/k9/tcia.2016.nQF4gpn2\">QIN GBM Treatment Response</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.&nbsp; &nbsp;&nbsp;</p>\n',120),(32,'qin_brain_dsc_mri','<div>\n	This collection contains MR images of both low and high grade glial brain lesions. Data includes post-contrast T1w images with co-registered volumes of dynamic susceptibility contrast (DSC) MR images in DICOM format. Binary regions of interest are also included, in DICOM format, of the lesion, arterial input function, normal appearing white matter, normal appearing cerebral cortex, and whole brain.&nbsp;</p>\n<div>\n	&nbsp;</p>\n<div>\n	Please see the <a href=\"https://doi.org/10.7937/K9/TCIA.2016.5DI84Js8\">Glioma DSC-MRI Perfusion Data with Standard Imaging and ROIs (QIN-BRAIN-DSC-MRI)</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.&nbsp;&nbsp;</p>\n',120),(33,'rider_neuro_mri','<p>\n	RIDER Neuro MRI is a subsection of the broader Reference Image Database to Evaluate Therapy Response (RIDER) collection.&nbsp; RIDER is a targeted data collection for the purpose of generating an initial consensus on how to harmonize data collection and analysis for quantitative imaging methods as applied to measure the response to drug or radiation therapy. The long term goal is to provide a resource to permit harmonized methods for data collection and analysis across different commercial imaging platforms, as required to support multi-site clinical trials, using imaging as a biomarker for therapy response.</p>\n<p>\n	More information about the RIDER Neuro MRI sub-collection can be found in the corresponding section of the <a href=\"https://wiki.cancerimagingarchive.net/x/EAA9\" target=\"_blank\">RIDER</a>&nbsp;wiki page.</p>\n',120),(34,'victre','<p >	The VICTRE Trial: Open-Source, In-Silico Clinical Trial For Evaluating Digital Breast Tomosynthesis</p>\n<p>\n	<span>A total of 2986 subjects, with breast sizes and radiographic densities representative of a screening population and compressed thicknesses from 3.5 to 6 cm, were simulated and imaged on in-silico versions of DM and DBT systems using fast Monte Carlo x-ray transport. Images were interpreted by a computational reader detecting the presence of lesions. The in-silico trial (VICTRE) was designed to replicate a comparative trial from a previous regulatory submission. The endpoint was the difference in area under the receiver-operating-characteristic curve between modalities (delta-AUC) for lesion detection.</span></p>\n<p>\n	See the <a href=\"\" url=\"http://doi.org/10.7937/TCIA.2019.ho23nxaw\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">VICTRE Collection <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(35,'rider_breast_mri','<span><p><a href=\"\" url=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.H1SXNUXL\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Breast MRI  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> is a subsection of the broader Reference Image  Database to Evaluate  Therapy Response (RIDER) collection. RIDER is a targeted data collection for the purpose of generating an initial consensus on how to harmonize data collection and analysis for quantitative imaging methods as applied to measure the response to drug or radiation therapy. The  long term goal is to provide a resource to permit harmonized methods for data collection and analysis across different commercial imaging  platforms, as required to support multi-site clinical trials, using  imaging as a biomarker for therapy  response.</p>\n<p>More information about the <a href=\"\" url=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.H1SXNUXL\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Breast MRI  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>sub-collection page can be found in the corresponding section of the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/EAA9\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page.</p>\n</span>',120),(36,'c4kc_kits','<p>\n	<strong>Data from the training set of the 2019 Kidney and Kidney Tumor Segmentation Challenge (C4KC-KiTS)</strong></p>\n<p>\n	This collection contains subjects from the training set of the&nbsp;<a href=\"\" url=\"https://kits19.grand-challenge.org/\" rel=\"nofollow\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">2019 Kidney and Kidney Tumor Segmentation Challenge <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;(KiTS19). The challenge aimed to accelerate progress in automatic 3D semantic segmentation by releasing a dataset of CT scans for 210 patients with manual semantic segmentations of the kidneys and tumors in the corticomedullary phase.<br />\n	<br />\n	The imaging was collected during routine care of patients who were treated by either partial or radical nephrectomy at the University of Minnesota Medical Center. Many of the CT scans were acquired at referring institutions and are therefore heterogeneous in terms of scanner manufacturers and acquisition protocols. Semantic segmentations were performed by students under the supervision of an experienced urologic cancer surgeon.</p>\n<p>\n	Please see the&nbsp;<a href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.IX49E8NX\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">C4KC-KiTS <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(37,'dro_toolkit','<p>\n	DRO Toolkit:&nbsp;This is a sample collection of synthetic 3D Digital Reference Objects (DROs) intended for standardization of quantitative imaging feature extraction pipelines. We have developed a software toolkit&nbsp;for the creation of DROs with customizable size, shape, intensity, texture, and margin sharpness values. Using user-supplied input parameters, these objects are defined mathematically as continuous functions, discretized, and then saved as DICOM objects. This collection includes objects with a range of values for the various feature categories and many combinations of these categories.</p>\n	<br /><p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/T062-8262\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">DRO Toolkit <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain the supporting metadata for this collection.</p>\n\n',120),(38,'cc_radiomics_phantom_2','<p>\n	<strong>Credence Cartridge Radiomics Phantom CT Scans with Controlled Scanning Approach (CC-Radiomics-Phantom-2)</strong></p>\n<p>\n	This collection consists of 251 CT scans of the Credence Cartridge Radiomic (CCR) phantom. This texture phantom was developed to investigate the feature robustness in the emerging field of radiomics. This phantom dataset was acquired on 4-8 CT scanners using a set of imaging parameters (e.g., reconstruction Field of View, Slice thickness, reconstruction kernels, mAs, and Pitch). A controlled scanning approach was employed to assess the variability in radiomic features due to each imaging parameter. This dataset will be useful to radiomic research community to identify a subset of robust radiomic features and for establishing the ground truths for future clinical investigations.</p>\n<p>\n	Using this dataset, the numerical values of radiomic features can be cross-validated by other research groups using their own feature extraction tools.</p>\n<p>\n	Please see the&nbsp;<a href=\"http://doi.org/10.7937/TCIA.2019.4l24tz5g\">CC-Radiomics-Phantom-2</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(39,'qin_pet_phantom','<p>\n	This collection consists of positron emission tomography (PET) phantom scans originally utilized by the Quantitative Imaging Network (QIN) PET Segmentation Challenge to assess the variability of segmentations and subsequently derived quantitative analysis results on phantom PET scans with known ground truth.</p>\n<p>\n	Please see the <a href=\"https://wiki.cancerimagingarchive.net/display/Public/QIN+PET+Phantom\" target=\"_blank\">QIN PET Phantom</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n	&nbsp;</p>\n',120),(40,'qiba_ct_1c','<p>Phantom data provided by the RSNA Quantitative Imaging Biomarkers Alliance,,s 1C group.&#160;</p>\n<p>&#10;&#9;Please see the <a href=\"\" url=\"http://qibawiki.rsna.org/index.php?title=VolCT_-_Group_1C#Downloadable_Images_and_Data_Sets\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIBA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page through RSNA, and TCIA,,s <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.YXGR4BLU\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIBA CT-1C <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page,&#160; to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(41,'rider_phantom_mri','<p>\n<p><span id=\"MAINbody:collectionDescriptionPopupForm:j_id767\" class=\"iceOutTxt\">\n<p>RIDER Phantom MRI is a subsection of the broader Reference Image   Database to  Evaluate Therapy Response (RIDER) collection.&nbsp; RIDER is a   targeted  data collection for the purpose of generating an initial   consensus on  how to harmonize data collection and analysis for   quantitative imaging  methods as applied to measure the response to drug   or radiation  therapy. The long term goal is to provide a resource to   permit  harmonized methods for data collection and analysis across   different  commercial imaging platforms, as required to support   multi-site  clinical trials, using imaging as a biomarker for therapy   response.</p>\n<p>More information about the RIDER Phantom MRI sub-collection can be  found in the corresponding section of the <a href=\"https://wiki.cancerimagingarchive.net/x/EAA9\" target=\"_blank\">RIDER</a>&nbsp;wiki  page.</p>\n</span></p>\n</p>',120),(42,'pseudo_phi_dicom_data','<p>\n	<span>The process of image de-identification is time consuming, requires significant human resources, and is prone to human error. &nbsp;Automated image de-identification algorithms have been developed but the research community requires some method of evaluation before such tools can be widely accepted.&nbsp; This evaluation requires a robust dataset that can be used as part of an evaluation process for de-identification algorithms.&nbsp;&nbsp;</span><span>We developed a DICOM dataset that can be used to evaluate the performance of de-identification algorithms.&nbsp;</span></p>\n<p>\n	&nbsp;</p>\n<p>\n	Please see the&nbsp;<a href=\"\" url=\"https://doi.org/10.7937/s17z-r072\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><u>Medical Imaging De-Identification Initiative (MIDI)</u> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(43,'naf_prostate','<p>\n	This is a collection of F-18 NaF positron emission tomography/computed tomography (PET/CT) images in patients with prostate cancer, with suspected or known bone involvement.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.ISOQTHKO\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">NaF PROSTATE <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(44,'prostate_mri','<p>\n	This is a collection of &nbsp;MRIs of the prostate obtained with an endorectal coil and phased array surface coil at 3T (Philips Achieva). &nbsp;Each patient had biopsy confirmation of cancer and then underwent robotic assisted radical prostatectomy. &nbsp;A mold was generated from each MRI and the prostatectomy specimen was first placed in the mold and then cut in the same plane as the MRI. &nbsp;The data was generated at the National Cancer Institute, Bethesda, Maryland, USA between 2008-2010.</p><br />\n<p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.6046GUDv\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prostate-MRI <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(45,'aapm_rt_mac','<p>\n	AAPM-RT-MAC: This data set was provided in association with a <a href=\"https://www.aapm.org/GrandChallenge/RT-MAC/\">challenge competition</a> conducted at the AAPM 2019 Annual Meeting. The MRI scanning protocol was designed for radiation treatment simulation, 55 cases with T2-weighted images are included. 31 are used as training cases, with the parotid glands, submandibular glands, level 2 and level 3 lymph nodes contoured. The images and contours were acquired from MD Anderson Cancer Center.</p>\n<p>\n	Please see the <a href=\"https://doi.org/10.7937/tcia.2019.bcfjqfqb\">AAPM-RT-MAC</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(46,'phantom_fda','<p>\n	A series of CT sets scanned at differing slice thicknesses and exposure parameters of a chest simulating phantom containing lung nodules. It was constructed by an NCI-FDA joint effort to provide data for nodule-detecting CAD processes.</p>\n<p>\n	Please see the<a href=\"\" url=\"https://doi.org/10.7937/k9/TCIA.2015.orbjkmux\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"> Phantom FDA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(47,'midrc_ricord_1b','<p>\n	<span><span><font color=\"#202124\">T</font><span>he first multi-institutional, multi-national expert annotated COVID-19 imaging dataset made freely available to the machine learning community as a research and educational resource for COVID-19 chest imaging.&nbsp;The Radiological Society of North America (RSNA) assembled the&nbsp;<a href=\"\" url=\"https://www.rsna.org/covid-19/covid-19-ricord\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><font color=\"#51a6fa\"><span>RSNA International COVID-19 Open Radiology Database (RICORD)</span></font> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;collection&nbsp;of COVID-related imaging datasets and expert annotations to support research and education.&nbsp;RICORD&nbsp;data will be incorporated in the&nbsp;<a href=\"\" url=\"https://www.midrc.org/\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><font color=\"#51a6fa\"><span>Medical Imaging&nbsp;and&nbsp;Data Resource Center (MIDRC)</span></font> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>,&nbsp;a&nbsp;multi-institutional research data repository funded by the National Institute of Biomedical Imaging and Bioengineering of the National Institutes of Health.</span></span></span></p>\n<p>\n	<span><span>Please see the<span>&nbsp;</span></span><a href=\"\" url=\"https://doi.org/10.7937/31V8-4A40\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><span><span>MIDRC-RICORD-1b</span></span> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>&nbsp;</span><span>wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n',120),(48,'midrc_ricord_1c','<p>\nThe first multi-institutional, multi-national expert annotated COVID-19 imaging dataset made freely available to the machine learning community as a research and educational resource for COVID-19 chest imaging. The Radiological Society of North America (RSNA) assembled the <a href=\"\" url=\"https://www.rsna.org/covid-19/covid-19-ricord\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RSNA International COVID-19 Open Radiology Database (RICORD) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> collection of COVID-related imaging datasets and expert annotations to support research and education. RICORD data will be incorporated in the <a href=\"\" url=\"https://www.midrc.org/\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Medical Imaging and Data Resource Center (MIDRC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>, a multi-institutional research data repository funded by the National Institute of Biomedical Imaging and Bioengineering of the National Institutes of Health. The collection contains 998 Chest x-ray examinations.</p><p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/91ah-v663\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">MIDRC-RICORD-1c <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(49,'midrc_ricord_1a','<p>\n	The Radiological Society of North America (RSNA) set out to create the <a href=\"\" url=\"https://www.rsna.org/covid-19/covid-19-ricord\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RSNA International COVID-19 Open Radiology Database (RICORD) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> collection of COVID-related imaging datasets and expert annotations to support research and education. RICORD data will be incorporated in the <a href=\"\" url=\"https://www.midrc.org/\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Medical Imaging and Data Resource Center (MIDRC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>, a multi-institutional research data repository funded by the National Institute of Biomedical Imaging and Bioengineering of the National Institutes of Health.</p></br>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/VTW4-X588\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">MIDRC-RICORD-1a <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> WIKI page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(50,'acrin_nsclc_fdg_pet','<p>\n        Thirty-seven institutions accrued 250 patients to the study ACRIN 6668.  Eligible patients were &gt;18 years with AJCC-criteria clinical <strong>stage IIB/III non-small cell lung carcinoma </strong>who were being planned for definitive concurrent chemoradiotherapy (inoperable disease). 173 had evaluable post-treatment PET, representing the analysis cohort for the primary end point.</p>\n<p>\n        Please see the <a href=\"\" url=\"https://doi.org/10.7937/tcia.2019.30ilqfcl\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ACRIN-NSCLC-FDG-PET <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(51,'cptac_cm','<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Cutaneous Melanoma (CPTAC-CM) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n	&nbsp;</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.ODU24GZE\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-CM <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n',120),(52,'acrin_dsc_mr_brain','<p>\n	ACRIN-DSC-MR-Brain: RTOG 0625/ACRIN 6677 in recurrent glioblastoma (GBM). FLAIR and postcontrast MRI.</p>\n<p>\n	Please see the wiki page <a href=\"https://doi.org/10.7937/tcia.2019.zr1pjf4i\">ACRIN-DSC-MR-Brain</a>&nbsp;to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(53,'acrin_fmiso_brain','<p class=\"p1\">\n	The objective of the ACRIN 6684 multi-center clinical trial was to determine baseline FMISO PET uptake (maximal tumor to blood ratio, hypoxic volume) and MRI parameters (K<sup>trans</sup>, CBV) with overall survival. After establishing eligibility and enrollment to the study, baseline imaging of both MR and PET was performed within 2 weeks of starting therapy.</p>\n<p>\n	Please see the&nbsp;<a href=\"https://doi.org/10.7937/K9/TCIA.2018.vohlekok\" target=\"_blank\">ACRIN-FMISO-Brain</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(54,'pdmr_997537_175_t','<div>\n	We identified and characterized a patient derived xenograft model with metastatic potential, adenocarcinoma colon xenograft 997537-175-T. In this study we performed a detailed imaging characterization of this model, which develops spontaneous lung metastases.</p>\n<div>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.BRY9-4N29\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">PDMR-997537-175-T <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<div>\n	&nbsp;</p>\n',120),(55,'breast_cancer_screening_dbt','<p>\n	Breast cancer is among the most common cancers and a common cause of death among women. Here, we share a curated dataset of digital breast tomosynthesis images that includes normal, actionable, biopsy-proven benign, and biopsy-proven cancer cases.&nbsp; The dataset contains four components: (1) DICOM images, (2) a spreadsheet indicating which group each case belongs to, and (3) annotation boxes.&nbsp;<span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/e4wt-cd02\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Breast-Cancer-Screening-DBT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a></span><span>&nbsp;to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n',120),(56,'acrin_flt_breast','<p><span><b>ACRIN 6688</b>, a [18F]-fluorothymidine (FLT) imaging study using positron emission tomography (PET), was designed to evaluate the relationship between [18F] FLT uptake parameters and pathologic complete response to neoadjuvant therapy of the primary tumor in patients with locally advanced breast cancer.</p>\n<p><u>Participant Eligibility and Enrollment</u>: Criteria for inclusion were patients with pathologically confirmed breast cancer and determined to be a candidate for primary systemic (neoadjuvant) therapy and for whom surgical resection of residual primary tumor following completion of neoadjuvant therapy is planned.</p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2017.ol20zmxg\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ACRIN-FLT-Breast <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>',120),(57,'pdmr_833975_119_r','<p><span><a href=\"\" url=\"\" url=\"https://doi.org/10.7937/TCIA.0ECK-C338\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">PDMR-833975-119-R <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> is a dataset of detailed MRI, PET/CT (FDG and FLT) in patient-derived xenograft in mouse model of pancreatic adenocarcinoma from the National Cancer Institute Patient-Derived Models Repository (<a href=\"https://pdmr.cancer.gov/\">https://pdmr.cancer.gov/</a>) with baseline and bi-weekly monitoring imaging.</span></p>\n<p>\n	<span>Please see the <a href=\"\" url=\"\" url=\"https://doi.org/10.7937/TCIA.0ECK-C338\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">PDMR-833975-119-R <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n',120),(58,'pdmr_292921_168_r','<div>\n	We identified and characterized a patient derived xenograft model with metastatic potential, adenocarcinoma pancreas xenograft 292921-168-R. In this study we performed a detailed imaging characterization of this model, which develops spontaneous lung metastases.</p>\n<div>\n	&nbsp;</p>\n<div>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.PCAK-8Z10\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Imaging characterization of a metastatic patient derived model of adenocarcinoma pancreas: PDMR-292921-168-R <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.&nbsp;</p>\n',120),(59,'pdmr_425362_245_t','<div>We identified and characterized a patient derived xenograft model with metastatic potential, melanoma xenograft 425362-245-T. In this study we performed a detailed imaging characterization of this model, which develops spontaneous lung metastases.</p>\n<div>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.7YRS-7J97\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">PDMR-425362-245-T <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(60,'pdmr_bl0293_f563','<span><p>We identified and characterized a patient-derived xenograft model with metastatic potential, bladder xenograft BL0293-F563. In this study we performed a detailed imaging characterization of this model, which develops spontaneous liver and bone metastases.</span></p>\n<p>	<span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/tcia.2019.b6u7wmqw\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">PDMR-BL0293-F563 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.\n</span></p>',120),(61,'cptac_pda','<p>\n	<p>\n		This collection contains subjects from the National Cancer Institute&rsquo;s <a href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a> Pancreatic Ductal Adenocarcinoma (CPTAC-PDA) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</p>\n	<p>\n		&nbsp;</p>\n	<p>\n		Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.SC20FO18\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-PDA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n</p>\n',120),(62,'head_neck_radiomics_hn1','<p>\nThis collection contains clinical data and computed tomography (CT) from 137 head and neck squamous cell carcinoma (HNSCC) patients treated by radiotherapy.</p>\n<p>\n	Please see the <a href=\"https://doi.org/10.7937/tcia.2019.8kap372n\">Head-Neck-Radiomics-HN1</a> wiki page for further detail and associated metadata.</p>\n',120),(63,'opc_radiomics','<div>\n	This data collection consists of 606 Radiotherapy planning CT scans retrieved for all non-metastatic p16-positive oropharynx squamous cell carcinoma patients treated with radiotherapy or chemoradiotherapy. Radiomic biomarkers were derived from each gross tumor volume (GTV). The role of quantitative imaging (i.e. radiomic) analysis was examined to determine whether there are primary tumor features discernible on imaging studies that associate with a higher risk of developing distant metastases.</p>\n<div>\n	&nbsp;</p>\n<div>\n	Please see the <a href=\"https://doi.org/10.7937/tcia.2019.8dho2gls\">OPC-Radiomics</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<div>\n	&nbsp;</p>\n',120),(64,'cptac_sar','<p>\n	This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u>&nbsp;Sarcomas (CPTAC-SAR) cohort.&nbsp; CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.9bt23r95\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-SAR <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(65,'icdc_glioma','<p><b><a href=\"\" url=\"https://doi.org/10.7937/TCIA.SVQT-Q016\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ICDC-Glioma <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;</b>contains treatment-na&#239;ve naturally-occurring <b>canine glioma</b> participants from the&#160;<a href=\"https://caninecommons.cancer.gov/#/study/GLIOMA01\">Integrated Canine Data Commons</a>. Brain radiology (57/81 participant animals) and H&amp;E-stained biopsy or necropsy pathology (76/81 participants) are classified by veterinary and physician neuropathologists.<br>&#10;&#9;<br>&#10;&#9;Please see the wiki <a href=\"\" url=\"https://doi.org/10.7937/TCIA.SVQT-Q016\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><b>ICDC-Glioma</b> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;to learn more about the images and to obtain any supporting metadata for this collection.</p>&#10;',120),(66,'lung_pet_ct_dx','<p>\nThis dataset consists of CT and PET-CT DICOM images of lung cancer subjects with XML Annotation files that indicate tumor location with bounding boxes. The images were retrospectively acquired from patients with suspicion of lung cancer, and who underwent standard-of-care lung biopsy and PET/CT. Subjects were grouped according to a tissue histopathological diagnosis. Patients with Names/IDs containing the letter ,,A,, were diagnosed with Adenocarcinoma, ,,B,, with Small Cell Carcinoma, ,,C,, with Large Cell Carcinoma, and ,,G,, with Squamous Cell Carcinoma.</p>\n<p></br>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.NNC2-0461\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">LUNG-PET-CT-Dx <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(67,'lung_fused_ct_pathology','<p>\n	This is the first attempt of mapping the extent of Invasive Adenocarcinoma onto&nbsp;in vivo&nbsp;lung CT. The mappings constitute ground truth of disease and may be used to further investigate the imaging signatures of Invasive Adenocarcinoma in ground glass pulmonary nodules.&nbsp;</p>\n<p>\n	Please see the&nbsp;<a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.SMT36LPN\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Lung-Fused-CT-Pathology <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.<o:p></o:p></p>\n',120),(68,'lctsc','<p>\n	This Lung CT and Radiation therapy data set was provided in association with an autocontouring challenge competition and <a href=\"\" url=\"http://www.aapm.org/meetings/2017AM/PRAbs.asp?mid=127&amp;aid=35318\" rel=\"nofollow\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">related conference session <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> conducted at the <a href=\"\" url=\"http://www.aapm.org/meetings/2017AM/\" rel=\"nofollow\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">AAPM 2017 Annual Meeting <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> </p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2017.3r3fvz08\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">LCTSC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection. </p>\n',120),(69,'qin_lung_ct','<p>\n	The Computed tomography (CT) Image data was obtained on patients diagnosed with Non-Small Cell Lung Cancer (NSCLC) with mixed stage &amp; histology from the H. Lee Moffitt Cancer Center and Research Institute. Scans were obtained from patients who underwent surgical resection and had corresponding pre-surgery diagnostic CTs.</p>\n<p>\n	&nbsp;</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.NPGZYZBZ\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIN LUNG CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>  page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(70,'4d_lung','<p>\n	<span>This data collection consists of images acquired during chemoradiotherapy of 20 locally-advanced, non-small cell lung cancer patients. The images include four-dimensional (4D) fan beam (4D-FBCT) and 4D cone beam CT (4D-CBCT). All patients underwent concurrent radiochemotherapy to a total dose of 64.8-70&nbsp;Gy using daily 1.8 or 2&nbsp;Gy fractions.</span></p>\n<p>\n	A single Radiation Oncologist delineated targets and organs at risk in all 4D-FBCT and a limited number of 4D-CBCT images. All patients underwent concurrent radiochemotherapy to a total dose of 64.8-70 Gy using daily 1.8 or 2 Gy fractions.</p>\n<p>\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2016.ELN8YGLE\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">4D-Lung <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(71,'brain_tumor_progression','<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection includes datasets from 20 subjects with primary newly diagnosed glioblastoma who were treated with surgery and standard concomitant chemo-radiation therapy (CRT) followed by adjuvant chemotherapy.&nbsp; Two MRI exams are included for each patient: within 90 days following CRT completion and at progression, including </span>Pre/post T1 images, FLAIR, Processed parameter maps such as rCBV, CBF, ADC</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id712\">Please see the <a href=\"https://doi.org/10.7937/K9/TCIA.2018.15quzvnb\">Brain-Tumor-Progression</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n',120),(72,'rider_lung_ct','<p><a href = \"https://doi.org/10.7937/K9/TCIA.2015.U1X8A5NR\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Lung CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>  is a subsection of the broader Reference Image   Database to  Evaluate Therapy Response (RIDER) collection.&nbsp; RIDER is a   targeted  data collection for the purpose of generating an initial   consensus on  how to harmonize data collection and analysis for   quantitative imaging  methods as applied to measure the response to drug   or radiation  therapy. The long term goal is to provide a resource to   permit  harmonized methods for data collection and analysis across   different  commercial imaging platforms, as required to support   multi-site  clinical trials, using imaging as a biomarker for therapy   response.</p>\n<p>More information about the <a href = \"https://doi.org/10.7937/K9/TCIA.2015.U1X8A5NR\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Lung CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> sub-collection page can be  found in the corresponding section of the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/EAA9\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>  page.</p>\n',120),(73,'ct_colonography','<p>The CT Colonography collection consists of a limited portion of The National CT Colonography Trial (<a href=\"\" url=\"http://www.acrin.org/TabID/151/Default.aspx\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ACRIN PROTOCOL 6664 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>). The collection is offered to enlarge the broader research community\'\'s ability to develop CAD algorithms for polyp detection. </p><br /><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.NWTESAY1\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CT COLONOGRAPHY <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>',120),(74,'cbis_ddsm','<p>\n	This CBIS-DDSM (Curated Breast Imaging Subset of DDSM) is an updated and standardized version of the <a href=\"http://marathon.csee.usf.edu/Mammography/Database.html\">Digital Database for Screening Mammography (DDSM)</a>.&nbsp; The DDSM is a database of 2,620 scanned film mammography studies. It contains normal, benign, and malignant cases with verified pathology information.</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id712\">Please see the <a href=\"https://wiki.cancerimagingarchive.net/display/Public/CBIS-DDSM\">CBIS-DDSM</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n',120),(75,'rider_phantom_pet_ct','<p>\n<p><span id=\"MAINbody:collectionDescriptionPopupForm:j_id767\" class=\"iceOutTxt\">\n<p>RIDER Phantom PET-CT is a subsection of the broader Reference Image   Database to  Evaluate Therapy Response (RIDER) collection.&nbsp; RIDER is a   targeted  data collection for the purpose of generating an initial   consensus on  how to harmonize data collection and analysis for   quantitative imaging  methods as applied to measure the response to drug   or radiation  therapy. The long term goal is to provide a resource to   permit  harmonized methods for data collection and analysis across   different  commercial imaging platforms, as required to support   multi-site  clinical trials, using imaging as a biomarker for therapy   response.</p>\n<p>More information about the RIDER Phantom PET-CT sub-collection can be  found in the corresponding section of the <a href=\"https://wiki.cancerimagingarchive.net/x/EAA9\" target=\"_blank\">RIDER</a>&nbsp;wiki  page.</p>\n</span></p>\n</p>',120),(76,'pancreas_ct','<p>\n	The National Institutes of Health Clinical Center performed 82 abdominal contrast enhanced 3D CT scans (~70 seconds after intravenous contrast injection in portal-venous) from 53 male and 27 female subjects.&nbsp; Seventeen of the subjects are healthy kidney donors scanned prior to nephrectomy.&nbsp; The remaining 65 patients were selected by a radiologist from patients who neither had major abdominal pathologies nor pancreatic cancer lesions.&nbsp;</p>\n<p>\n	A medical student manually performed slice-by-slice segmentations of the pancreas as ground-truth and these were verified/modified by an experienced radiologist.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.tNB1kqBU\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Pancreas-CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(77,'qin_prostate_repeatability','<p>This is a dataset with multiparametric prostate MRI applied in a test-retest setting, allowing to evaluate repeatability of the MRI-based measurements in the prostate. There is very limited data about the repeatability in mpMRI of the prostate, while such information is critical for establishing technical characteristics of mpMRI as imaging biomarker of prostate cancer.</p><p>Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2018.MR1CKGND\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIN-PROSTATE-Repeatability  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(78,'prostate_mri_us_biopsy','<p>\n	This dataset was derived from tracked biopsy sessions using the Artemis biopsy system, many of which included image fusion with MRI targets. Patients received a 3D transrectal ultrasound scan, after which nonrigid registration (e.g. &ldquo;fusion&rdquo;)&nbsp; was performed between real-time ultrasound and preoperative MRI, enabling biopsy cores to be sampled from MR regions of interest.&nbsp;Most cases also included sampling of systematic biopsy cores using a 12-core digital template.&nbsp;The Artemis system tracked targeted and systematic core locations using encoder kinematics of a mechanical arm, and recorded locations relative to the Ultrasound scan. MRI biopsy coordinates were also recorded for most cases.&nbsp;</span></p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.A61IOC1A\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prostate-MRI-US-Biopsy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(79,'prostatex','<p>SPIE, along with the support of the American Association of Physicists in Medicine (AAPM) and the National Cancer Institute (NCI), will conduct a &#8220;Grand Challenge&#8221; on quantitative image analysis methods for the diagnostic classification of clinically significant prostate lesions.  As part of the 2017 SPIE Medical Imaging Symposium, the PROSTATEx Challenge will provide a unique opportunity for participants to compare their algorithms with those of others from academia, industry, and government in a structured, direct way using the same data sets.  For more details, go to <a href=\"\" url=\"https://www.spie.org/search?&term=PROSTATEx\" rel=\"nofollow\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">https://www.spie.org/search?&term=PROSTATEx <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>. rel=\"nofollow\">http://spiechallenges.cloudapp.net/</a>. </p>&#10;<ul>&#10;&#9;<li>&#10;&#9;&#9;Release date of training set cases with truth:<strong> &#160;November 21, 2016</strong></li>&#10;&#9;<li>&#10;&#9;&#9;Release date of test set cases without truth:<strong> &#160;December 12, 2016</strong></li>&#10;&#9;<li>&#10;&#9;&#9;Submission date for participants&#8217; test set classification output:<strong> &#160;January 15, 2017</strong></li>&#10;&#9;<li>&#10;&#9;&#9;Challenge results released to participants:&#160; <strong>January 20, 2017</strong></li>&#10;&#9;<li>&#10;&#9;&#9;SPIE Medical Imaging Symposium: &#160;<strong>February 13-16, 2017</strong></li></span></p>\n<br /><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9TCIA.2017.MURS5CL\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"> PROSTATEx <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(80,'prostate_fused_mri_pathology','<p>\n	This collection comprises a total of 28 3 Tesla T1-weighted, T2-weighted, Diffusion weighted and Dynamic Contrast Enhanced prostate MRI along with accompanying digitized histopathology (H&amp;Estained) images of corresponding radical prostatectomy specimens.</p>\n<p>\n	Fused MATLAB objects are also provided combining both the MR and Pathology Images.</p>\n<p>\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2016.TLPMR1AM\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prostate Fused-MRI-Pathology <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(81,'prostate_3t','<p>\n	Prostate transversal T2 -weighted MRIs acquired on a 3.0T Siemens TrioTim using only a pelvic phased array coil. Images were acquired for prostate cancer detection.&nbsp;</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.QJTV5IL5\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prostate-3T  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(82,'hnscc_3dct_rt','<p>\n	This data collection contains three dimensional high-resolution (3D) fan-beam CT scans collected during pre-treatment, mid-treatment, and post-treatment using a Siemens 16-slice CT scanner with standard clinical protocol for 31 head-and-neck squamous cell carcinoma (HNSCC) patients.&nbsp;</p>\n<p>\n	&nbsp;</p>\n<p>\n	Please see the&nbsp;<a href=\"https://doi.org/10.7937/K9/TCIA.2018.13upr2xf\">HNSCC-3DCT-RT</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.<o:p></o:p></p>\n',120),(83,'head_neck_cetuximab','<p>\n	<p>\n		<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id767\">The Head-Neck Cetuximab collection consists of a subset of image data from ACRIN 4500/RTOG 0522, which was randomized phase III Trial of Radiation Therapy and Chemotherapy for stage III and IV Head and Neck carcinomas. Data was provided through two indpendent channels:</span></p>\n	<ul>\n		<li>\n			<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id767\">RTOG 0522: CT, Structures, RT Doses, RT Plans sent to ITC</span></li>\n		<li>\n			<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id767\">ACRIN 4500: Quantitative PET (PET/CT) sent to ACRIN</span></li>\n	</ul>\n	<p>\n		<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id767\">Please see the <a href=\"https://wiki.cancerimagingarchive.net/x/xwxp\" target=\"_blank\">Head-Neck Cetuximab wiki page</a> </span><span class=\"iceOutTxt\">to learn more about the images and to obtain any supporting metadata for this collection</span>.</p>\n</p>\n',120),(84,'lung_phantom','<p>\n	The FDA anthropomorphic thorax phantom with 12 phantom lesions of different sizes (10 and 20 mm in effective diameter), shapes (spherical, elliptical, lobulated, and spiculated), and densities (&minus;630,&minus;10, and +100 HU) was scanned at Columbia University Medical Center on a 64-detector row scanner (LightSpeed VCT, GE Healthcare, Milwaukee, WI). The CT scanning parameters were 120 kVp, 100 mAs, 64x0.625 collimation, and pitch of 1.375. The images were reconstructed with the lung kernel using 1.25 mm slice thickness.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.08A1IXOO\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Lung Phantom <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(85,'ct_lymph_nodes','<p>\n	This collection consists of Computed Tomography (CT) images of the mediastinum and abdomen in which lymph node positions are marked by radiologists at the National Institutes of Health, Clinical Center. <a href=\"https://www.cc.nih.gov/meet-our-doctors/rsummers.html\">Radiologists at the Imaging Biomarkers and Computer-Aided Diagnosis Laboratory</a> labeled a total of 388 mediastinal lymph nodes in CT images of 90 patients and a total of 595 abdominal lymph nodes in 86 patients.<br /></p>\n	<br /><p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.AQIIDCNM\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CT Lymph Nodes  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>wiki page to learn more about the images and to obtain supporting data from this collection.</p>\n',120),(86,'cc_radiomics_phantom_3','<p>\n	<strong>CT Phantom Scans for Head, Chest, and Controlled Protocols on 100 Scanners (CC-Radiomics-Phantom-3)</strong></p>\n<p>\n	This data collection contains one physical phantom, imaged across three protocols, on 100 scanners. CT scans were acquired on 100 scanners at 35 clinics: 51 GE scanners, 20 Philips scanners, 11 Toshiba scanners, and 1 Philips and Neusoft Medical System scanner. The commonly used chest and head protocols of the local clinic were acquired without changing the protocol parameters. Additionally, a controlled protocol was acquired that was designed to minimize radiomics feature differences between manufacturers.&nbsp;</p>\n<p>\n	Please see the&nbsp;<a href=\"http://doi.org/10.7937/tcia.2019.j71i4fah\">CC-Radiomics-Phantom-3</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(87,'pelvic_reference_data','<p>\n	This collection links expertly selected landmark points on clinical image pairs to provide a basis for rigid registration validation. Using combinatorial rigid registration optimization (CORRO) we provide a statistically characterized reference data set for image registration of the pelvis by estimating the optimal ground truth. Landmark points for each CT/CBCT image pair for 58 pelvic cases were identified. The mean and the standard deviation of the registration were used as the final registration for each image pair.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.woskq5oo\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Pelvic-Reference-Data <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n	&nbsp;</p>\n',120),(88,'vestibular_schwannoma_seg','	This collection contains a labelled dataset of MRI images collected on 242 consecutive patients with vestibular schwannoma (VS) undergoing Gamma Knife stereotactic radiosurgery (GK SRS). The structural images included contrast-enhanced T1-weighted (ceT1) images and high-resolution T2-weighted (hrT2) images. Each imaging dataset is accompanied by the patient&rsquo;s radiation therapy (RT) dataset including the RTDose, RTStructures and RTPlan.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.9YTJ-5Q73\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Vestibular-Schwannoma-SEG <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(89,'covid_19_ar','<p>\n	<span>Radiology imaging&nbsp; is playing an increasingly vital role in the diagnosis of COVID-19 patients and determining therapeutic options, patient care management and new research directions.&nbsp;</span><span>We have published a collection of radiographic and CT imaging studies for patients who tested positive for COVID-19. Each patient is described by a limited set of clinical data correlates that includes demographics, comorbidities, selected lab data and key radiology findings.&nbsp;These data are cross-linked to SARS-COV-2 cDNA sequence data extracted from clinical isolates from the same population, uploaded to the Genbank repository.</span></p>\n\n<br /> <p>\n\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/tcia.2020.py71-5978\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">COVID-19-AR <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(90,'nsclc_radiomics_genomics','<p>This collection contains images from 89 non-small cell lung cancer (NSCLC) patients that were treated with surgery. For these patients pretreatment CT scans, gene expression, and clinical data are available. This dataset refers to the Lung3 dataset of the study published in Nature Communications.</p><br />\n<p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.L4FRET6Z\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">NSCLC Radiomics Genomics <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection. </p>\n',120),(91,'ivygap','<p>\n	This data collection consists of MRI/CT scan data for brain tumor patients that form the cohort for the resource Ivy Glioblastoma Atlas Project (Ivy GAP). There are 390 studies for 39 patients that include pre-surgery, post-surgery and follow up scans.</p>\n<p>\n	The Ivy Glioblastoma Atlas Project (Ivy GAP) is a collaborative partnership between the Ben and Catherine Ivy Foundation, which generously provided the financial support, the Allen Institute for Brain Science, and the Ben and Catherine Ivy Center for Advanced Brain Tumor Treatment. The goal of the project is to provide online resources to scientists and physicians dedicated to the development of innovative treatments and diagnostics that will enhance the quality of life and survival of patients with brain cancer.</p>\n<p>\n	See the <a href=\"https://doi.org/10.7937/K9/TCIA.2016.XLwaN6nL\">Ivy GAP</a> wiki page for more information and supporting metadata.</p>\n',120),(92,'mri_dir','<div>\n	Two sets of images were created to evaluate deformable image registration accuracy. The first set contains CT, T1-, and T2-weighted images from a porcine phantom implanted with gold markers, immobilized, compressed in 4 different ways and images were acquired in each position.&nbsp;</p>\n<p>\n	The second set of images are synthetic images derived from 28 head and neck squamous cell carcinoma patients who had pre-, mid-, and post-radiotherapy treatment MR scans. From these patients, inter- and intra-patient models were created.</p>\n<p>\n	Please see the <a href=\"https://10.7937/K9/TCIA.2018.3f08iejt\">MRI-DIR</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection</p>\n',120),(93,'lungct_diagnosis','<p>\n	This is a small set of 61 diagnostic contrast enhanced CT scans with known tumors.&nbsp; The images were retrospectively acquired, to ensure sufficient patient follow-up. Slice thickness is variable : between 3 and 6 mm. All images were done at diagnosis and prior to surgery.<br />\n	&nbsp;</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.A6V7JIWX\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">LungCT-Diagnosis <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(94,'spie_aapm_lung_ct_challenge','<p>\n	As part of the 2015 SPIE Medical Imaging Conference, SPIE &ndash; with the support of American Association of Physicists in Medicine (AAPM) and the National Cancer Institute (NCI) &ndash; conducted a &ldquo;Grand Challenge&rdquo; on quantitative image analysis methods for the diagnostic classification of malignant and benign lung nodules. The LUNGx Challenge provided a unique opportunity for participants to compare their algorithms to those of others from academia, industry, and government in a structured, direct way using the same data sets.</p><br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.UZLSU3FL\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">SPIE-AAPM Lung CT Challenge <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(95,'qin_breast_dce_mri','<p>\n	This collection of breast dynamic contrast-enhanced (DCE) MRI data contains images from a longitudinal study to assess breast cancer response to neoadjuvant chemotherapy. Images were acquired at four time points: prior to the start of treatment (Visit 1, V1), after the first cycle of treatment (Visit 2, V2), at midpoint of treatment course (Visit 3, V3), and after completion of treatment (prior to surgery) (Visit 4, V4). The value of this collection is to provide clinical imaging data for the development and validation of quantitative imaging methods for assessment of breast cancer response to treatment.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/k9/tcia.2014.a2n1ixox\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIN Breast DCE-MRI <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(96,'breast_diagnosis','<p>\n	The Breast-Diagnosis collection contains cases that are high-risk normals, DCIS, fibroids and lobular carcinomas. Each case has 3 or more distinct MR pulse sequences from a Phillips 1.5 T (usual sequences are labeled T2, STIR and BLISS but may occasionally include other pulse sequences and digital mammogram of tumor specimen).&nbsp; Multiple time point studies on the same patient are possible.&nbsp;</p>\n<p>\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2015.SDNRQXXR\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">BREAST-DIAGNOSIS <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(97,'hnscc','<p>\n	This collection contains imaging, radiation therapy, and clinical data from 627 head and neck squamous cell carcinoma (HNSCC) patients&nbsp;at MD Anderson Cancer Center.&nbsp; Researchers at MDACC analyzed these patients&#39; data as part of two separate research projects.</p>\n<p>\n	&nbsp;Please see Grossberg et al: Data from Head and Neck Cancer CT Atlas (<a href=\"https://doi.org/10.7937/k9/tcia.2020.a8sh-7363\">HNSCC</a>) wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n	<br />\n	&nbsp;</p>\n',120),(98,'prostate_diagnosis','<p>\n	Prostate, cancer-confirmed, MRIs obtained using endorectal and phased array surface coils at 1.5T (Philips Achieva). Pulse sequences include T1, T2 and DCE (&ldquo;BLISS&rdquo; sequence using Magnevist injected i.v.). Linked clinical information, including matched patient demographics, radiology and pathology reports in XLS format are available.</p>\n<p>\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2015.FOQEUJVT\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prostate-Diagnosis <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n	Data was provided by Boston Medical Center courtesy Dr. Nicolas Bloch.<br />\n	&nbsp;</p>\n',120),(99,'lgg_1p19qdeletion','<p>\n	These MRIs are pre-operative examinations performed in 159 subjects with Low Grade Gliomas (WHO grade II &amp; III). T1, T2, and Segmentations are provided in DICOM format here.&nbsp;</p>\n<p>\n	Please see the <a href=\"https://doi.org/10.7937/K9/TCIA.2017.dwehtz9v\">LGG-1p19qDeletion</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n	<o:p></o:p></p>\n',120),(100,'cptac_lscc','<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Lung Squamous Cell Carcinoma (CPTAC-LSCC) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n	&nbsp;</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.6EMUB5L2\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-LSCC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n',120),(101,'cptac_gbm','\n<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Glioblastoma Multiforme (CPTAC-GBM) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n	&nbsp;</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.3RJE41Q1\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-GBM <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>',120),(102,'cptac_hnscc','<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Head-and-Neck cancer (CPTAC-HNSCC) cohort.</span><span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\"><span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\"> </span>CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\"><span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.UW45NH81\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-HNSCC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n',120),(103,'cptac_luad','	<p>	This collection contains subjects from the National Cancer Institute&rsquo;s <a href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a> Lung Adenocarcinoma (CPTAC-LUAD) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</p>\n	<p>	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.PAT12TBS\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-LUAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n\n',120),(104,'cptac_ccrcc','<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Clear Cell Renal Cell Carcinoma (CPTAC-CCRCC) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n	&nbsp;</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.OBLAMN27\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-CCRCC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n',120),(105,'cptac_ucec','This collection contains subjects from the National Cancer Institute\'\'s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Uterine Corpus Endometrial Carcinoma (CPTAC-UCEC) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</p><p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.3R3JUISW\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-UCEC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(106,'rembrandt','<p>\n	This image collection was created as a companion data set to augment the larger <a class=\"external-link\" href=\"http://caintegrator-info.nci.nih.gov/rembrandt\" rel=\"nofollow\">REMBRANDT</a> project. It consists of 130 cases of MR multi-sequence images of pre-surgical brain tumor patients including cases of glioblastoma, astrocytomas and oligodendrogliomas from the REMBRANDT patient population.</p>\n<p>\n	Please see the <a href=\"https://doi.org/10.7937/K9/TCIA.2015.588OZUZB\" target=\"_blank\">REMBRANDT</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(107,'nsclc_radiogenomics','<p>\n	This collection contains images from patients with non-small cell lung cancer imaged prior to surgical excision with both thin-section CT and whole body PET/CT scans acquired under IRB approval from Stanford University and the Veterans Administration Palo Alto Health Care System.</p>\n<p>\n	The first installment of 26 cases (see shared list &quot;NSCLC Radiogenomics: Initial Stanford Study of 26 Cases&quot;) corresponds to microarray data acquired from the excised samples, which is available on the NCBI Gene Expression Omnibus (<a class=\"external-link\" href=\"http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE28827\" rel=\"nofollow\">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE28827</a>), where DICOM patient names are identical to microarray sample names.</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id487\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2017.7hs46erv\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">NSCLC Radiogenomics <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n',120),(108,'head_neck_pet_ct','<p>\n	<span>This collection contains FDG-PET/CT and radiotherapy planning CT imaging data of&nbsp;<span>298</span><span>&nbsp;</span>patients from four different institutions in Qu&eacute;bec with histologically proven head-and-neck cancer (H&amp;N) All patients had pre-treatment FDG-PET/CT scans between April 2006 and November 2014, and within a median of 18 days (range: 6-66) before treatment. </span></p>\n<p>\n	<span>Please see the <a href=\"https://doi.org/10.7937/K9/TCIA.2017.8oje5q00\">Head-Neck-PET-CT</a> wiki page <span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n',120),(109,'soft_tissue_sarcoma','<p>\n	In this study, a cohort of 51 patients with histologically proven soft-tissue sarcomas (STSs) of the extremities was retrospectively evaluated. During the follow-up period, 19 patients developed lung metastases. Patients with metastatic and/or recurrent STSs at presentation were excluded from the study. All patients had pre-treatment FDG-PET/CT and MRI scans.</p>\n<p>\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2015.7GO2GSKS\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Soft Tissue Sarcoma <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(110,'mouse_mammary','<p>\n	This collection consists of Magnetic Resonance Imaging (MRI) of mouse breast cancer in 32 mice. &nbsp;</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.9P42KSE6\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Mouse-Mammary <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(111,'nsclc_radiomics_interobserver1','<b><a href=\"\" url=\"\" url=\"https://doi.org/10.7937/tcia.2019.cwvlpd26\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">NSCLC-Radiomics-Interobserver1 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> </b>contains clinical data and computed tomography (CT) from 22 non-small cell lung cancer (NSCLC) radiotherapy patients.  This dataset refers to the &#34;Multiple delineation&#34; dataset of the study published in Nature Communications (<a href=\"\" url=\"http://doi.org/10.1038/ncomms5006\" rel=\"nofollow\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">https://doi.org/10.1038/ncomms5006 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>). In short, the publication used a radiomics approach to computed tomography data of 1,019 patients with lung or head-and-neck cancer.</span><div><br></p><div>Please see the <a href=\"\" url=\"\" url=\"https://doi.org/10.7937/tcia.2019.cwvlpd26\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">NSCLC-Radiomics-Interobserver1 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(112,'qin_breast','<div>\n	This collection contains longitudinal PET/CT and quantitative MR images collected for the purpose of studying treatment assessment in breast cancer in the neoadjuvant setting. Images were acquired at three time points: prior to the start of treatment (t1), after the first cycle of treatment (t2), and either after the second cycle of treatment or at the completion of all treatments (prior to surgery) (t3). The PET/CT images were acquired with a support device built in-house to allow the patient to be in the prone position to facilitate registration with the MRI data. The value of this collection is to provide clinical imaging data for the development and evaluation of quantitative imaging methods for treatment assessment early in the course of therapy for breast cancer.&nbsp;</p>\n<div>\n	&nbsp;</p>\n<div>\n	An extension of this dataset is available via the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.4cfm06rr\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIN-BREAST-02 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;collection.</p>\n<div>\n	&nbsp;</p>\n<div>\n	Please see the&nbsp;<a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.21JUebH0\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIN-Breast <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.&nbsp; &nbsp;</p>\n',120),(113,'breast_mri_nact_pilot','<p>\n	<span>This pilot study to investigate the use of serial DCE MRI examinations during neoadjuvant chemotherapy (NACT) for invasive breast cancer recruited 68 patients with stage II or III locally advanced breast cancer. L<span>ongitudinal DCE MRI studies of 64 patients are available.</span></span></p>\n<p>\n	<span>Please see the wiki page <span><a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.QHSYHJKY\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Breast-MRI-NACT-Pilot <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n',120),(114,'apollo_5_lscc','<p class=\"MsoNormal\"></p><p><span>This collection contains subjects from the National Cancer&#10;Institute&#8217;s Applied Proteogenomics OrganizationaL Learning and Outcomes&#10;(APOLLO) network Lung Squamous Cell Carcinoma cohort. The APOLLO network is a&#10;collaboration between NCI, the Department of Defense (DoD), and the Department&#10;of Veterans Affairs (VA) to incorporate proteogenomics into patient care as a&#10;way of looking beyond the genome, to the activity and expression of the&#10;proteins that the genome encodes.</span>&#160;</p><p><span>Please see the&#160;</span><a href=\"https://doi.org/10.7937/TCIA.QQ0G-EB24\" target=\"_blank\">APOLLO-5-LSCC</a><span>&#160;wiki page to learn&#10;more about the images and to obtain any supporting metadata for this&#10;collection.</span></p><p class=\"MsoListParagraph\"></p>&#10;<p>&#160; &#160;</p>&#10;',120),(115,'nlst','<p>This collection contains all CT exams from the National Lung Screening Trial (NLST), which  was conducted to determine whether screening with low-dose CT could reduce mortality from lung cancer for persons at high risk for lung cancer at 33 U.S. medical centers.  Participants were randomly assigned to undergo three annual screenings between 2002-2004.&#160;</p><p><i>NOTE: The full dataset occupies over 11 Terabytes on disk. Ensure you have what you need before downloading.</i></p><br><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.HMQ8-J677\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">NLST <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(116,'acrin_hnscc_fdg_pet_ct','<p>\n	<span lang=\"EN\">A Multicenter Trial of FDG-PET/CT Staging of Head and Neck Cancer&nbsp;</span><span lang=\"EN\">and its Impact on the N0 Neck Surgical Treatment in Head and Neck Cancer Patients.&nbsp;</span><span>The objective of this study was to determine the negative predictive value (NPV)&nbsp;of positron emission tomography (PET)/computed tomography CT for clinically N0 neck based upon pathologic sampling of the neck lymph nodes and to&nbsp;determine PET/CT&rsquo;s potential to change treatment of the N0 neck.&nbsp;</span></p>\n',120),(117,'b_mode_and_ceus_liver','<p>This Collection contains <u><strong>c</strong></u>ontrast-<u><strong>e</strong></u>nhanced <u><strong>u</strong></u>ltra<u><strong>s</strong></u>ound to a) characterize liver lesions and b) monitor treatment response. Sagittal and transverse plane ultrasound data was obtained with curvilinear probe.</p><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2021.v4z7-tc39\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">B-mode-and-CEUS-Liver <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(118,'cmmd','<div><span>This collection contains clinical data and 3,728 mammographies (MG) from 1,775 patients, with biopsy-confirmed benign or malignant breast tumors. For a subset of 749 patients (D2-####), molecular subtype is included.&#160;</span></p><div><span>Please see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/tcia.eqde-4b16\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CMMD&#160; <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>wiki page for further detail and associated metadata.</span></p>',120),(119,'covid_19_ny_sbu','<p>&#10;&#9;The collection of cases was acquired at Stony Brook University from patients who tested positive for COVID-19. The collection includes images from different modalities and organ sites (chest radiographs, chest CTs, brain MRIs, etc.). Radiology imaging data is extremely important in COVID-19 from both a diagnostic and a monitoring perspective, given the crucial nature of COVID-19 pulmonary disease and its rapid phenotypic changes.</p><p><br></p>&#10;<p>&#10;&#9;Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.BBAG-2923\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Stony Brook University COVID-19 Positive Cases <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>&#10;',120),(120,'duke_breast_cancer_mri','<p>\n	<strong>Dynamic contrast-enhanced magnetic resonance images of breast cancer patients with tumor locations (Duke-Breast-Cancer-MRI)</strong><br />\n	<br />\n	This collection contains MRI imaging and other data for 922 patients with invasive breast cancer. Prone position axial breast MRI images were acquired by 1.5T or 3T scanners;&nbsp; non-fat saturated T1, fat-saturated gradient echo T1 pre-contrast, and 3-4 post-contrast sequences are included.<br />\n	<br />\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.e3sv-re93\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Duke-Breast-Cancer-MRI <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain the supporting metadata for this collection.</p>\n',120),(121,'apollo_5_esca','<p class=\"MsoNormal\">This collection contains subjects from the National Cancer&#10;Institute&#8217;s Applied Proteogenomics OrganizationaL Learning and Outcomes&#10;(APOLLO) network&#160;<span>Esophageal Squamous Cell Carcinoma cohort.&#160;</span><span>The APOLLO network is a collaboration between&#10;NCI, the Department of Defense (DoD), and the Department of Veterans Affairs&#10;(VA) to incorporate proteogenomics into patient care as a way of looking beyond&#10;the genome, to the activity and expression of the proteins that the genome&#10;encodes.</span></p><p class=\"MsoNormal\"></p><div><span><font face=\"-apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji\">P</font>lease see the&#160;</span><a href=\"https://wiki.cancerimagingarchive.net/x/uAC_BQ\" target=\"_blank\">APOLLO-5-ESCA</a><span>&#160;wiki page to learn&#10;more about the images and to obtain any supporting metadata for this&#10;collection.&#160;</span><br></p><div><div><p class=\"MsoListParagraph\"></p><div><span><br></span></p></p></p>',120),(122,'apollo_5_luad','<p class=\"MsoNormal\">This collection contains subjects from the National Cancer&#10;Institute&#8217;s Applied Proteogenomics OrganizationaL Learning and Outcomes&#10;(APOLLO) network Lung Adenocarcinoma&#160;<span>cohort</span>. The APOLLO network is a collaboration between NCI,&#10;the Department of Defense (DoD), and the Department of Veterans Affairs (VA) to&#10;incorporate proteogenomics into patient care as a way of looking beyond the&#10;genome, to the activity and expression of the proteins that the genome encodes.</p><p class=\"MsoNormal\"><span>Please see the&#160;</span><a href=\"https://wiki.cancerimagingarchive.net/x/CgC_BQ\" target=\"_blank\">APOLLO-5-LUAD</a><span>&#160;wiki page to learn more about the&#10;images and to obtain any supporting metadata for this collection.&#160;</span><br></p><p class=\"MsoNormal\"></p><p class=\"MsoNormal\"><br></p><p class=\"MsoNormal\"><br></p>',120),(123,'apollo_5_paad','<p class=\"MsoNormal\">This collection contains subjects from the National Cancer&#10;Institute&#8217;s Applied Proteogenomics OrganizationaL Learning and Outcomes&#10;(APOLLO) network&#160;<span><span>Pancreatic Adenocarcinoma</span></span><span>&#160;cohort.&#160;</span>&#160;The APOLLO network is a collaboration between&#10;NCI, the Department of Defense (DoD), and the Department of Veterans Affairs&#10;(VA) to incorporate proteogenomics into patient care as a way of looking beyond&#10;the genome, to the activity and expression of the proteins that the genome&#10;encodes.</p><p class=\"MsoNormal\">Please see the&#160;<a href=\"https://wiki.cancerimagingarchive.net/x/xgC_BQ\" target=\"_blank\">APOLLO-5-PAAD</a>&#160;<span>wiki page to learn more about the&#10;images and to obtain any supporting metadata for this collection.</span></p><p class=\"MsoNormal\"></p>',120),(124,'apollo_5_thym','<p class=\"MsoNormal\"><span>This collection contains&#10;subjects from the&#160;</span><span>National Cancer Institute&#8217;s Applied&#10;Proteogenomics OrganizationaL Learning and Outcomes (APOLLO) network</span><span>&#160;</span><span>Thymoma&#160;cohort.&#160;The APOLLO network is&#10;a collaboration between&#160;</span><span>NCI</span><span>, the Department of Defense (DoD), and the&#10;Department of Veterans Affairs (VA) to incorporate proteogenomics into patient&#10;care as a way of looking beyond the genome, to the activity and expression of&#10;the proteins that the genome encodes.&#160;</span><b></b></p><div><div><p class=\"MsoListParagraph\">Please see the&#160;<a href=\"https://wiki.cancerimagingarchive.net/x/DYYTBg\" target=\"_blank\">APOLLO-5-THYM</a>&#160;wiki page to learn&#10;more about the images and to obtain any supporting metadata for this&#10;collection.&#160;</p></p></p>',120),(125,'pancreatic_ct_cbct_seg','<span>The objective of this dataset is to provide a means of evaluating the performance of CT-to-CBCT deformable registration and auto-segmentation algorithms for delineating OARs from 40 patients who received ablative radiation therapy for locally advanced pancreatic cancer.</span><span>The images (one treatment planning CT and two CBCT scans for each of 40 patients) were acquired during a deep inspiration breath-hold verified with an external respiratory monitor.</span><br><span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/tcia.eshq-4d90\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Pancreatic-CT-CBCT-SEG <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span>',120),(126,'pediatric_ct_seg','<p>\n	<span>This dataset was collected by a collaboration of researchers from Children&rsquo;s Wisconsin, Marquette University, Varian Medical Systems, Medical College of Wisconsin, and Stanford University as part of a project funded by the National Institute of Biomedical Imaging and Bioengineering (U01EB023822) to develop tools for rapid, patient-specific CT organ dose estimation. The collection consists of CT images in DICOM format of 359 pediatric chest-abdomen-pelvis or abdomen-pelvis exams acquired from three CT scanners.&nbsp;The datasets represent random pediatric cases based upon routine clinical indications. Each dataset contains expert contours of&nbsp;up to&nbsp;twenty-nine structures in DICOM RTSS format.&nbsp;</span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.X0H0-1706\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Pediatric-CT-SEG <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(127,'tcga_acc','<p>\n      Adrenocortical carcinoma is a rare cancer that develops in the outer layer of tissue of the adrenal glands, organs that lie on top of each kidney. This outer layer known as the adrenal cortex produces important hormones called steroids that help the body deal with stress, regulate blood pressure and the amount of salt in the blood, as well as cause the body to acquire masculine or feminine characteristics. A tumor of the adrenal cortex can produce either no hormones or excess hormones.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/adrenocortical\" target=\"_blank\">TCGA-ACC</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>',120),(128,'tcga_chol','<p>\n      Cholangiocarcinoma is a cancer that develops in the bile duct. The bile duct is a network of tubes that carry bile from the liver and gallbladder to the small intestine. Tumors that start in bile duct branches that lie inside the liver are called intrahepatic bile duct cancer, while those that form outside the liver are called extrahepatic bile duct cancer. About 10% of all cholangiocarcinoma are intrahepatic and 90% are extrahepatic. TCGA studied both subtypes of cholangiocarcinoma.</p>\n<p>\n      Although cholangiocarcinoma is a rare cancer, the incidence and mortality rates for the disease have been increasing worldwide in the last three decades. Between 2,000 and 3,000 Americans are diagnosed with cholangiocarcinoma each year, the majority of them with tumors at advanced stages. This cancer is more prevalent in Asia and the Middle East, where parasitic infection of the bile duct increases the risk of cholangiocarcinoma. Other diseases of the bile duct or liver, such as bile duct stones and liver disease, obesity, diabetes, and smoking are also risk factors. When intrahepatic and extrahepatic cholangiocarcinoma spread to other parts of the body, only 2% of patients survive five years after diagnosis.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/cholangiocarcinoma\" target=\"_blank\">TCGA-CHOL</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>',120),(129,'tcga_dlbc','<p>\n       There is no available description page for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>',120),(130,'tcga_meso','<p>\n      Mesothelioma is a rare cancer that affects the thin layer of tissue that lines the chest, abdominal cavities, and most of the organs within them. This lining is called the mesothelium.\nIn the United States, there are roughly 3,000 new cases diagnosed each year. Exposure to asbestos is the main risk factor for developing this disease, and men tend to be more commonly affected— a fact that most likely correlates with men holding jobs where they are more likely to come in contact with it.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/mesothelioma\" target=\"_blank\">TCGA-MESO</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(131,'tcga_paad','<p>\n      Pancreatic ductal adenocarcinoma is the most common form of pancreatic cancer, making up more than 80% of cases. The disease begins in the cells of the pancreas’s ducts, which transport juices containing digestive enzymes into the small intestine.</p>\n<p>\n      Pancreatic cancer is the fourth most common cause of global cancer-related deaths and is almost always fatal. In 2012, it was estimated that around 44,000 new cases of pancreatic cancer were diagnosed and more than 37,000 deaths from this disease occurred in the United States alone, affecting both men and women.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/pancreatic\" target=\"_blank\">TCGA-PAAD</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>',120),(132,'tcga_pcpg','<p>\n      Paraganglioma is a rare cancer that originates in the nerve cells of the adrenal glands, organs on top of each kidney that produce important hormones. Paraganglioma that develops in the center of the adrenal gland is called pheochromocytoma. Paraganglioma that forms outside of the adrenal gland, often along blood vessels and nerves in the head and neck, is called extra-adrenal paraganglioma, or simply paraganglioma.\nEach year, between 2 and 8 people per million worldwide are diagnosed with paraganglioma and pheochromocytoma. 10% of all cases occur in children. In both adults and children, pheochromocytoma is more common than paraganglioma. No known environmental, dietary, or lifestyle risk factors have been associated with these cancers.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/paraganglioma\" target=\"_blank\">TCGA-PCPG</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>',120),(133,'tcga_skcm','<p>\n      Melanoma is a cancer in a type of skin cells called melanocytes. Melanocyes are the cells that produce melanin, which colors the skin. When exposed to sun, these cells make more melanin, causing the skin to darken or tan. Melanoma can occur anywhere on the body and risk factors include fair complexion, family history of melanoma, and being exposed to natural or artificial sunlight over long periods of time. Melanoma is most often discovered because it has metastasized, or spread, to another organ, such as the lymph nodes. In many cases, the primary skin melanoma site is never found. Because of this challenge, TCGA is studying primarily metastatic cases (in contrast to other cancers selected for study, where metastatic cases are excluded). For 2011, it was estimated that there were 70,230 new cases of melanoma and 8,790 deaths from the disease.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/pancreatic\" target=\"_blank\">TCGA-SKCM</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>',120),(134,'tcga_tgct','</p>\n      More than 90% of testicular cancer start in the germ cells, which are cells in the testicles and develop into sperm. This type of cancer is known as testicular germ cell cancer. Testicular germ cell cancer can be classified as either seminomas or nonseminomas, which may be identified by microscopy. Nonseminomas typically grow and spread more quickly than seminomas. A testicular germ cell tumor that contains a mix of both these subtypes is classified as a nonseminoma. TCGA studied both seminomas and nonseminomas.</p>\n<p>\n      Testicular germ cell cancer is rare, comprising 1-2% of all tumors in males. However, it is the most common cancer in men ages 15 to 35. The incidence of testicular germ cell cancer has been continuously rising in many countries, including Europe and the U.S. In 2013, about 8,000 American men were estimated to be diagnosed with the cancer. Of those, 370 are predicted to die from the disease. Men who are Caucasian, have an undescended testicle, abnormally developed testicles, or a family history of testicular cancer have a greater risk of developing testicular cancer. Fortunately, testicular germ cell cancer is highly treatable.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/testicular-germ-cell\" target=\"_blank\">TCGA-TGCT</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>',120),(135,'tcga_thym','<p>\n      This cancer develops in the outer surface of the thymus, a gland behind the breastbone that produces T-cells, a type of white blood cells. Thymoma is rare, but it is the most common tumor in adults affecting the mediastinum, which is the cavity between the lungs containing the heart, esophagus, and trachea. A tumor of the thymus tends to grow slowly and rarely spreads to other parts of the body. However, of the estimated 400 Americans who develop this cancer each year, half are diagnosed with metastatic thymoma. When the cancer metastasizes, only 45% of patients survive five years after their diagnosis.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/thymoma\" target=\"_blank\">TCGA-THYM</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>',120),(136,'tcga_ucs','<p>\n      Uterine carcinosarcoma (UCS) is a cancer that develops in the uterus. Carcinosarcoma signifies that, when looked at under a microscope, the tumor displays histological features of both endometrial carcinoma and sarcoma. Endometrial carcinoma starts in the endometrium, the inner layer of tissue lining the uterus, while sarcoma begins in the outer layer of muscle of the uterus.</p\n<p>\n      A rare cancer, UCS makes up less than 5% of all uterine cancers. In the U.S., about two per 100,000 women develop UCS annually. Roughly only 35% of patients survive five years after diagnosis. </p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/uterine-carcinosarcoma\" target=\"_blank\">TCGA-UCS</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>',120),(137,'tcga_uvm','<p>\n      Uveal (intraocular or eye) melanoma develops in the pigment cells of the uvea, which is the middle layer of the eye. The uvea consists of three main parts: the iris, ciliary body, and choroid. Compared to tumors of the iris, tumors of the ciliary body and choroid tend to be larger and more likely to spread to other parts of the body. TCGA studied tumors from all three parts of the uvea.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/melanoma-eye\" target=\"_blank\">TCGA-UVM</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>',120),(138,'cptac_aml','\n<p>\n    <span>This collection contains subjects from the National Cancer Institute&rsquo;s <u><a href=\"https://proteomics.cancer.gov/programs/cptac\" class=\"external-link\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Acute Myeloid Leukemia (CPTAC-AML) cohort.&nbsp;<span>CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics.</span></p>\n<p>\n    Please see the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-AML\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-AML <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(139,'cptac_brca','\n<p>\n    <span>This collection contains subjects from the National Cancer Institute&rsquo;s <u><a href=\"https://proteomics.cancer.gov/programs/cptac\" class=\"external-link\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> CPTAC Breast Invasive Carcinoma cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n    Please see the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-BRCA\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-BRCA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(140,'cptac_coad','\n<p>\n    <span>This collection contains subjects from the National Cancer Institute&rsquo;s <u><a href=\"https://proteomics.cancer.gov/programs/cptac\" class=\"external-link\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> CPTAC&nbsp;Colon Adenocarcinoma cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics.</span></p>\n<p>\n    Please see the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-COAD\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-COAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(141,'cptac_ov','\n<p>\n    <span>This collection contains subjects from the National Cancer Institute&rsquo;s <u><a href=\"https://proteomics.cancer.gov/programs/cptac\" class=\"external-link\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> CPTAC&nbsp;Ovarian Serous Cystadenocarcinoma cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics.</span></p>\n<p>\n    Please see the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-OV\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-OV <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(142,'acrin_contralateral_breast_mr','<p>\n	<span>Even after careful clinical and mammographic evaluation, cancer is found in the contralateral breast in up to 10% of women who have received treatment for unilateral breast cancer. ACRIN 6667 was conducted to determine whether magnetic resonance imaging (MRI) could improve on clinical breast examination and mammography in detecting contralateral breast cancer soon after the initial diagnosis of unilateral breast cancer. </span> Please see the <a href=\"\" url=\"https://doi.org/10.7937/Q1EE-J082\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ACRIN Contralateral Breast MR (ACRIN 6667) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(143,'stageii_colorectal_ct','<span>This dataset includes abdominal or pelvic enhanced CT images within 10 days before surgery of 230 patients with stage II colorectal cancer (CRC). Images were acquired on Sensation 64 (Siemens Healthcare, Erlangen, Germany) CT scanner or Brilliance (Philips Healthcare, Best, The Netherlands) CT scanner.&#160;</span><div><div>Please see the <a href=\"\" url=\"https://doi.org/10.7937/p5k5-tg43\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">StageII-Colorectal-CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki to learn more about the images and to obtain any supporting metadata for this collection.</p></p>',120),(144,'DICOM-LIDC-IDRI-Nodules','<p> This dataset contains standardized DICOM representation of the volumetric nodule annotations and characterizations collected by the LIDC/IDRI initiative.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/TCIA.2018.h7umfurq\" data-toggle=\"modal\" data-target=\"#external-web-warning\">DICOM-LIDC-IDRI-Nodules <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(145,'QIN-LungCT-Seg','<p> This collection contains lung nodule segmentations obtained using segmentation tools that participated in the QIN challenge “moist run” initiative.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.1BUVFJR7\" data-toggle=\"modal\" data-target=\"#external-web-warning\">QIN-LungCT-Seg <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(146,'DICOM-SR-Breast-Clinical','<p> This collection contains clinical metadata corresponding to several breast imaging collections harmonized into DICOM SR representation, generated by the Data Integration & Imaging Informatics (DI-Cubed) project.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.wgllssg1\" data-toggle=\"modal\" data-target=\"#external-web-warning\">DICOM-SR-Breast-Clinical <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(147,'PROSTATEx-Seg-Zones','<p> This collection contains prostate’s CNN-based automatic zonal segmentation for 98 cases randomly selected from the SPIE-AAPM-NCI PROSTATEx Challenges (PROSTATEx). The four-class segmentation encompasses the peripheral zone, transition zone, fibromuscular stroma and the distal prostatic urethra.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/tcia.nbb4-4655\" data-toggle=\"modal\" data-target=\"#external-web-warning\">PROSTATEx-Seg-Zones <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(148,'PROSTATEx-Seg-HiRes','<p> This collection contains 66 high resolution automatic CNN-based segmentations of the prostate gland for randomly chosen T2-weighted volumes of the SPIE-AAPM-NCI PROSTATEx Challenges (PROSTATEx).</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.DEG7ZG1U\" data-toggle=\"modal\" data-target=\"#external-web-warning\">PROSTATEx-Seg-HiRes <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(149,'RIDER-LungCT-Seg','<p> This collection contains manual and automatic segmentations of the gross tumor volume for the RIDER Lung CT collection.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/tcia.2020.jit9grk8\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER-LungCT-Seg <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(150,'DICOM-Glioma-SEG','<p> This dataset contains DICOM-SEG (DSO) conversions of the Segmentation Labels and Radiomic Features for the Pre-operative Scans of the TCGA-LGG collection and Segmentation Labels and Radiomic Features for the Pre-operative Scans of the TCGA-GBM collection  analysis datasets.</p> <p>  Please see the <a href=https://doi.org/10.7937/TCIA.2018.ow6ce3ml target=_blank>DICOM-Glioma-SEG</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(151,'acrin_6698','<span>The American College of Radiology Imaging Network (ACRIN) trial 6698 was a multi-center study to evaluate the effectiveness of quantitative diffusion weighted imaging (DWI) for assessing breast cancer response to neoadjuvant chemotherapy (NAC).&#160; &#160;ACRIN 6698 was performed as a sub-study of the ongoing I-SPY 2 TRIAL (Investigation of Serial studies to Predict Your Therapeutic Response with Imaging And moLecular Analysis 2), an adaptive, multi-agent phase II trial designed to quickly identify new agents for breast cancer.&#160;&#160;</span><div><span><br></span></p><div><span>Please see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/TCIA.KK02-6D95\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ACRIN 6698/I-SPY2 Breast DWI&#160; <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>wiki page to learn more about the images and to obtain any supporting metadata for this collection.&#160;</span><div><span>&#160;</span></p></p>',120),(152,'htan_hms','<p>\nWe are constructing a multi-dimensional atlas of pre-melanoma focused on understanding genetic and epigenetic events that transform melanocytes into invasive tumors. Melanoma is a cancer of increasing prevalence that is curable with minor surgery if detected early but life-threatening when it metastasizes. Melanomas metastasize when still small, making early detection essential but challenging. Our atlas will delineate the precise sequence of events leading up to pre-melanoma through detailed spatial analysis of cell-autonomous events such as oncogene mutation and non-autonomous events such as escape from immune surveillance. The atlas is based on highly-multiplexed tissue imaging and single cell sequencing and focused on samples in which the full sequence of events from atypia to invasive melanoma can be visualized in a single specimen. The atlas will serve as a publicly accessible resource for research scientists, physicians, and patients and improve our ability to (i) highlight lesions likely to progress to cancer, (ii) identify high-risk patients to inform decisions on surgery, (iii) identify low-risk patients to reduce unnecessary procedures, (iv) design improved procedures for routine screening of all individuals, and (v) inform treatment options when surgery is insufficient. Complementary studies with similar goals (but not supported by HTAN) are studying later stage melanomas.</p>\n<p>\n       Please see the <a href=\"\" url=\"https://humantumoratlas.org/hta7/\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN-HMS <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"\" url=\"https://docs.google.com/document/d/1seSp9yo8mqMK5sFXd4LEf_t_4SEofZi-1p-M75R1lhk/edit#heading=h.ybiv33wzts40\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN Publication Policy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page.</p>',120),(153,'htan_ohsu','<p>\nThe overall goal of the HTAN OMS Atlas Center is to elucidate mechanisms by which metastatic breast cancers become resistant to current generation pathway- and immune checkpoint-targeted treatments. The OMS Atlas is motivated by the appreciation that these treatments are often effective in primary tumors but only transiently effective in the metastatic setting. Possible resistance mechanisms include tumor-intrinsic genomic instability and epigenomic plasticity, as well as events extrinsic to the cancer cells, including chemical and mechanical signals from the microenvironments, production of mechanical extracellular matrix barriers and/or changes in vasculature that reduce drug and/or immune cell access, nanoscale cancer cell-microenvironment interactions that reduce drug efficacy, and a plethora of immune resistance mechanisms, such as loss of HLA expression and antigen presentation, and immune exhaustion. These mechanisms likely vary between patients and within individual patients and change with time as tumors respond to therapeutic attack. The OMS Atlas will focus on elucidating resistance mechanisms in two specific current generation clinical trial scenarios: (a) hormone receptor-positive breast cancer (HRBC) undergoing treatment with a CDK4/6 inhibitor in combination with endocrine therapy and (b) triple negative breast cancer (TNBC) undergoing treatment with a PARP inhibitor and an immunomodulatory agent.</p>\n<p>\n       Please see the <a href=\"\" url=\"https://humantumoratlas.org/hta9/\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN-OHSU <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"\" url=\"https://docs.google.com/document/d/1seSp9yo8mqMK5sFXd4LEf_t_4SEofZi-1p-M75R1lhk/edit#heading=h.ybiv33wzts40\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN Publication Policy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page.</p>\n',120),(154,'htan_wustl','<p>\n       Please see the <a href=\"\" url=\"https://humantumoratlas.org/hta12/\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN-WUSTL <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"\" url=\"https://docs.google.com/document/d/1seSp9yo8mqMK5sFXd4LEf_t_4SEofZi-1p-M75R1lhk/edit#heading=h.ybiv33wzts40\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN Publication Policy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page.</p>',120),(155,'ispy2','<p>\n	<span ><span><span >I-SPY 2 (Investigation of Serial studies to Predict Your Therapeutic Response with Imaging And moLecular analysis 2) is an ongoing, multi-center trial designed to quickly evaluate the efficacy of new agents for breast cancer in neoadjuvant chemotherapy (NAC) setting. Women aged &ge;18 years diagnosed with locally advanced breast cancer (tumor size &ge;2.5 cm) without distant metastasis are eligible to enroll in the trial. </span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.D8Z0-9T85\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">I-SPY 2 Breast Dynamic Contrast Enhanced MRI (I-SPY2 TRIAL) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n',120),(156,'upenn_gbm','<p  > <span>This collection comprises multi-parametric magnetic resonance imaging (mpMRI) scans for de novo Glioblastoma (GBM) patients coupled with patient demographics, clinical outcome (e.g., overall survival, genomic information, tumor progression), as well as computer-aided and manually-corrected segmentation labels of multiple histologically distinct tumor sub-regions, computer-aided and manually-corrected segmentations of the whole brain, a rich panel of radiomic features along with their corresponding co-registered mpMRI volumes in NIfTI format.</span></p> <p  > Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.709X-DN49\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Multi-parametric magnetic resonance imaging (mpMRI) scans for de novo Glioblastoma (GBM) patients from the University of Pennsylvania Health System <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>wiki page to learn more about the images and to obtain any supporting metadata for this collection. </p> </span>',120),(157,'None','<p>Selects studies which contain series that are <b>NOT</b> associated with an analysis result.</p>',165),(158,'cmb_crc','<p>The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">Cancer Moonshot Biobank</a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\" class=\"external-link\" rel=\"nofollow\">standard of care cancer treatment</a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">NCI Community Oncology Research Program (NCORP)</a>&#160;sites.</p><p>This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank-Colorectal Cancer (CMB-CRC) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\" class=\"external-link\" rel=\"nofollow\">The Database of Genotypes and Phenotypes (dbGaP)</a>&#160;and other NCI databases.</p><p>Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/DJG7-GZ87\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank-Colorectal Cancer Collection (CMB-CRC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;<span>page to learn more about the images and to obtain any supporting metadata for this collection.</span></p><p class=\"MsoListParagraph\"><span></span></p>',120),(159,'cmb_gec','<span class=\"conf-macro output-inline\"><p><font face=\"-apple-system, system-ui, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji\">The&#160;</font><a href=\"https://moonshotbiobank.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">Cancer Moonshot Biobank</a><font face=\"-apple-system, system-ui, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji\">&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;</font><a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\" class=\"external-link\" rel=\"nofollow\">standard of care cancer treatment</a><font face=\"-apple-system, system-ui, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji\">&#160;at multiple&#160;</font><a href=\"https://ncorp.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">NCI Community Oncology Research Program (NCORP)</a><font face=\"-apple-system, system-ui, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji\">&#160;sites.&#160;</font><span>This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank - Gastroesophageal Cancer&#160; (CMB-GEC) cohort.</span><span>&#160;</span><span>Associated genomic, phenotypic and clinical data will be hosted by</span><span>&#160;</span><a href=\"https://www.ncbi.nlm.nih.gov/gap/\" class=\"external-link\" rel=\"nofollow\">The Database of Genotypes and Phenotypes (dbGaP)</a><span>&#160;and other NCI databases.&#160;</span><span>Please see the&#160;</span><span><a href=\"\" url=\"https://doi.org/10.7937/E7KH-R486\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank - Gastroesophageal Cancer Collection (CMB-GEC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><font color=\"#222222\" face=\"Calibri, sans-serif\"><span>&#160;page to learn more about the images and to obtain any supporting metadata for this collection.</span></font></span></p></span><span></span>',120),(160,'cmb_lca','<p class=\"p1\">The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\"><span class=\"s1\">Cancer Moonshot Biobank</span></a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\"><span class=\"s1\">standard of care cancer treatment</span></a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\"><span class=\"s1\">NCI Community Oncology Research Program (NCORP)</span></a>&#160;sites.</p><p class=\"p1\">This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank-Colorectal Cancer (CMB-CRC) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\"><span class=\"s1\">The Database of Genotypes and Phenotypes (dbGaP)</span></a>&#160;and other NCI databases.</p><p class=\"p1\">Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/3CX3-S132\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank-Lung Cancer Collection (CMB-LCA) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;page to learn more about the images and to obtain any supporting metadata for this collection.</p><span></span>',120),(161,'cmb_mel','<p class=\"p1\">The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\"><span class=\"s1\">Cancer Moonshot Biobank</span></a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\"><span class=\"s1\">standard of care cancer treatment</span></a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\"><span class=\"s1\">NCI Community Oncology Research Program (NCORP)</span></a>&#160;sites.</p><p class=\"p1\">This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank-Colorectal Cancer (CMB-CRC) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\"><span class=\"s1\">The Database of Genotypes and Phenotypes (dbGaP)</span></a>&#160;and other NCI databases.&#160;<span>Please see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/GWSP-WH72\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank - Melanoma Collection (CMB-MEL) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span><span>&#160;page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>',120),(162,'cmb_mml','<p class=\"p1\">The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\"><span class=\"s1\">Cancer Moonshot Biobank</span></a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\"><span class=\"s1\">standard of care cancer treatment</span></a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\"><span class=\"s1\">NCI Community Oncology Research Program (NCORP)</span></a>&#160;sites.</p><p class=\"p1\">This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank-Colorectal Cancer (CMB-CRC) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\"><span class=\"s1\">The Database of Genotypes and Phenotypes (dbGaP)</span></a>&#160;and other NCI databases.</p><p class=\"p1\">Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/SZKB-SW39\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank-Multiple Myeloma Collection (CMB-MML) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;<span>page to learn more about the images and to obtain any supporting metadata for this collection.</span></p><span></span>',120),(163,'cmb_pca','<p>The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">Cancer Moonshot Biobank</a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\" class=\"external-link\" rel=\"nofollow\">standard of care cancer treatment</a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">NCI Community Oncology Research Program (NCORP)</a>&#160;sites.&#160;This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank - Prostate Cancer (CMB-PCA) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\" class=\"external-link\" rel=\"nofollow\">The Database of Genotypes and Phenotypes (dbGaP)</a>&#160;and other NCI databases.&#160;<span>Please see the&#160;</span><span><a href=\"\" url=\"https://doi.org/10.7937/25T7-6Y12\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank-Prostate Cancer Collection (CMB-PCA) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;page to learn more about the images and to obtain any supporting metadata for this collection.</span></p><p class=\"MsoListParagraph\"><span></span></p>',120),(164,'gbm_dsc_mri_dro','The standardization of dynamic susceptibility contrast (DSC)-magnetic resonance imaging (MRI) have been confounded by a lack of consensus on DSC-MRI methodology for preventing potential relative cerebral blood volume (CBV) inaccuracies, including the choice of acquisition protocols and postprocessing algorithms. Therefore, a digital reference object (DRO) was developed using physiological and kinetic parameters derived from a patient database, unique voxel-wise 3-dimensional tissue structures, and a validated MRI signal computational approach. The primary, intended use of the DRO is to validate image acquisition and analysis methods for accurately measuring relative cerebral blood volume in glioblastomas.<div>Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.RMWVZWIX\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">GBM-DSC-MRI-DRO <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;wiki page to learn more about the images and to obtain any supporting metadata for this collection.<br></p>',120),(165,'hcc_tace_seg','<p>Despite surveillance, the majority of HCC cases are diagnosed at advanced stages that can be treated only using (Transarterial chemoembolization) TACE, or systemic therapy. TACE failure can occur to 60% of patients receiving the procedure, with subsequent financial and emotional burden. Radiomics have emerged as a new tool capable of predicting tumor response to TACE from pre-procedural CT study.</p><p>This retrospectively acquired data collection includes pre- and post-procedure CT imaging studies of 105&#160;confirmed HCC patients who underwent TACE between 2002 and 2012 with an available treatment outcome, in the form of time-to-progression and overall survival.</p><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.5FNA-0924\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HCC-TACE-Seg <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(166,'pdmr_521955_158_r4','<div><div><div><span>We identified and characterized a patient derived xenograft model with metastatic potential, adenocarcinoma pancreas xenograft 521955-158-R4 from the National Cancer Institute Patient-Derived Models Repository (</span><a href=\"https://pdmr.cancer.gov\"><span>https://pdmr.cancer.gov/</span></a><span>). In this study we performed a detailed imaging characterization of this model, which develops spontaneous lung metastases (bi-weekly MRI) and PET/CT characterization of the primary tumor.</span><div><br><span></span><p></p><div>Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/q37d-vh79\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">PDMR-521955-158-R4 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;wiki page to learn more about the images and to obtain any supporting metadata for this collection.&#160;<span><br></span></p></p></p></p></p>',120),(167,'ct_vs_pet_ventilation_imaging','<span>The purpose of the </span><span><a href=\"\" url=\"https://doi.org/10.7937/3ppx-7s22\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CT Ventilation as a Functional Imaging Modality for Lung Cancer Radiotherapy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> </span><span>study was to enable comparisons between: (i) CT ventilation images derived from exhale/inhale BHCT scans, (ii) CT ventilation images derived from free-breathing 4DCT scans, and (iii) Galligas PET (nuclear medicine) ventilation scans. This dataset can build the international capacity for prototyping and evaluating new CT ventilation imaging technologies. For this study, 20 lung cancer patients underwent exhale/inhale breath hold CT (BHCT), free-breathing four-dimensional CT (4DCT) and Galligas PET ventilation scans in a single session on a combined 4DPET/CT scanner.</p><p>All image acquisitions were performed on a Siemens Biograph mCT.S/64 PET/CT scanner (Siemens, Knoxville, USA) at the Royal North Shore Hospital between 2013 and 2015. A total of 20 4DCT scans, 20 inhale/exhale BHCT scans, 20 Galligas PET scans and 19 attenuation CT scans (missing for CT-PET-VI-07) were successfully acquired for the 20 patients and included in this dataset.</span><br><span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/3ppx-7s22\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CT-vs-PET-Ventilation-Imaging <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a></span><span> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>',120),(168,'ctpred_sunitinib_pannet','<span>The </span><span><a href=\"\" url=\"https://doi.org/10.7937/SPGK-0P94\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prediction of Sunitinib Efficacy using Computed Tomography in Patients with Pancreatic Neuroendocrine Tumors <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a></span><span> study aimed to develop and validate a computed tomography (CT)-based method to predict the efficacy of sunitinib in Pretreatment CT images of 171 lesions from 38 patients with metastatic or locally advanced pancreatic neuroendocrine tumors (panNET).</span><br><span>The dataset can be used to validate the findings of the study. More importantly, researchers can use this dataset to study the imaging characteristics of panNET.</span><br><span>Please see the </span><span><a href=\"\" url=\"https://doi.org/10.7937/SPGK-0P94\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CTpred-Sunitinib-panNET <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a></span><span> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span>',120),(169,'nnU-Net-BPR-annotations','<p>Public imaging datasets are critical for the development and evaluation of automated tools in cancer imaging. Unfortunately, many of the available datasets do not provide annotations of tumors or&nbsp; organs-at-risk, crucial for the assessment of these tools. This is due to the fact that annotation of medical images is time consuming and requires domain expertise. It has been demonstrated that artificial intelligence (AI) based annotation tools can achieve acceptable performance and thus can be used to automate the annotation of large datasets. As part of the effort to enrich the public data available within NCI Imaging Data Commons (IDC) (<a href=\"\"https://imaging.datacommons.cancer.gov/\">https://imaging.datacommons.cancer.gov/</a>) [1],&nbsp;we introduce this dataset that consists of such AI-generated annotations for two publicly available medical imaging collections of Computed Tomography (CT) images of the chest.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.5281/zenodo.7975081\" data-toggle=\"modal\" data-target=\"#external-web-warning\">nnU-Net-BPR-annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> description page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(170,'adrenal_acc_ki67_seg','<p>\nThe Ki-67 index is one of the most important established prognostic markers for local recurrence of ACC. Radiomic feature extraction showed a significant association between radiomic signature and Ki-67 expression status in ACC.</p><p>\nThis retrospectively acquired data includes contrast enhanced CT imaging studies of 53 confirmed ACC patients between 2006 to 2018 with available clinical and pathological data, including Ki-67 index. Semi-automatic segmentation of the adrenal tumor was created using AMIRA, then manually refined by an experienced radiologist. The segmentations of each contrast-enhanced CT were done for the purpose of radiomic features extraction.</p>\n<p>\n        Please see the <a href=\"\" url=\"https://doi.org/10.7937/1FPG-VM46\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Adrenal-ACC-Ki67-Seg <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',120),(171,'cc_tumor_heterogeneity','The functional and biological properties of the tumor microenvironment are fundamentally important determinants of tumor response and therapy outcome in cancer. Oxygenation status and vascularity are known to influence radiation response, and molecular energy metabolism and proliferation impact on recurrence and metastatic progression. &#10;Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/ERZ5-QZ59\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cervical Cancer &#8211; Tumor Heterogeneity: Serial Functional and Molecular Imaging Across the Radiation Therapy Course in Advanced Cervical Cancer Cervical Cancer&#160; <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>wiki page to learn more about the images and to obtain any supporting metadata for this collection. ',120),(172,'colorectal_liver_metastases','<span>This collection consists of DICOM images and DICOM Segmentation Objects (DSOs) for 197 patients with Colorectal Liver Metastases (CRLM). The collection consists of a large, single-institution consecutive series of patients that underwent resection of CRLM and matched preoperative computed tomography (CT) scans for quantitative image analysis. Inclusion criteria were (a) pathologically confirmed resected CRLM, (b) available data from pathologic analysis of the underlying non-tumoral liver parenchyma and hepatic tumor, (c) available preoperative conventional portal venous contrast-enhanced multi-detector computed tomography (MDCT) performed within 6 weeks of hepatic resection. Patients with 90-day mortality or that had less than 24 months of follow-up were excluded.</span>&#160;Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/QXK2-QG03\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Preoperative CT and Recurrence for Patients Undergoing Resection of Colorectal Liver Metastases (Colorectal Liver Metastases) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;wiki page to learn more about the images and to obtain any supporting metadata for this collection.',120),(173,'htan_vanderbilt','<p>Colorectal cancer (CRC) is among the top three most prevalent cancers in global incidence and mortality. Most of these cancers develop from pre-cancerous adenomas. There is an unmet need to develop new preventive strategies and risk stratification models to decrease incidence, improve early detection, and prevent deaths from CRC.</p>\n<p>We believe that the ability to provide the most effective precision diagnostics and preventive strategies can only be achieved with single-cell analysis. As such, we will map spatial relationships across the spectrum of normal colon, early polyps, and late adenomas, including their unique stromal and microbial microenvironments to identify unique molecular phenotypes.</p>\n<p>Our goal will be accomplished through prospective, standardized collection and analysis of colorectal tissue, associated biospecimens, and related clinical and epidemiological data from participants undergoing colonoscopy or surgical resection. The biospecimens from these participants will be used for single-cell RNA sequencing, whole exome sequencing, multiplex immunofluorescence, species-specific bacterial fluorescence in situ hybridization, and other approaches. Finally, the information from these approaches will be integrated to develop a single-cell pre-cancer atlas with defined molecular phenotypes for dissemination to the broader scientific community.</p>\n<p>\n       Please see the <a href=\"\" url=\"https://humantumoratlas.org/hta11/\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN-Vanderbilt <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"\" url=\"https://docs.google.com/document/d/1seSp9yo8mqMK5sFXd4LEf_t_4SEofZi-1p-M75R1lhk/edit#heading=h.ybiv33wzts40\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN Publication Policy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page.</p>',120),(174,'nlm_visible_human_project','<p>The NLM Visible Human Project has created publicly-available complete, anatomically detailed, three-dimensional representations of a human male body and a human female body. Specifically, the VHP provides a public-domain library of cross-sectional cryosection, CT, and MRI images obtained from one male cadaver and one female cadaver. The Visible Man data set was publicly released in 1994 and the Visible Woman in 1995.</p>\n<p>The data sets were designed to serve as (1) a reference for the study of human anatomy, (2) public-domain data for testing medical imaging algorithms, and (3) a test bed and model for the construction of network-accessible image libraries. The VHP data sets have been applied to a wide range of educational, diagnostic, treatment planning, virtual reality, artistic, mathematical, and industrial uses. About 4,000 licensees from 66 countries were authorized to access the datasets. As of 2019, a license is no longer required to access the VHP datasets.</p>\n<p>\n       Courtesy of the U.S. National Library of Medicine. Release of this collection by IDC does not indicate or imply that NLM has endorsed its products/services/applications. Please see the <a href=\"https://www.nlm.nih.gov/research/visible/visible_human.html\" target=\"_blank\">Visible Human Project</a> information page to learn more about the images and to obtain any supporting metadata for this collection. Note that this collection may not reflect the most current/accurate data available from NLM.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.nlm.nih.gov/databases/download/terms_and_conditions.html\" target=\"_blank\">National Library of Medicine Terms and Conditions</a> information page.</p>',120),(175,'prostate_anatomical_edge_cases','<span>In this single institution retrospective study, we reviewed 950 consecutive patients with prostate adenocarcinoma receiving definitive radiotherapy between 2011 and 2019, and identified among them 112 patients with anatomic variations (edge cases) seen on simulation CT and/or MRI imaging. These variations included hip arthroplasty, prostate median lobe hypertrophy, so-called &#8220;droopy&#8221; seminal vesicles, presence of a urinary catheter, and others. A separate cohort of 19 &#8220;normal&#8221; cases were randomly selected for inclusion. Prostate, rectum, bladder, and bilateral femoral heads were manually segmented on all CT simulation images (where present) and were ultimately used clinically for radiation treatment planning.&#160;</span><span>Please see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/QSTF-ST65\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Stress-Testing Pelvic Autosegmentation Algorithms Using Anatomical Edge Cases (Prostate-Anatomical-Edge-Cases) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>&#160;</span><span>wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span><p class=\"MsoListParagraph\"><span></span></p>',120),(176,'rider_pilot','<span>This data collection was originally supported under supplemental funding for the Lung Image Database Consortium (LIDC) U01 project and focused on the collection of longitudinal studies using X-ray CT for monitoring the response to therapy. The data came primarily from the M.D. Anderson Cancer Center and several of the LIDC academic sites. The data is not annotated. A small subset of the </span><a href=\"\" url=\"https://doi.org/10.7937/m87f-mz83 \" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER Pilot <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span> data (8 subjects) which were analyzed in </span><a href=\"\" url=\"https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=70228818\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIBA-VolCT-1B <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>&#160;have been re-released on TCIA.</span><br><br><span>More information about the <a href=\"\" url=\"https://doi.org/10.7937/m87f-mz83\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER Pilot <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> sub-collection page can be found in the corresponding section of the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/EAA9\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page.</span>',120),(177,'pdmr_texture_analysis','<p class=\"MsoNormal\"><span>This collection contains serial non-contrast T2w MRI of&#10;18 patient derived xenograft cancer models (514 images) for researchers to&#10;develop various algorithms using neural networks, and classification techniques&#10;to improve tissue characterization (morphological changes).&#160;&#160;</span></p>&#10;&#10;<p class=\"MsoNormal\"><span>Please&#10;see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/3KQ0-YK19\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">PDMR-Texture-Analysis&#160; <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>wiki page to learn more about the images and&#10;to obtain any supporting metadata for this collection.</span></p>',120),(178,'rms_mutation_prediction','<p>RMS is an aggressive soft tissue tumor which is associated with a broad range of pathological mutations. Multiple studies have identified recurrent mutations and chromosomal translocations which are associated with poor outcome in RMS patients. Since not all patients are screened for these mutations with molecular testing, there is a need to develop tools which accurately identify these alterations at diagnosis. Our study associated with these images applied deep learning with convolutional neural networks to learn mutation-associated histological features from H&E images from diagnostic RMS specimens.</p>\n<p>\n       Please see the <a href=\"\" url=\"https://doi.org/10.5281/zenodo.8225131\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RMS-Mutation-Prediction <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"\" url=\"https://doi.org/10.5281/zenodo.8225131\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RMS-Mutation-Prediction <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page.</p>',120),(179,'CPTAC-PDA-Tumor-Annotations','<p>This dataset contains image annotations derived from <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.SC20FO18\" class=\"external-link\" rel=\"nofollow\">The Clinical Proteomic Tumor Analysis Consortium Pancreatic Ductal Adenocarcinoma Collection (CPTAC-PDA) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>. This dataset was generated as part of a National Cancer Institute project to augment images from The Cancer Imaging Archive with annotations that will improve their value for cancer researchers and artificial intelligence experts.</p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/bw9v-bx61\" data-toggle=\"modal\" data-target=\"#external-web-warning\">CPTAC-PDA-Tumor-Annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(180,'CPTAC-UCEC-Tumor-Annotations','<p>This dataset contains image annotations derived from <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.3R3JUISW\" class=\"external-link\" rel=\"nofollow\">The Clinical Proteomic Tumor Analysis Consortium Uterine Corpus Endometrial Carcinoma Collection (CPTAC-UCEC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>. This dataset was generated as part of a National Cancer Institute project to augment images from The Cancer Imaging Archive with annotations that will improve their value for cancer researchers and artificial intelligence experts.</p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/89m3-kq43\" data-toggle=\"modal\" data-target=\"#external-web-warning\">CPTAC-UCEC-Tumor-Annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(181,'QIBA-VolCT-1B','<p>The purpose of this analysis is to determine the variability of lesion size measurements in computed tomography data sets of patients imaged under a “no change” (“coffee break”) condition and to determine the impact of two reading paradigms on measurement variability.</p> \n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/tcia.2020.1c3h-vp70\" data-toggle=\"modal\" data-target=\"#external-web-warning\">QIBA-VolCT-1B <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(182,'cmb_aml','<p>The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">Cancer Moonshot Biobank</a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\" class=\"external-link\" rel=\"nofollow\">standard of care cancer treatment</a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">NCI Community Oncology Research Program (NCORP)</a>&#160;sites.</p><p>This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank-Colorectal Cancer (CMB-CRC) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\" class=\"external-link\" rel=\"nofollow\">The Database of Genotypes and Phenotypes (dbGaP)</a>&#160;and other NCI databases.</p><p>Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/PCTE-6M66\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank - Acute Myeloid Leukemia Cancer Collection (CMB-AML)<i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;<span>page to learn more about the images and to obtain any supporting metadata for this collection.</span></p><p class=\"MsoListParagraph\"><span></span></p>',120),(183,'ct_phantom4radiomics','<p>The aims of this dataset are to determine the stability of radiomics features against computed tomography (CT) parameter variations and to study their discriminative power concerning tissue classification using <a href=\"\" url=\"https://doi.org/10.1002/mp.16373\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">a 3D-printed CT phantom based on real patient data <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>. 4 tissue types (normal, benign cyst, hemangioma, and metastasis) were manually annotated. 238 CT series with 8 parameter variations of reconstruction algorithms, reconstruction kernels, slice thickness, and slice spacing are available.</p><br><br><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/a1v1-rc66\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CT-Phantom4Radiomics <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',120),(184,'ea1141','<span><span>This collection contains data from the Eastern Cooperative Oncology Group (ECOG) Clinical Trial</span><span>&#160;</span><span><a title=\"\" href=\"https://clinicaltrials.gov/study/NCT02933489\">NCT02933489</a><span>, &#8220;</span>Abbreviated Breast MRI and Digital Tomosynthesis Mammography in Screening Women With Dense Breasts<span>&#34;. Principle Investigator: Christopher Comstock.</span></span><span>&#160;&#160;</span><span>It was sponsored by NCI and performed by the<span>&#160;</span></span><span>Eastern Cooperative Oncology Group&#160;<span>under study number<span>&#160;</span></span></span><span>EA1141</span><span>.<span>&#160;</span></span><span>This randomized phase II trial enrolled 1,516 subjects to study how well abbreviated breast magnetic resonance imaging (MRI) and digital tomosynthesis mammography work in detecting cancer in women with dense breasts. Imaging and clinical data from 500 of those subjects are being made available through this collection. Please see the </span></span><span><a href=\"\" url=\"https://doi.org/10.7937/2BAS-HR33\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Abbreviated Breast MRI and Digital Tomosynthesis Mammography in Screening Women With Dense Breasts (EA1141)</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span><span></span>',120),(185,'remind','<span class=\"x_ContentPasted0\">The Brain Resection Multimodal Imaging Database (ReMIND) contains pre- and intra-operative data collected on 114 consecutive patients who were surgically treated with image-guided tumor resection between<span> 2018 and 2022</span>. For all patients, preoperative MRI, 3D intraoperative ultrasound series, and intraoperative MRI are available. <span>Additionally, each case typically contains segmentations,</span> including the preoperative tumor, the pre-resection cerebrum, the previous resection cavity derived from the preoperative MRI (if applicable), and any residual tumor identified on the intraoperative MRI. In total, this collection contains <span>369</span> preoperative MRI series, 320 3D intraoperative ultrasound series,<span> 301</span> intraoperative MRI series, and 356 segmentations. We expect this data to be a resource for computational research in brain shift and image analysis as well as for neurosurgical training in the interpretation of intraoperative ultrasound and intraoperative MRI. Please see the&#160;</span><a href=\'\'\" url=\"https://doi.org/10.7937/3RAG-D070\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">The Brain Resection Multimodal Imaging Database (ReMIND)  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span>',120),(186,'vestibular_schwannoma_mc_rc','<p><span>This multi-center routine clinical (MC-RC) dataset consists of 160 patients with a single sporadic Vestibular Schwannoma (VS) who were referred from 10 medical sites and consecutively seen at a single center. These routine clinical datasets are more diverse in terms of the tumor manifestation as well as the acquisition parameters. Using this dataset, researchers can develop and validate methods for automatic surveillance of Vestibular Schwannoma, which work robustly on images acquired at different hospitals.&#160;</span><span>Patients had multiple time points resulting in a total of 428 timepoints and 487 3D-images.&#160;</span><span>Manual ground truth segmentations were obtained in an iterative process in which segmentations were: 1) produced or amended by a specialized company; and 2) reviewed by one of three trained radiologists; and 3) validated by an expert team. Compared to the existing&#160;<a href=\"\" url=\"https://doi.org/10.7937/TCIA.9YTJ-5Q73\" class=\"external-link\" rel=\"nofollow\" data-toggle=\"modal\" data-target=\"#external-web-warning\">Vestibular-Schwannoma-SEG <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;dataset on TCIA that was obtained from a single scanner, this dataset was acquired from multiple scanners manufactured by different vendors. This dataset also provides a refined segmentation of intrameatal and extrameatal components of the VS.&#160;</span><span>Please see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/HRZH-2N82\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Segmentation of Vestibular Schwannoma from Magnetic Resonance Imaging: An Annotated Multi-Center Routine Clinical Dataset (Vestibular-Schwannoma-MC-RC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>&#160;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p><p class=\"MsoListParagraph\"><span></span></p>',120),(187,'BAMF-AIMI-Annotations','<p>Many of the collections in IDC have limited annotations due to the expense and effort required to create these manually. The increased capabilities of AI analysis of radiology images provides an opportunity to augment existing IDC collections with new annotation data. To further this goal, we trained several nnU-Net based models for a variety of radiology segmentation tasks from public datasets and used them to generate segmentations for IDC collections.</p>\n<p>To validate the models performance, roughly 10% of the predictions were manually reviewed and corrected by both a board certified radiologist and a medical student (non-expert). Additionally, this non-expert looked at all the ai predictions and rated them on a 5 point Likert scale .</p>\n<p>This record provides AI segmentations, manually corrected segmentations, and manual scores for the inspected IDC Collection images. Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.5281/zenodo.10081112\" data-toggle=\"modal\" data-target=\"#external-web-warning\">BAMF-AIMI-Annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n',165),(188,'CPTAC-CCRCC-Tumor-Annotations','<p>This dataset contains image annotations derived from <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.OBLAMN27\" class=\"external-link\" rel=\"nofollow\">The Clinical Proteomic Tumor Analysis Consortium Clear Cell Renal Cell Carcinoma Collection (CPTAC-CCRCC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>. This dataset was generated as part of a National Cancer Institute project to augment images from The Cancer Imaging Archive with annotations that will improve their value for cancer researchers and artificial intelligence experts.</p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"\"https://doi.org/10.7937/skq4-qx48\"data-toggle=\"modal\" data-target=\"#external-web-warning\">CPTAC-CCRCC-Tumor-Annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>',165),(189,'Prostate-MRI-US-Biopsy-DICOM-Annotations','<p>This dataset contributes DICOM-converted annotations to the publicly available National Cancer Institute Imaging Data Commons Prostate-MRI-US-Biopsy collection.  Originally released by TCIA, the collection was accompanied by 1017 semi-automatic segmentations of the prostate and 1317 manual segmentations of target lesions in the STL format. Although STL is a common and practical format for 3D printing, it is not interoperable with many visualization and analysis tools commonly used in medical imaging research and does not provide any standard means to communicate metadata, among other limitations. To alleviate those deficiencies, we created DICOM Encapsulated 3D Manufacturing Model objects (M3D modality) that includes the original STL content enriched with the DICOM metadata. Furthermore, we created an alternative encoding of the surface segmentations by rasterizing them and saving the result as a DICOM Segmentation object (SEG modality). As a result, the contributed DICOM objects can be stored in any DICOM server that supports those objects (including Google Healthcare DICOM stores), and the DICOM Segmentations can be visualized using off-the-shelf tools, such as OHIF Viewer. </p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.5281/zenodo.10069910\" data-toggle=\"modal\" data-target=\"#external-web-warning\">Prostate-MRI-US-Biopsy-DICOM-annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the data and any supporting metadata for this collection, and to learn about attribution/citation requirements.</p>\n',165),(190,'advanced_mri_breast_lesions','<p>The application of deep-learning methods to breast MRI has a potential to improve diagnosis as well as predict pathological features of the tumors and their response to therapy, and is currently widely investigated. Standard contrast-enhanced MRI of the breast provides high sensitivity but variable specificity in detecting breast cancer, and may lead to excessive benign biopsies. Large, well annotated datasets are needed to improve the current results in this field.&nbsp;<span>This dataset is a single-institutional, retrospective collection of 632 breast-MRI imaging sessions acquired on a 1.5T MR system between 2018-2021. Supporting data, collected for 200 patients from clinical notes, radiology report and pathology reports, is included: patient age, lesion location, malignant/benign outcome, pathology and grade (if applicable), receptor status and KI67 (if applicable). The main uniqueness of this data is that it contains the standard protocol series (T1-weighted DCE sequence with 5 time points- before and 4 time points after contrast injection, and T2-weighted MRI with/without fat suppression) as well as the delayed contrast T1-images acquired 20-28 minutes after contrast administration and the calculated treatment response assessment maps (TRAMs).Please see the&nbsp;</span><span><a href=\"\" url=\"https://doi.org/10.7937/C7X1-YN57\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Advanced-MRI-Breast-Lesions  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> collection page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>',120),(191,'RMS-Mutation-Prediction-Expert-Annotations','<p>The RMS-Mutation-Prediction-Expert-Annotations collections contains expert-generated annotations of the regions of interest identified in the pathology slides from the RMS-Mutation-Prediction collection. Annotations were generated manually, using the ImageScope tool, to delineate regions of ARMS (alveolar rhabdomyosarcoma), ERMS (embryonal rhabdomyosarcoma), stroma, and necrosis and stored as planar contours, consequently converted into Digital Imaging and Communications in Medicine (DICOM) Structured Report (SR) representation. </p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.5281/zenodo.10462857\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RMS-Mutation-Prediction-Expert-Annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and any supporting metadata for this collection, and to learn about attribution/citation requirements.</p>',165),(192,'TotalSegmentator-CT-Segmentations','<p>This collection contains results of automatic analysis of the CT images using the TotalSegmentator model to generate volumetric segmentations of anatomic organs, with the subsequent application of the pyradiomics library to extract first order and shape features for the regions defined by those segmentations. Initial release of this collection contains these analysis results for the National Lung Screening Trial (NLST) collection. </p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.5281/zenodo.8347011\" data-toggle=\"modal\" data-target=\"#external-web-warning\"  target=\"_blank\">TotalSegmentator-CT-Segmentations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the data and any supporting metadata for this collection, and to learn about attribution/citation requirements.</p>\n',165);
/*!40000 ALTER TABLE `idc_collections_attribute_tooltips` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_collection`
--

DROP TABLE IF EXISTS `idc_collections_collection`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_collection` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(255) DEFAULT NULL,
  `tcia_collection_id` varchar(255) DEFAULT NULL,
  `nbia_collection_id` varchar(255) DEFAULT NULL,
  `collection_id` varchar(255) DEFAULT NULL,
  `description` longtext,
  `date_updated` date DEFAULT NULL,
  `status` varchar(40) DEFAULT NULL,
  `access` varchar(40) DEFAULT NULL,
  `subject_count` int DEFAULT NULL,
  `image_types` varchar(255) DEFAULT NULL,
  `cancer_type` varchar(512) DEFAULT NULL,
  `doi` varchar(255) DEFAULT NULL,
  `supporting_data` varchar(255) DEFAULT NULL,
  `analysis_artifacts` varchar(255) DEFAULT NULL,
  `species` varchar(64) DEFAULT NULL,
  `location` varchar(255) DEFAULT NULL,
  `active` tinyint(1) NOT NULL,
  `is_public` tinyint(1) NOT NULL,
  `collection_type` varchar(1) NOT NULL,
  `collections` varchar(255) DEFAULT NULL,
  `owner_id` int NOT NULL,
  `program_id` int DEFAULT NULL,
  `collection_uuid` varchar(255) DEFAULT NULL,
  `source_url` varchar(512) DEFAULT NULL,
  PRIMARY KEY (`id`),
  KEY `idc_collections_collection_owner_id_2f0ec021_fk_auth_user_id` (`owner_id`),
  KEY `idc_collections_coll_program_id_ddfbb0dd_fk_idc_colle` (`program_id`),
  CONSTRAINT `idc_collections_coll_program_id_ddfbb0dd_fk_idc_colle` FOREIGN KEY (`program_id`) REFERENCES `idc_collections_program` (`id`),
  CONSTRAINT `idc_collections_collection_owner_id_2f0ec021_fk_auth_user_id` FOREIGN KEY (`owner_id`) REFERENCES `auth_user` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=201 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_collection`
--

LOCK TABLES `idc_collections_collection` WRITE;
/*!40000 ALTER TABLE `idc_collections_collection` DISABLE KEYS */;
INSERT INTO `idc_collections_collection` VALUES (1,'Standardized representation of the TCIA LIDC-IDRI annotations using DICOM','','','DICOM-LIDC-IDRI-Nodules','<p> This dataset contains standardized DICOM representation of the volumetric nodule annotations and characterizations collected by the LIDC/IDRI initiative.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/TCIA.2018.h7umfurq\" data-toggle=\"modal\" data-target=\"#external-web-warning\">DICOM-LIDC-IDRI-Nodules <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',875,'','Lung','10.7937/TCIA.2018.h7umfurq','','Tumor segmentations, image features, Software/Source Code','','Chest',1,0,'A','lidc_idri',1,NULL,'ab83115d-efcd-446e-b76c-b6682998b34e','https://doi.org/10.7937/tcia.2018.h7umfurq'),(2,'QIN multi-site collection of Lung CT data with Nodule Segmentations','','','QIN-LungCT-Seg','<p> This collection contains lung nodule segmentations obtained using segmentation tools that participated in the QIN challenge “moist run” initiative.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.1BUVFJR7\" data-toggle=\"modal\" data-target=\"#external-web-warning\">QIN-LungCT-Seg <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',31,'','Lung','10.7937/k9/tcia.2015.1buvfjr7','','Tumor segmentations','','Chest',1,0,'A','lidc_idri,lung_phantom,qin_lung_ct,rider_lung_ct',1,NULL,'58c5c36b-6a64-4034-9128-cb981a9c7f2b','https://doi.org/10.7937/k9/tcia.2015.1buvfjr7'),(3,'DICOM SR of clinical data and measurement for breast cancer collections to TCIA','','','DICOM-SR-Breast-Clinical','<p> This collection contains clinical metadata corresponding to several breast imaging collections harmonized into DICOM SR representation, generated by the Data Integration & Imaging Informatics (DI-Cubed) project.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.wgllssg1\" data-toggle=\"modal\" data-target=\"#external-web-warning\">DICOM-SR-Breast-Clinical <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',474,'','Breast Cancer','10.7937/TCIA.2019.wgllssg1','','DICOM SR descriptions of patient characteristics, histopathology, receptor status and clinical findings including measurements.','','Breast',1,0,'A','breast_diagnosis,breast_mri_nact_pilot,ispy1,tcga_brca',1,NULL,'fec41175-9a45-456f-b789-0f60958b9482','https://doi.org/10.7937/tcia.2019.wgllssg1'),(4,'TCGA-PRAD','TCGA-PRAD','TCGA-PRAD','tcga_prad','<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>.&nbsp;Currently this image collection of prostate adenocarcinoma (PRAD) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.<br /><br /></p>\n <p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.YXOGLM4Y\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-PRAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',500,'CT, MR, PT, SM','Prostate Cancer','10.7937/K9/TCIA.2016.YXOGLM4Y','Clinical, Genomics','','Human','Prostate',1,0,'O','',1,1,'e1cb37dc-87d9-4074-9925-788be68a9ffa','https://doi.org/10.7937/K9/TCIA.2016.YXOGLM4Y'),(5,'TCGA-BLCA','TCGA-BLCA','TCGA-BLCA','tcga_blca','<p>\n	The Cancer Genome Atlas-Bladder Endothelial Carcinoma (TCGA-BLCA) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP), with the cooperation of several of the TCGA tissue-contributing institutions, has archived a large portion of the radiological images of the genetically-analyzed BLCA cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.8LNG8XDR\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-BLCA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',412,'CR, CT, DX, MR, PT, SM','Bladder Endothelial Carcinoma','10.7937/K9/TCIA.2016.8LNG8XDR','Clinical, Genomics','','Human','Bladder',1,0,'O','',1,1,'6157f2cb-aaf8-4f42-9a2c-ce1261863785','https://doi.org/10.7937/K9/TCIA.2016.8LNG8XDR'),(6,'TCGA-UCEC','TCGA-UCEC','TCGA-UCEC','tcga_ucec','<p>\n	The Cancer Genome Atlas-Uterine Corpus Endometrial Carcinoma (TCGA-UCEC) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed UCEC cases.</p><br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.GKJ0ZWAC\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-UCEC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',560,'CR, CT, MR, PT, SM','Uterine Corpus Endometrial Carcinoma','10.7937/K9/TCIA.2016.GKJ0ZWAC','Clinical, Genomics','','Human','Uterus',1,0,'O','',1,1,'a92cd421-c314-4bdf-8315-94a092065239','https://doi.org/10.7937/K9/TCIA.2016.GKJ0ZWAC'),(7,'TCGA-HNSC','TCGA-HNSC','TCGA-HNSC','tcga_hnsc','\n<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>. Currently this large PET/CT multi-sequence image collection of  head and neck squamous cell carcinoma (HNSC) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.</p>\n</br>\n<p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.LXKQ47MS\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-HNSC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',523,'SM','Head and Neck Squamous Cell Carcinoma','10.7937/K9/TCIA.2016.LXKQ47MS','Clinical, Genomics','','Human','Head-Neck',1,0,'O','',1,1,'c5ef1e90-2919-44b9-bee9-93d938792651','https://doi.org/10.7937/K9/TCIA.2016.LXKQ47MS'),(8,'TCGA-LUSC','TCGA-LUSC','TCGA-LUSC','tcga_lusc','<p>\n	The Cancer Genome Atlas-Lung Squamous Cell Carcinoma (TCGA-LUSC) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the LUSC cases.</p>\n<br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.TYGKKFMQ\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-LUSC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',504,'CT, NM, PT, SEG, SM','Lung Squamous Cell Carcinoma','10.7937/K9/TCIA.2016.TYGKKFMQ','Clinical, Genomics','','Human','Lung',1,0,'O','',1,1,'a9027ab7-6de6-46b6-99ea-d88341d89fa6','https://doi.org/10.7937/K9/TCIA.2016.TYGKKFMQ'),(9,'TCGA-KIRP','TCGA-KIRP','TCGA-KIRP','tcga_kirp','\n<p> The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>.&nbsp;Currently this large CT and MR multi-sequence image collection of  kidney renal papillary cell carcinoma (KIRP) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.&nbsp;<br />\n</br><p><a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.ACWOGBEF\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-KIRP <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',291,'CT, MR, PT, SM','Kidney Renal Papillary Cell Carcinoma','10.7937/K9/TCIA.2016.ACWOGBEF','Clinical, Genomics','','Human','Kidney',1,0,'O','',1,1,'ff623c57-e3eb-48d6-9122-d9ff5ddaa3df','https://doi.org/10.7937/K9/TCIA.2016.ACWOGBEF'),(10,'TCGA-THCA','TCGA-THCA','TCGA-THCA','tcga_thca','<p>\n	The Cancer Genome Atlas-Thyroid Cancer (TCGA-THCA) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed THCA cases.</p><br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.9ZFRVF1B\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-THCA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',507,'CT, PT, SM','Thyroid Cancer','10.7937/K9/TCIA.2016.9ZFRVF1B','Clinical, Genomics','','Human','Thyroid',1,0,'O','',1,1,'fc6ede17-e136-4b32-8e99-06fbc7d23584','https://doi.org/10.7937/K9/TCIA.2016.9ZFRVF1B'),(11,'TCGA-SARC','TCGA-SARC','TCGA-SARC','tcga_sarc','<p>\n	The Cancer Genome Atlas-Sarcoma Cancer (TCGA-SARC) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed SARC cases.</p><br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.CX6YLSUX\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-SARC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',261,'CT, MR, SM','Sarcomas','10.7937/K9/TCIA.2016.CX6YLSUX','Clinical, Genomics','','Human','Chest-Abdomen-Pelvis, Leg, TSpine',1,0,'O','',1,1,'35ff19b3-f523-4ab6-978c-a8a4303372c9','https://doi.org/10.7937/K9/TCIA.2016.CX6YLSUX'),(12,'TCGA-ESCA','TCGA-ESCA','TCGA-ESCA','tcga_esca','<p>\n	The Cancer Genome Atlas-Esophageal Carcinoma (TCGA-ESCA) data collection is part of a larger effort to enhance the <a href=\"http://cancergenome.nih.gov/\">TCGA&nbsp;</a> data set with characterized radiological images. The <a href=\"https://imaging.cancer.gov/\">Cancer Imaging Program (CIP)</a> with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed ESCA cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.VPTNRGFY\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-ESCA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',185,'CT, SM','Esophageal Carcinoma','10.7937/K9/TCIA.2016.VPTNRGFY','Clinical, Genomics','','Human','Esophagus',1,0,'O','',1,1,'4cf748f3-1f68-4165-a6a1-d1631f079ab1','https://doi.org/10.7937/K9/TCIA.2016.VPTNRGFY'),(13,'TCGA-CESC','TCGA-CESC','TCGA-CESC','tcga_cesc','<p>\n	The Cancer Genome Atlas-Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma (TCGA-CESC) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed CESC cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.SQ4M8YP4\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-CESC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',307,'MR, SM','Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma','10.7937/K9/TCIA.2016.SQ4M8YP4','Clinical, Genomics','','Human','Cervix',1,0,'O','',1,1,'d2482ed6-9ca5-4e2a-a054-b85f880f8664','https://doi.org/10.7937/K9/TCIA.2016.SQ4M8YP4'),(14,'TCGA-STAD','TCGA-STAD','TCGA-STAD','tcga_stad','<p>\n	The Cancer Genome Atlas-Stomach Adenoarcinoma (TCGA-STAD) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed STAD cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.GDHL9KIM\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-STAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',443,'CT, SM','Stomach Adenocarcinoma','10.7937/K9/TCIA.2016.GDHL9KIM','Clinical, Genomics','','Human','Stomach',1,0,'O','',1,1,'4a99c410-08f1-472c-8689-2eebe54c10ae','https://doi.org/10.7937/K9/TCIA.2016.GDHL9KIM'),(15,'TCGA-COAD','TCGA-COAD','TCGA-COAD','tcga_coad','<p>\n	The Cancer Genome Atlas-Colon Adenocarcinoma (TCGA-COAD) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP), with the cooperation of several of the TCGA tissue-contributing institutions, has archived a large portion of the radiological images of the COAD cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.HJJHBOXZ\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-COAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',464,'CT, OT, SM','Colon adenocarcinoma','10.7937/K9/TCIA.2016.HJJHBOXZ','Clinical, Genomics','','Human','Colon',1,0,'O','',1,1,'7458addc-caa4-4270-bb92-bc6745cb99a0','https://doi.org/10.7937/K9/TCIA.2016.HJJHBOXZ'),(16,'TCGA-KICH','TCGA-KICH','TCGA-KICH','tcga_kich','<p>\n	The Cancer Genome Atlas-Kidney Chromophobe (TCGA-KICH) data collection is part of a larger effort to enhance the The Cancer Genome Atlas (TCGA) <a href=\"https://cancergenome.nih.gov/\">https://cancergenome.nih.gov/</a> data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions has archived a large portion of the radiological images of the genetically-analyzed KICH cases.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.YU3RBCZN\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-KICH <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',113,'CT, MR, SM','Kidney Chromophobe','10.7937/K9/TCIA.2016.YU3RBCZN','Clinical, Genomics','','Human','Kidney',1,0,'O','',1,1,'b656ee38-9449-4bce-91f3-7789742afcba','https://doi.org/10.7937/K9/TCIA.2016.YU3RBCZN'),(17,'TCGA-READ','TCGA-READ','TCGA-READ','tcga_read','<p>\n	The Cancer Genome Atlas-Rectum Adenocarcinoma (TCGA-READ) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP), with the cooperation of several TCGA tissue-contributing institutions, has archived a large portion of the radiological images of the genetically-analyzed READ cases.</p>\n<br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.F7PPNPNU\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-READ <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',171,'CT, MR, SM','Rectum Adenocarcinoma','10.7937/K9/TCIA.2016.F7PPNPNU','Clinical, Genomics','','Human','Rectum',1,0,'O','',1,1,'3a8c8279-819e-4d59-bdba-310eaa1144b6','https://doi.org/10.7937/K9/TCIA.2016.F7PPNPNU'),(18,'ISPY1','ISPY1','ISPY1','ispy1','<p>\n	ISPY1/ACRIN 6657 was designed as a prospective study to test MRI for ability to predict response to treatment and risk-of-recurrence in patients with stage 2 or 3 breast cancer receiving neoadjuvant chemotherapy (NACT). ACRIN 6657 was conducted as a companion study to CALGB 150007, a correlative science study evaluating tissue-based biomarkers in the setting of neoadjuvant treatment of breast cancer. Collectively, CALGB 150007 and ACRIN 6657 formed the basis of the multicenter Investigation of Serial Studies to Predict Your Therapeutic Response with Imaging and moLecular Analysis (I-SPY TRIAL) breast cancer trial, a study of imaging and tissue-based biomarkers for predicting pathologic complete response (pCR) and recurrence-free survival (RFS).<br />\n	<br />\n	The collection consists of 847 MR studies from 222 subjects.<br />\n	<br />\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2016.HdHpgJLK\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ISPY1 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',222,'MR, SEG, SR','Breast Cancer','10.7937/K9/TCIA.2016.HdHpgJLK','Clinical, Image Analyses','','Human','Breast',1,0,'O','',1,19,'c3e65be0-2de0-4d01-9eea-aab87dd9bbb7','https://doi.org/10.7937/K9/TCIA.2016.HdHpgJLK'),(19,'QIN-HEADNECK','QIN-HeadNeck','QIN-HeadNeck','qin_headneck','<p>\n	This collection is a set of head and neck cancer patients each of whom has had multiple PET/CT FDG scans &ndash; before and after therapy &ndash; and with follow up scans where clinically indicated.&nbsp; The data were provided to help facilitate research activities of <a href=\"https://wiki.cancerimagingarchive.net/display/Public/Quantitative+Imaging+Network+Collections\" target=\"_blank\">NCI&#39;s Quantitative Imaging Network</a>.</p>\n<p>\n	Please see the <a href=\"https://wiki.cancerimagingarchive.net/display/Public/QIN-HEADNECK\" target=\"_blank\">QIN-HeadNeck</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2022-04-21',NULL,'Limited',279,'CT, PT, RWV, SEG, SR','Head and Neck Carcinomas','10.7937/K9/TCIA.2015.K0F5CGLI','Clinical, Image Analyses','','Human','Head-Neck',1,0,'O','',1,2,'26798942-f95f-41d6-a59e-001dfd04e2c4','https://doi.org/10.7937/K9/TCIA.2015.K0F5CGLI'),(20,'LIDC-IDRI','LIDC-IDRI','LIDC-IDRI','lidc_idri','<div><span>The Lung Image Database Consortium (LIDC-IDRI) image collection consists of diagnostic and lung cancer screening thoracic CT scans with marked-up annotated lesions. It is a web-accessible international resource for development, training, and evaluation of computer-assisted diagnostic (CAD) methods for lung cancer detection and diagnosis. </span><span><br>&#10;<br>&#10;For more information about the LIDC-IDRI images please visit the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.LO9QL9SX\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><u>LIDC-IDRI</u> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page.&#160; For comprehensive information about utilizing the associated metadata please see the </span><span>Research Project page for the</span><span> <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/XwA2\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Lung Image Database Consortium <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>.</span></p>','2024-03-08',NULL,'Public',1010,'CR, CT, DX, SEG, SR','Lung Cancer','10.7937/K9/TCIA.2015.LO9QL9SX','Clinical, Image Analyses, Software/Source Code','','Human','Chest',1,0,'O','',1,18,'55f13d12-d5d5-46f2-92a4-83494293380b','https://doi.org/10.7937/K9/TCIA.2015.LO9QL9SX'),(21,'TCGA-LUAD','TCGA-LUAD','TCGA-LUAD','tcga_luad','<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a>  is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a> Currently this large CT multi-sequence image collection of lung adenocarcinoma (LUAD) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.</p><br />\n\n<p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.JGNIHEP5\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-LUAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',560,'CT, NM, PT, SEG, SM','Lung Adenocarcinoma','10.7937/K9/TCIA.2016.JGNIHEP5','Clinical, Genomics, Image Analyses','','Human','Chest',1,0,'O','',1,1,'f1d3290e-b761-4bc9-a269-48d040d9cfcc','https://doi.org/10.7937/K9/TCIA.2016.JGNIHEP5'),(22,'TCGA-LIHC','TCGA-LIHC','TCGA-LIHC','tcga_lihc','<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>.&nbsp;Currently this CT and MR multi-sequence image collection of liver hepatocellular carcinoma (LIHC) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.</p><br />\n<p><a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.IMMQW8UQ\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-LIHC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>   page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',377,'CT, MR, PT, SEG, SM','Liver Hepatocellular Carcinoma','10.7937/K9/TCIA.2016.IMMQW8UQ','Clinical, Genomics, Image Analyses','','Human','Liver',1,0,'O','',1,1,'fb43620b-f951-4232-b2ab-78f9a1f518d3','https://doi.org/10.7937/K9/TCIA.2016.IMMQW8UQ'),(23,'TCGA-BRCA','TCGA-BRCA','TCGA-BRCA','tcga_brca','The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a></span> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>.&nbsp;Currently this MR multi-sequence image collection of breast invasive carcinoma patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.&nbsp;<br />\n\n<p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.AB2NAZRP\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-BRCA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',1098,'MG, MR, SM, SR','Breast Cancer','10.7937/K9/TCIA.2016.AB2NAZRP','Clinical, Genomics, Image Analyses','','Human','Breast',1,0,'O','',1,1,'1eae6aaa-4816-4b04-a1f8-809c10949d19','https://doi.org/10.7937/K9/TCIA.2016.AB2NAZRP'),(24,'TCGA-OV','TCGA-OV','TCGA-OV','tcga_ov','<p>\n	The Cancer Genome Atlas-Ovarian Cancer (TCGA-OV) data collection is part of a larger effort to enhance the TCGA http://cancergenome.nih.gov/ data set with characterized radiological images. The Cancer Imaging Program (CIP) with the cooperation of several of the TCGA tissue-contributing institutions are working to archive a large portion of the radiological images of the genetically-analyzed OV cases.</p>\n<br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.NDO1MDFQ\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-OV <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',591,'CT, MR, OT, SM','Ovarian Serous Cystadenocarcinoma','10.7937/K9/TCIA.2016.NDO1MDFQ','Clinical, Genomics, Image Analyses, Pathology','','Human','Ovary',1,0,'O','',1,1,'fd6a4aec-20cb-4bab-a841-6fad7f6e8361','https://doi.org/10.7937/K9/TCIA.2016.NDO1MDFQ'),(25,'TCGA-KIRC','TCGA-KIRC','TCGA-KIRC','tcga_kirc','<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>.&nbsp;Currently this large CT and MR multi-sequence image collection of kidney renal clear cell carcinoma (KIRC) patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.</p><br />\n</br><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.V6PBVTDR\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA -KIRC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>  page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',537,'CR, CT, MR, SEG, SM','Kidney Renal Clear Cell Carcinoma','10.7937/K9/TCIA.2016.V6PBVTDR','Clinical, Genomics, Image Analyses','','Human','Kidney',1,0,'O','',1,1,'602285fe-931c-4dc0-9c93-5dc0f9d7e069','https://doi.org/10.7937/K9/TCIA.2016.V6PBVTDR'),(26,'TCGA-LGG','TCGA-LGG','TCGA-LGG','tcga_lgg','\n<div>\n	<strong>Note:&nbsp;This collection has special restrictions on its usage. See <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/c4hF\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Data Usage Policies and Restrictions <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>.</strong></p>\n</br><p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a>. Currently this large MR multi-sequence image collection of low grade glioma patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome.</p><br />\n<p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.L4LTD3TK\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-LGG <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',516,'SM','Low Grade Glioma','10.7937/K9/TCIA.2016.L4LTD3TK','Clinical, Genomics, Image Analyses','','Human','Brain',1,0,'O','',1,1,'a884797b-a368-47e3-b09a-5dd35c6f71d3','https://doi.org/10.7937/K9/TCIA.2016.L4LTD3TK'),(27,'TCGA-GBM','TCGA-GBM','TCGA-GBM','tcga_gbm','\n<p>The <a href=\"http://imaging.cancer.gov/\" target=\"_blank\"><u>Cancer Imaging Program (CIP)</u></a> is working directly with primary investigators from institutes participating in TCGA to obtain and load images relating to the genomic, clinical, and pathological data being stored within the <a href=\"http://tcga-data.nci.nih.gov/\" target=\"_blank\">TCGA Data Portal</a> Currently this large MR multi-sequence image collection of glioblastoma patients can be matched by each unique case identifier with the extensive gene and expression data of the same case from The Cancer Genome Atlas Data Portal to research the link between clinical phenome and tissue genome. </p><br />\n<a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.RNYFUYE9\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">TCGA-GBM <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a></span> page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',607,'SM','Glioblastoma Multiforme','10.7937/K9/TCIA.2016.RNYFUYE9','Clinical, Genomics, Image Analyses','','Human','Brain',1,0,'O','',1,1,'3e398ab7-a4e8-42a8-9ca9-668f782b9148','https://doi.org/10.7937/K9/TCIA.2016.RNYFUYE9'),(28,'NSCLC-Radiomics','NSCLC-Radiomics','NSCLC-Radiomics','nsclc_radiomics','<p>\n	This collection contains images from 422 non-small cell lung cancer (NSCLC) patients. For these patients pretreatment CT scans and clinical outcome data are available. This dataset refers to the Lung1 dataset of the study published in Nature Communications</p><br />\n<p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.PF0M9REI\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">NSCLC-Radiomics <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',422,'CT, RTSTRUCT, SEG, SR','Lung Cancer','10.7937/K9/TCIA.2015.PF0M9REI','Clinical, Image Analyses','','Human','Lung',1,0,'O','',1,18,'2753a18a-2fa2-4358-a682-82afa2cf4077','https://doi.org/10.7937/K9/TCIA.2015.PF0M9REI'),(29,'PROSTATEx Zone Segmentations','','','PROSTATEx-Seg-Zones','<p> This collection contains prostate’s CNN-based automatic zonal segmentation for 98 cases randomly selected from the SPIE-AAPM-NCI PROSTATEx Challenges (PROSTATEx). The four-class segmentation encompasses the peripheral zone, transition zone, fibromuscular stroma and the distal prostatic urethra.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/tcia.nbb4-4655\" data-toggle=\"modal\" data-target=\"#external-web-warning\">PROSTATEx-Seg-Zones <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',98,'','Prostate','10.7937/TCIA.NBB4-4655','','Organ segmentations','','Prostate',1,0,'A','prostatex',1,NULL,'6662e02b-efc1-4217-98e7-e2804edb0b71','https://doi.org/10.7937/tcia.nbb4-4655'),(30,'High Resolution Prostate Segmentations for the ProstateX-Challenge','','','PROSTATEx-Seg-HiRes','<p> This collection contains 66 high resolution automatic CNN-based segmentations of the prostate gland for randomly chosen T2-weighted volumes of the SPIE-AAPM-NCI PROSTATEx Challenges (PROSTATEx).</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.DEG7ZG1U\" data-toggle=\"modal\" data-target=\"#external-web-warning\">PROSTATEx-Seg-HiRes <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',66,'','Prostate','10.7937/TCIA.2019.DEG7ZG1U','','Organ segmentations','','Prostate',1,0,'A','prostatex',1,NULL,'5ae19e5b-59c9-437e-ba1a-389f18eaea4e','https://doi.org/10.7937/tcia.2019.deg7zg1u'),(31,'RIDER Lung CT Segmentation Labels from: Decoding tumour phenotype by noninvasive imaging using a quantitative radiomics approach','','','RIDER-LungCT-Seg','<p> This collection contains manual and automatic segmentations of the gross tumor volume for the RIDER Lung CT collection.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/tcia.2020.jit9grk8\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER-LungCT-Seg <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',31,'','Lung','10.7937/tcia.2020.jit9grk8','','Tumor segmentations','','Chest',1,0,'A','rider_lung_ct',1,NULL,'8ac16fb5-5710-4667-a02c-12e632a80f1d','https://doi.org/10.7937/tcia.2020.jit9grk8'),(32,'Mouse-Astrocytoma','Mouse-Astrocytoma','Mouse-Astrocytoma','mouse_astrocytoma','<p>\n	This collection consists of magnetic resonance images (MRI) of genetically engineered mouse models (GEMMs) of high grade astrocytoma, including glioblastoma multiforme (GBM).</p>\n<p>\n	The MRI data contained herein includes anatomic T2 weighted images and dynamic contrast enhanced MRI.</p><br />\n<p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/K9TCIA.2017.SGW7CAQW\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Mouse-Astrocytoma <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>  page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',48,'MR','Glioblastoma Multiforme','10.7937/K9TCIA.2017.SGW7CAQW','','','Mouse','Head',1,0,'O','',1,18,'66056b56-240b-448f-acb1-3be6aad2dcf4','https://doi.org/10.7937/K9TCIA.2017.SGW7CAQW'),(33,'Anti-PD-1_Lung','Anti-PD-1_Lung','Anti-PD-1_Lung','anti_pd_1_lung','<p>\n	This collection includes 46 lung cases treated with&nbsp;anti-PD1 immunotherapy in 2016, each with pre-treatment imaging (CT,PT,SC) and most with 1 imaging follow-up timepoint.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/tcia.2019.zjjwb9ip\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Anti-PD-1_Lung <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',46,'CT, PT, SC, SEG','Lung Cancer','10.7937/tcia.2019.zjjwb9ip','','','Human','Lung',1,0,'O','',1,18,'a7b014c4-2bef-4388-98ff-4aaf3e8798f8','https://doi.org/10.7937/tcia.2019.zjjwb9ip'),(34,'APOLLO-5-LSCC','APOLLO-5-LSCC','APOLLO-5-LSCC','apollo_5_lscc','<p class=\"MsoNormal\"></p><p><span>This collection contains subjects from the National Cancer&#10;Institute&#8217;s Applied Proteogenomics OrganizationaL Learning and Outcomes&#10;(APOLLO) network Lung Squamous Cell Carcinoma cohort. The APOLLO network is a&#10;collaboration between NCI, the Department of Defense (DoD), and the Department&#10;of Veterans Affairs (VA) to incorporate proteogenomics into patient care as a&#10;way of looking beyond the genome, to the activity and expression of the&#10;proteins that the genome encodes.</span>&#160;</p><p><span>Please see the&#160;</span><a href=\"https://doi.org/10.7937/TCIA.QQ0G-EB24\" target=\"_blank\">APOLLO-5-LSCC</a><span>&#160;wiki page to learn&#10;more about the images and to obtain any supporting metadata for this&#10;collection.</span></p><p class=\"MsoListParagraph\"></p>&#10;<p>&#160; &#160;</p>&#10;','2022-04-21',NULL,'Public',7,'CT, PT','Lung Squamous Cell Carcinoma','10.7937/TCIA.QQ0G-EB24','','','Human','Lung',0,0,'O','',1,6,'9ac0c23d-484a-49cf-b135-bed563b7bb11','https://doi.org/10.7937/TCIA.QQ0G-EB24'),(35,'RIDER Lung PET-CT','RIDER Lung PET-CT','RIDER Lung PET-CT','rider_lung_pet_ct','<p><a href = \"https://doi.org/10.7937/K9/TCIA.2015.OFIP7TVM\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Lung PET-CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> is a subsection of the broader Reference Image   Database to  Evaluate Therapy Response (RIDER) collection.&nbsp; RIDER is a   targeted  data collection for the purpose of generating an initial   consensus on  how to harmonize data collection and analysis for   quantitative imaging  methods as applied to measure the response to drug   or radiation  therapy. The long term goal is to provide a resource to   permit  harmonized methods for data collection and analysis across   different  commercial imaging platforms, as required to support   multi-site  clinical trials, using imaging as a biomarker for therapy   response.</p>\n<p>More information about the <a href = \"https://doi.org/10.7937/K9/TCIA.2015.OFIP7TVM\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Lung PET-CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> sub-collection can be  found in the corresponding section of the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/EAA9\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;wiki  page.</p>','2024-03-08',NULL,'Public',243,'CT, PT, SEG','Lung Cancer','10.7937/k9/tcia.2015.ofip7tvm','','','Human','Lung',1,0,'O','',1,7,'b37db341-36d4-4c5d-82f4-0e048075fc07','https://doi.org/10.7937/k9/tcia.2015.ofip7tvm'),(36,'Anti-PD-1_MELANOMA','Anti-PD-1_MELANOMA','Anti-PD-1_MELANOMA','anti_pd_1_melanoma','<p>\n	This collection includes 47 melanoma cases treated with anti-PD1 immunotherapy in 2016, each with pre-treatment and 1 or more imaging (CT, PET, MRI) follow-up timepoints.</p>\n<p>\n	<span _ngcontent-cmj-c43=\"\" class=\"toolTip step show\"><span _ngcontent-cmj-c43=\"\">Please see the <a href=\"https://doi.org/10.7937/tcia.2019.1ae0qtcu\">Anti-PD-1_MELANOMA</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n','2022-04-21',NULL,'Limited',47,'CT, MR, PT','Melanoma','10.7937/tcia.2019.1ae0qtcu','','','Human','Skin',1,0,'O','',1,18,'7beaaad5-bcbf-4c65-a34e-020a1c7a12af','https://doi.org/10.7937/tcia.2019.1ae0qtcu'),(37,'QIN GBM Treatment Response','QIN GBM Treatment Response','QIN GBM Treatment Response','qin_gbm_treatment_response','<div>\n	This collection contains &ldquo;double baseline&rdquo; multi-parametric MRI images collected on participants with newly diagnosed glioblastoma,&nbsp; scanned after surgery but prior to the start of therapy, typically 2-5 days apart. The structural images include T1-weighted pre- and post-contrast , T2-weighted , FLAIR and MEMPRAGE images.&nbsp; Advanced MRI sequences included Diffusion Weighted (DW) Imaging, Dynamic Contrast-Enhanced MRI (DCE-MRI), and Dynamic Susceptibility Contrast MRI (DSC-MRI).</p>\n<div>\n	The value of this collection is to provide clinical image data to establish the test-retest characteristics of parameters calculated from DW-MRI, DCE-MRI, and DSC-MRI such as ADC, Ktrans and rCBV.&nbsp;</p>\n<div>\n	&nbsp;</p>\n<div>\n	Please see the&nbsp;<a href=\"https://doi.org/10.7937/k9/tcia.2016.nQF4gpn2\">QIN GBM Treatment Response</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.&nbsp; &nbsp;&nbsp;</p>\n','2022-04-21',NULL,'Limited',54,'MR','Glioblastoma Multiforme','10.7937/K9/TCIA.2016.nQF4gpn2','','','Human','Brain',1,0,'O','',1,2,'a35fca0f-6450-41fd-955d-2aeedea5c0f7','https://doi.org/10.7937/K9/TCIA.2016.nQF4gpn2'),(38,'QIN-BRAIN-DSC-MRI','QIN-BRAIN-DSC-MRI','QIN-BRAIN-DSC-MRI','qin_brain_dsc_mri','<div>\n	This collection contains MR images of both low and high grade glial brain lesions. Data includes post-contrast T1w images with co-registered volumes of dynamic susceptibility contrast (DSC) MR images in DICOM format. Binary regions of interest are also included, in DICOM format, of the lesion, arterial input function, normal appearing white matter, normal appearing cerebral cortex, and whole brain.&nbsp;</p>\n<div>\n	&nbsp;</p>\n<div>\n	Please see the <a href=\"https://doi.org/10.7937/K9/TCIA.2016.5DI84Js8\">Glioma DSC-MRI Perfusion Data with Standard Imaging and ROIs (QIN-BRAIN-DSC-MRI)</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.&nbsp;&nbsp;</p>\n','2022-04-21',NULL,'Limited',49,'MR','Low & High Grade Glioma','10.7937/K9/TCIA.2016.5DI84Js8','','','Human','Brain',1,0,'O','',1,2,'8ea305f2-7484-43fc-a775-ebe08b6046b9','https://doi.org/10.7937/K9/TCIA.2016.5DI84Js8'),(39,'RIDER NEURO MRI','RIDER Neuro MRI','RIDER Neuro MRI','rider_neuro_mri','<p>\n	RIDER Neuro MRI is a subsection of the broader Reference Image Database to Evaluate Therapy Response (RIDER) collection.&nbsp; RIDER is a targeted data collection for the purpose of generating an initial consensus on how to harmonize data collection and analysis for quantitative imaging methods as applied to measure the response to drug or radiation therapy. The long term goal is to provide a resource to permit harmonized methods for data collection and analysis across different commercial imaging platforms, as required to support multi-site clinical trials, using imaging as a biomarker for therapy response.</p>\n<p>\n	More information about the RIDER Neuro MRI sub-collection can be found in the corresponding section of the <a href=\"https://wiki.cancerimagingarchive.net/x/EAA9\" target=\"_blank\">RIDER</a>&nbsp;wiki page.</p>\n','2022-04-21',NULL,'Limited',19,'MR','Brain Cancer','10.7937/K9/TCIA.2015.VOSN3HN1','','','Human','Brain',1,0,'O','',1,7,'85b8db42-1a7a-47e4-b690-56d325735ce8','https://doi.org/10.7937/K9/TCIA.2015.VOSN3HN1'),(40,'VICTRE','VICTRE','VICTRE','victre','<p >	The VICTRE Trial: Open-Source, In-Silico Clinical Trial For Evaluating Digital Breast Tomosynthesis</p>\n<p>\n	<span>A total of 2986 subjects, with breast sizes and radiographic densities representative of a screening population and compressed thicknesses from 3.5 to 6 cm, were simulated and imaged on in-silico versions of DM and DBT systems using fast Monte Carlo x-ray transport. Images were interpreted by a computational reader detecting the presence of lesions. The in-silico trial (VICTRE) was designed to replicate a comparative trial from a previous regulatory submission. The endpoint was the difference in area under the receiver-operating-characteristic curve between modalities (delta-AUC) for lesion detection.</span></p>\n<p>\n	See the <a href=\"\" url=\"http://doi.org/10.7937/TCIA.2019.ho23nxaw\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">VICTRE Collection <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',2994,'MG','Breast Cancer','10.7937/TCIA.2019.ho23nxaw','Software/Source Code','','Human','Breast',1,0,'O','',1,21,'d95d2e9d-19bd-40f8-bd48-9cbed8108dc1','https://doi.org/10.7937/TCIA.2019.ho23nxaw'),(41,'RIDER Breast MRI','RIDER Breast MRI','RIDER Breast MRI','rider_breast_mri','<span><p><a href=\"\" url=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.H1SXNUXL\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Breast MRI  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> is a subsection of the broader Reference Image  Database to Evaluate  Therapy Response (RIDER) collection. RIDER is a targeted data collection for the purpose of generating an initial consensus on how to harmonize data collection and analysis for quantitative imaging methods as applied to measure the response to drug or radiation therapy. The  long term goal is to provide a resource to permit harmonized methods for data collection and analysis across different commercial imaging  platforms, as required to support multi-site clinical trials, using  imaging as a biomarker for therapy  response.</p>\n<p>More information about the <a href=\"\" url=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.H1SXNUXL\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Breast MRI  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>sub-collection page can be found in the corresponding section of the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/EAA9\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page.</p>\n</span>','2024-03-08',NULL,'Public',5,'MR','Breast Cancer','10.7937/K9/TCIA.2015.H1SXNUXL','','','Human','Breast',1,0,'O','',1,7,'709f2931-21dc-4b46-abf4-995c0dd3b022','https://doi.org/10.7937/K9/TCIA.2015.H1SXNUXL'),(42,'C4KC-KiTS','C4KC-KiTS','C4KC-KiTS','c4kc_kits','<p>\n	<strong>Data from the training set of the 2019 Kidney and Kidney Tumor Segmentation Challenge (C4KC-KiTS)</strong></p>\n<p>\n	This collection contains subjects from the training set of the&nbsp;<a href=\"\" url=\"https://kits19.grand-challenge.org/\" rel=\"nofollow\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">2019 Kidney and Kidney Tumor Segmentation Challenge <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;(KiTS19). The challenge aimed to accelerate progress in automatic 3D semantic segmentation by releasing a dataset of CT scans for 210 patients with manual semantic segmentations of the kidneys and tumors in the corticomedullary phase.<br />\n	<br />\n	The imaging was collected during routine care of patients who were treated by either partial or radical nephrectomy at the University of Minnesota Medical Center. Many of the CT scans were acquired at referring institutions and are therefore heterogeneous in terms of scanner manufacturers and acquisition protocols. Semantic segmentations were performed by students under the supervision of an experienced urologic cancer surgeon.</p>\n<p>\n	Please see the&nbsp;<a href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.IX49E8NX\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">C4KC-KiTS <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',210,'CT, SEG','Kidney Cancer','10.7937/TCIA.2019.IX49E8NX','','','Human','Kidney',1,0,'O','',1,18,'7b8d6061-1a26-4958-96c1-1150c6a01140','https://doi.org/10.7937/TCIA.2019.IX49E8NX'),(43,'Mouse-Mammary','Mouse-Mammary','Mouse-Mammary','mouse_mammary','<p>\n	This collection consists of Magnetic Resonance Imaging (MRI) of mouse breast cancer in 32 mice. &nbsp;</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.9P42KSE6\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Mouse-Mammary <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',32,'MR','Breast Cancer','10.7937/K9/TCIA.2015.9P42KSE6','','','Mouse','Abdomen',1,0,'O','',1,18,'3cdade5f-6ee9-4eed-a324-d8c3bad133cd','https://doi.org/10.7937/K9/TCIA.2015.9P42KSE6'),(44,'DRO-Toolkit','DRO-Toolkit','DRO-Toolkit','dro_toolkit','<p>\n	DRO Toolkit:&nbsp;This is a sample collection of synthetic 3D Digital Reference Objects (DROs) intended for standardization of quantitative imaging feature extraction pipelines. We have developed a software toolkit&nbsp;for the creation of DROs with customizable size, shape, intensity, texture, and margin sharpness values. Using user-supplied input parameters, these objects are defined mathematically as continuous functions, discretized, and then saved as DICOM objects. This collection includes objects with a range of values for the various feature categories and many combinations of these categories.</p>\n	<br /><p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/T062-8262\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">DRO Toolkit <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain the supporting metadata for this collection.</p>\n\n','2024-03-08',NULL,'Public',32,'CT, SEG','Phantom','10.7937/T062-8262','','','Human','Phantom',1,0,'O','',1,18,'fb413f9f-935c-471c-8950-2be80c40c00f','https://doi.org/10.7937/T062-8262'),(45,'CC-Radiomics-Phantom-2','CC-Radiomics-Phantom-2','CC-Radiomics-Phantom-2','cc_radiomics_phantom_2','<p>\n	<strong>Credence Cartridge Radiomics Phantom CT Scans with Controlled Scanning Approach (CC-Radiomics-Phantom-2)</strong></p>\n<p>\n	This collection consists of 251 CT scans of the Credence Cartridge Radiomic (CCR) phantom. This texture phantom was developed to investigate the feature robustness in the emerging field of radiomics. This phantom dataset was acquired on 4-8 CT scanners using a set of imaging parameters (e.g., reconstruction Field of View, Slice thickness, reconstruction kernels, mAs, and Pitch). A controlled scanning approach was employed to assess the variability in radiomic features due to each imaging parameter. This dataset will be useful to radiomic research community to identify a subset of robust radiomic features and for establishing the ground truths for future clinical investigations.</p>\n<p>\n	Using this dataset, the numerical values of radiomic features can be cross-validated by other research groups using their own feature extraction tools.</p>\n<p>\n	Please see the&nbsp;<a href=\"http://doi.org/10.7937/TCIA.2019.4l24tz5g\">CC-Radiomics-Phantom-2</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Excluded',251,'CT','Phantom','10.7937/tcia.2019.4l24tz5g','','','Human','Phantom',0,0,'O','',1,18,'1524fc24-3171-42c5-8407-644e4337d3b0','https://doi.org/10.7937/tcia.2019.4l24tz5g'),(46,'QIN PET Phantom','QIN PET Phantom','QIN PET Phantom','qin_pet_phantom','<p>\n	This collection consists of positron emission tomography (PET) phantom scans originally utilized by the Quantitative Imaging Network (QIN) PET Segmentation Challenge to assess the variability of segmentations and subsequently derived quantitative analysis results on phantom PET scans with known ground truth.</p>\n<p>\n	Please see the <a href=\"https://wiki.cancerimagingarchive.net/display/Public/QIN+PET+Phantom\" target=\"_blank\">QIN PET Phantom</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n	&nbsp;</p>\n','2024-03-08',NULL,'Excluded',2,'PT','Phantom','10.7937/k9/tcia.2015.zpukhckb','','','Human','Phantom',0,0,'O','',1,2,'d093b3ea-6e97-4ecb-95ff-0d808c6177fe','https://doi.org/10.7937/k9/tcia.2015.zpukhckb'),(47,'QIBA CT-1C','QIBA CT-1C','QIBA CT-1C','qiba_ct_1c','<p>Phantom data provided by the RSNA Quantitative Imaging Biomarkers Alliance,,s 1C group.&#160;</p>\n<p>&#10;&#9;Please see the <a href=\"\" url=\"http://qibawiki.rsna.org/index.php?title=VolCT_-_Group_1C#Downloadable_Images_and_Data_Sets\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIBA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page through RSNA, and TCIA,,s <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.YXGR4BLU\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIBA CT-1C <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page,&#160; to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',1,'CT, PR, SEG, SR','Phantom','10.7937/k9/tcia.2016.yxgr4blu','','','Human','Phantom',1,0,'O','',1,25,'1d49db8a-20cb-4995-b716-105cb4baf88d','https://doi.org/10.7937/k9/tcia.2016.yxgr4blu'),(48,'RIDER PHANTOM MRI','RIDER Phantom MRI','RIDER Phantom MRI','rider_phantom_mri','<p>\n<p><span id=\"MAINbody:collectionDescriptionPopupForm:j_id767\" class=\"iceOutTxt\">\n<p>RIDER Phantom MRI is a subsection of the broader Reference Image   Database to  Evaluate Therapy Response (RIDER) collection.&nbsp; RIDER is a   targeted  data collection for the purpose of generating an initial   consensus on  how to harmonize data collection and analysis for   quantitative imaging  methods as applied to measure the response to drug   or radiation  therapy. The long term goal is to provide a resource to   permit  harmonized methods for data collection and analysis across   different  commercial imaging platforms, as required to support   multi-site  clinical trials, using imaging as a biomarker for therapy   response.</p>\n<p>More information about the RIDER Phantom MRI sub-collection can be  found in the corresponding section of the <a href=\"https://wiki.cancerimagingarchive.net/x/EAA9\" target=\"_blank\">RIDER</a>&nbsp;wiki  page.</p>\n</span></p>\n</p>','2024-03-08',NULL,'Excluded',10,'MR','Phantom','10.7937/k9/tcia.2015.mi4qddhu','','','Human','Phantom',0,0,'O','',1,7,'dedb5edc-691e-4d4c-a85c-47164fb53bb4','https://doi.org/10.7937/k9/tcia.2015.mi4qddhu'),(49,'Pseudo-PHI-DICOM-Data','Pseudo-PHI-DICOM-Data','Pseudo-PHI-DICOM-Data','pseudo_phi_dicom_data','<p>\n	<span>The process of image de-identification is time consuming, requires significant human resources, and is prone to human error. &nbsp;Automated image de-identification algorithms have been developed but the research community requires some method of evaluation before such tools can be widely accepted.&nbsp; This evaluation requires a robust dataset that can be used as part of an evaluation process for de-identification algorithms.&nbsp;&nbsp;</span><span>We developed a DICOM dataset that can be used to evaluate the performance of de-identification algorithms.&nbsp;</span></p>\n<p>\n	&nbsp;</p>\n<p>\n	Please see the&nbsp;<a href=\"\" url=\"https://doi.org/10.7937/s17z-r072\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><u>Medical Imaging De-Identification Initiative (MIDI)</u> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',42,'CR, CT, DX, MG, MR, PT','Various','10.7937/s17z-r072','','','Human','Various',1,0,'O','',1,20,'01e9e88f-89a7-445e-92b0-06b7489e2967','https://doi.org/10.7937/s17z-r072'),(50,'NaF PROSTATE','NaF PROSTATE','NaF PROSTATE','naf_prostate','<p>\n	This is a collection of F-18 NaF positron emission tomography/computed tomography (PET/CT) images in patients with prostate cancer, with suspected or known bone involvement.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.ISOQTHKO\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">NaF PROSTATE <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',9,'CT, PT','Prostate Cancer','10.7937/K9/TCIA.2015.ISOQTHKO','','','Human','Prostate',1,0,'O','',1,18,'2cce1b86-8780-4706-a354-7086078bb127','https://doi.org/10.7937/K9/TCIA.2015.ISOQTHKO'),(51,'PROSTATE-MRI','PROSTATE-MRI','PROSTATE-MRI','prostate_mri','<p>\n	This is a collection of &nbsp;MRIs of the prostate obtained with an endorectal coil and phased array surface coil at 3T (Philips Achieva). &nbsp;Each patient had biopsy confirmation of cancer and then underwent robotic assisted radical prostatectomy. &nbsp;A mold was generated from each MRI and the prostatectomy specimen was first placed in the mold and then cut in the same plane as the MRI. &nbsp;The data was generated at the National Cancer Institute, Bethesda, Maryland, USA between 2008-2010.</p><br />\n<p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.6046GUDv\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prostate-MRI <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',26,'MR','Prostate Cancer','10.7937/K9/TCIA.2016.6046GUDv','','','Human','Prostate',1,0,'O','',1,18,'68ecd60d-d5b7-433a-9a0d-7609b21b31d4','https://doi.org/10.7937/K9/TCIA.2016.6046GUDv'),(52,'AAPM-RT-MAC','AAPM RT-MAC Grand Challenge 2019','AAPM RT-MAC Grand Challenge 2019','aapm_rt_mac','<p>\n	AAPM-RT-MAC: This data set was provided in association with a <a href=\"https://www.aapm.org/GrandChallenge/RT-MAC/\">challenge competition</a> conducted at the AAPM 2019 Annual Meeting. The MRI scanning protocol was designed for radiation treatment simulation, 55 cases with T2-weighted images are included. 31 are used as training cases, with the parotid glands, submandibular glands, level 2 and level 3 lymph nodes contoured. The images and contours were acquired from MD Anderson Cancer Center.</p>\n<p>\n	Please see the <a href=\"https://doi.org/10.7937/tcia.2019.bcfjqfqb\">AAPM-RT-MAC</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2022-04-21',NULL,'Limited',55,'MR, RTSTRUCT','Head and Neck Cancer','10.7937/tcia.2019.bcfjqfqb','','','Human','Head-Neck',1,0,'O','',1,18,'7ca6295c-6c2a-4de9-bf42-82e49830b0a5','https://doi.org/10.7937/tcia.2019.bcfjqfqb'),(53,'Phantom FDA','Phantom FDA','Phantom FDA','phantom_fda','<p>\n	A series of CT sets scanned at differing slice thicknesses and exposure parameters of a chest simulating phantom containing lung nodules. It was constructed by an NCI-FDA joint effort to provide data for nodule-detecting CAD processes.</p>\n<p>\n	Please see the<a href=\"\" url=\"https://doi.org/10.7937/k9/TCIA.2015.orbjkmux\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"> Phantom FDA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',7,'CT','Phantom','10.7937/k9/TCIA.2015.orbjkmux','','','Human','Lung Phantom',1,0,'O','',1,21,'b0140041-af49-46e8-a5a5-20612b3f315e','https://doi.org/10.7937/k9/TCIA.2015.orbjkmux'),(54,'MIDRC-RICORD-1B','MIDRC-RICORD-1B','MIDRC-RICORD-1B','midrc_ricord_1b','<p>\n	<span><span><font color=\"#202124\">T</font><span>he first multi-institutional, multi-national expert annotated COVID-19 imaging dataset made freely available to the machine learning community as a research and educational resource for COVID-19 chest imaging.&nbsp;The Radiological Society of North America (RSNA) assembled the&nbsp;<a href=\"\" url=\"https://www.rsna.org/covid-19/covid-19-ricord\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><font color=\"#51a6fa\"><span>RSNA International COVID-19 Open Radiology Database (RICORD)</span></font> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;collection&nbsp;of COVID-related imaging datasets and expert annotations to support research and education.&nbsp;RICORD&nbsp;data will be incorporated in the&nbsp;<a href=\"\" url=\"https://www.midrc.org/\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><font color=\"#51a6fa\"><span>Medical Imaging&nbsp;and&nbsp;Data Resource Center (MIDRC)</span></font> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>,&nbsp;a&nbsp;multi-institutional research data repository funded by the National Institute of Biomedical Imaging and Bioengineering of the National Institutes of Health.</span></span></span></p>\n<p>\n	<span><span>Please see the<span>&nbsp;</span></span><a href=\"\" url=\"https://doi.org/10.7937/31V8-4A40\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><span><span>MIDRC-RICORD-1b</span></span> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>&nbsp;</span><span>wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n','2024-03-08',NULL,'Public',117,'CT','COVID-19 (non-cancer)','10.7937/31V8-4A40','Clinical','','Human','Lung',1,0,'O','',1,24,'b3bdf3d0-fb88-46b2-bbad-d9b4b96572df','https://doi.org/10.7937/31V8-4A40'),(55,'MIDRC-RICORD-1C','MIDRC-RICORD-1C','MIDRC-RICORD-1C','midrc_ricord_1c','<p>\nThe first multi-institutional, multi-national expert annotated COVID-19 imaging dataset made freely available to the machine learning community as a research and educational resource for COVID-19 chest imaging. The Radiological Society of North America (RSNA) assembled the <a href=\"\" url=\"https://www.rsna.org/covid-19/covid-19-ricord\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RSNA International COVID-19 Open Radiology Database (RICORD) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> collection of COVID-related imaging datasets and expert annotations to support research and education. RICORD data will be incorporated in the <a href=\"\" url=\"https://www.midrc.org/\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Medical Imaging and Data Resource Center (MIDRC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>, a multi-institutional research data repository funded by the National Institute of Biomedical Imaging and Bioengineering of the National Institutes of Health. The collection contains 998 Chest x-ray examinations.</p><p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/91ah-v663\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">MIDRC-RICORD-1c <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',361,'CR, DX','COVID-19 (non-cancer)','10.7937/91ah-v663','Clinical','','Human','Lung',1,0,'O','',1,24,'ecbeb9f8-3a31-49a5-b1c5-b1096980f186','https://doi.org/10.7937/91ah-v663'),(56,'MIDRC-RICORD-1A','MIDRC-RICORD-1A','MIDRC-RICORD-1A','midrc_ricord_1a','<p>\n	The Radiological Society of North America (RSNA) set out to create the <a href=\"\" url=\"https://www.rsna.org/covid-19/covid-19-ricord\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RSNA International COVID-19 Open Radiology Database (RICORD) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> collection of COVID-related imaging datasets and expert annotations to support research and education. RICORD data will be incorporated in the <a href=\"\" url=\"https://www.midrc.org/\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Medical Imaging and Data Resource Center (MIDRC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>, a multi-institutional research data repository funded by the National Institute of Biomedical Imaging and Bioengineering of the National Institutes of Health.</p></br>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/VTW4-X588\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">MIDRC-RICORD-1a <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> WIKI page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',110,'CT','COVID-19 (non-cancer)','10.7937/VTW4-X588','Clinical, Software/Source Code','','Human','Lung',1,0,'O','',1,24,'8c619e0a-5140-4e1c-a28d-601802cef123','https://doi.org/10.7937/VTW4-X588'),(57,'NSCLC-Radiomics-Interobserver1','NSCLC-Radiomics-Interobserver1','NSCLC-Radiomics-Interobserver1','nsclc_radiomics_interobserver1','<b><a href=\"\" url=\"\" url=\"https://doi.org/10.7937/tcia.2019.cwvlpd26\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">NSCLC-Radiomics-Interobserver1 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> </b>contains clinical data and computed tomography (CT) from 22 non-small cell lung cancer (NSCLC) radiotherapy patients.  This dataset refers to the &#34;Multiple delineation&#34; dataset of the study published in Nature Communications (<a href=\"\" url=\"http://doi.org/10.1038/ncomms5006\" rel=\"nofollow\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">https://doi.org/10.1038/ncomms5006 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>). In short, the publication used a radiomics approach to computed tomography data of 1,019 patients with lung or head-and-neck cancer.</span><div><br></p><div>Please see the <a href=\"\" url=\"\" url=\"https://doi.org/10.7937/tcia.2019.cwvlpd26\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">NSCLC-Radiomics-Interobserver1 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',22,'CT, RTSTRUCT, SEG','Non-small Cell Lung Cancer','10.7937/tcia.2019.cwvlpd26','Clinical','','Human','Lung',1,0,'O','',1,18,'62a7c0bf-75f0-4a39-895b-5e070d58124b','https://doi.org/10.7937/tcia.2019.cwvlpd26'),(58,'ACRIN-NSCLC-FDG-PET','ACRIN-NSCLC-FDG-PET','ACRIN-NSCLC-FDG-PET','acrin_nsclc_fdg_pet','<p>\n        Thirty-seven institutions accrued 250 patients to the study ACRIN 6668.  Eligible patients were &gt;18 years with AJCC-criteria clinical <strong>stage IIB/III non-small cell lung carcinoma </strong>who were being planned for definitive concurrent chemoradiotherapy (inoperable disease). 173 had evaluable post-treatment PET, representing the analysis cohort for the primary end point.</p>\n<p>\n        Please see the <a href=\"\" url=\"https://doi.org/10.7937/tcia.2019.30ilqfcl\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ACRIN-NSCLC-FDG-PET <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',242,'CR, CT, DX, MR, NM, PT, SC, SEG','Non-small Cell Lung Cancer','10.7937/tcia.2019.30ilqfcl','Clinical','','Human','Lung',1,0,'O','',1,19,'4ac4f400-fa51-4246-b880-be8604e03dc2','https://doi.org/10.7937/tcia.2019.30ilqfcl'),(59,'CPTAC-CM','CPTAC-CM','CPTAC-CM','cptac_cm','<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Cutaneous Melanoma (CPTAC-CM) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n	&nbsp;</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.ODU24GZE\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-CM <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n','2024-03-08',NULL,'Public',95,'CT, MR, PT, SM','Cutaneous Melanoma','10.7937/K9/TCIA.2018.ODU24GZE','Clinical','','Human','Skin',1,0,'O','',1,11,'52feb597-7298-463a-9a71-a7bf28262518','https://doi.org/10.7937/K9/TCIA.2018.ODU24GZE'),(60,'ACRIN-DSC-MR-Brain','ACRIN-DSC-MR-Brain (ACRIN 6677/RTOG 0625)','ACRIN-DSC-MR-Brain (ACRIN 6677/RTOG 0625)','acrin_dsc_mr_brain','<p>\n	ACRIN-DSC-MR-Brain: RTOG 0625/ACRIN 6677 in recurrent glioblastoma (GBM). FLAIR and postcontrast MRI.</p>\n<p>\n	Please see the wiki page <a href=\"https://doi.org/10.7937/tcia.2019.zr1pjf4i\">ACRIN-DSC-MR-Brain</a>&nbsp;to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2022-04-21',NULL,'Limited',123,'CT, MR','Glioblastoma Multiforme','10.7937/tcia.2019.zr1pjf4i','Clinical','','Human','Brain',1,0,'O','',1,19,'2e66054a-addd-4f16-930d-9e32d74c22b8','https://doi.org/10.7937/tcia.2019.zr1pjf4i'),(61,'ACRIN-FMISO-Brain','ACRIN-FMISO-Brain (ACRIN 6684)','ACRIN-FMISO-Brain (ACRIN 6684)','acrin_fmiso_brain','<p class=\"p1\">\n	The objective of the ACRIN 6684 multi-center clinical trial was to determine baseline FMISO PET uptake (maximal tumor to blood ratio, hypoxic volume) and MRI parameters (K<sup>trans</sup>, CBV) with overall survival. After establishing eligibility and enrollment to the study, baseline imaging of both MR and PET was performed within 2 weeks of starting therapy.</p>\n<p>\n	Please see the&nbsp;<a href=\"https://doi.org/10.7937/K9/TCIA.2018.vohlekok\" target=\"_blank\">ACRIN-FMISO-Brain</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2022-04-21',NULL,'Limited',45,'CT, MR, PT','Glioblastoma Multiforme','10.7937/K9/TCIA.2018.vohlekok','Clinical','','Human','Brain',1,0,'O','',1,19,'4a5a2e42-9fc5-4b3f-afd3-88b67ca11564','https://doi.org/10.7937/K9/TCIA.2018.vohlekok'),(62,'PDMR-997537-175-T','PDMR-997537-175-T','PDMR-997537-175-T','pdmr_997537_175_t','<div>\n	We identified and characterized a patient derived xenograft model with metastatic potential, adenocarcinoma colon xenograft 997537-175-T. In this study we performed a detailed imaging characterization of this model, which develops spontaneous lung metastases.</p>\n<div>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.BRY9-4N29\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">PDMR-997537-175-T <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<div>\n	&nbsp;</p>\n','2024-03-08',NULL,'Public',24,'MR, SR','Adenocarcinoma Colon','10.7937/TCIA.2020.BRY9-4N29','Clinical','','Mouse','Colon',1,0,'O','',1,12,'d2a36131-50e5-4354-99bf-e0e52862c207','https://doi.org/10.7937/TCIA.2020.BRY9-4N29'),(63,'Breast-Cancer-Screening-DBT','Breast-Cancer-Screening-DBT','Breast-Cancer-Screening-DBT','breast_cancer_screening_dbt','<p>\n	Breast cancer is among the most common cancers and a common cause of death among women. Here, we share a curated dataset of digital breast tomosynthesis images that includes normal, actionable, biopsy-proven benign, and biopsy-proven cancer cases.&nbsp; The dataset contains four components: (1) DICOM images, (2) a spreadsheet indicating which group each case belongs to, and (3) annotation boxes.&nbsp;<span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/e4wt-cd02\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Breast-Cancer-Screening-DBT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a></span><span>&nbsp;to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n','2024-03-08',NULL,'Public',5060,'MG','Breast Cancer','10.7937/E4WT-CD02','Clinical, Image Analyses, Software/Source Code','','Human','Breast',1,0,'O','',1,18,'edc2e55d-3a50-4bce-b0e0-7bd6dc214b3c','https://doi.org/10.7937/E4WT-CD02'),(64,'ACRIN-FLT-Breast','ACRIN-FLT-Breast','ACRIN-FLT-Breast','acrin_flt_breast','<p><span><b>ACRIN 6688</b>, a [18F]-fluorothymidine (FLT) imaging study using positron emission tomography (PET), was designed to evaluate the relationship between [18F] FLT uptake parameters and pathologic complete response to neoadjuvant therapy of the primary tumor in patients with locally advanced breast cancer.</p>\n<p><u>Participant Eligibility and Enrollment</u>: Criteria for inclusion were patients with pathologically confirmed breast cancer and determined to be a candidate for primary systemic (neoadjuvant) therapy and for whom surgical resection of residual primary tumor following completion of neoadjuvant therapy is planned.</p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2017.ol20zmxg\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ACRIN-FLT-Breast <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>','2024-03-08',NULL,'Public',83,'CT, OT, PT','Breast Cancer','10.7937/K9/TCIA.2017.ol20zmxg','Clinical','','Human','Breast',1,0,'O','',1,19,'a901ae13-7be2-4624-a7b6-f4ed6aee6f89','https://doi.org/10.7937/K9/TCIA.2017.ol20zmxg'),(65,'QIN-BREAST','QIN-BREAST','QIN-BREAST','qin_breast','<div>\n	This collection contains longitudinal PET/CT and quantitative MR images collected for the purpose of studying treatment assessment in breast cancer in the neoadjuvant setting. Images were acquired at three time points: prior to the start of treatment (t1), after the first cycle of treatment (t2), and either after the second cycle of treatment or at the completion of all treatments (prior to surgery) (t3). The PET/CT images were acquired with a support device built in-house to allow the patient to be in the prone position to facilitate registration with the MRI data. The value of this collection is to provide clinical imaging data for the development and evaluation of quantitative imaging methods for treatment assessment early in the course of therapy for breast cancer.&nbsp;</p>\n<div>\n	&nbsp;</p>\n<div>\n	An extension of this dataset is available via the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.4cfm06rr\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIN-BREAST-02 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;collection.</p>\n<div>\n	&nbsp;</p>\n<div>\n	Please see the&nbsp;<a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.21JUebH0\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIN-Breast <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.&nbsp; &nbsp;</p>\n','2024-03-08',NULL,'Public',68,'CT, MR, PT, SEG','Breast Cancer','10.7937/K9/TCIA.2016.21JUEBH0','Clinical','','Human','Breast',1,0,'O','',1,2,'6b2fc557-be74-4999-9cb0-0f204b13a2ad','https://doi.org/10.7937/K9/TCIA.2016.21JUEBH0'),(66,'PDMR-833975-119-R','PDMR-833975-119-R','PDMR-833975-119-R','pdmr_833975_119_r','<p><span><a href=\"\" url=\"\" url=\"https://doi.org/10.7937/TCIA.0ECK-C338\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">PDMR-833975-119-R <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> is a dataset of detailed MRI, PET/CT (FDG and FLT) in patient-derived xenograft in mouse model of pancreatic adenocarcinoma from the National Cancer Institute Patient-Derived Models Repository (<a href=\"https://pdmr.cancer.gov/\">https://pdmr.cancer.gov/</a>) with baseline and bi-weekly monitoring imaging.</span></p>\n<p>\n	<span>Please see the <a href=\"\" url=\"\" url=\"https://doi.org/10.7937/TCIA.0ECK-C338\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">PDMR-833975-119-R <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n','2024-03-08',NULL,'Public',20,'MR, SR','Pancreatic Ductal Adenocarcinoma','10.7937/TCIA.0ECK-C338','Clinical','','Mouse','Abdomen',1,0,'O','',1,12,'c4087a4d-5f1a-477c-8042-55d99545f218','https://doi.org/10.7937/TCIA.0ECK-C338'),(67,'PDMR-292921-168-R','PDMR-292921-168-R','PDMR-292921-168-R','pdmr_292921_168_r','<div>\n	We identified and characterized a patient derived xenograft model with metastatic potential, adenocarcinoma pancreas xenograft 292921-168-R. In this study we performed a detailed imaging characterization of this model, which develops spontaneous lung metastases.</p>\n<div>\n	&nbsp;</p>\n<div>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.PCAK-8Z10\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Imaging characterization of a metastatic patient derived model of adenocarcinoma pancreas: PDMR-292921-168-R <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.&nbsp;</p>\n','2024-03-08',NULL,'Public',20,'MR, SR','Adenocarcinoma Pancreas','10.7937/TCIA.2020.PCAK-8Z10','Clinical','','Mouse','Abdomen',1,0,'O','',1,12,'21c86a17-7d6a-41ee-9065-86fb470c1e86','https://doi.org/10.7937/TCIA.2020.PCAK-8Z10'),(68,'PDMR-425362-245-T','PDMR-425362-245-T','PDMR-425362-245-T','pdmr_425362_245_t','<div>We identified and characterized a patient derived xenograft model with metastatic potential, melanoma xenograft 425362-245-T. In this study we performed a detailed imaging characterization of this model, which develops spontaneous lung metastases.</p>\n<div>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.7YRS-7J97\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">PDMR-425362-245-T <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',20,'MR, SR','Melanoma','10.7937/TCIA.2020.7YRS-7J97','Clinical','','Mouse','Abdomen',1,0,'O','',1,12,'a0a21a2e-4ba7-4204-b4e6-0b16de99f67e','https://doi.org/10.7937/TCIA.2020.7YRS-7J97'),(69,'PDMR-BL0293-F563','PDMR-BL0293-F563','PDMR-BL0293-F563','pdmr_bl0293_f563','<span><p>We identified and characterized a patient-derived xenograft model with metastatic potential, bladder xenograft BL0293-F563. In this study we performed a detailed imaging characterization of this model, which develops spontaneous liver and bone metastases.</span></p>\n<p>	<span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/tcia.2019.b6u7wmqw\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">PDMR-BL0293-F563 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.\n</span></p>','2024-03-08',NULL,'Public',19,'MR','Bladder Cancer','10.7937/tcia.2019.b6u7wmqw','Clinical','','Mouse','Bladder',1,0,'O','',1,12,'eef46450-6326-4670-a6a9-8c0e9c97ce27','https://doi.org/10.7937/tcia.2019.b6u7wmqw'),(70,'CPTAC-PDA','CPTAC-PDA','CPTAC-PDA','cptac_pda','<p>\n	<p>\n		This collection contains subjects from the National Cancer Institute&rsquo;s <a href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a> Pancreatic Ductal Adenocarcinoma (CPTAC-PDA) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</p>\n	<p>\n		&nbsp;</p>\n	<p>\n		Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.SC20FO18\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-PDA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n</p>\n','2024-03-08',NULL,'Public',194,'CT, MR, PT, RTSTRUCT, SM, US','Ductal Adenocarcinoma','10.7937/K9/TCIA.2018.SC20FO18','Clinical','','Human','Pancreas',1,0,'O','',1,11,'90c6cad3-94be-4bb3-851a-f1f5e73f3c40','https://doi.org/10.7937/K9/TCIA.2018.SC20FO18'),(71,'HEAD-NECK-RADIOMICS-HN1','Head-Neck-Radiomics-HN1','Head-Neck-Radiomics-HN1','head_neck_radiomics_hn1','<p>\nThis collection contains clinical data and computed tomography (CT) from 137 head and neck squamous cell carcinoma (HNSCC) patients treated by radiotherapy.</p>\n<p>\n	Please see the <a href=\"https://doi.org/10.7937/tcia.2019.8kap372n\">Head-Neck-Radiomics-HN1</a> wiki page for further detail and associated metadata.</p>\n','2022-04-21',NULL,'Limited',137,'CT, PT, RTSTRUCT, SEG','Head and Neck Cancer','10.7937/tcia.2019.8kap372n','Clinical','','Human','Head-Neck',1,0,'O','',1,18,'d7e26892-e237-4df0-9b6a-35ec6ac75a54','https://doi.org/10.7937/tcia.2019.8kap372n'),(72,'OPC-Radiomics','OPC-Radiomics','OPC-Radiomics','opc_radiomics','<div>\n	This data collection consists of 606 Radiotherapy planning CT scans retrieved for all non-metastatic p16-positive oropharynx squamous cell carcinoma patients treated with radiotherapy or chemoradiotherapy. Radiomic biomarkers were derived from each gross tumor volume (GTV). The role of quantitative imaging (i.e. radiomic) analysis was examined to determine whether there are primary tumor features discernible on imaging studies that associate with a higher risk of developing distant metastases.</p>\n<div>\n	&nbsp;</p>\n<div>\n	Please see the <a href=\"https://doi.org/10.7937/tcia.2019.8dho2gls\">OPC-Radiomics</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<div>\n	&nbsp;</p>\n','2022-04-21',NULL,'Limited',606,'CT, RTSTRUCT','Oropharyngeal','10.7937/tcia.2019.8dho2gls','Clinical','','Human','Head-Neck',1,0,'O','',1,18,'eb470a1f-b9f6-4d44-ba00-4c93e1d1da87','https://doi.org/10.7937/tcia.2019.8dho2gls'),(73,'CPTAC-SAR','CPTAC-SAR','CPTAC-SAR','cptac_sar','<p>\n	This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u>&nbsp;Sarcomas (CPTAC-SAR) cohort.&nbsp; CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.9bt23r95\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-SAR <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',88,'CT, MR, PT, SM','Sarcomas','10.7937/tcia.2019.9bt23r95','Clinical','','Human','Abdomen, Arm, Bladder, Chest, Head-Neck, Kidney, Leg, Retroperitoneum, Stomach, Uterus',1,0,'O','',1,11,'32ee8b06-fea5-4432-9693-57e170282923','https://doi.org/10.7937/tcia.2019.9bt23r95'),(74,'ICDC-Glioma','ICDC-Glioma','ICDC-Glioma','icdc_glioma','<p><b><a href=\"\" url=\"https://doi.org/10.7937/TCIA.SVQT-Q016\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ICDC-Glioma <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;</b>contains treatment-na&#239;ve naturally-occurring <b>canine glioma</b> participants from the&#160;<a href=\"https://caninecommons.cancer.gov/#/study/GLIOMA01\">Integrated Canine Data Commons</a>. Brain radiology (57/81 participant animals) and H&amp;E-stained biopsy or necropsy pathology (76/81 participants) are classified by veterinary and physician neuropathologists.<br>&#10;&#9;<br>&#10;&#9;Please see the wiki <a href=\"\" url=\"https://doi.org/10.7937/TCIA.SVQT-Q016\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"><b>ICDC-Glioma</b> <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;to learn more about the images and to obtain any supporting metadata for this collection.</p>&#10;','2024-03-08',NULL,'Public',80,'MR, SM','Glioma','10.7937/TCIA.SVQT-Q016','Genomics','','Canine','Head',1,0,'O','',1,22,'430415db-eeb8-41d4-b4ca-ebb568010c48','https://doi.org/10.7937/TCIA.SVQT-Q016'),(75,'Lung-PET-CT-Dx','Lung-PET-CT-Dx','Lung-PET-CT-Dx','lung_pet_ct_dx','<p>\nThis dataset consists of CT and PET-CT DICOM images of lung cancer subjects with XML Annotation files that indicate tumor location with bounding boxes. The images were retrospectively acquired from patients with suspicion of lung cancer, and who underwent standard-of-care lung biopsy and PET/CT. Subjects were grouped according to a tissue histopathological diagnosis. Patients with Names/IDs containing the letter ,,A,, were diagnosed with Adenocarcinoma, ,,B,, with Small Cell Carcinoma, ,,C,, with Large Cell Carcinoma, and ,,G,, with Squamous Cell Carcinoma.</p>\n<p></br>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.NNC2-0461\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">LUNG-PET-CT-Dx <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',355,'CT, PT, SEG','Lung Cancer','10.7937/TCIA.2020.NNC2-0461','Image Analyses, Software/Source Code','','Human','Lung',1,0,'O','',1,18,'1fd8e102-aa5a-4e53-8cb0-3de4ea774d74','https://doi.org/10.7937/TCIA.2020.NNC2-0461'),(76,'Lung-Fused-CT-Pathology','Lung-Fused-CT-Pathology','Lung-Fused-CT-Pathology','lung_fused_ct_pathology','<p>\n	This is the first attempt of mapping the extent of Invasive Adenocarcinoma onto&nbsp;in vivo&nbsp;lung CT. The mappings constitute ground truth of disease and may be used to further investigate the imaging signatures of Invasive Adenocarcinoma in ground glass pulmonary nodules.&nbsp;</p>\n<p>\n	Please see the&nbsp;<a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.SMT36LPN\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Lung-Fused-CT-Pathology <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.<o:p></o:p></p>\n','2024-03-08',NULL,'Public',6,'CT, FUSION, SEG','Lung Cancer','10.7937/k9/tcia.2018.smt36lpn','Image Analyses, Software/Source Code','','Human','Lung',1,0,'O','',1,18,'6ea91b16-f6a9-4b95-9107-6474d352e887','https://doi.org/10.7937/k9/tcia.2018.smt36lpn'),(77,'LCTSC','LCTSC','LCTSC','lctsc','<p>\n	This Lung CT and Radiation therapy data set was provided in association with an autocontouring challenge competition and <a href=\"\" url=\"http://www.aapm.org/meetings/2017AM/PRAbs.asp?mid=127&amp;aid=35318\" rel=\"nofollow\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">related conference session <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> conducted at the <a href=\"\" url=\"http://www.aapm.org/meetings/2017AM/\" rel=\"nofollow\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">AAPM 2017 Annual Meeting <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> </p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2017.3r3fvz08\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">LCTSC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection. </p>\n','2024-03-08',NULL,'Public',60,'CT, RTSTRUCT','Lung Cancer','10.7937/K9/TCIA.2017.3R3FVZ08','Image Analyses','','Human','Lung',1,0,'O','',1,18,'8a00b548-cba4-42e1-869d-0f5ab498371f','https://doi.org/10.7937/K9/TCIA.2017.3R3FVZ08'),(78,'QIN LUNG CT','QIN LUNG CT','QIN LUNG CT','qin_lung_ct','<p>\n	The Computed tomography (CT) Image data was obtained on patients diagnosed with Non-Small Cell Lung Cancer (NSCLC) with mixed stage &amp; histology from the H. Lee Moffitt Cancer Center and Research Institute. Scans were obtained from patients who underwent surgical resection and had corresponding pre-surgery diagnostic CTs.</p>\n<p>\n	&nbsp;</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.NPGZYZBZ\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIN LUNG CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>  page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',47,'CT, SEG','Non-small Cell Lung Cancer','10.7937/K9/TCIA.2015.NPGZYZBZ','','','Human','Lung',1,0,'O','',1,2,'7ef5f7dc-a2f2-4ab7-b26f-2e8c141d52a2','https://doi.org/10.7937/K9/TCIA.2015.NPGZYZBZ'),(79,'4D-Lung','4D-Lung','4D-Lung','4d_lung','<p>\n	<span>This data collection consists of images acquired during chemoradiotherapy of 20 locally-advanced, non-small cell lung cancer patients. The images include four-dimensional (4D) fan beam (4D-FBCT) and 4D cone beam CT (4D-CBCT). All patients underwent concurrent radiochemotherapy to a total dose of 64.8-70&nbsp;Gy using daily 1.8 or 2&nbsp;Gy fractions.</span></p>\n<p>\n	A single Radiation Oncologist delineated targets and organs at risk in all 4D-FBCT and a limited number of 4D-CBCT images. All patients underwent concurrent radiochemotherapy to a total dose of 64.8-70 Gy using daily 1.8 or 2 Gy fractions.</p>\n<p>\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2016.ELN8YGLE\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">4D-Lung <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',20,'CT, RTSTRUCT','Non-small Cell Lung Cancer','10.7937/K9/TCIA.2016.ELN8YGLE','Image Analyses','','Human','Lung',1,0,'O','',1,18,'89264efe-02e2-48ad-a177-a5d930b4fc65','https://doi.org/10.7937/K9/TCIA.2016.ELN8YGLE'),(80,'Brain-Tumor-Progression','Brain-Tumor-Progression','Brain-Tumor-Progression','brain_tumor_progression','<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection includes datasets from 20 subjects with primary newly diagnosed glioblastoma who were treated with surgery and standard concomitant chemo-radiation therapy (CRT) followed by adjuvant chemotherapy.&nbsp; Two MRI exams are included for each patient: within 90 days following CRT completion and at progression, including </span>Pre/post T1 images, FLAIR, Processed parameter maps such as rCBV, CBF, ADC</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id712\">Please see the <a href=\"https://doi.org/10.7937/K9/TCIA.2018.15quzvnb\">Brain-Tumor-Progression</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n','2022-04-21',NULL,'Limited',20,'MR','Brain Cancer','10.7937/K9/TCIA.2018.15quzvnb','Image Analyses','','Human','Brain',1,0,'O','',1,18,'0fbc2ae0-0e8e-4c80-ba29-90223332be4d','https://doi.org/10.7937/K9/TCIA.2018.15quzvnb'),(81,'RIDER Lung CT','RIDER Lung CT','RIDER Lung CT','rider_lung_ct','<p><a href = \"https://doi.org/10.7937/K9/TCIA.2015.U1X8A5NR\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Lung CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>  is a subsection of the broader Reference Image   Database to  Evaluate Therapy Response (RIDER) collection.&nbsp; RIDER is a   targeted  data collection for the purpose of generating an initial   consensus on  how to harmonize data collection and analysis for   quantitative imaging  methods as applied to measure the response to drug   or radiation  therapy. The long term goal is to provide a resource to   permit  harmonized methods for data collection and analysis across   different  commercial imaging platforms, as required to support   multi-site  clinical trials, using imaging as a biomarker for therapy   response.</p>\n<p>More information about the <a href = \"https://doi.org/10.7937/K9/TCIA.2015.U1X8A5NR\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RIDER Lung CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> sub-collection page can be  found in the corresponding section of the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/EAA9\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>  page.</p>\n','2024-03-08',NULL,'Public',32,'CT, PR, RTSTRUCT, SEG, SR','Lung Cancer','10.7937/k9/tcia.2015.u1x8a5nr','Image Analyses','','Human','Chest',1,0,'O','',1,7,'40d20584-b728-446a-ab7a-3062cbf151ad','https://doi.org/10.7937/k9/tcia.2015.u1x8a5nr'),(82,'CT COLONOGRAPHY','CT COLONOGRAPHY','CT COLONOGRAPHY','ct_colonography','<p>The CT Colonography collection consists of a limited portion of The National CT Colonography Trial (<a href=\"\" url=\"http://www.acrin.org/TabID/151/Default.aspx\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ACRIN PROTOCOL 6664 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>). The collection is offered to enlarge the broader research community\'\'s ability to develop CAD algorithms for polyp detection. </p><br /><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.NWTESAY1\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CT COLONOGRAPHY <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>','2024-03-08',NULL,'Public',825,'CT','Colon Cancer','10.7937/K9/TCIA.2015.NWTESAY1','Image Analyses','','Human','Colon',1,0,'O','',1,19,'362cb3b9-80eb-4a0d-92ef-d5e8140d1bc4','https://doi.org/10.7937/K9/TCIA.2015.NWTESAY1'),(83,'CBIS-DDSM','CBIS-DDSM','CBIS-DDSM','cbis_ddsm','<p>\n	This CBIS-DDSM (Curated Breast Imaging Subset of DDSM) is an updated and standardized version of the <a href=\"http://marathon.csee.usf.edu/Mammography/Database.html\">Digital Database for Screening Mammography (DDSM)</a>.&nbsp; The DDSM is a database of 2,620 scanned film mammography studies. It contains normal, benign, and malignant cases with verified pathology information.</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id712\">Please see the <a href=\"https://wiki.cancerimagingarchive.net/display/Public/CBIS-DDSM\">CBIS-DDSM</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n','2024-03-08',NULL,'Excluded',6671,'MG','Breast Cancer','10.7937/k9/tcia.2016.7o02s9cy','Image Analyses','','Human','Breast',0,0,'O','',1,18,'3313e20d-5e44-494b-b40d-d0b65acbfe65','https://doi.org/10.7937/k9/tcia.2016.7o02s9cy'),(84,'RIDER PHANTOM PET-CT','RIDER Phantom PET-CT','RIDER Phantom PET-CT','rider_phantom_pet_ct','<p>\n<p><span id=\"MAINbody:collectionDescriptionPopupForm:j_id767\" class=\"iceOutTxt\">\n<p>RIDER Phantom PET-CT is a subsection of the broader Reference Image   Database to  Evaluate Therapy Response (RIDER) collection.&nbsp; RIDER is a   targeted  data collection for the purpose of generating an initial   consensus on  how to harmonize data collection and analysis for   quantitative imaging  methods as applied to measure the response to drug   or radiation  therapy. The long term goal is to provide a resource to   permit  harmonized methods for data collection and analysis across   different  commercial imaging platforms, as required to support   multi-site  clinical trials, using imaging as a biomarker for therapy   response.</p>\n<p>More information about the RIDER Phantom PET-CT sub-collection can be  found in the corresponding section of the <a href=\"https://wiki.cancerimagingarchive.net/x/EAA9\" target=\"_blank\">RIDER</a>&nbsp;wiki  page.</p>\n</span></p>\n</p>','2024-03-08',NULL,'Excluded',20,'PT, CT','Phantom','10.7937/k9/tcia.2015.8wg2kn4w','Image Analyses','','Human','Phantom',0,0,'O','',1,7,'fcc8910a-ab1e-450c-b6bf-04da46dffa19','https://doi.org/10.7937/k9/tcia.2015.8wg2kn4w'),(85,'Pancreas-CT','Pancreas-CT','Pancreas-CT','pancreas_ct','<p>\n	The National Institutes of Health Clinical Center performed 82 abdominal contrast enhanced 3D CT scans (~70 seconds after intravenous contrast injection in portal-venous) from 53 male and 27 female subjects.&nbsp; Seventeen of the subjects are healthy kidney donors scanned prior to nephrectomy.&nbsp; The remaining 65 patients were selected by a radiologist from patients who neither had major abdominal pathologies nor pancreatic cancer lesions.&nbsp;</p>\n<p>\n	A medical student manually performed slice-by-slice segmentations of the pancreas as ground-truth and these were verified/modified by an experienced radiologist.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.tNB1kqBU\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Pancreas-CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',80,'CT','Healthy Controls (non-cancer)','10.7937/K9/TCIA.2016.tNB1kqBU','Image Analyses','','Human','Pancreas',1,0,'O','',1,18,'3f938fbc-8308-42ad-a101-b401410c908f','https://doi.org/10.7937/K9/TCIA.2016.tNB1kqBU'),(86,'QIN-PROSTATE-Repeatability','QIN-PROSTATE-Repeatability','QIN-PROSTATE-Repeatability','qin_prostate_repeatability','<p>This is a dataset with multiparametric prostate MRI applied in a test-retest setting, allowing to evaluate repeatability of the MRI-based measurements in the prostate. There is very limited data about the repeatability in mpMRI of the prostate, while such information is critical for establishing technical characteristics of mpMRI as imaging biomarker of prostate cancer.</p><p>Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2018.MR1CKGND\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIN-PROSTATE-Repeatability  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',15,'MR, SEG, SR','Prostate Cancer','10.7937/K9/TCIA.2018.MR1CKGND','Image Analyses','','Human','Prostate',1,0,'O','',1,2,'2e43e1fc-0fea-46c4-9bec-9a90129db099','https://doi.org/10.7937/K9/TCIA.2018.MR1CKGND'),(87,'Prostate-MRI-US-Biopsy','Prostate-MRI-US-Biopsy','Prostate-MRI-US-Biopsy','prostate_mri_us_biopsy','<p>\n	This dataset was derived from tracked biopsy sessions using the Artemis biopsy system, many of which included image fusion with MRI targets. Patients received a 3D transrectal ultrasound scan, after which nonrigid registration (e.g. &ldquo;fusion&rdquo;)&nbsp; was performed between real-time ultrasound and preoperative MRI, enabling biopsy cores to be sampled from MR regions of interest.&nbsp;Most cases also included sampling of systematic biopsy cores using a 12-core digital template.&nbsp;The Artemis system tracked targeted and systematic core locations using encoder kinematics of a mechanical arm, and recorded locations relative to the Ultrasound scan. MRI biopsy coordinates were also recorded for most cases.&nbsp;</span></p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.A61IOC1A\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prostate-MRI-US-Biopsy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',1151,'M3D, MR, SEG, US','Prostate Cancer','10.7937/TCIA.2020.A61IOC1A','Image Analyses','','Human','Prostate',1,0,'O','',1,18,'c60c1882-61e3-464c-ab84-e24e590fc4aa','https://doi.org/10.7937/TCIA.2020.A61IOC1A'),(88,'PROSTATEx','PROSTATEx','PROSTATEx','prostatex','<p>SPIE, along with the support of the American Association of Physicists in Medicine (AAPM) and the National Cancer Institute (NCI), will conduct a &#8220;Grand Challenge&#8221; on quantitative image analysis methods for the diagnostic classification of clinically significant prostate lesions.  As part of the 2017 SPIE Medical Imaging Symposium, the PROSTATEx Challenge will provide a unique opportunity for participants to compare their algorithms with those of others from academia, industry, and government in a structured, direct way using the same data sets.  For more details, go to <a href=\"\" url=\"https://www.spie.org/search?&term=PROSTATEx\" rel=\"nofollow\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">https://www.spie.org/search?&term=PROSTATEx <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>. rel=\"nofollow\">http://spiechallenges.cloudapp.net/</a>. </p>&#10;<ul>&#10;&#9;<li>&#10;&#9;&#9;Release date of training set cases with truth:<strong> &#160;November 21, 2016</strong></li>&#10;&#9;<li>&#10;&#9;&#9;Release date of test set cases without truth:<strong> &#160;December 12, 2016</strong></li>&#10;&#9;<li>&#10;&#9;&#9;Submission date for participants&#8217; test set classification output:<strong> &#160;January 15, 2017</strong></li>&#10;&#9;<li>&#10;&#9;&#9;Challenge results released to participants:&#160; <strong>January 20, 2017</strong></li>&#10;&#9;<li>&#10;&#9;&#9;SPIE Medical Imaging Symposium: &#160;<strong>February 13-16, 2017</strong></li></span></p>\n<br /><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9TCIA.2017.MURS5CL\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\"> PROSTATEx <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',346,'MR, SEG','Prostate Cancer','10.7937/K9TCIA.2017.MURS5CL','Image Analyses','','Human','Prostate',1,0,'O','',1,18,'41eab4f7-68f6-4b36-b358-33bc0312e66e','https://doi.org/10.7937/K9TCIA.2017.MURS5CL'),(89,'Prostate Fused-MRI-Pathology','Prostate Fused-MRI-Pathology','Prostate Fused-MRI-Pathology','prostate_fused_mri_pathology','<p>\n	This collection comprises a total of 28 3 Tesla T1-weighted, T2-weighted, Diffusion weighted and Dynamic Contrast Enhanced prostate MRI along with accompanying digitized histopathology (H&amp;Estained) images of corresponding radical prostatectomy specimens.</p>\n<p>\n	Fused MATLAB objects are also provided combining both the MR and Pathology Images.</p>\n<p>\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2016.TLPMR1AM\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prostate Fused-MRI-Pathology <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',28,'MR','Prostate Cancer','10.7937/k9/TCIA.2016.tlpmr1am','Image Analyses','','Human','Prostate',1,0,'O','',1,18,'eb69d4f3-1833-4153-b102-cf38d541a8e0','https://doi.org/10.7937/k9/TCIA.2016.tlpmr1am'),(90,'Prostate-3T','Prostate-3T','Prostate-3T','prostate_3t','<p>\n	Prostate transversal T2 -weighted MRIs acquired on a 3.0T Siemens TrioTim using only a pelvic phased array coil. Images were acquired for prostate cancer detection.&nbsp;</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.QJTV5IL5\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prostate-3T  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',64,'MR','Prostate Cancer','10.7937/K9/TCIA.2015.QJTV5IL5','Image Analyses','','Human','Prostate',1,0,'O','',1,18,'6c0e7d31-8a05-480c-b9c2-0cb287204c6c','https://doi.org/10.7937/K9/TCIA.2015.QJTV5IL5'),(91,'HNSCC-3DCT-RT','HNSCC-3DCT-RT','HNSCC-3DCT-RT','hnscc_3dct_rt','<p>\n	This data collection contains three dimensional high-resolution (3D) fan-beam CT scans collected during pre-treatment, mid-treatment, and post-treatment using a Siemens 16-slice CT scanner with standard clinical protocol for 31 head-and-neck squamous cell carcinoma (HNSCC) patients.&nbsp;</p>\n<p>\n	&nbsp;</p>\n<p>\n	Please see the&nbsp;<a href=\"https://doi.org/10.7937/K9/TCIA.2018.13upr2xf\">HNSCC-3DCT-RT</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.<o:p></o:p></p>\n','2022-04-21',NULL,'Limited',31,'CT, RTDOSE, RTSTRUCT','Head and Neck Squamous Cell Carcinoma','10.7937/K9/TCIA.2018.13upr2xf','Image Analyses','','Human','Head-Neck',1,0,'O','',1,18,'e149d553-81fd-479a-9a95-8a6f0bb4db8e','https://doi.org/10.7937/K9/TCIA.2018.13upr2xf'),(92,'Head-Neck Cetuximab','Head-Neck Cetuximab (RTOG 0522)','Head-Neck Cetuximab (RTOG 0522)','head_neck_cetuximab','<p>\n	<p>\n		<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id767\">The Head-Neck Cetuximab collection consists of a subset of image data from ACRIN 4500/RTOG 0522, which was randomized phase III Trial of Radiation Therapy and Chemotherapy for stage III and IV Head and Neck carcinomas. Data was provided through two indpendent channels:</span></p>\n	<ul>\n		<li>\n			<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id767\">RTOG 0522: CT, Structures, RT Doses, RT Plans sent to ITC</span></li>\n		<li>\n			<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id767\">ACRIN 4500: Quantitative PET (PET/CT) sent to ACRIN</span></li>\n	</ul>\n	<p>\n		<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id767\">Please see the <a href=\"https://wiki.cancerimagingarchive.net/x/xwxp\" target=\"_blank\">Head-Neck Cetuximab wiki page</a> </span><span class=\"iceOutTxt\">to learn more about the images and to obtain any supporting metadata for this collection</span>.</p>\n</p>\n','2022-04-21',NULL,'Limited',111,'CT, PT, RTDOSE, RTPLAN, RTSTRUCT','Head and Neck Carcinomas','10.7937/K9/TCIA.2015.7AKGJUPZ','Image Analyses','','Human','Head-Neck',1,0,'O','',1,19,'d06d4158-96ed-4d6c-bd00-fd1071f5a520','https://doi.org/10.7937/K9/TCIA.2015.7AKGJUPZ'),(93,'CC-Radiomics-Phantom','CC-Radiomics-Phantom','CC-Radiomics-Phantom','cc_radiomics_phantom','<p>\n	This collection consists of 17 CT scans of the Credence Cartridge Radiomics (CCR) phantom, which was designed for use in studies of texture feature robustness. The scans were acquired at four medical centers using each center&rsquo;s chest protocol and were taken using GE (7 scans), Philips (5 scans), Siemens (2 scans), and Toshiba (3 scans) scanners. The CCR phantom has 10 cartridges, each with a unique texture</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id712\">Please see the <a href=\"https://doi.org/10.7937/K9/TCIA.2017.zuzrml5b\">CC-Radiomics-Phantom</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n','2024-03-08',NULL,'Excluded',17,'CT, RTSTRUCT','Phantom','10.7937/k9/tcia.2017.zuzrml5b','Image Analyses','','Human','Lung Phantom',0,0,'O','',1,18,'f8bfe3a0-359b-4459-942c-68c7a7a76ac7','https://doi.org/10.7937/k9/tcia.2017.zuzrml5b'),(94,'Lung Phantom','Lung Phantom','Lung Phantom','lung_phantom','<p>\n	The FDA anthropomorphic thorax phantom with 12 phantom lesions of different sizes (10 and 20 mm in effective diameter), shapes (spherical, elliptical, lobulated, and spiculated), and densities (&minus;630,&minus;10, and +100 HU) was scanned at Columbia University Medical Center on a 64-detector row scanner (LightSpeed VCT, GE Healthcare, Milwaukee, WI). The CT scanning parameters were 120 kVp, 100 mAs, 64x0.625 collimation, and pitch of 1.375. The images were reconstructed with the lung kernel using 1.25 mm slice thickness.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.08A1IXOO\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Lung Phantom <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',1,'CT, SEG','Phantom','10.7937/k9/tcia.2015.08a1ixoo','Image Analyses','','Human','Lung Phantom',1,0,'O','',1,2,'33d897ef-68b7-4a6e-b8bc-71171c200243','https://doi.org/10.7937/k9/tcia.2015.08a1ixoo'),(95,'CT Lymph Nodes','CT Lymph Nodes','CT Lymph Nodes','ct_lymph_nodes','<p>\n	This collection consists of Computed Tomography (CT) images of the mediastinum and abdomen in which lymph node positions are marked by radiologists at the National Institutes of Health, Clinical Center. <a href=\"https://www.cc.nih.gov/meet-our-doctors/rsummers.html\">Radiologists at the Imaging Biomarkers and Computer-Aided Diagnosis Laboratory</a> labeled a total of 388 mediastinal lymph nodes in CT images of 90 patients and a total of 595 abdominal lymph nodes in 86 patients.<br /></p>\n	<br /><p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.AQIIDCNM\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CT Lymph Nodes  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>wiki page to learn more about the images and to obtain supporting data from this collection.</p>\n','2024-03-08',NULL,'Public',176,'CT, SEG','Lymphadenopathy (non-cancer)','10.7937/K9/TCIA.2015.AQIIDCNM','Image Analyses','','Human','Abdomen, Mediastinum',1,0,'O','',1,18,'0bcbd7b7-4e4a-4301-8eb2-eaa0fe8b4e7c','https://doi.org/10.7937/K9/TCIA.2015.AQIIDCNM'),(96,'CC-Radiomics-Phantom-3','CC-Radiomics-Phantom-3','CC-Radiomics-Phantom-3','cc_radiomics_phantom_3','<p>\n	<strong>CT Phantom Scans for Head, Chest, and Controlled Protocols on 100 Scanners (CC-Radiomics-Phantom-3)</strong></p>\n<p>\n	This data collection contains one physical phantom, imaged across three protocols, on 100 scanners. CT scans were acquired on 100 scanners at 35 clinics: 51 GE scanners, 20 Philips scanners, 11 Toshiba scanners, and 1 Philips and Neusoft Medical System scanner. The commonly used chest and head protocols of the local clinic were acquired without changing the protocol parameters. Additionally, a controlled protocol was acquired that was designed to minimize radiomics feature differences between manufacturers.&nbsp;</p>\n<p>\n	Please see the&nbsp;<a href=\"http://doi.org/10.7937/tcia.2019.j71i4fah\">CC-Radiomics-Phantom-3</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Excluded',95,'RTSTRUCT, CT','Phantom','10.7937/tcia.2019.j71i4fah','Image Analyses','','Human','Head & Chest Phantom',0,0,'O','',1,18,'0e47fe59-e5e7-46b9-9a04-2068cf23a910','https://doi.org/10.7937/tcia.2019.j71i4fah'),(97,'Pelvic-Reference-Data','Pelvic-Reference-Data','Pelvic-Reference-Data','pelvic_reference_data','<p>\n	This collection links expertly selected landmark points on clinical image pairs to provide a basis for rigid registration validation. Using combinatorial rigid registration optimization (CORRO) we provide a statistically characterized reference data set for image registration of the pelvis by estimating the optimal ground truth. Landmark points for each CT/CBCT image pair for 58 pelvic cases were identified. The mean and the standard deviation of the registration were used as the final registration for each image pair.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2019.woskq5oo\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Pelvic-Reference-Data <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n	&nbsp;</p>\n','2024-03-08',NULL,'Public',58,'CT','Prostate, Anal','10.7937/TCIA.2019.WOSKQ5OO','Image Analyses','','Human','Pelvis, Prostate, Anus',1,0,'O','',1,18,'4da1d50f-5018-43cc-8243-37e4d215583a','https://doi.org/10.7937/TCIA.2019.WOSKQ5OO'),(98,'Vestibular-Schwannoma-SEG','Vestibular-Schwannoma-SEG','Vestibular-Schwannoma-SEG','vestibular_schwannoma_seg','	This collection contains a labelled dataset of MRI images collected on 242 consecutive patients with vestibular schwannoma (VS) undergoing Gamma Knife stereotactic radiosurgery (GK SRS). The structural images included contrast-enhanced T1-weighted (ceT1) images and high-resolution T2-weighted (hrT2) images. Each imaging dataset is accompanied by the patient&rsquo;s radiation therapy (RT) dataset including the RTDose, RTStructures and RTPlan.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.9YTJ-5Q73\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Vestibular-Schwannoma-SEG <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',242,'MR, RTDOSE, RTPLAN, RTSTRUCT','Vestibular Schwannoma (non-cancer)','10.7937/TCIA.9YTJ-5Q73','Image Analyses, Software/Source Code','','Human','Ear',1,0,'O','',1,18,'e2e518d3-fa44-44b6-a2a6-a675286a3846','https://doi.org/10.7937/TCIA.9YTJ-5Q73'),(99,'COVID-19-AR','COVID-19-AR','COVID-19-AR','covid_19_ar','<p>\n	<span>Radiology imaging&nbsp; is playing an increasingly vital role in the diagnosis of COVID-19 patients and determining therapeutic options, patient care management and new research directions.&nbsp;</span><span>We have published a collection of radiographic and CT imaging studies for patients who tested positive for COVID-19. Each patient is described by a limited set of clinical data correlates that includes demographics, comorbidities, selected lab data and key radiology findings.&nbsp;These data are cross-linked to SARS-COV-2 cDNA sequence data extracted from clinical isolates from the same population, uploaded to the Genbank repository.</span></p>\n\n<br /> <p>\n\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/tcia.2020.py71-5978\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">COVID-19-AR <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',105,'CR, CT, DX','COVID-19 (non-cancer)','10.7937/tcia.2020.py71-5978','Clinical, Genomics','','Human','Lung',1,0,'O','',1,24,'8acd5933-499d-43a2-8dc1-de9a9a34756e','https://doi.org/10.7937/tcia.2020.py71-5978'),(100,'NSCLC-Radiomics-Genomics','NSCLC-Radiomics-Genomics','NSCLC-Radiomics-Genomics','nsclc_radiomics_genomics','<p>This collection contains images from 89 non-small cell lung cancer (NSCLC) patients that were treated with surgery. For these patients pretreatment CT scans, gene expression, and clinical data are available. This dataset refers to the Lung3 dataset of the study published in Nature Communications.</p><br />\n<p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.L4FRET6Z\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">NSCLC Radiomics Genomics <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection. </p>\n','2024-03-08',NULL,'Public',89,'CT','Lung Cancer','10.7937/K9/TCIA.2015.L4FRET6Z','Clinical, Genomics','','Human','Lung',1,0,'O','',1,18,'f162aa14-8f48-4897-b607-c16fdb5051b4','https://doi.org/10.7937/K9/TCIA.2015.L4FRET6Z'),(101,'IvyGAP','IvyGAP','IvyGAP','ivygap','<p>\n	This data collection consists of MRI/CT scan data for brain tumor patients that form the cohort for the resource Ivy Glioblastoma Atlas Project (Ivy GAP). There are 390 studies for 39 patients that include pre-surgery, post-surgery and follow up scans.</p>\n<p>\n	The Ivy Glioblastoma Atlas Project (Ivy GAP) is a collaborative partnership between the Ben and Catherine Ivy Foundation, which generously provided the financial support, the Allen Institute for Brain Science, and the Ben and Catherine Ivy Center for Advanced Brain Tumor Treatment. The goal of the project is to provide online resources to scientists and physicians dedicated to the development of innovative treatments and diagnostics that will enhance the quality of life and survival of patients with brain cancer.</p>\n<p>\n	See the <a href=\"https://doi.org/10.7937/K9/TCIA.2016.XLwaN6nL\">Ivy GAP</a> wiki page for more information and supporting metadata.</p>\n','2022-04-21',NULL,'Limited',39,'MR','Glioblastoma','10.7937/K9/TCIA.2016.XLwaN6nL','Clinical, Genomics','','Human','Brain',1,0,'O','',1,18,'11de5f0f-51a6-4c56-bdd3-fc01559d97ab','https://doi.org/10.7937/K9/TCIA.2016.XLwaN6nL'),(102,'MRI-DIR','MRI-DIR','MRI-DIR','mri_dir','<div>\n	Two sets of images were created to evaluate deformable image registration accuracy. The first set contains CT, T1-, and T2-weighted images from a porcine phantom implanted with gold markers, immobilized, compressed in 4 different ways and images were acquired in each position.&nbsp;</p>\n<p>\n	The second set of images are synthetic images derived from 28 head and neck squamous cell carcinoma patients who had pre-, mid-, and post-radiotherapy treatment MR scans. From these patients, inter- and intra-patient models were created.</p>\n<p>\n	Please see the <a href=\"https://10.7937/K9/TCIA.2018.3f08iejt\">MRI-DIR</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection</p>\n','2022-04-21',NULL,'Limited',9,'CT, MR, RTSTRUCT','Squamous Cell Carcinoma','10.7937/K9/TCIA.2018.3f08iejt','Image Registrations','','Human','Head-Neck',1,0,'O','',1,18,'bda548ca-55d3-4951-adc2-8957ba7ccc20','https://doi.org/10.7937/K9/TCIA.2018.3f08iejt'),(103,'LungCT-Diagnosis','LungCT-Diagnosis','LungCT-Diagnosis','lungct_diagnosis','<p>\n	This is a small set of 61 diagnostic contrast enhanced CT scans with known tumors.&nbsp; The images were retrospectively acquired, to ensure sufficient patient follow-up. Slice thickness is variable : between 3 and 6 mm. All images were done at diagnosis and prior to surgery.<br />\n	&nbsp;</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.A6V7JIWX\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">LungCT-Diagnosis <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',61,'CT','Lung Cancer','10.7937/K9/TCIA.2015.A6V7JIWX','Clinical, Image Analyses','','Human','Lung',1,0,'O','',1,18,'70734e72-2288-48d0-b1e0-e708805c170b','https://doi.org/10.7937/K9/TCIA.2015.A6V7JIWX'),(104,'SPIE-AAPM Lung CT Challenge','SPIE-AAPM Lung CT Challenge','SPIE-AAPM Lung CT Challenge','spie_aapm_lung_ct_challenge','<p>\n	As part of the 2015 SPIE Medical Imaging Conference, SPIE &ndash; with the support of American Association of Physicists in Medicine (AAPM) and the National Cancer Institute (NCI) &ndash; conducted a &ldquo;Grand Challenge&rdquo; on quantitative image analysis methods for the diagnostic classification of malignant and benign lung nodules. The LUNGx Challenge provided a unique opportunity for participants to compare their algorithms to those of others from academia, industry, and government in a structured, direct way using the same data sets.</p><br />\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2015.UZLSU3FL\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">SPIE-AAPM Lung CT Challenge <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',70,'CT','Lung Cancer','10.7937/K9/TCIA.2015.UZLSU3FL','Clinical, Image Analyses','','Human','Lung',1,0,'O','',1,18,'d901aedc-faf6-433e-a618-ad8674ef8631','https://doi.org/10.7937/K9/TCIA.2015.UZLSU3FL'),(105,'Breast-MRI-NACT-Pilot','Breast-MRI-NACT-Pilot','Breast-MRI-NACT-Pilot','breast_mri_nact_pilot','<p>\n	<span>This pilot study to investigate the use of serial DCE MRI examinations during neoadjuvant chemotherapy (NACT) for invasive breast cancer recruited 68 patients with stage II or III locally advanced breast cancer. L<span>ongitudinal DCE MRI studies of 64 patients are available.</span></span></p>\n<p>\n	<span>Please see the wiki page <span><a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2016.QHSYHJKY\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Breast-MRI-NACT-Pilot <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n','2024-03-08',NULL,'Public',64,'MR, SEG, SR','Breast Cancer','10.7937/K9/TCIA.2016.QHsyhJKy','Clinical, Image Analyses','','Human','Breast',1,0,'O','',1,18,'3a7ec9f3-353d-4ddd-bc2e-af242739c79f','https://doi.org/10.7937/K9/TCIA.2016.QHsyhJKy'),(106,'QIN Breast DCE-MRI','QIN Breast DCE-MRI','QIN Breast DCE-MRI','qin_breast_dce_mri','<p>\n	This collection of breast dynamic contrast-enhanced (DCE) MRI data contains images from a longitudinal study to assess breast cancer response to neoadjuvant chemotherapy. Images were acquired at four time points: prior to the start of treatment (Visit 1, V1), after the first cycle of treatment (Visit 2, V2), at midpoint of treatment course (Visit 3, V3), and after completion of treatment (prior to surgery) (Visit 4, V4). The value of this collection is to provide clinical imaging data for the development and validation of quantitative imaging methods for assessment of breast cancer response to treatment.</p>\n<p>\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/k9/tcia.2014.a2n1ixox\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIN Breast DCE-MRI <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',10,'KO, MR','Breast Cancer','10.7937/k9/tcia.2014.a2n1ixox','Clinical, Image Analyses','','Human','Breast',1,0,'O','',1,2,'f0070dec-d624-46fe-a177-20b58eaae9d3','https://doi.org/10.7937/k9/tcia.2014.a2n1ixox'),(107,'BREAST-DIAGNOSIS','BREAST-DIAGNOSIS','BREAST-DIAGNOSIS','breast_diagnosis','<p>\n	The Breast-Diagnosis collection contains cases that are high-risk normals, DCIS, fibroids and lobular carcinomas. Each case has 3 or more distinct MR pulse sequences from a Phillips 1.5 T (usual sequences are labeled T2, STIR and BLISS but may occasionally include other pulse sequences and digital mammogram of tumor specimen).&nbsp; Multiple time point studies on the same patient are possible.&nbsp;</p>\n<p>\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2015.SDNRQXXR\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">BREAST-DIAGNOSIS <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',88,'CT, MG, MR, PT, SR','Breast Cancer','10.7937/K9/TCIA.2015.SDNRQXXR','Clinical, Image Analyses','','Human','Breast',1,0,'O','',1,18,'1361075d-d05d-47c7-9563-2d137773b8bf','https://doi.org/10.7937/K9/TCIA.2015.SDNRQXXR'),(108,'HNSCC','HNSCC','HNSCC','hnscc','<p>\n	This collection contains imaging, radiation therapy, and clinical data from 627 head and neck squamous cell carcinoma (HNSCC) patients&nbsp;at MD Anderson Cancer Center.&nbsp; Researchers at MDACC analyzed these patients&#39; data as part of two separate research projects.</p>\n<p>\n	&nbsp;Please see Grossberg et al: Data from Head and Neck Cancer CT Atlas (<a href=\"https://doi.org/10.7937/k9/tcia.2020.a8sh-7363\">HNSCC</a>) wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n	<br />\n	&nbsp;</p>\n','2022-04-21',NULL,'Limited',627,'CT, MR, PT, RTDOSE, RTPLAN, RTSTRUCT','Head and Neck Squamous Cell Carcinoma','10.7937/k9/tcia.2020.a8sh-7363','Clinical, Image Analyses','','Human','Head-Neck',1,0,'O','',1,18,'cb49bb92-ab4d-486b-af3d-af1d3b99cae8','https://doi.org/10.7937/k9/tcia.2020.a8sh-7363'),(109,'PROSTATE-DIAGNOSIS','PROSTATE-DIAGNOSIS','PROSTATE-DIAGNOSIS','prostate_diagnosis','<p>\n	Prostate, cancer-confirmed, MRIs obtained using endorectal and phased array surface coils at 1.5T (Philips Achieva). Pulse sequences include T1, T2 and DCE (&ldquo;BLISS&rdquo; sequence using Magnevist injected i.v.). Linked clinical information, including matched patient demographics, radiology and pathology reports in XLS format are available.</p>\n<p>\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2015.FOQEUJVT\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prostate-Diagnosis <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n	Data was provided by Boston Medical Center courtesy Dr. Nicolas Bloch.<br />\n	&nbsp;</p>\n','2024-03-08',NULL,'Public',92,'MR','Prostate Cancer','10.7937/K9/TCIA.2015.FOQEUJVT','Clinical, Image Analyses','','Human','Prostate',1,0,'O','',1,18,'52c26655-8f66-4485-ad33-be412023acfb','https://doi.org/10.7937/K9/TCIA.2015.FOQEUJVT'),(110,'LGG-1p19qDeletion','LGG-1p19qDeletion','LGG-1p19qDeletion','lgg_1p19qdeletion','<p>\n	These MRIs are pre-operative examinations performed in 159 subjects with Low Grade Gliomas (WHO grade II &amp; III). T1, T2, and Segmentations are provided in DICOM format here.&nbsp;</p>\n<p>\n	Please see the <a href=\"https://doi.org/10.7937/K9/TCIA.2017.dwehtz9v\">LGG-1p19qDeletion</a>&nbsp;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n	<o:p></o:p></p>\n','2022-04-21',NULL,'Limited',159,'MR, SEG','Low Grade Glioma','10.7937/K9/TCIA.2017.dwehtz9v','Image Analyses, Genomics','','Human','Brain',1,0,'O','',1,18,'f7c87160-33cf-4152-a72a-d7f6c07eb4dd','https://doi.org/10.7937/K9/TCIA.2017.dwehtz9v'),(111,'CPTAC-LSCC','CPTAC-LSCC','CPTAC-LSCC','cptac_lscc','<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Lung Squamous Cell Carcinoma (CPTAC-LSCC) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n	&nbsp;</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.6EMUB5L2\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-LSCC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n','2024-03-08',NULL,'Public',212,'CT, PT, SM','Squamous Cell Carcinoma','10.7937/K9/TCIA.2018.6EMUB5L2','Clinical, Genomics, Proteomics','','Human','Lung',1,0,'O','',1,11,'500cfb50-4c79-4024-b835-2234a439cd39','https://doi.org/10.7937/K9/TCIA.2018.6EMUB5L2'),(112,'CPTAC-GBM','CPTAC-GBM','CPTAC-GBM','cptac_gbm','\n<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Glioblastoma Multiforme (CPTAC-GBM) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n	&nbsp;</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.3RJE41Q1\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-GBM <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>','2024-03-08',NULL,'Public',178,'SM','Glioblastoma Multiforme','10.7937/K9/TCIA.2018.3RJE41Q1','Clinical, Genomics, Proteomics','','Human','Brain',1,0,'O','',1,11,'fe522a1e-9494-45da-8d4d-86a1c55edf53','https://doi.org/10.7937/K9/TCIA.2018.3RJE41Q1'),(113,'CPTAC-HNSCC','CPTAC-HNSCC','CPTAC-HNSCC','cptac_hnscc','<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Head-and-Neck cancer (CPTAC-HNSCC) cohort.</span><span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\"><span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\"> </span>CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\"><span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.UW45NH81\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-HNSCC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n','2024-03-08',NULL,'Public',112,'SM','Head and Neck Cancer','10.7937/K9/TCIA.2018.UW45NH81','Clinical, Genomics, Proteomics','','Human','Head-Neck',1,0,'O','',1,11,'d20cea38-e918-470b-8a6a-b0fc3cd9f7c4','https://doi.org/10.7937/K9/TCIA.2018.UW45NH81'),(114,'CPTAC-LUAD','CPTAC-LUAD','CPTAC-LUAD','cptac_luad','	<p>	This collection contains subjects from the National Cancer Institute&rsquo;s <a href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a> Lung Adenocarcinoma (CPTAC-LUAD) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</p>\n	<p>	Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.PAT12TBS\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-LUAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n\n','2024-03-08',NULL,'Public',244,'CT, MR, PT, SM','Adenocarcinoma','10.7937/K9/TCIA.2018.PAT12TBS','Clinical, Genomics, Proteomics','','Human','Lung',1,0,'O','',1,11,'6540c5f9-2502-4b14-8829-1852a87838c9','https://doi.org/10.7937/K9/TCIA.2018.PAT12TBS'),(115,'CPTAC-CCRCC','CPTAC-CCRCC','CPTAC-CCRCC','cptac_ccrcc','<p>\n	<span class=\"conf-macro output-inline\" data-hasbody=\"true\" data-macro-name=\"excerpt\">This collection contains subjects from the National Cancer Institute&rsquo;s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Clear Cell Renal Cell Carcinoma (CPTAC-CCRCC) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n	&nbsp;</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.OBLAMN27\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-CCRCC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n','2024-03-08',NULL,'Public',227,'CT, MR, RTSTRUCT, SM','Clear Cell Carcinoma','10.7937/K9/TCIA.2018.OBLAMN27','Clinical, Genomics, Proteomics','','Human','Kidney',1,0,'O','',1,11,'6f9f2f15-1e3d-4c8e-bfef-45afa99d25c5','https://doi.org/10.7937/K9/TCIA.2018.OBLAMN27'),(116,'CPTAC-UCEC','CPTAC-UCEC','CPTAC-UCEC','cptac_ucec','This collection contains subjects from the National Cancer Institute\'\'s <u><a class=\"external-link\" href=\"https://proteomics.cancer.gov/programs/cptac\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Uterine Corpus Endometrial Carcinoma (CPTAC-UCEC) cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC Phase 3 patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</p><p>\nPlease see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.3R3JUISW\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-UCEC <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',250,'CT, MR, PT, RTSTRUCT, SM, US','Corpus Endometrial Carcinoma','10.7937/K9/TCIA.2018.3R3JUISW','Clinical, Genomics, Proteomics','','Human','Uterus',1,0,'O','',1,11,'33680334-8dcb-4495-9022-fccec60ec9c0','https://doi.org/10.7937/K9/TCIA.2018.3R3JUISW'),(117,'REMBRANDT','REMBRANDT','REMBRANDT','rembrandt','<p>\n	This image collection was created as a companion data set to augment the larger <a class=\"external-link\" href=\"http://caintegrator-info.nci.nih.gov/rembrandt\" rel=\"nofollow\">REMBRANDT</a> project. It consists of 130 cases of MR multi-sequence images of pre-surgical brain tumor patients including cases of glioblastoma, astrocytomas and oligodendrogliomas from the REMBRANDT patient population.</p>\n<p>\n	Please see the <a href=\"https://doi.org/10.7937/K9/TCIA.2015.588OZUZB\" target=\"_blank\">REMBRANDT</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2022-04-21',NULL,'Limited',130,'MR','Low & High Grade Glioma','10.7937/K9/TCIA.2015.588OZUZB','Clinical, Genomics, Image Analyses','','Human','Brain',1,0,'O','',1,18,'05009b88-a8e6-4251-ba6c-0b8355b61a28','https://doi.org/10.7937/K9/TCIA.2015.588OZUZB'),(118,'NSCLC Radiogenomics','NSCLC Radiogenomics','NSCLC Radiogenomics','nsclc_radiogenomics','<p>\n	This collection contains images from patients with non-small cell lung cancer imaged prior to surgical excision with both thin-section CT and whole body PET/CT scans acquired under IRB approval from Stanford University and the Veterans Administration Palo Alto Health Care System.</p>\n<p>\n	The first installment of 26 cases (see shared list &quot;NSCLC Radiogenomics: Initial Stanford Study of 26 Cases&quot;) corresponds to microarray data acquired from the excised samples, which is available on the NCBI Gene Expression Omnibus (<a class=\"external-link\" href=\"http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE28827\" rel=\"nofollow\">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE28827</a>), where DICOM patient names are identical to microarray sample names.</p>\n<p>\n	<span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id487\">Please see the <a href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2017.7hs46erv\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">NSCLC Radiogenomics <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n','2024-03-08',NULL,'Public',211,'CT, PT, SEG','Non-small Cell Lung Cancer','10.7937/K9/TCIA.2017.7hs46erv','Clinical, Image Analyses, Genomics','','Human','Chest',1,0,'O','',1,18,'6525e140-0773-46c9-bc34-d88922a7397c','https://doi.org/10.7937/K9/TCIA.2017.7hs46erv'),(119,'Head-Neck-PET-CT','Head-Neck-PET-CT','Head-Neck-PET-CT','head_neck_pet_ct','<p>\n	<span>This collection contains FDG-PET/CT and radiotherapy planning CT imaging data of&nbsp;<span>298</span><span>&nbsp;</span>patients from four different institutions in Qu&eacute;bec with histologically proven head-and-neck cancer (H&amp;N) All patients had pre-treatment FDG-PET/CT scans between April 2006 and November 2014, and within a median of 18 days (range: 6-66) before treatment. </span></p>\n<p>\n	<span>Please see the <a href=\"https://doi.org/10.7937/K9/TCIA.2017.8oje5q00\">Head-Neck-PET-CT</a> wiki page <span class=\"iceOutTxt\" id=\"MAINbody:collectionDescriptionPopupForm:j_id716\">to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n','2022-04-21',NULL,'Limited',298,'CT, PT, REG, RTDOSE, RTPLAN, RTSTRUCT','Head and Neck Cancer','10.7937/K9/TCIA.2017.8oje5q00','Clinical, Image Analyses, Software/Source Code','','Human','Head-Neck',1,0,'O','',1,18,'b5c83a06-c293-4d86-a204-5e01b7b62cda','https://doi.org/10.7937/K9/TCIA.2017.8oje5q00'),(120,'Soft-tissue-Sarcoma','Soft-tissue-Sarcoma','Soft-tissue-Sarcoma','soft_tissue_sarcoma','<p>\n	In this study, a cohort of 51 patients with histologically proven soft-tissue sarcomas (STSs) of the extremities was retrospectively evaluated. During the follow-up period, 19 patients developed lung metastases. Patients with metastatic and/or recurrent STSs at presentation were excluded from the study. All patients had pre-treatment FDG-PET/CT and MRI scans.</p>\n<p>\n	Please see the <a href=\"\" url=\"http://doi.org/10.7937/K9/TCIA.2015.7GO2GSKS\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Soft Tissue Sarcoma <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',51,'CT, MR, PT, RTSTRUCT','Soft-tissue Sarcoma','10.7937/K9/TCIA.2015.7GO2GSKS','Clinical, Image Analyses, Software/Source Code','','Human','Extremities',1,0,'O','',1,18,'1bf83011-6e12-470a-8db9-0e1123d41076','https://doi.org/10.7937/K9/TCIA.2015.7GO2GSKS'),(121,'NLST','NLST','NLST','nlst','<p>This collection contains all CT exams from the National Lung Screening Trial (NLST), which  was conducted to determine whether screening with low-dose CT could reduce mortality from lung cancer for persons at high risk for lung cancer at 33 U.S. medical centers.  Participants were randomly assigned to undergo three annual screenings between 2002-2004.&#160;</p><p><i>NOTE: The full dataset occupies over 11 Terabytes on disk. Ensure you have what you need before downloading.</i></p><br><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.HMQ8-J677\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">NLST <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',26408,'CT, SEG, SM, SR','Lung Cancer','10.7937/TCIA.HMQ8-J677','Clinical','','Human','Chest',1,0,'O','',1,19,'d0b6918b-71ad-41b9-8da3-ceb005ce1a40','https://doi.org/10.7937/TCIA.HMQ8-J677'),(128,'Outcome Prediction in Patients with Glioblastoma by Using Imaging, Clinical, and Genomic Biomarkers: Focus on the Nonenhancing Component of the Tumor','','','GBM-MR-NER-Outcomes','','2022-03-02',NULL,'Limited',45,'','Glioblastoma','10.7937/K9/TCIA.2014.FAB7YRPZ','','Radiologist assessments of image features and hemodynamic parameters','','Brain',0,0,'A','tcga_gbm',1,NULL,'3e743048-d76e-425e-8d0a-75ff17752f29',NULL),(129,'DICOM-SEG Conversions for TCGA-LGG and TCGA-GBM Segmentation Datasets','','','DICOM-Glioma-SEG','<p> This dataset contains DICOM-SEG (DSO) conversions of the Segmentation Labels and Radiomic Features for the Pre-operative Scans of the TCGA-LGG collection and Segmentation Labels and Radiomic Features for the Pre-operative Scans of the TCGA-GBM collection  analysis datasets.</p> <p>  Please see the <a href=https://doi.org/10.7937/TCIA.2018.ow6ce3ml target=_blank>DICOM-Glioma-SEG</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2022-04-21',NULL,'Limited',167,'','Glioblastoma, Low Grade Glioma','10.7937/TCIA.2018.ow6ce3ml','','Tumor segmentations','','Brain',1,0,'A','tcga_lgg,tcga_gbm',1,NULL,'034a127d-ee85-49c3-a007-3eb0d6672009',''),(130,'LDCT-and-Projection-data','LDCT-and-Projection-data','LDCT-and-Projection-data','ldct_and_projection_data','<p>&#10;Investigators at the Mayo Clinic, with funding from the National Institute of Biomedical Imaging and Bioengineering (EB 017095 and EB 017185), have built a library of CT patient projection data in an open and vendor-neutral format. This format, referred to as DICOM-CT-PD (1), is an extended DICOM format that contains CT projection data and acquisition geometry. The de-identified patient projection data in the library were decoded with help of the manufacturer and have been converted into an open standardized format.</p></br><p> Please see the <a href=\"https://doi.org/10.7937/9npb-2637\">LDCT-and-Projection</a> wiki for further details.</p>','2024-03-08',NULL,'Excluded',151,'CT','Various','10.7937/9npb-2637','Clinical','','Human','Head, Chest, Abdomen',0,0,'O','',1,18,'299a8ebf-1b7f-4d68-852c-9b5544e02113','https://doi.org/10.7937/9npb-2637'),(131,'QIBA-CT-Liver-Phantom','QIBA-CT-Liver-Phantom','QIBA-CT-Liver-Phantom','qiba_ct_liver_phantom','<p>\n	<span><span>This database contains a collection of three sets of CT scan images acquired from an anthropomorphic abdominal phantom with removable liver inserts.&nbsp;</span></span></p>\n<p>\n	<span>Please see the</span>&nbsp;<span>wiki page&nbsp;<span><a href=\"https://doi.org/10.7937/TCIA.RMV0-9Y95\">QIBA Anthropomorphic Abdominal Phantom CT Scans</a>&nbsp;</span></span><span>to learn more about the images and to obtain any supporting metadata for this collection.</span></p>\n','2024-03-08',NULL,'Excluded',3,'CT','Phantom','10.7937/tcia.rmv0-9y95','Image Analyses','','Human','Liver Phantom',0,0,'O','',1,25,'85e6aed0-b0db-49d7-949a-c139435d7129','https://doi.org/10.7937/tcia.rmv0-9y95'),(132,'ACRIN-HNSCC-FDG-PET-CT','ACRIN-HNSCC-FDG-PET/CT (ACRIN 6685)','ACRIN-HNSCC-FDG-PET/CT (ACRIN 6685)','acrin_hnscc_fdg_pet_ct','<p>\n	<span lang=\"EN\">A Multicenter Trial of FDG-PET/CT Staging of Head and Neck Cancer&nbsp;</span><span lang=\"EN\">and its Impact on the N0 Neck Surgical Treatment in Head and Neck Cancer Patients.&nbsp;</span><span>The objective of this study was to determine the negative predictive value (NPV)&nbsp;of positron emission tomography (PET)/computed tomography CT for clinically N0 neck based upon pathologic sampling of the neck lymph nodes and to&nbsp;determine PET/CT&rsquo;s potential to change treatment of the N0 neck.&nbsp;</span></p>\n','2022-04-21',NULL,'Limited',260,'CT, MR, NM, PT','Head and Neck Squamous Cell Carcinoma','10.7937/K9/TCIA.2016.JQEJZZNG','Clinical','','Human','Head-Neck',1,0,'O','',1,19,'3797d354-3f30-48c7-9920-90bc4969f474','https://doi.org/10.7937/K9/TCIA.2016.JQEJZZNG'),(133,'B-mode-and-CEUS-Liver','B-mode-and-CEUS-Liver','B-mode-and-CEUS-Liver','b_mode_and_ceus_liver','<p>This Collection contains <u><strong>c</strong></u>ontrast-<u><strong>e</strong></u>nhanced <u><strong>u</strong></u>ltra<u><strong>s</strong></u>ound to a) characterize liver lesions and b) monitor treatment response. Sagittal and transverse plane ultrasound data was obtained with curvilinear probe.</p><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.2021.v4z7-tc39\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">B-mode-and-CEUS-Liver <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',120,'US','Liver Cancer','10.7937/TCIA.2021.v4z7-tc39','Clinical','','Human','Liver',1,0,'O','',1,18,'231992f6-55a1-4cd7-8ec8-36538b1f59f5','https://doi.org/10.7937/TCIA.2021.v4z7-tc39'),(134,'CMMD','CMMD','CMMD','cmmd','<div><span>This collection contains clinical data and 3,728 mammographies (MG) from 1,775 patients, with biopsy-confirmed benign or malignant breast tumors. For a subset of 749 patients (D2-####), molecular subtype is included.&#160;</span></p><div><span>Please see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/tcia.eqde-4b16\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CMMD&#160; <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>wiki page for further detail and associated metadata.</span></p>','2024-03-08',NULL,'Public',1775,'MG','Breast Cancer','10.7937/tcia.eqde-4b16','Clinical','','Human','Breast',1,0,'O','',1,18,'43519397-01bb-4cf1-8aad-77250ee6d14a','https://doi.org/10.7937/tcia.eqde-4b16'),(135,'COVID-19-NY-SBU','COVID-19-NY-SBU','COVID-19-NY-SBU','covid_19_ny_sbu','<p>&#10;&#9;The collection of cases was acquired at Stony Brook University from patients who tested positive for COVID-19. The collection includes images from different modalities and organ sites (chest radiographs, chest CTs, brain MRIs, etc.). Radiology imaging data is extremely important in COVID-19 from both a diagnostic and a monitoring perspective, given the crucial nature of COVID-19 pulmonary disease and its rapid phenotypic changes.</p><p><br></p>&#10;<p>&#10;&#9;Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.BBAG-2923\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Stony Brook University COVID-19 Positive Cases <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>&#10;','2024-03-08',NULL,'Public',1384,'CR, CT, DX, MR, NM, OT, PT, SR','COVID-19 (non-cancer)','10.7937/TCIA.BBAG-2923','Clinical, Image Analyses','','Human','Lung',1,0,'O','',1,24,'886c9ffa-a567-4a01-8953-6322e5525c2c','https://doi.org/10.7937/TCIA.BBAG-2923'),(136,'Duke-Breast-Cancer-MRI','Duke-Breast-Cancer-MRI','Duke-Breast-Cancer-MRI','duke_breast_cancer_mri','<p>\n	<strong>Dynamic contrast-enhanced magnetic resonance images of breast cancer patients with tumor locations (Duke-Breast-Cancer-MRI)</strong><br />\n	<br />\n	This collection contains MRI imaging and other data for 922 patients with invasive breast cancer. Prone position axial breast MRI images were acquired by 1.5T or 3T scanners;&nbsp; non-fat saturated T1, fat-saturated gradient echo T1 pre-contrast, and 3-4 post-contrast sequences are included.<br />\n	<br />\n	Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.e3sv-re93\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Duke-Breast-Cancer-MRI <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain the supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',922,'MR, SEG','Breast Cancer','10.7937/TCIA.e3sv-re93','Clinical, Genomics, Image Analyses','','Human','Breast',1,0,'O','',1,18,'7d9bb85c-b8e0-449c-996d-8b0d2385792f','https://doi.org/10.7937/TCIA.e3sv-re93'),(137,'APOLLO-5-ESCA','APOLLO-5-ESCA','APOLLO-5-ESCA','apollo_5_esca','<p class=\"MsoNormal\">This collection contains subjects from the National Cancer&#10;Institute&#8217;s Applied Proteogenomics OrganizationaL Learning and Outcomes&#10;(APOLLO) network&#160;<span>Esophageal Squamous Cell Carcinoma cohort.&#160;</span><span>The APOLLO network is a collaboration between&#10;NCI, the Department of Defense (DoD), and the Department of Veterans Affairs&#10;(VA) to incorporate proteogenomics into patient care as a way of looking beyond&#10;the genome, to the activity and expression of the proteins that the genome&#10;encodes.</span></p><p class=\"MsoNormal\"></p><div><span><font face=\"-apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji\">P</font>lease see the&#160;</span><a href=\"https://wiki.cancerimagingarchive.net/x/uAC_BQ\" target=\"_blank\">APOLLO-5-ESCA</a><span>&#160;wiki page to learn&#10;more about the images and to obtain any supporting metadata for this&#10;collection.&#160;</span><br></p><div><div><p class=\"MsoListParagraph\"></p><div><span><br></span></p></p></p>','2022-04-21',NULL,'Public',2,'CT','Esophageal Squamous Cell Carcinoma','10.7937/n69a-7a26','','','Human','Esophagus',0,0,'O','',1,6,'4cbdc12f-1386-4209-b128-f1e93e6513b4','https://doi.org/10.7937/n69a-7a26'),(138,'APOLLO-5-LUAD','APOLLO-5-LUAD','APOLLO-5-LUAD','apollo_5_luad','<p class=\"MsoNormal\">This collection contains subjects from the National Cancer&#10;Institute&#8217;s Applied Proteogenomics OrganizationaL Learning and Outcomes&#10;(APOLLO) network Lung Adenocarcinoma&#160;<span>cohort</span>. The APOLLO network is a collaboration between NCI,&#10;the Department of Defense (DoD), and the Department of Veterans Affairs (VA) to&#10;incorporate proteogenomics into patient care as a way of looking beyond the&#10;genome, to the activity and expression of the proteins that the genome encodes.</p><p class=\"MsoNormal\"><span>Please see the&#160;</span><a href=\"https://wiki.cancerimagingarchive.net/x/CgC_BQ\" target=\"_blank\">APOLLO-5-LUAD</a><span>&#160;wiki page to learn more about the&#10;images and to obtain any supporting metadata for this collection.&#160;</span><br></p><p class=\"MsoNormal\"></p><p class=\"MsoNormal\"><br></p><p class=\"MsoNormal\"><br></p>','2022-04-21',NULL,'Public',1,'CT','Lung Adenocarcinoma','10.7937/BDM9-4623','','','Human','Lung',0,0,'O','',1,6,'30685a1b-c03b-4d7b-9708-c61550433683','https://doi.org/10.7937/BDM9-4623'),(139,'APOLLO-5-PAAD','APOLLO-5-PAAD','APOLLO-5-PAAD','apollo_5_paad','<p class=\"MsoNormal\">This collection contains subjects from the National Cancer&#10;Institute&#8217;s Applied Proteogenomics OrganizationaL Learning and Outcomes&#10;(APOLLO) network&#160;<span><span>Pancreatic Adenocarcinoma</span></span><span>&#160;cohort.&#160;</span>&#160;The APOLLO network is a collaboration between&#10;NCI, the Department of Defense (DoD), and the Department of Veterans Affairs&#10;(VA) to incorporate proteogenomics into patient care as a way of looking beyond&#10;the genome, to the activity and expression of the proteins that the genome&#10;encodes.</p><p class=\"MsoNormal\">Please see the&#160;<a href=\"https://wiki.cancerimagingarchive.net/x/xgC_BQ\" target=\"_blank\">APOLLO-5-PAAD</a>&#160;<span>wiki page to learn more about the&#10;images and to obtain any supporting metadata for this collection.</span></p><p class=\"MsoNormal\"></p>','2022-04-21',NULL,'Public',1,'CT','Pancreatic Adenocarcinoma','10.7937/tcia.1yeg-5740','','','Human','Pancreas ',0,0,'O','',1,6,'2623aa2a-9e38-4c1b-9e0f-97445d9d35cc','https://doi.org/10.7937/tcia.1yeg-5740'),(140,'APOLLO-5-THYM','APOLLO-5-THYM','APOLLO-5-THYM','apollo_5_thym','<p class=\"MsoNormal\"><span>This collection contains&#10;subjects from the&#160;</span><span>National Cancer Institute&#8217;s Applied&#10;Proteogenomics OrganizationaL Learning and Outcomes (APOLLO) network</span><span>&#160;</span><span>Thymoma&#160;cohort.&#160;The APOLLO network is&#10;a collaboration between&#160;</span><span>NCI</span><span>, the Department of Defense (DoD), and the&#10;Department of Veterans Affairs (VA) to incorporate proteogenomics into patient&#10;care as a way of looking beyond the genome, to the activity and expression of&#10;the proteins that the genome encodes.&#160;</span><b></b></p><div><div><p class=\"MsoListParagraph\">Please see the&#160;<a href=\"https://wiki.cancerimagingarchive.net/x/DYYTBg\" target=\"_blank\">APOLLO-5-THYM</a>&#160;wiki page to learn&#10;more about the images and to obtain any supporting metadata for this&#10;collection.&#160;</p></p></p>','2022-04-21',NULL,'Public',1,'CT','Thymoma Cancer','10.7937/tcia.0pg4-nh82','','','Human','Thymus',0,0,'O','',1,6,'f9b426ca-e3ac-4342-8894-dbbfe2746324','https://doi.org/10.7937/tcia.0pg4-nh82'),(141,'CPTAC-AML','CPTAC-AML','CPTAC-AML','cptac_aml','\n<p>\n    <span>This collection contains subjects from the National Cancer Institute&rsquo;s <u><a href=\"https://proteomics.cancer.gov/programs/cptac\" class=\"external-link\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> Acute Myeloid Leukemia (CPTAC-AML) cohort.&nbsp;<span>CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics.</span></p>\n<p>\n    Please see the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-AML\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-AML <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',89,'SM','Acute Myeloid Leukemia','10.7937/TCIA.2019.B6FOE619','Clinical','','Human','Marrow, Blood',1,0,'O','',1,11,'87b7f6cf-6ef9-4bf3-bd90-ceae007d41e5','https://doi.org/10.7937/TCIA.2019.B6FOE619'),(142,'CPTAC-BRCA','CPTAC-BRCA','CPTAC-BRCA','cptac_brca','\n<p>\n    <span>This collection contains subjects from the National Cancer Institute&rsquo;s <u><a href=\"https://proteomics.cancer.gov/programs/cptac\" class=\"external-link\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> CPTAC Breast Invasive Carcinoma cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics. Radiology and pathology images from CPTAC patients are being collected and made publicly available by The Cancer Imaging Archive to enable researchers to investigate cancer phenotypes which may correlate to corresponding proteomic, genomic and clinical data.</span></p>\n<p>\n    Please see the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-BRCA\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-BRCA <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',198,'SM','Breast Cancer','10.7937/TCIA.CAEM-YS80','Clinical, Genomics, Proteomics','','Human','Breast',1,0,'O','',1,11,'e4f7e425-084c-41d8-9378-1fde90d5c58f','https://doi.org/10.7937/TCIA.CAEM-YS80'),(143,'CPTAC-COAD','CPTAC-COAD','CPTAC-COAD','cptac_coad','\n<p>\n    <span>This collection contains subjects from the National Cancer Institute&rsquo;s <u><a href=\"https://proteomics.cancer.gov/programs/cptac\" class=\"external-link\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> CPTAC&nbsp;Colon Adenocarcinoma cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics.</span></p>\n<p>\n    Please see the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-COAD\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-COAD <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',178,'SM','Colon Cancer','10.7937/TCIA.YZWQ-ZZ63','Clinical, Genomics, Proteomics','','Human','Colon',1,0,'O','',1,11,'d320916f-1b37-409a-8be3-2072afce7037','https://doi.org/10.7937/TCIA.YZWQ-ZZ63'),(144,'CPTAC-OV','CPTAC-OV','CPTAC-OV','cptac_ov','\n<p>\n    <span>This collection contains subjects from the National Cancer Institute&rsquo;s <u><a href=\"https://proteomics.cancer.gov/programs/cptac\" class=\"external-link\" rel=\"nofollow\">Clinical Proteomic Tumor Analysis Consortium</a></u> CPTAC&nbsp;Ovarian Serous Cystadenocarcinoma cohort. CPTAC is a national effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome and genome analysis, or proteogenomics.</span></p>\n<p>\n    Please see the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/display/Public/CPTAC-OV\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CPTAC-OV <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',73,'SM','Ovarian Cancer','10.7937/TCIA.ZS4A-JD58','Clinical, Genomics, Proteomics','','Human','Ovary',1,0,'O','',1,11,'49bf9eb5-ada6-4b13-b5be-5000156949fb','https://doi.org/10.7937/TCIA.ZS4A-JD58'),(145,'Pancreatic-CT-CBCT-SEG','Pancreatic-CT-CBCT-SEG','Pancreatic-CT-CBCT-SEG','pancreatic_ct_cbct_seg','<span>The objective of this dataset is to provide a means of evaluating the performance of CT-to-CBCT deformable registration and auto-segmentation algorithms for delineating OARs from 40 patients who received ablative radiation therapy for locally advanced pancreatic cancer.</span><span>The images (one treatment planning CT and two CBCT scans for each of 40 patients) were acquired during a deep inspiration breath-hold verified with an external respiratory monitor.</span><br><span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/tcia.eshq-4d90\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Pancreatic-CT-CBCT-SEG <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span>','2024-03-08',NULL,'Public',40,'CT, RTDOSE, RTSTRUCT','Pancreatic Cancer','10.7937/TCIA.ESHQ-4D90','Image Analyses','','Human','Pancreas',1,0,'O','',1,18,'f810105e-82fa-4be5-bdbc-9778cb982c88','https://doi.org/10.7937/TCIA.ESHQ-4D90'),(146,'Pediatric-CT-SEG','Pediatric-CT-SEG','Pediatric-CT-SEG','pediatric_ct_seg','<p>\n	<span>This dataset was collected by a collaboration of researchers from Children&rsquo;s Wisconsin, Marquette University, Varian Medical Systems, Medical College of Wisconsin, and Stanford University as part of a project funded by the National Institute of Biomedical Imaging and Bioengineering (U01EB023822) to develop tools for rapid, patient-specific CT organ dose estimation. The collection consists of CT images in DICOM format of 359 pediatric chest-abdomen-pelvis or abdomen-pelvis exams acquired from three CT scanners.&nbsp;The datasets represent random pediatric cases based upon routine clinical indications. Each dataset contains expert contours of&nbsp;up to&nbsp;twenty-nine structures in DICOM RTSS format.&nbsp;</span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.X0H0-1706\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Pediatric-CT-SEG <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',359,'CT, RTSTRUCT','Non-Cancer','10.7937/TCIA.X0H0-1706','','','Human','Various',1,0,'O','',1,18,'9da5ed25-738a-4c9a-bd0b-c681d50e63d1','https://doi.org/10.7937/TCIA.X0H0-1706'),(147,'TCGA-ACC','TCGA-ACC','TCGA-ACC','tcga_acc','<p>\n      Adrenocortical carcinoma is a rare cancer that develops in the outer layer of tissue of the adrenal glands, organs that lie on top of each kidney. This outer layer known as the adrenal cortex produces important hormones called steroids that help the body deal with stress, regulate blood pressure and the amount of salt in the blood, as well as cause the body to acquire masculine or feminine characteristics. A tumor of the adrenal cortex can produce either no hormones or excess hormones.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/adrenocortical\" target=\"_blank\">TCGA-ACC</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>','2024-03-08',NULL,'Public',92,'SM','Adrenocortical Carcinoma','','','','Human','Adrenal Glands',1,0,'O','',1,1,'31653e7a-c448-48f1-9608-dabe8dc61ada','https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/adrenocortical'),(148,'TCGA-CHOL','TCGA-CHOL','TCGA-CHOL','tcga_chol','<p>\n      Cholangiocarcinoma is a cancer that develops in the bile duct. The bile duct is a network of tubes that carry bile from the liver and gallbladder to the small intestine. Tumors that start in bile duct branches that lie inside the liver are called intrahepatic bile duct cancer, while those that form outside the liver are called extrahepatic bile duct cancer. About 10% of all cholangiocarcinoma are intrahepatic and 90% are extrahepatic. TCGA studied both subtypes of cholangiocarcinoma.</p>\n<p>\n      Although cholangiocarcinoma is a rare cancer, the incidence and mortality rates for the disease have been increasing worldwide in the last three decades. Between 2,000 and 3,000 Americans are diagnosed with cholangiocarcinoma each year, the majority of them with tumors at advanced stages. This cancer is more prevalent in Asia and the Middle East, where parasitic infection of the bile duct increases the risk of cholangiocarcinoma. Other diseases of the bile duct or liver, such as bile duct stones and liver disease, obesity, diabetes, and smoking are also risk factors. When intrahepatic and extrahepatic cholangiocarcinoma spread to other parts of the body, only 2% of patients survive five years after diagnosis.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/cholangiocarcinoma\" target=\"_blank\">TCGA-CHOL</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>','2024-03-08',NULL,'Public',51,'SM','Cholangiocarcinoma ','','','','Human','Bile Duct',1,0,'O','',1,1,'aa0b3958-38cf-4a33-b586-f87dab8bc369','https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/cholangiocarcinoma'),(149,'TCGA-DLBC','TCGA-DLBC','TCGA-DLBC','tcga_dlbc','<p>\n       There is no available description page for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>','2024-03-08',NULL,'Public',48,'SM','Lymphoid Neoplasm Diffuse Large B-cell Lymphoma','','','','Human','Various',1,0,'O','',1,1,'48129a9f-f35c-4fc3-bbe6-3cc1c1f6e679','TCGA-DLBC has no description page'),(150,'TCGA-MESO','TCGA-MESO','TCGA-MESO','tcga_meso','<p>\n      Mesothelioma is a rare cancer that affects the thin layer of tissue that lines the chest, abdominal cavities, and most of the organs within them. This lining is called the mesothelium.\nIn the United States, there are roughly 3,000 new cases diagnosed each year. Exposure to asbestos is the main risk factor for developing this disease, and men tend to be more commonly affected— a fact that most likely correlates with men holding jobs where they are more likely to come in contact with it.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/mesothelioma\" target=\"_blank\">TCGA-MESO</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',87,'SM','Mesothelioma','','','','Human','Mesothelium',1,0,'O','',1,1,'c59cc9a6-d82b-4005-a7cd-b4dd14103177','https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/mesothelioma'),(151,'TCGA-PAAD','TCGA-PAAD','TCGA-PAAD','tcga_paad','<p>\n      Pancreatic ductal adenocarcinoma is the most common form of pancreatic cancer, making up more than 80% of cases. The disease begins in the cells of the pancreas’s ducts, which transport juices containing digestive enzymes into the small intestine.</p>\n<p>\n      Pancreatic cancer is the fourth most common cause of global cancer-related deaths and is almost always fatal. In 2012, it was estimated that around 44,000 new cases of pancreatic cancer were diagnosed and more than 37,000 deaths from this disease occurred in the United States alone, affecting both men and women.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/pancreatic\" target=\"_blank\">TCGA-PAAD</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>','2024-03-08',NULL,'Public',185,'SM','Pancreatic ductal adenocarcinoma','','','','Human','Pancreas',1,0,'O','',1,1,'4bacf1ec-e314-438d-a8cb-b3c17c927588','https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/pancreatic'),(152,'TCGA-PCPG','TCGA-PCPG','TCGA-PCPG','tcga_pcpg','<p>\n      Paraganglioma is a rare cancer that originates in the nerve cells of the adrenal glands, organs on top of each kidney that produce important hormones. Paraganglioma that develops in the center of the adrenal gland is called pheochromocytoma. Paraganglioma that forms outside of the adrenal gland, often along blood vessels and nerves in the head and neck, is called extra-adrenal paraganglioma, or simply paraganglioma.\nEach year, between 2 and 8 people per million worldwide are diagnosed with paraganglioma and pheochromocytoma. 10% of all cases occur in children. In both adults and children, pheochromocytoma is more common than paraganglioma. No known environmental, dietary, or lifestyle risk factors have been associated with these cancers.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/paraganglioma\" target=\"_blank\">TCGA-PCPG</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>','2024-03-08',NULL,'Public',179,'SM','Paraganglioma, Pheochromocytoma','','','','Human','Adrenal Glands',1,0,'O','',1,1,'49a6f4fd-5627-48e8-bcf3-cf00ac47c0e8','https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/paraganglioma'),(153,'TCGA-SKCM','TCGA-SKCM','TCGA-SKCM','tcga_skcm','<p>\n      Melanoma is a cancer in a type of skin cells called melanocytes. Melanocyes are the cells that produce melanin, which colors the skin. When exposed to sun, these cells make more melanin, causing the skin to darken or tan. Melanoma can occur anywhere on the body and risk factors include fair complexion, family history of melanoma, and being exposed to natural or artificial sunlight over long periods of time. Melanoma is most often discovered because it has metastasized, or spread, to another organ, such as the lymph nodes. In many cases, the primary skin melanoma site is never found. Because of this challenge, TCGA is studying primarily metastatic cases (in contrast to other cancers selected for study, where metastatic cases are excluded). For 2011, it was estimated that there were 70,230 new cases of melanoma and 8,790 deaths from the disease.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/pancreatic\" target=\"_blank\">TCGA-SKCM</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>','2024-03-08',NULL,'Public',470,'SM','Cutaneous Melanoma','','','','Human','Skin',1,0,'O','',1,1,'7b667349-9db0-4eb8-b8ae-fa407a458796','https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/melanoma-skin'),(154,'TCGA-TGCT','TCGA-TGCT','TCGA-TGCT','tcga_tgct','</p>\n      More than 90% of testicular cancer start in the germ cells, which are cells in the testicles and develop into sperm. This type of cancer is known as testicular germ cell cancer. Testicular germ cell cancer can be classified as either seminomas or nonseminomas, which may be identified by microscopy. Nonseminomas typically grow and spread more quickly than seminomas. A testicular germ cell tumor that contains a mix of both these subtypes is classified as a nonseminoma. TCGA studied both seminomas and nonseminomas.</p>\n<p>\n      Testicular germ cell cancer is rare, comprising 1-2% of all tumors in males. However, it is the most common cancer in men ages 15 to 35. The incidence of testicular germ cell cancer has been continuously rising in many countries, including Europe and the U.S. In 2013, about 8,000 American men were estimated to be diagnosed with the cancer. Of those, 370 are predicted to die from the disease. Men who are Caucasian, have an undescended testicle, abnormally developed testicles, or a family history of testicular cancer have a greater risk of developing testicular cancer. Fortunately, testicular germ cell cancer is highly treatable.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/testicular-germ-cell\" target=\"_blank\">TCGA-TGCT</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>','2024-03-08',NULL,'Public',150,'SM','Testicular Germ Cell','','','','Human','Testicles',1,0,'O','',1,1,'a10c1548-464e-4d39-8de2-e697a25f15f1','https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/testicular-germ-cell'),(155,'TCGA-THYM','TCGA-THYM','TCGA-THYM','tcga_thym','<p>\n      This cancer develops in the outer surface of the thymus, a gland behind the breastbone that produces T-cells, a type of white blood cells. Thymoma is rare, but it is the most common tumor in adults affecting the mediastinum, which is the cavity between the lungs containing the heart, esophagus, and trachea. A tumor of the thymus tends to grow slowly and rarely spreads to other parts of the body. However, of the estimated 400 Americans who develop this cancer each year, half are diagnosed with metastatic thymoma. When the cancer metastasizes, only 45% of patients survive five years after their diagnosis.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/thymoma\" target=\"_blank\">TCGA-THYM</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>','2024-03-08',NULL,'Public',124,'SM','Thymoma','','','','Human','Thymus',1,0,'O','',1,1,'ee5caf78-da94-4b5f-aa94-a688a905f713','https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/thymoma'),(156,'TCGA-UCS','TCGA-UCS','TCGA-UCS','tcga_ucs','<p>\n      Uterine carcinosarcoma (UCS) is a cancer that develops in the uterus. Carcinosarcoma signifies that, when looked at under a microscope, the tumor displays histological features of both endometrial carcinoma and sarcoma. Endometrial carcinoma starts in the endometrium, the inner layer of tissue lining the uterus, while sarcoma begins in the outer layer of muscle of the uterus.</p\n<p>\n      A rare cancer, UCS makes up less than 5% of all uterine cancers. In the U.S., about two per 100,000 women develop UCS annually. Roughly only 35% of patients survive five years after diagnosis. </p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/uterine-carcinosarcoma\" target=\"_blank\">TCGA-UCS</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>','2024-03-08',NULL,'Public',57,'SM','Uterine Carcinosarcoma','','','','Human','Uterus',1,0,'O','',1,1,'91b0e019-41f8-4907-823c-e9d9a4b01660','https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/uterine-carcinosarcoma'),(157,'TCGA-UVM','TCGA-UVM','TCGA-UVM','tcga_uvm','<p>\n      Uveal (intraocular or eye) melanoma develops in the pigment cells of the uvea, which is the middle layer of the eye. The uvea consists of three main parts: the iris, ciliary body, and choroid. Compared to tumors of the iris, tumors of the ciliary body and choroid tend to be larger and more likely to spread to other parts of the body. TCGA studied tumors from all three parts of the uvea.</p>\n<p>\n       Please see the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/melanoma-eye\" target=\"_blank\">TCGA-UVM</a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/using-tcga/citing-tcga\" target=\"_blank\">Citing TCGA in Publications and Presentations</a> information page.</p>','2024-03-08',NULL,'Public',80,'SM','Uveal Melanoma','','','','Human','Intraocular',1,0,'O','',1,1,'f5e36be1-957a-4685-a20d-d2910340c5ec','https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga/studied-cancers/melanoma-eye'),(158,'ACRIN-Contralateral-Breast-MR','ACRIN-Contralateral-Breast-MR','ACRIN-Contralateral-Breast-MR','acrin_contralateral_breast_mr','<p>\n	<span>Even after careful clinical and mammographic evaluation, cancer is found in the contralateral breast in up to 10% of women who have received treatment for unilateral breast cancer. ACRIN 6667 was conducted to determine whether magnetic resonance imaging (MRI) could improve on clinical breast examination and mammography in detecting contralateral breast cancer soon after the initial diagnosis of unilateral breast cancer. </span> Please see the <a href=\"\" url=\"https://doi.org/10.7937/Q1EE-J082\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ACRIN Contralateral Breast MR (ACRIN 6667) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',984,'CR, MR','Breast Cancer','10.7937/Q1EE-J082','Clinical','','Human','Breast',1,0,'O','',1,19,'b097511d-56a3-4e74-946d-c71403787fc2','https://doi.org/10.7937/Q1EE-J082'),(159,'StageII-Colorectal-CT','StageII-Colorectal-CT','StageII-Colorectal-CT','stageii_colorectal_ct','<span>This dataset includes abdominal or pelvic enhanced CT images within 10 days before surgery of 230 patients with stage II colorectal cancer (CRC). Images were acquired on Sensation 64 (Siemens Healthcare, Erlangen, Germany) CT scanner or Brilliance (Philips Healthcare, Best, The Netherlands) CT scanner.&#160;</span><div><div>Please see the <a href=\"\" url=\"https://doi.org/10.7937/p5k5-tg43\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">StageII-Colorectal-CT <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki to learn more about the images and to obtain any supporting metadata for this collection.</p></p>','2024-03-08',NULL,'Public',230,'CT','Colorectal Cancer','10.7937/p5k5-tg43','Software/Source Code','','Human','Abdomen, Pelvis',1,0,'O','',1,18,'6f8f6800-2eaa-43d0-8aea-a74400768ab0','https://doi.org/10.7937/p5k5-tg43'),(160,'ACRIN-6698','ACRIN-6698','ACRIN-6698','acrin_6698','<span>The American College of Radiology Imaging Network (ACRIN) trial 6698 was a multi-center study to evaluate the effectiveness of quantitative diffusion weighted imaging (DWI) for assessing breast cancer response to neoadjuvant chemotherapy (NAC).&#160; &#160;ACRIN 6698 was performed as a sub-study of the ongoing I-SPY 2 TRIAL (Investigation of Serial studies to Predict Your Therapeutic Response with Imaging And moLecular Analysis 2), an adaptive, multi-agent phase II trial designed to quickly identify new agents for breast cancer.&#160;&#160;</span><div><span><br></span></p><div><span>Please see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/TCIA.KK02-6D95\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">ACRIN 6698/I-SPY2 Breast DWI&#160; <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>wiki page to learn more about the images and to obtain any supporting metadata for this collection.&#160;</span><div><span>&#160;</span></p></p>','2024-03-08',NULL,'Public',385,'MR, SEG','Breast Cancer','10.7937/TCIA.KK02-6D95','Clinical, Image Analyses','','Human','Breast',1,0,'O','',1,19,'50befbe5-9bd6-4d08-b766-b8825a8b7bb3','https://doi.org/10.7937/TCIA.KK02-6D95'),(161,'HTAN-HMS','HTAN-HMS','HTAN-HMS','htan_hms','<p>\nWe are constructing a multi-dimensional atlas of pre-melanoma focused on understanding genetic and epigenetic events that transform melanocytes into invasive tumors. Melanoma is a cancer of increasing prevalence that is curable with minor surgery if detected early but life-threatening when it metastasizes. Melanomas metastasize when still small, making early detection essential but challenging. Our atlas will delineate the precise sequence of events leading up to pre-melanoma through detailed spatial analysis of cell-autonomous events such as oncogene mutation and non-autonomous events such as escape from immune surveillance. The atlas is based on highly-multiplexed tissue imaging and single cell sequencing and focused on samples in which the full sequence of events from atypia to invasive melanoma can be visualized in a single specimen. The atlas will serve as a publicly accessible resource for research scientists, physicians, and patients and improve our ability to (i) highlight lesions likely to progress to cancer, (ii) identify high-risk patients to inform decisions on surgery, (iii) identify low-risk patients to reduce unnecessary procedures, (iv) design improved procedures for routine screening of all individuals, and (v) inform treatment options when surgery is insufficient. Complementary studies with similar goals (but not supported by HTAN) are studying later stage melanomas.</p>\n<p>\n       Please see the <a href=\"\" url=\"https://humantumoratlas.org/hta7/\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN-HMS <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"\" url=\"https://docs.google.com/document/d/1seSp9yo8mqMK5sFXd4LEf_t_4SEofZi-1p-M75R1lhk/edit#heading=h.ybiv33wzts40\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN Publication Policy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page.</p>','2024-03-08',NULL,'Public',16,'PR, SM','Adenocarcinoma','','','','Human','Colon',1,0,'O','',1,26,'ee4f8dcc-b03d-4307-8439-da3f6ab3d83d','https://humantumoratlas.org/hta7/'),(162,'HTAN-OHSU','HTAN-OHSU','HTAN-OHSU','htan_ohsu','<p>\nThe overall goal of the HTAN OMS Atlas Center is to elucidate mechanisms by which metastatic breast cancers become resistant to current generation pathway- and immune checkpoint-targeted treatments. The OMS Atlas is motivated by the appreciation that these treatments are often effective in primary tumors but only transiently effective in the metastatic setting. Possible resistance mechanisms include tumor-intrinsic genomic instability and epigenomic plasticity, as well as events extrinsic to the cancer cells, including chemical and mechanical signals from the microenvironments, production of mechanical extracellular matrix barriers and/or changes in vasculature that reduce drug and/or immune cell access, nanoscale cancer cell-microenvironment interactions that reduce drug efficacy, and a plethora of immune resistance mechanisms, such as loss of HLA expression and antigen presentation, and immune exhaustion. These mechanisms likely vary between patients and within individual patients and change with time as tumors respond to therapeutic attack. The OMS Atlas will focus on elucidating resistance mechanisms in two specific current generation clinical trial scenarios: (a) hormone receptor-positive breast cancer (HRBC) undergoing treatment with a CDK4/6 inhibitor in combination with endocrine therapy and (b) triple negative breast cancer (TNBC) undergoing treatment with a PARP inhibitor and an immunomodulatory agent.</p>\n<p>\n       Please see the <a href=\"\" url=\"https://humantumoratlas.org/hta9/\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN-OHSU <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"\" url=\"https://docs.google.com/document/d/1seSp9yo8mqMK5sFXd4LEf_t_4SEofZi-1p-M75R1lhk/edit#heading=h.ybiv33wzts40\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN Publication Policy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page.</p>\n','2024-03-08',NULL,'Public',1,'PR, SM','Breast','','','','Human','Breast',1,0,'O','',1,26,'7a3e0201-5436-4f48-9db5-9d8612e9a2c0','https://humantumoratlas.org/hta9/'),(163,'HTAN-WUSTL','HTAN-WUSTL','HTAN-WUSTL','htan_wustl','<p>\n       Please see the <a href=\"\" url=\"https://humantumoratlas.org/hta12/\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN-WUSTL <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"\" url=\"https://docs.google.com/document/d/1seSp9yo8mqMK5sFXd4LEf_t_4SEofZi-1p-M75R1lhk/edit#heading=h.ybiv33wzts40\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN Publication Policy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page.</p>','2024-03-08',NULL,'Public',21,'PR, SM','Ductal adenocarcinoma','','','','Human','Pancreas',1,0,'O','',1,26,'5ed563b3-8521-4202-ab84-ea88c8920b70','https://humantumoratlas.org/hta12/'),(164,'ISPY2','ISPY2','ISPY2','ispy2','<p>\n	<span ><span><span >I-SPY 2 (Investigation of Serial studies to Predict Your Therapeutic Response with Imaging And moLecular analysis 2) is an ongoing, multi-center trial designed to quickly evaluate the efficacy of new agents for breast cancer in neoadjuvant chemotherapy (NAC) setting. Women aged &ge;18 years diagnosed with locally advanced breast cancer (tumor size &ge;2.5 cm) without distant metastasis are eligible to enroll in the trial. </span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.D8Z0-9T85\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">I-SPY 2 Breast Dynamic Contrast Enhanced MRI (I-SPY2 TRIAL) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></span></p>\n','2024-03-08',NULL,'Public',719,'MR, SEG','Breast Cancer','10.7937/TCIA.D8Z0-9T85','Clinical, Image Analyses','','Human','Breast',1,0,'O','',1,19,'cc608e04-d37b-4faa-86d8-5b17068c6f80','https://doi.org/10.7937/TCIA.D8Z0-9T85'),(165,'UPENN-GBM','UPENN-GBM','UPENN-GBM','upenn_gbm','<p  > <span>This collection comprises multi-parametric magnetic resonance imaging (mpMRI) scans for de novo Glioblastoma (GBM) patients coupled with patient demographics, clinical outcome (e.g., overall survival, genomic information, tumor progression), as well as computer-aided and manually-corrected segmentation labels of multiple histologically distinct tumor sub-regions, computer-aided and manually-corrected segmentations of the whole brain, a rich panel of radiomic features along with their corresponding co-registered mpMRI volumes in NIfTI format.</span></p> <p  > Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.709X-DN49\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Multi-parametric magnetic resonance imaging (mpMRI) scans for de novo Glioblastoma (GBM) patients from the University of Pennsylvania Health System <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>wiki page to learn more about the images and to obtain any supporting metadata for this collection. </p> </span>','2024-03-08',NULL,'Public',630,'MR','Glioblastoma','10.7937/TCIA.709X-DN49','Clinical, Image Analyses','','Human','Brain',1,0,'O','',1,18,'17cb6856-c1cf-4a51-8480-8458959a4091','https://doi.org/10.7937/TCIA.709X-DN49'),(166,'CMB-CRC','CMB-CRC','CMB-CRC','cmb_crc','<p>The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">Cancer Moonshot Biobank</a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\" class=\"external-link\" rel=\"nofollow\">standard of care cancer treatment</a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">NCI Community Oncology Research Program (NCORP)</a>&#160;sites.</p><p>This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank-Colorectal Cancer (CMB-CRC) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\" class=\"external-link\" rel=\"nofollow\">The Database of Genotypes and Phenotypes (dbGaP)</a>&#160;and other NCI databases.</p><p>Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/DJG7-GZ87\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank-Colorectal Cancer Collection (CMB-CRC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;<span>page to learn more about the images and to obtain any supporting metadata for this collection.</span></p><p class=\"MsoListParagraph\"><span></span></p>','2024-03-08',NULL,'Public',24,'CT, DX, MR, PT, US, XA','Colorectal Cancer','10.7937/DJG7-GZ87','Clinical','','Human','Colon',1,0,'O','',1,27,'40cd9a0a-487a-4e48-9fc6-0dacbd0df63a','https://doi.org/10.7937/DJG7-GZ87'),(167,'CMB-GEC','CMB-GEC','CMB-GEC','cmb_gec','<span class=\"conf-macro output-inline\"><p><font face=\"-apple-system, system-ui, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji\">The&#160;</font><a href=\"https://moonshotbiobank.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">Cancer Moonshot Biobank</a><font face=\"-apple-system, system-ui, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji\">&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;</font><a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\" class=\"external-link\" rel=\"nofollow\">standard of care cancer treatment</a><font face=\"-apple-system, system-ui, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji\">&#160;at multiple&#160;</font><a href=\"https://ncorp.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">NCI Community Oncology Research Program (NCORP)</a><font face=\"-apple-system, system-ui, Segoe UI, Roboto, Helvetica Neue, Arial, Noto Sans, sans-serif, Apple Color Emoji, Segoe UI Emoji, Segoe UI Symbol, Noto Color Emoji\">&#160;sites.&#160;</font><span>This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank - Gastroesophageal Cancer&#160; (CMB-GEC) cohort.</span><span>&#160;</span><span>Associated genomic, phenotypic and clinical data will be hosted by</span><span>&#160;</span><a href=\"https://www.ncbi.nlm.nih.gov/gap/\" class=\"external-link\" rel=\"nofollow\">The Database of Genotypes and Phenotypes (dbGaP)</a><span>&#160;and other NCI databases.&#160;</span><span>Please see the&#160;</span><span><a href=\"\" url=\"https://doi.org/10.7937/E7KH-R486\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank - Gastroesophageal Cancer Collection (CMB-GEC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><font color=\"#222222\" face=\"Calibri, sans-serif\"><span>&#160;page to learn more about the images and to obtain any supporting metadata for this collection.</span></font></span></p></span><span></span>','2024-03-08',NULL,'Public',1,'CT, PT','Gastroesophageal Cancer','10.7937/E7KH-R486','Clinical','','Human','Esophagus',1,0,'O','',1,27,'d9f3f5b8-aec9-4854-a95b-3ba88bdb1b71','https://doi.org/10.7937/E7KH-R486'),(168,'CMB-LCA','CMB-LCA','CMB-LCA','cmb_lca','<p class=\"p1\">The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\"><span class=\"s1\">Cancer Moonshot Biobank</span></a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\"><span class=\"s1\">standard of care cancer treatment</span></a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\"><span class=\"s1\">NCI Community Oncology Research Program (NCORP)</span></a>&#160;sites.</p><p class=\"p1\">This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank-Colorectal Cancer (CMB-CRC) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\"><span class=\"s1\">The Database of Genotypes and Phenotypes (dbGaP)</span></a>&#160;and other NCI databases.</p><p class=\"p1\">Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/3CX3-S132\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank-Lung Cancer Collection (CMB-LCA) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;page to learn more about the images and to obtain any supporting metadata for this collection.</p><span></span>','2024-03-08',NULL,'Public',16,'CT, DX, MR, NM, PT, US','Lung Cancer','10.7937/3CX3-S132','Clinical','','Human','Lung',1,0,'O','',1,27,'b7dc2145-3ca8-41d7-9219-83aea3cdcb1d','https://doi.org/10.7937/3CX3-S132'),(169,'CMB-MEL','CMB-MEL','CMB-MEL','cmb_mel','<p class=\"p1\">The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\"><span class=\"s1\">Cancer Moonshot Biobank</span></a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\"><span class=\"s1\">standard of care cancer treatment</span></a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\"><span class=\"s1\">NCI Community Oncology Research Program (NCORP)</span></a>&#160;sites.</p><p class=\"p1\">This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank-Colorectal Cancer (CMB-CRC) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\"><span class=\"s1\">The Database of Genotypes and Phenotypes (dbGaP)</span></a>&#160;and other NCI databases.&#160;<span>Please see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/GWSP-WH72\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank - Melanoma Collection (CMB-MEL) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span><span>&#160;page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>','2024-03-08',NULL,'Public',15,'CT, PT, US','Melanoma','10.7937/GWSP-WH72','Clinical','','Human','Various',1,0,'O','',1,27,'4c7c9e4f-397b-4fc9-b674-169827e57e83','https://doi.org/10.7937/GWSP-WH72'),(170,'CMB-MML','CMB-MML','CMB-MML','cmb_mml','<p class=\"p1\">The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\"><span class=\"s1\">Cancer Moonshot Biobank</span></a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\"><span class=\"s1\">standard of care cancer treatment</span></a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\"><span class=\"s1\">NCI Community Oncology Research Program (NCORP)</span></a>&#160;sites.</p><p class=\"p1\">This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank-Colorectal Cancer (CMB-CRC) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\"><span class=\"s1\">The Database of Genotypes and Phenotypes (dbGaP)</span></a>&#160;and other NCI databases.</p><p class=\"p1\">Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/SZKB-SW39\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank-Multiple Myeloma Collection (CMB-MML) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;<span>page to learn more about the images and to obtain any supporting metadata for this collection.</span></p><span></span>','2024-03-08',NULL,'Public',12,'CT, MR, PT, XA','Multiple Myeloma','10.7937/SZKB-SW39','Clinical','','Human','Blood, Bone',1,0,'O','',1,27,'28e78f90-734b-4715-adf3-4eaecf7f29f3','https://doi.org/10.7937/SZKB-SW39'),(171,'CMB-PCA','CMB-PCA','CMB-PCA','cmb_pca','<p>The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">Cancer Moonshot Biobank</a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\" class=\"external-link\" rel=\"nofollow\">standard of care cancer treatment</a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">NCI Community Oncology Research Program (NCORP)</a>&#160;sites.&#160;This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank - Prostate Cancer (CMB-PCA) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\" class=\"external-link\" rel=\"nofollow\">The Database of Genotypes and Phenotypes (dbGaP)</a>&#160;and other NCI databases.&#160;<span>Please see the&#160;</span><span><a href=\"\" url=\"https://doi.org/10.7937/25T7-6Y12\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank-Prostate Cancer Collection (CMB-PCA) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;page to learn more about the images and to obtain any supporting metadata for this collection.</span></p><p class=\"MsoListParagraph\"><span></span></p>','2024-03-08',NULL,'Public',6,'CT, DX, MR, NM, PT, RF','Prostate Cancer','10.7937/25T7-6Y12','Clinical','','Human','Prostate',1,0,'O','',1,27,'d358a754-065c-4c0c-b387-287ca8629a64','https://doi.org/10.7937/25T7-6Y12'),(172,'GBM-DSC-MRI-DRO','GBM-DSC-MRI-DRO','GBM-DSC-MRI-DRO','gbm_dsc_mri_dro','The standardization of dynamic susceptibility contrast (DSC)-magnetic resonance imaging (MRI) have been confounded by a lack of consensus on DSC-MRI methodology for preventing potential relative cerebral blood volume (CBV) inaccuracies, including the choice of acquisition protocols and postprocessing algorithms. Therefore, a digital reference object (DRO) was developed using physiological and kinetic parameters derived from a patient database, unique voxel-wise 3-dimensional tissue structures, and a validated MRI signal computational approach. The primary, intended use of the DRO is to validate image acquisition and analysis methods for accurately measuring relative cerebral blood volume in glioblastomas.<div>Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/TCIA.2020.RMWVZWIX\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">GBM-DSC-MRI-DRO <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;wiki page to learn more about the images and to obtain any supporting metadata for this collection.<br></p>','2024-03-08',NULL,'Public',3,'MR','Phantom','10.7937/TCIA.2020.RMWVZWIX','','','Human','Brain Phantom',1,0,'O','',1,2,'c3ba2fb4-5f97-4bec-a7e6-235e4dbfe0db','https://doi.org/10.7937/TCIA.2020.RMWVZWIX'),(173,'HCC-TACE-Seg','HCC-TACE-Seg','HCC-TACE-Seg','hcc_tace_seg','<p>Despite surveillance, the majority of HCC cases are diagnosed at advanced stages that can be treated only using (Transarterial chemoembolization) TACE, or systemic therapy. TACE failure can occur to 60% of patients receiving the procedure, with subsequent financial and emotional burden. Radiomics have emerged as a new tool capable of predicting tumor response to TACE from pre-procedural CT study.</p><p>This retrospectively acquired data collection includes pre- and post-procedure CT imaging studies of 105&#160;confirmed HCC patients who underwent TACE between 2002 and 2012 with an available treatment outcome, in the form of time-to-progression and overall survival.</p><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/TCIA.5FNA-0924\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HCC-TACE-Seg <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',105,'CT, SEG','Hepatocellular carcinoma','10.7937/TCIA.5FNA-0924','Clinical, Image Analyses, Software/Source Code','','Human','Liver',1,0,'O','',1,18,'0d3c0f7b-1545-4881-ad94-7781ed54d1a9','https://doi.org/10.7937/TCIA.5FNA-0924'),(174,'PDMR-521955-158-R4','PDMR-521955-158-R4','PDMR-521955-158-R4','pdmr_521955_158_r4','<div><div><div><span>We identified and characterized a patient derived xenograft model with metastatic potential, adenocarcinoma pancreas xenograft 521955-158-R4 from the National Cancer Institute Patient-Derived Models Repository (</span><a href=\"https://pdmr.cancer.gov\"><span>https://pdmr.cancer.gov/</span></a><span>). In this study we performed a detailed imaging characterization of this model, which develops spontaneous lung metastases (bi-weekly MRI) and PET/CT characterization of the primary tumor.</span><div><br><span></span><p></p><div>Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/q37d-vh79\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">PDMR-521955-158-R4 <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;wiki page to learn more about the images and to obtain any supporting metadata for this collection.&#160;<span><br></span></p></p></p></p></p>','2024-03-08',NULL,'Public',20,'MR, SR','Adenocarcinoma Pancreas','10.7937/q37d-vh79','Clinical','','Mouse','Abdomen',1,0,'O','',1,12,'703cc0a3-2899-4daf-9dae-d23e470f7619','https://doi.org/10.7937/q37d-vh79'),(175,'CT-vs-PET-Ventilation-Imaging','CT-vs-PET-Ventilation-Imaging','CT-vs-PET-Ventilation-Imaging','ct_vs_pet_ventilation_imaging','<span>The purpose of the </span><span><a href=\"\" url=\"https://doi.org/10.7937/3ppx-7s22\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CT Ventilation as a Functional Imaging Modality for Lung Cancer Radiotherapy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> </span><span>study was to enable comparisons between: (i) CT ventilation images derived from exhale/inhale BHCT scans, (ii) CT ventilation images derived from free-breathing 4DCT scans, and (iii) Galligas PET (nuclear medicine) ventilation scans. This dataset can build the international capacity for prototyping and evaluating new CT ventilation imaging technologies. For this study, 20 lung cancer patients underwent exhale/inhale breath hold CT (BHCT), free-breathing four-dimensional CT (4DCT) and Galligas PET ventilation scans in a single session on a combined 4DPET/CT scanner.</p><p>All image acquisitions were performed on a Siemens Biograph mCT.S/64 PET/CT scanner (Siemens, Knoxville, USA) at the Royal North Shore Hospital between 2013 and 2015. A total of 20 4DCT scans, 20 inhale/exhale BHCT scans, 20 Galligas PET scans and 19 attenuation CT scans (missing for CT-PET-VI-07) were successfully acquired for the 20 patients and included in this dataset.</span><br><span>Please see the <a href=\"\" url=\"https://doi.org/10.7937/3ppx-7s22\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CT-vs-PET-Ventilation-Imaging <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a></span><span> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>','2024-03-08',NULL,'Public',20,'CT, PT, RWV','Lung Cancer','10.7937/3ppx-7s22','','','Human','Lung',1,0,'O','',1,18,'c89f5a56-87b5-4c30-a3d0-0bb70a3d4705','https://doi.org/10.7937/3ppx-7s22'),(176,'CTpred-Sunitinib-panNET','CTpred-Sunitinib-panNET','CTpred-Sunitinib-panNET','ctpred_sunitinib_pannet','<span>The </span><span><a href=\"\" url=\"https://doi.org/10.7937/SPGK-0P94\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Prediction of Sunitinib Efficacy using Computed Tomography in Patients with Pancreatic Neuroendocrine Tumors <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a></span><span> study aimed to develop and validate a computed tomography (CT)-based method to predict the efficacy of sunitinib in Pretreatment CT images of 171 lesions from 38 patients with metastatic or locally advanced pancreatic neuroendocrine tumors (panNET).</span><br><span>The dataset can be used to validate the findings of the study. More importantly, researchers can use this dataset to study the imaging characteristics of panNET.</span><br><span>Please see the </span><span><a href=\"\" url=\"https://doi.org/10.7937/SPGK-0P94\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CTpred-Sunitinib-panNET <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a></span><span> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span>','2024-03-08',NULL,'Public',38,'CT','Pancreas Cancer','10.7937/SPGK-0P94','','','Human','Pancreas',1,0,'O','',1,18,'7bb71159-c19a-4adc-9d93-848e4e65eb68','https://doi.org/10.7937/SPGK-0P94'),(177,'AI-derived annotations for the NLST and NSCLC-Radiomics computed tomography imaging collections','','','nnU-Net-BPR-annotations','<p>Public imaging datasets are critical for the development and evaluation of automated tools in cancer imaging. Unfortunately, many of the available datasets do not provide annotations of tumors or&nbsp; organs-at-risk, crucial for the assessment of these tools. This is due to the fact that annotation of medical images is time consuming and requires domain expertise. It has been demonstrated that artificial intelligence (AI) based annotation tools can achieve acceptable performance and thus can be used to automate the annotation of large datasets. As part of the effort to enrich the public data available within NCI Imaging Data Commons (IDC) (<a href=\"\"https://imaging.datacommons.cancer.gov/\">https://imaging.datacommons.cancer.gov/</a>) [1],&nbsp;we introduce this dataset that consists of such AI-generated annotations for two publicly available medical imaging collections of Computed Tomography (CT) images of the chest.</p> \n<p>  Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.5281/zenodo.7975081\" data-toggle=\"modal\" data-target=\"#external-web-warning\">nnU-Net-BPR-annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> description page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',985,'','Lung Cancer','10.5281/zenodo.7975081','','Organ segmentations, slice-level qualitative annotations, shape radiomics features','','Chest',1,0,'A','nlst,nsclc_radiomics',1,NULL,'15d3aa43-fe60-479b-8633-2a5ef73d62af','https://doi.org/10.5281/zenodo.7975081'),(178,'Adrenal-ACC-Ki67-Seg','Adrenal-ACC-Ki67-Seg','Adrenal-ACC-Ki67-Seg','adrenal_acc_ki67_seg','<p>\nThe Ki-67 index is one of the most important established prognostic markers for local recurrence of ACC. Radiomic feature extraction showed a significant association between radiomic signature and Ki-67 expression status in ACC.</p><p>\nThis retrospectively acquired data includes contrast enhanced CT imaging studies of 53 confirmed ACC patients between 2006 to 2018 with available clinical and pathological data, including Ki-67 index. Semi-automatic segmentation of the adrenal tumor was created using AMIRA, then manually refined by an experienced radiologist. The segmentations of each contrast-enhanced CT were done for the purpose of radiomic features extraction.</p>\n<p>\n        Please see the <a href=\"\" url=\"https://doi.org/10.7937/1FPG-VM46\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Adrenal-ACC-Ki67-Seg <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',53,'CT, SEG','Adrenocortical Carcinoma','10.7937/1FPG-VM46','Clinical, Image Analyses','','Human','Adrenal',1,0,'O','',1,18,'a8ba574c-adae-42fe-ace4-b3eaa3bdd598','https://doi.org/10.7937/1FPG-VM46'),(179,'CC-Tumor-Heterogeneity','CC-Tumor-Heterogeneity','CC-Tumor-Heterogeneity','cc_tumor_heterogeneity','The functional and biological properties of the tumor microenvironment are fundamentally important determinants of tumor response and therapy outcome in cancer. Oxygenation status and vascularity are known to influence radiation response, and molecular energy metabolism and proliferation impact on recurrence and metastatic progression. &#10;Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/ERZ5-QZ59\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cervical Cancer &#8211; Tumor Heterogeneity: Serial Functional and Molecular Imaging Across the Radiation Therapy Course in Advanced Cervical Cancer Cervical Cancer&#160; <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>wiki page to learn more about the images and to obtain any supporting metadata for this collection. ','2024-03-08',NULL,'Public',23,'CT, MR, PT, REG, RTSTRUCT','Cervical Cancer','10.7937/ERZ5-QZ59','Clinical','','Human','Cervix',1,0,'O','',1,18,'c9ad6d83-1e4a-4d1a-a91e-c775946d0e53','https://doi.org/10.7937/ERZ5-QZ59'),(180,'Colorectal-Liver-Metastases','Colorectal-Liver-Metastases','Colorectal-Liver-Metastases','colorectal_liver_metastases','<span>This collection consists of DICOM images and DICOM Segmentation Objects (DSOs) for 197 patients with Colorectal Liver Metastases (CRLM). The collection consists of a large, single-institution consecutive series of patients that underwent resection of CRLM and matched preoperative computed tomography (CT) scans for quantitative image analysis. Inclusion criteria were (a) pathologically confirmed resected CRLM, (b) available data from pathologic analysis of the underlying non-tumoral liver parenchyma and hepatic tumor, (c) available preoperative conventional portal venous contrast-enhanced multi-detector computed tomography (MDCT) performed within 6 weeks of hepatic resection. Patients with 90-day mortality or that had less than 24 months of follow-up were excluded.</span>&#160;Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/QXK2-QG03\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Preoperative CT and Recurrence for Patients Undergoing Resection of Colorectal Liver Metastases (Colorectal Liver Metastases) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;wiki page to learn more about the images and to obtain any supporting metadata for this collection.','2024-03-08',NULL,'Public',197,'CT, SEG','Colorectal Cancer','10.7937/QXK2-QG03','Clinical','','Human','Colon',1,0,'O','',1,18,'bd8951df-a75e-44fb-b1c7-5f53ae56e5a9','https://doi.org/10.7937/QXK2-QG03'),(181,'HTAN-VANDERBILT','HTAN-VANDERBILT','HTAN-VANDERBILT','htan_vanderbilt','<p>Colorectal cancer (CRC) is among the top three most prevalent cancers in global incidence and mortality. Most of these cancers develop from pre-cancerous adenomas. There is an unmet need to develop new preventive strategies and risk stratification models to decrease incidence, improve early detection, and prevent deaths from CRC.</p>\n<p>We believe that the ability to provide the most effective precision diagnostics and preventive strategies can only be achieved with single-cell analysis. As such, we will map spatial relationships across the spectrum of normal colon, early polyps, and late adenomas, including their unique stromal and microbial microenvironments to identify unique molecular phenotypes.</p>\n<p>Our goal will be accomplished through prospective, standardized collection and analysis of colorectal tissue, associated biospecimens, and related clinical and epidemiological data from participants undergoing colonoscopy or surgical resection. The biospecimens from these participants will be used for single-cell RNA sequencing, whole exome sequencing, multiplex immunofluorescence, species-specific bacterial fluorescence in situ hybridization, and other approaches. Finally, the information from these approaches will be integrated to develop a single-cell pre-cancer atlas with defined molecular phenotypes for dissemination to the broader scientific community.</p>\n<p>\n       Please see the <a href=\"\" url=\"https://humantumoratlas.org/hta11/\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN-Vanderbilt <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"\" url=\"https://docs.google.com/document/d/1seSp9yo8mqMK5sFXd4LEf_t_4SEofZi-1p-M75R1lhk/edit#heading=h.ybiv33wzts40\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">HTAN Publication Policy <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page.</p>','2024-03-08',NULL,'Public',30,'SM','Colon Precancer','','','','Human','Colon',1,0,'O','',1,26,'707d7395-8ccf-4aa2-a39d-6290b2541738','https://humantumoratlas.org/hta11/'),(182,'NLM-Visible-Human-Project','NLM-Visible-Human-Project','NLM-Visible-Human-Project','nlm_visible_human_project','<p>The NLM Visible Human Project has created publicly-available complete, anatomically detailed, three-dimensional representations of a human male body and a human female body. Specifically, the VHP provides a public-domain library of cross-sectional cryosection, CT, and MRI images obtained from one male cadaver and one female cadaver. The Visible Man data set was publicly released in 1994 and the Visible Woman in 1995.</p>\n<p>The data sets were designed to serve as (1) a reference for the study of human anatomy, (2) public-domain data for testing medical imaging algorithms, and (3) a test bed and model for the construction of network-accessible image libraries. The VHP data sets have been applied to a wide range of educational, diagnostic, treatment planning, virtual reality, artistic, mathematical, and industrial uses. About 4,000 licensees from 66 countries were authorized to access the datasets. As of 2019, a license is no longer required to access the VHP datasets.</p>\n<p>\n       Courtesy of the U.S. National Library of Medicine. Release of this collection by IDC does not indicate or imply that NLM has endorsed its products/services/applications. Please see the <a href=\"https://www.nlm.nih.gov/research/visible/visible_human.html\" target=\"_blank\">Visible Human Project</a> information page to learn more about the images and to obtain any supporting metadata for this collection. Note that this collection may not reflect the most current/accurate data available from NLM.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"https://www.nlm.nih.gov/databases/download/terms_and_conditions.html\" target=\"_blank\">National Library of Medicine Terms and Conditions</a> information page.</p>','2024-03-08',NULL,'Public',2,'CT, MR, XC','Normal (non-cancer)','','','','Human','Whole Body',1,0,'O','',1,28,'a77eb27a-1009-4b16-b650-6ca1532bb8e0','https://www.nlm.nih.gov/research/visible/visible_human.html'),(183,'Prostate-Anatomical-Edge-Cases','Prostate-Anatomical-Edge-Cases','Prostate-Anatomical-Edge-Cases','prostate_anatomical_edge_cases','<span>In this single institution retrospective study, we reviewed 950 consecutive patients with prostate adenocarcinoma receiving definitive radiotherapy between 2011 and 2019, and identified among them 112 patients with anatomic variations (edge cases) seen on simulation CT and/or MRI imaging. These variations included hip arthroplasty, prostate median lobe hypertrophy, so-called &#8220;droopy&#8221; seminal vesicles, presence of a urinary catheter, and others. A separate cohort of 19 &#8220;normal&#8221; cases were randomly selected for inclusion. Prostate, rectum, bladder, and bilateral femoral heads were manually segmented on all CT simulation images (where present) and were ultimately used clinically for radiation treatment planning.&#160;</span><span>Please see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/QSTF-ST65\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Stress-Testing Pelvic Autosegmentation Algorithms Using Anatomical Edge Cases (Prostate-Anatomical-Edge-Cases) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>&#160;</span><span>wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span><p class=\"MsoListParagraph\"><span></span></p>','2024-03-08',NULL,'Public',131,'CT, RTSTRUCT','Prostate Cancer','10.7937/QSTF-ST65','','','Human','Prostate',1,0,'O','',1,18,'2e28e5ee-03a4-4111-a07e-f1ae94bd2441','https://doi.org/10.7937/QSTF-ST65'),(184,'RIDER Pilot','RIDER Pilot','RIDER Pilot','rider_pilot','<span>This data collection was originally supported under supplemental funding for the Lung Image Database Consortium (LIDC) U01 project and focused on the collection of longitudinal studies using X-ray CT for monitoring the response to therapy. The data came primarily from the M.D. Anderson Cancer Center and several of the LIDC academic sites. The data is not annotated. A small subset of the </span><a href=\"\" url=\"https://doi.org/10.7937/m87f-mz83 \" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER Pilot <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span> data (8 subjects) which were analyzed in </span><a href=\"\" url=\"https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=70228818\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">QIBA-VolCT-1B <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>&#160;have been re-released on TCIA.</span><br><br><span>More information about the <a href=\"\" url=\"https://doi.org/10.7937/m87f-mz83\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER Pilot <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> sub-collection page can be found in the corresponding section of the <a href=\"\" url=\"https://wiki.cancerimagingarchive.net/x/EAA9\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">RIDER <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page.</span>','2024-03-08',NULL,'Public',8,'CR, CT, DX, PR, SEG, SR','Lung Cancer','10.7937/m87f-mz83','','','Human','Lung',1,0,'O','',1,7,'30d3f223-e008-42b3-b495-922164fa460d','https://doi.org/10.7937/m87f-mz83'),(185,'Annotations for The Clinical Proteomic Tumor Analysis Consortium Pancreatic Ductal Adenocarcinoma Collection','','','CPTAC-PDA-Tumor-Annotations','<p>This dataset contains image annotations derived from <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.SC20FO18\" class=\"external-link\" rel=\"nofollow\">The Clinical Proteomic Tumor Analysis Consortium Pancreatic Ductal Adenocarcinoma Collection (CPTAC-PDA) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>. This dataset was generated as part of a National Cancer Institute project to augment images from The Cancer Imaging Archive with annotations that will improve their value for cancer researchers and artificial intelligence experts.</p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/bw9v-bx61\" data-toggle=\"modal\" data-target=\"#external-web-warning\">CPTAC-PDA-Tumor-Annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',103,'','Ductal Adenocarcinoma','10.7937/BW9V-BX61','','Tumor segmentations, Seed points','','Pancreas',1,0,'A','cptac_pda',1,NULL,'7d291fd7-33cb-4964-8050-9d446d0f2ed6','https://doi.org/10.7937/bw9v-bx61'),(186,'Annotations for The Clinical Proteomic Tumor Analysis Consortium Uterine Corpus Endometrial Carcinoma Collection','','','CPTAC-UCEC-Tumor-Annotations','<p>This dataset contains image annotations derived from <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.3R3JUISW\" class=\"external-link\" rel=\"nofollow\">The Clinical Proteomic Tumor Analysis Consortium Uterine Corpus Endometrial Carcinoma Collection (CPTAC-UCEC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>. This dataset was generated as part of a National Cancer Institute project to augment images from The Cancer Imaging Archive with annotations that will improve their value for cancer researchers and artificial intelligence experts.</p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/89m3-kq43\" data-toggle=\"modal\" data-target=\"#external-web-warning\">CPTAC-UCEC-Tumor-Annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',72,'','Corpus Endometrial Carcinoma','10.7937/89M3-KQ43','','Tumor segmentations, Seed points','','Uterus',1,0,'A','cptac_ucec',1,NULL,'7d5362a0-e23f-453b-9204-0a0a7a636741','https://doi.org/10.7937/89m3-kq43'),(187,'QIBA VolCT Group 1B Round 2 No Change Size Measurements','','','QIBA-VolCT-1B','<p>The purpose of this analysis is to determine the variability of lesion size measurements in computed tomography data sets of patients imaged under a “no change” (“coffee break”) condition and to determine the impact of two reading paradigms on measurement variability.</p> \n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/tcia.2020.1c3h-vp70\" data-toggle=\"modal\" data-target=\"#external-web-warning\">QIBA-VolCT-1B <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',40,'','Lung','10.7937/tcia.2020.1c3h-vp70','','Tumor segmentations, image features','','Lung',1,0,'A','rider_lung_ct,rider_pilot',1,NULL,'a24de43e-a21f-4649-8084-0b726ad1a14f','https://doi.org/10.7937/tcia.2020.1c3h-vp70'),(188,'PDMR-Texture-Analysis','PDMR-Texture-Analysis','PDMR-Texture-Analysis','pdmr_texture_analysis','<p class=\"MsoNormal\"><span>This collection contains serial non-contrast T2w MRI of&#10;18 patient derived xenograft cancer models (514 images) for researchers to&#10;develop various algorithms using neural networks, and classification techniques&#10;to improve tissue characterization (morphological changes).&#160;&#160;</span></p>&#10;&#10;<p class=\"MsoNormal\"><span>Please&#10;see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/3KQ0-YK19\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">PDMR-Texture-Analysis&#160; <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>wiki page to learn more about the images and&#10;to obtain any supporting metadata for this collection.</span></p>','2024-03-08',NULL,'Public',175,'MR, SR','Adenocarcinoma of colon, Adenocarcinoma of pancreas, Adenocarcinoma of rectum, Ewing sarcoma - Peripheral PNET, Melanoma, Neuroendocrine cancer (NOS), Osteosarcoma, Squamous cell carcinoma of anus, Squamous cell carcinoma of lung, Urothelial - bladder cancer (NOS)','10.7937/3KQ0-YK19','','','Mouse','Arm, Bladder, Buttock, Colon, Liver, Myometrium, Pancreas, Rectum, Shoulder, Scapula',1,0,'O','',1,12,'23cfc436-0f5c-4bb8-818f-b6d7be984b96','https://doi.org/10.7937/3KQ0-YK19'),(189,'RMS-Mutation-Prediction','RMS-Mutation-Prediction','RMS-Mutation-Prediction','rms_mutation_prediction','<p>RMS is an aggressive soft tissue tumor which is associated with a broad range of pathological mutations. Multiple studies have identified recurrent mutations and chromosomal translocations which are associated with poor outcome in RMS patients. Since not all patients are screened for these mutations with molecular testing, there is a need to develop tools which accurately identify these alterations at diagnosis. Our study associated with these images applied deep learning with convolutional neural networks to learn mutation-associated histological features from H&E images from diagnostic RMS specimens.</p>\n<p>\n       Please see the <a href=\"\" url=\"https://doi.org/10.5281/zenodo.8225131\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RMS-Mutation-Prediction <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n<p>\n      Citation guidelines can be found on the <a href=\"\" url=\"https://doi.org/10.5281/zenodo.8225131\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RMS-Mutation-Prediction <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page.</p>','2024-03-08',NULL,'Public',403,'SM, SR','Rhabdomyosarcoma','10.5281/zenodo.8225131','Clinical, Genomics, Software source code','','Human','Various',1,0,'O','',1,29,'98beec44-4bf3-4b92-85c9-cf1c14dc7ad0','https://doi.org/10.5281/zenodo.8225131'),(190,'Image segmentations produced by the AIMI Annotations initiative','','','BAMF-AIMI-Annotations','<p>Many of the collections in IDC have limited annotations due to the expense and effort required to create these manually. The increased capabilities of AI analysis of radiology images provides an opportunity to augment existing IDC collections with new annotation data. To further this goal, we trained several nnU-Net based models for a variety of radiology segmentation tasks from public datasets and used them to generate segmentations for IDC collections.</p>\n<p>To validate the models performance, roughly 10% of the predictions were manually reviewed and corrected by both a board certified radiologist and a medical student (non-expert). Additionally, this non-expert looked at all the ai predictions and rated them on a 5 point Likert scale .</p>\n<p>This record provides AI segmentations, manually corrected segmentations, and manual scores for the inspected IDC Collection images. Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.5281/zenodo.10081112\" data-toggle=\"modal\" data-target=\"#external-web-warning\">BAMF-AIMI-Annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>\n','2024-03-08',NULL,'Public',1178,'','Various','10.5281/zenodo.10081112','','Annotations','','Various',1,0,'A','acrin_nsclc_fdg_pet,anti_pd_1_lung,lung_pet_ct_dx,nsclc_radiogenomics,prostatex,qin_breast,rider_lung_pet_ct,tcga_kirc,tcga_lihc,tcga_luad,tcga_lusc',1,NULL,'eea1344f-d53c-4248-ba6a-2e31578d741d','https://doi.org/10.5281/zenodo.10081112'),(191,'Annotations for The Clinical Proteomic Tumor Analysis Consortium Clear Cell Renal Cell Carcinoma Collection','','','CPTAC-CCRCC-Tumor-Annotations','<p>This dataset contains image annotations derived from <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.7937/K9/TCIA.2018.OBLAMN27\" class=\"external-link\" rel=\"nofollow\">The Clinical Proteomic Tumor Analysis Consortium Clear Cell Renal Cell Carcinoma Collection (CPTAC-CCRCC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>. This dataset was generated as part of a National Cancer Institute project to augment images from The Cancer Imaging Archive with annotations that will improve their value for cancer researchers and artificial intelligence experts.</p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"\"https://doi.org/10.7937/skq4-qx48\"data-toggle=\"modal\" data-target=\"#external-web-warning\">CPTAC-CCRCC-Tumor-Annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',60,'','Clear Cell Carcinoma','10.7937/SKQ4-QX48','','Tumor segmentations, Seed points','','Kidney',1,0,'A','cptac_ccrcc',1,NULL,'62e6a98a-2114-49e7-9a40-7c93b6b80e8f','https://doi.org/10.7937/skq4-qx48'),(192,'DICOM converted annotations for the Prostate-MRI-US-Biopsy collection','','','Prostate-MRI-US-Biopsy-DICOM-Annotations','<p>This dataset contributes DICOM-converted annotations to the publicly available National Cancer Institute Imaging Data Commons Prostate-MRI-US-Biopsy collection.  Originally released by TCIA, the collection was accompanied by 1017 semi-automatic segmentations of the prostate and 1317 manual segmentations of target lesions in the STL format. Although STL is a common and practical format for 3D printing, it is not interoperable with many visualization and analysis tools commonly used in medical imaging research and does not provide any standard means to communicate metadata, among other limitations. To alleviate those deficiencies, we created DICOM Encapsulated 3D Manufacturing Model objects (M3D modality) that includes the original STL content enriched with the DICOM metadata. Furthermore, we created an alternative encoding of the surface segmentations by rasterizing them and saving the result as a DICOM Segmentation object (SEG modality). As a result, the contributed DICOM objects can be stored in any DICOM server that supports those objects (including Google Healthcare DICOM stores), and the DICOM Segmentations can be visualized using off-the-shelf tools, such as OHIF Viewer. </p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.5281/zenodo.10069910\" data-toggle=\"modal\" data-target=\"#external-web-warning\">Prostate-MRI-US-Biopsy-DICOM-annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the data and any supporting metadata for this collection, and to learn about attribution/citation requirements.</p>\n','2024-03-08',NULL,'Public',842,'','Prostate Cancer','10.5281/zenodo.10069911','','Annotations','','Prostate ',1,0,'A','prostate_mri_us_biopsy',1,NULL,'d4dbc08f-5fe1-4287-9d94-3bf3ff424aee','https://doi.org/10.5281/zenodo.10069911'),(193,'CMB-AML','CMB-AML','CMB-AML','cmb_aml','<p>The&#160;<a href=\"https://moonshotbiobank.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">Cancer Moonshot Biobank</a>&#160;is a National Cancer Institute initiative to support current and future investigations into drug resistance and sensitivity and other NCI-sponsored cancer research initiatives, with an aim of improving researchers\'\' understanding of cancer and how to intervene in cancer initiation and progression. During the course of this study, biospecimens (blood and tissue removed during medical procedures) and associated data will be collected longitudinally from at least 1000 patients across at least 10 cancer types, who represent the demographic diversity of the U.S. and receiving&#160;<a href=\"https://www.cancer.gov/publications/dictionaries/cancer-terms/def/standard-of-care\" class=\"external-link\" rel=\"nofollow\">standard of care cancer treatment</a>&#160;at multiple&#160;<a href=\"https://ncorp.cancer.gov/\" class=\"external-link\" rel=\"nofollow\">NCI Community Oncology Research Program (NCORP)</a>&#160;sites.</p><p>This collection contains de-identified radiology and histopathology imaging procured from subjects in NCI&#8217;s Cancer Moonshot Biobank-Colorectal Cancer (CMB-CRC) cohort.&#160;Associated genomic, phenotypic and clinical data will be hosted by&#160;<a href=\"https://www.ncbi.nlm.nih.gov/gap/\" class=\"external-link\" rel=\"nofollow\">The Database of Genotypes and Phenotypes (dbGaP)</a>&#160;and other NCI databases.</p><p>Please see the&#160;<a href=\"\" url=\"https://doi.org/10.7937/PCTE-6M66\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Cancer Moonshot Biobank - Acute Myeloid Leukemia Cancer Collection (CMB-AML)<i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;<span>page to learn more about the images and to obtain any supporting metadata for this collection.</span></p><p class=\"MsoListParagraph\"><span></span></p>','2024-03-08',NULL,'Public',1,'CT, XA','Acute Myeloid Leukemia','10.7937/PCTE-6M66','Clinical','','Human','Blood',1,0,'O','',1,27,'98647393-3de7-4101-ada6-2c5506f4b003','https://doi.org/10.7937/PCTE-6M66'),(194,'CT-Phantom4Radiomics','CT-Phantom4Radiomics','CT-Phantom4Radiomics','ct_phantom4radiomics','<p>The aims of this dataset are to determine the stability of radiomics features against computed tomography (CT) parameter variations and to study their discriminative power concerning tissue classification using <a href=\"\" url=\"https://doi.org/10.1002/mp.16373\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">a 3D-printed CT phantom based on real patient data <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>. 4 tissue types (normal, benign cyst, hemangioma, and metastasis) were manually annotated. 238 CT series with 8 parameter variations of reconstruction algorithms, reconstruction kernels, slice thickness, and slice spacing are available.</p><br><br><p>Please see the <a href=\"\" url=\"https://doi.org/10.7937/a1v1-rc66\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">CT-Phantom4Radiomics <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</p>','2024-03-08',NULL,'Public',1,'CT, SEG','Phantom','10.7937/a1v1-rc66','Image Analyses, Software/Source Code','','Human','Phantom',1,0,'O','',1,18,'e021bd39-cb21-4999-ace3-9383fb950510','https://doi.org/10.7937/a1v1-rc66'),(195,'EA1141','EA1141','EA1141','ea1141','<span><span>This collection contains data from the Eastern Cooperative Oncology Group (ECOG) Clinical Trial</span><span>&#160;</span><span><a title=\"\" href=\"https://clinicaltrials.gov/study/NCT02933489\">NCT02933489</a><span>, &#8220;</span>Abbreviated Breast MRI and Digital Tomosynthesis Mammography in Screening Women With Dense Breasts<span>&#34;. Principle Investigator: Christopher Comstock.</span></span><span>&#160;&#160;</span><span>It was sponsored by NCI and performed by the<span>&#160;</span></span><span>Eastern Cooperative Oncology Group&#160;<span>under study number<span>&#160;</span></span></span><span>EA1141</span><span>.<span>&#160;</span></span><span>This randomized phase II trial enrolled 1,516 subjects to study how well abbreviated breast magnetic resonance imaging (MRI) and digital tomosynthesis mammography work in detecting cancer in women with dense breasts. Imaging and clinical data from 500 of those subjects are being made available through this collection. Please see the </span></span><span><a href=\"\" url=\"https://doi.org/10.7937/2BAS-HR33\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Abbreviated Breast MRI and Digital Tomosynthesis Mammography in Screening Women With Dense Breasts (EA1141)</a> wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span><span></span>','2024-03-08',NULL,'Public',500,'CT, MG, MR','Breast Cancer','10.7937/2BAS-HR33','Clinical','','Human','Breast',1,0,'O','',1,23,'1c5f13d2-c8fa-4fa0-ac89-04e6662865cb','https://doi.org/10.7937/2BAS-HR33'),(196,'ReMIND','ReMIND','ReMIND','remind','<span class=\"x_ContentPasted0\">The Brain Resection Multimodal Imaging Database (ReMIND) contains pre- and intra-operative data collected on 114 consecutive patients who were surgically treated with image-guided tumor resection between<span> 2018 and 2022</span>. For all patients, preoperative MRI, 3D intraoperative ultrasound series, and intraoperative MRI are available. <span>Additionally, each case typically contains segmentations,</span> including the preoperative tumor, the pre-resection cerebrum, the previous resection cavity derived from the preoperative MRI (if applicable), and any residual tumor identified on the intraoperative MRI. In total, this collection contains <span>369</span> preoperative MRI series, 320 3D intraoperative ultrasound series,<span> 301</span> intraoperative MRI series, and 356 segmentations. We expect this data to be a resource for computational research in brain shift and image analysis as well as for neurosurgical training in the interpretation of intraoperative ultrasound and intraoperative MRI. Please see the&#160;</span><a href=\'\'\" url=\"https://doi.org/10.7937/3RAG-D070\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">The Brain Resection Multimodal Imaging Database (ReMIND)  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span>','2024-03-08',NULL,'Public',114,'MR, SEG, US','Brain Cancer','10.7937/3RAG-D070','Clinical','','Human','Brain',1,0,'O','',1,18,'6a8cdb98-7db6-47d2-bf45-da8cbcdeec53','https://doi.org/10.7937/3RAG-D070'),(197,'Vestibular-Schwannoma-MC-RC','Vestibular-Schwannoma-MC-RC','Vestibular-Schwannoma-MC-RC','vestibular_schwannoma_mc_rc','<p><span>This multi-center routine clinical (MC-RC) dataset consists of 160 patients with a single sporadic Vestibular Schwannoma (VS) who were referred from 10 medical sites and consecutively seen at a single center. These routine clinical datasets are more diverse in terms of the tumor manifestation as well as the acquisition parameters. Using this dataset, researchers can develop and validate methods for automatic surveillance of Vestibular Schwannoma, which work robustly on images acquired at different hospitals.&#160;</span><span>Patients had multiple time points resulting in a total of 428 timepoints and 487 3D-images.&#160;</span><span>Manual ground truth segmentations were obtained in an iterative process in which segmentations were: 1) produced or amended by a specialized company; and 2) reviewed by one of three trained radiologists; and 3) validated by an expert team. Compared to the existing&#160;<a href=\"\" url=\"https://doi.org/10.7937/TCIA.9YTJ-5Q73\" class=\"external-link\" rel=\"nofollow\" data-toggle=\"modal\" data-target=\"#external-web-warning\">Vestibular-Schwannoma-SEG <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a>&#160;dataset on TCIA that was obtained from a single scanner, this dataset was acquired from multiple scanners manufactured by different vendors. This dataset also provides a refined segmentation of intrameatal and extrameatal components of the VS.&#160;</span><span>Please see the&#160;</span><a href=\"\" url=\"https://doi.org/10.7937/HRZH-2N82\" data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Segmentation of Vestibular Schwannoma from Magnetic Resonance Imaging: An Annotated Multi-Center Routine Clinical Dataset (Vestibular-Schwannoma-MC-RC) <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a><span>&#160;wiki page to learn more about the images and to obtain any supporting metadata for this collection.</span></p><p class=\"MsoListParagraph\"><span></span></p>','2024-03-08',NULL,'Public',124,'MR','Vestibular Schwannoma (non-cancer)','10.7937/HRZH-2N82','','','Human','Ear',1,0,'O','',1,18,'6c3b6191-c371-42d6-989a-d9e520239dcd','https://doi.org/10.7937/HRZH-2N82'),(198,'Expert annotations of the tissue types for the RMS-Mutation-Prediction microscopy images','','','RMS-Mutation-Prediction-Expert-Annotations','<p>The RMS-Mutation-Prediction-Expert-Annotations collections contains expert-generated annotations of the regions of interest identified in the pathology slides from the RMS-Mutation-Prediction collection. Annotations were generated manually, using the ImageScope tool, to delineate regions of ARMS (alveolar rhabdomyosarcoma), ERMS (embryonal rhabdomyosarcoma), stroma, and necrosis and stored as planar contours, consequently converted into Digital Imaging and Communications in Medicine (DICOM) Structured Report (SR) representation. </p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.5281/zenodo.10462857\" data-toggle=\"modal\" data-target=\"#external-web-warning\">RMS-Mutation-Prediction-Expert-Annotations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the images and any supporting metadata for this collection, and to learn about attribution/citation requirements.</p>','2024-03-08',NULL,'Public',95,'','Rhabdomyosarcoma','10.5281/zenodo.10462858','','Annotations','','Various',1,0,'A','rms_mutation_prediction',1,NULL,'8764bf29-2c2a-4994-9434-e54bf8ead2cd','https://doi.org/10.5281/zenodo.10462858'),(199,'AI-driven enrichment of NCI Imaging Data Commons CT images with volumetric segmentations and radiomics features','','','TotalSegmentator-CT-Segmentations','<p>This collection contains results of automatic analysis of the CT images using the TotalSegmentator model to generate volumetric segmentations of anatomic organs, with the subsequent application of the pyradiomics library to extract first order and shape features for the regions defined by those segmentations. Initial release of this collection contains these analysis results for the National Lung Screening Trial (NLST) collection. </p>\n<p>Please see the <a class=\"external-link\" href=\"\" url=\"https://doi.org/10.5281/zenodo.8347011\" data-toggle=\"modal\" data-target=\"#external-web-warning\"  target=\"_blank\">TotalSegmentator-CT-Segmentations <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> information page to learn more about the data and any supporting metadata for this collection, and to learn about attribution/citation requirements.</p>\n','2024-03-08',NULL,'Public',26194,'','Lung Cancer','10.5281/zenodo.8347012','','Segmentations','','Chest',1,0,'A','nlst',1,NULL,'a63eef07-3e01-4f6a-afb6-a97f23bde926','https://doi.org/10.5281/zenodo.8347012'),(200,'Advanced-MRI-Breast-Lesions','Advanced-MRI-Breast-Lesions','Advanced-MRI-Breast-Lesions','advanced_mri_breast_lesions','<p>The application of deep-learning methods to breast MRI has a potential to improve diagnosis as well as predict pathological features of the tumors and their response to therapy, and is currently widely investigated. Standard contrast-enhanced MRI of the breast provides high sensitivity but variable specificity in detecting breast cancer, and may lead to excessive benign biopsies. Large, well annotated datasets are needed to improve the current results in this field.&nbsp;<span>This dataset is a single-institutional, retrospective collection of 632 breast-MRI imaging sessions acquired on a 1.5T MR system between 2018-2021. Supporting data, collected for 200 patients from clinical notes, radiology report and pathology reports, is included: patient age, lesion location, malignant/benign outcome, pathology and grade (if applicable), receptor status and KI67 (if applicable). The main uniqueness of this data is that it contains the standard protocol series (T1-weighted DCE sequence with 5 time points- before and 4 time points after contrast injection, and T2-weighted MRI with/without fat suppression) as well as the delayed contrast T1-images acquired 20-28 minutes after contrast administration and the calculated treatment response assessment maps (TRAMs).Please see the&nbsp;</span><span><a href=\"\" url=\"https://doi.org/10.7937/C7X1-YN57\"  data-toggle=\"modal\" data-target=\"#external-web-warning\" class=\"external-link\">Advanced-MRI-Breast-Lesions  <i class=\"fa-solid fa-external-link external-link-icon\" aria-hidden=\"true\"></i></a> collection page to learn more about the images and to obtain any supporting metadata for this collection.</span></p>','2024-03-08',NULL,'Public',632,'MR, SEG','Breast Cancer','10.7937/C7X1-YN57','Clinical','','Human','Breast',1,0,'O','',1,18,'665a6902-e4ab-4f2c-8cb2-e63a18c01822','https://doi.org/10.7937/C7X1-YN57');
/*!40000 ALTER TABLE `idc_collections_collection` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_collection_data_versions`
--

DROP TABLE IF EXISTS `idc_collections_collection_data_versions`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_collection_data_versions` (
  `id` int NOT NULL AUTO_INCREMENT,
  `collection_id` int NOT NULL,
  `dataversion_id` int NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `idc_collections_collecti_collection_id_dataversio_18a9d571_uniq` (`collection_id`,`dataversion_id`),
  KEY `idc_collections_coll_dataversion_id_ac489275_fk_idc_colle` (`dataversion_id`),
  CONSTRAINT `idc_collections_coll_collection_id_172df0a7_fk_idc_colle` FOREIGN KEY (`collection_id`) REFERENCES `idc_collections_collection` (`id`),
  CONSTRAINT `idc_collections_coll_dataversion_id_ac489275_fk_idc_colle` FOREIGN KEY (`dataversion_id`) REFERENCES `idc_collections_dataversion` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_collection_data_versions`
--

LOCK TABLES `idc_collections_collection_data_versions` WRITE;
/*!40000 ALTER TABLE `idc_collections_collection_data_versions` DISABLE KEYS */;
/*!40000 ALTER TABLE `idc_collections_collection_data_versions` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_datasettype`
--

DROP TABLE IF EXISTS `idc_collections_datasettype`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_datasettype` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(128) NOT NULL,
  `data_type` varchar(1) NOT NULL,
  `set_type` varchar(1) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=4 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_datasettype`
--

LOCK TABLES `idc_collections_datasettype` WRITE;
/*!40000 ALTER TABLE `idc_collections_datasettype` DISABLE KEYS */;
INSERT INTO `idc_collections_datasettype` VALUES (1,'IDC Source Data','I','O'),(2,'Clinical, Biospecimen, and Mutation Data','A','C'),(3,'Derived Data','D','R');
/*!40000 ALTER TABLE `idc_collections_datasettype` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_datasource`
--

DROP TABLE IF EXISTS `idc_collections_datasource`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_datasource` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(128) NOT NULL,
  `count_col` varchar(128) NOT NULL,
  `source_type` varchar(1) NOT NULL,
  `aggregate_level` varchar(128) NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `idc_collections_datasource_name_source_type_9026c259_uniq` (`name`,`source_type`)
) ENGINE=InnoDB AUTO_INCREMENT=62 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_datasource`
--

LOCK TABLES `idc_collections_datasource` WRITE;
/*!40000 ALTER TABLE `idc_collections_datasource` DISABLE KEYS */;
INSERT INTO `idc_collections_datasource` VALUES (1,'dicom_derived_all','PatientID','S','SOPInstanceUID'),(2,'tcga_clin','case_barcode','S','case_barcode'),(3,'tcga_bios','case_barcode','S','sample_barcode'),(4,'idc-dev.metadata.dicom_pivot_wave0','PatientID','B','SOPInstanceUID'),(5,'bigquery-public-data.idc_v4.tcga_biospecimen_rel9','case_barcode','B','sample_barcode'),(6,'bigquery-public-data.idc_v4.tcga_clinical_rel9','case_barcode','B','case_barcode'),(7,'bigquery-public-data.idc_tcia_views_mvp_wave0.segmentations','PatientID','B','SOPInstanceUID'),(8,'bigquery-public-data.idc_tcia_views_mvp_wave0.qualitative_measurements','PatientID','B','SOPInstanceUID'),(9,'bigquery-public-data.idc_tcia_views_mvp_wave0.quantitative_measurements','PatientID','B','SOPInstanceUID'),(10,'bigquery-public-data.idc_v1.dicom_pivot_v1','PatientID','B','SOPInstanceUID'),(11,'dicom_derived_series_v2','PatientID','S','SeriesInstanceUID'),(12,'dicom_derived_study_v2','PatientID','S','StudyInstanceUID'),(13,'bigquery-public-data.idc_v2.dicom_pivot_v2','PatientID','B','SOPInstanceUID'),(14,'dicom_derived_series_v3','PatientID','S','SeriesInstanceUID'),(15,'dicom_derived_study_v3','PatientID','S','StudyInstanceUID'),(16,'bigquery-public-data.idc_v3.dicom_pivot_v3','PatientID','B','SOPInstanceUID'),(17,'dicom_derived_series_v4','PatientID','S','SeriesInstanceUID'),(18,'dicom_derived_study_v4','PatientID','S','StudyInstanceUID'),(19,'bigquery-public-data.idc_v4.dicom_pivot_v4','PatientID','B','SOPInstanceUID'),(20,'dicom_derived_series_v5','PatientID','S','SeriesInstanceUID'),(21,'dicom_derived_study_v5','PatientID','S','StudyInstanceUID'),(22,'bigquery-public-data.idc_v5.dicom_pivot_v5','PatientID','B','SOPInstanceUID'),(23,'dicom_derived_series_v6','PatientID','S','SeriesInstanceUID'),(24,'dicom_derived_study_v6','PatientID','S','StudyInstanceUID'),(25,'bigquery-public-data.idc_v6.dicom_pivot_v6','PatientID','B','SOPInstanceUID'),(26,'dicom_derived_series_v7','PatientID','S','SeriesInstanceUID'),(27,'dicom_derived_study_v7','PatientID','S','StudyInstanceUID'),(28,'bigquery-public-data.idc_v7.dicom_pivot_v7','PatientID','B','SOPInstanceUID'),(29,'dicom_derived_series_v8','PatientID','S','SeriesInstanceUID'),(30,'dicom_derived_study_v8','PatientID','S','StudyInstanceUID'),(31,'bigquery-public-data.idc_v8.dicom_pivot_v8','PatientID','B','SOPInstanceUID'),(32,'dicom_derived_series_v9','PatientID','S','SeriesInstanceUID'),(33,'dicom_derived_study_v9','PatientID','S','StudyInstanceUID'),(34,'bigquery-public-data.idc_v9.dicom_pivot_v9','PatientID','B','SOPInstanceUID'),(35,'dicom_derived_series_v10','PatientID','S','SeriesInstanceUID'),(36,'dicom_derived_study_v10','PatientID','S','StudyInstanceUID'),(37,'bigquery-public-data.idc_v10.dicom_pivot_v10','PatientID','B','SOPInstanceUID'),(38,'dicom_derived_series_v11','PatientID','S','SeriesInstanceUID'),(39,'dicom_derived_study_v11','PatientID','S','StudyInstanceUID'),(40,'bigquery-public-data.idc_v11.dicom_pivot_v11','PatientID','B','SOPInstanceUID'),(41,'dicom_derived_series_v12','PatientID','S','SeriesInstanceUID'),(42,'dicom_derived_study_v12','PatientID','S','StudyInstanceUID'),(43,'bigquery-public-data.idc_v12.dicom_pivot_v12','PatientID','B','SOPInstanceUID'),(44,'dicom_derived_series_v13','PatientID','S','SeriesInstanceUID'),(45,'dicom_derived_study_v13','PatientID','S','StudyInstanceUID'),(46,'bigquery-public-data.idc_v13.dicom_pivot_v13','PatientID','B','SOPInstanceUID'),(47,'dicom_derived_series_v14','PatientID','S','SeriesInstanceUID'),(48,'dicom_derived_study_v14','PatientID','S','StudyInstanceUID'),(49,'bigquery-public-data.idc_v14.dicom_pivot_v14','PatientID','B','SOPInstanceUID'),(50,'dicom_derived_series_v15','PatientID','S','SeriesInstanceUID'),(51,'dicom_derived_study_v15','PatientID','S','StudyInstanceUID'),(52,'bigquery-public-data.idc_v15.dicom_pivot','PatientID','B','SOPInstanceUID'),(53,'dicom_derived_series_v16','PatientID','S','SeriesInstanceUID'),(54,'dicom_derived_study_v16','PatientID','S','StudyInstanceUID'),(55,'bigquery-public-data.idc_v16.dicom_pivot','PatientID','B','SOPInstanceUID'),(56,'dicom_derived_series_v17','PatientID','S','SeriesInstanceUID'),(57,'dicom_derived_study_v17','PatientID','S','StudyInstanceUID'),(58,'idc-dev-etl.idc_v17_pub.dicom_pivot','PatientID','B','SOPInstanceUID'),(59,'dicom_derived_series_v18','PatientID','S','SeriesInstanceUID'),(60,'dicom_derived_study_v18','PatientID','S','StudyInstanceUID'),(61,'idc-dev-etl.idc_v18_pub.dicom_pivot','PatientID','B','SOPInstanceUID');
/*!40000 ALTER TABLE `idc_collections_datasource` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_datasource_data_sets`
--

DROP TABLE IF EXISTS `idc_collections_datasource_data_sets`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_datasource_data_sets` (
  `id` int NOT NULL AUTO_INCREMENT,
  `datasource_id` int NOT NULL,
  `datasettype_id` int NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `idc_collections_datasour_datasource_id_datasettyp_e8533e41_uniq` (`datasource_id`,`datasettype_id`),
  KEY `idc_collections_data_datasettype_id_8d1688e2_fk_idc_colle` (`datasettype_id`),
  CONSTRAINT `idc_collections_data_datasettype_id_8d1688e2_fk_idc_colle` FOREIGN KEY (`datasettype_id`) REFERENCES `idc_collections_datasettype` (`id`),
  CONSTRAINT `idc_collections_data_datasource_id_488d9869_fk_idc_colle` FOREIGN KEY (`datasource_id`) REFERENCES `idc_collections_datasource` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=116 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_datasource_data_sets`
--

LOCK TABLES `idc_collections_datasource_data_sets` WRITE;
/*!40000 ALTER TABLE `idc_collections_datasource_data_sets` DISABLE KEYS */;
INSERT INTO `idc_collections_datasource_data_sets` VALUES (1,1,1),(2,1,3),(3,2,2),(4,3,2),(5,4,1),(11,4,3),(6,5,2),(7,6,2),(12,10,1),(13,10,3),(14,11,1),(15,11,3),(16,12,1),(17,12,3),(18,13,1),(19,13,3),(20,14,1),(21,14,3),(22,15,1),(23,15,3),(24,16,1),(25,16,3),(26,17,1),(27,17,3),(28,18,1),(29,18,3),(30,19,1),(31,19,3),(32,20,1),(33,20,3),(34,21,1),(35,21,3),(36,22,1),(37,22,3),(38,23,1),(39,23,3),(40,24,1),(41,24,3),(42,25,1),(43,25,3),(44,26,1),(45,26,3),(46,27,1),(47,27,3),(48,28,1),(49,28,3),(50,29,1),(51,29,3),(52,30,1),(53,30,3),(54,31,1),(55,31,3),(56,32,1),(57,32,3),(58,33,1),(59,33,3),(60,34,1),(61,34,3),(62,35,1),(63,35,3),(64,36,1),(65,36,3),(66,37,1),(67,37,3),(68,38,1),(69,38,3),(70,39,1),(71,39,3),(72,40,1),(73,40,3),(74,41,1),(75,41,3),(76,42,1),(77,42,3),(78,43,1),(79,43,3),(80,44,1),(81,44,3),(82,45,1),(83,45,3),(84,46,1),(85,46,3),(86,47,1),(87,47,3),(88,48,1),(89,48,3),(90,49,1),(91,49,3),(92,50,1),(93,50,3),(94,51,1),(95,51,3),(96,52,1),(97,52,3),(98,53,1),(99,53,3),(100,54,1),(101,54,3),(102,55,1),(103,55,3),(104,56,1),(105,56,3),(106,57,1),(107,57,3),(108,58,1),(109,58,3),(110,59,1),(111,59,3),(112,60,1),(113,60,3),(114,61,1),(115,61,3);
/*!40000 ALTER TABLE `idc_collections_datasource_data_sets` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_datasource_programs`
--

DROP TABLE IF EXISTS `idc_collections_datasource_programs`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_datasource_programs` (
  `id` int NOT NULL AUTO_INCREMENT,
  `datasource_id` int NOT NULL,
  `program_id` int NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `idc_collections_datasour_datasource_id_program_id_dcf2f16a_uniq` (`datasource_id`,`program_id`),
  KEY `idc_collections_data_program_id_98e3479c_fk_idc_colle` (`program_id`),
  CONSTRAINT `idc_collections_data_datasource_id_22e6f616_fk_idc_colle` FOREIGN KEY (`datasource_id`) REFERENCES `idc_collections_datasource` (`id`),
  CONSTRAINT `idc_collections_data_program_id_98e3479c_fk_idc_colle` FOREIGN KEY (`program_id`) REFERENCES `idc_collections_program` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_datasource_programs`
--

LOCK TABLES `idc_collections_datasource_programs` WRITE;
/*!40000 ALTER TABLE `idc_collections_datasource_programs` DISABLE KEYS */;
INSERT INTO `idc_collections_datasource_programs` VALUES (1,4,1),(2,4,2),(3,4,3),(4,4,4);
/*!40000 ALTER TABLE `idc_collections_datasource_programs` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_datasource_versions`
--

DROP TABLE IF EXISTS `idc_collections_datasource_versions`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_datasource_versions` (
  `id` int NOT NULL AUTO_INCREMENT,
  `datasource_id` int NOT NULL,
  `dataversion_id` int NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `idc_collections_datasour_datasource_id_dataversio_2d31db35_uniq` (`datasource_id`,`dataversion_id`),
  KEY `idc_collections_data_dataversion_id_3e6d1aa3_fk_idc_colle` (`dataversion_id`),
  CONSTRAINT `idc_collections_data_datasource_id_acde347b_fk_idc_colle` FOREIGN KEY (`datasource_id`) REFERENCES `idc_collections_datasource` (`id`),
  CONSTRAINT `idc_collections_data_dataversion_id_3e6d1aa3_fk_idc_colle` FOREIGN KEY (`dataversion_id`) REFERENCES `idc_collections_dataversion` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=116 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_datasource_versions`
--

LOCK TABLES `idc_collections_datasource_versions` WRITE;
/*!40000 ALTER TABLE `idc_collections_datasource_versions` DISABLE KEYS */;
INSERT INTO `idc_collections_datasource_versions` VALUES (1,1,4),(2,1,5),(3,2,1),(4,3,1),(5,4,2),(11,4,3),(6,5,1),(7,6,1),(12,10,4),(13,10,5),(14,11,6),(15,11,7),(16,12,6),(17,12,7),(18,13,6),(19,13,7),(20,14,8),(21,14,9),(22,15,8),(23,15,9),(24,16,8),(25,16,9),(26,17,10),(27,17,11),(28,18,10),(29,18,11),(30,19,10),(31,19,11),(32,20,12),(33,20,13),(34,21,12),(35,21,13),(36,22,12),(37,22,13),(38,23,14),(39,23,15),(40,24,14),(41,24,15),(42,25,14),(43,25,15),(44,26,16),(45,26,17),(46,27,16),(47,27,17),(48,28,16),(49,28,17),(50,29,18),(51,29,19),(52,30,18),(53,30,19),(54,31,18),(55,31,19),(56,32,20),(57,32,21),(58,33,20),(59,33,21),(60,34,20),(61,34,21),(62,35,22),(63,35,23),(64,36,22),(65,36,23),(66,37,22),(67,37,23),(68,38,24),(69,38,25),(70,39,24),(71,39,25),(72,40,24),(73,40,25),(74,41,26),(75,41,27),(76,42,26),(77,42,27),(78,43,26),(79,43,27),(80,44,28),(81,44,29),(82,45,28),(83,45,29),(84,46,28),(85,46,29),(86,47,30),(87,47,31),(88,48,30),(89,48,31),(90,49,30),(91,49,31),(92,50,32),(93,50,33),(94,51,32),(95,51,33),(96,52,32),(97,52,33),(98,53,34),(99,53,35),(100,54,34),(101,54,35),(102,55,34),(103,55,35),(104,56,36),(105,56,37),(106,57,36),(107,57,37),(108,58,36),(109,58,37),(110,59,38),(111,59,39),(112,60,38),(113,60,39),(114,61,38),(115,61,39);
/*!40000 ALTER TABLE `idc_collections_datasource_versions` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_datasourcejoin`
--

DROP TABLE IF EXISTS `idc_collections_datasourcejoin`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_datasourcejoin` (
  `id` int NOT NULL AUTO_INCREMENT,
  `from_src_col` varchar(64) NOT NULL,
  `to_src_col` varchar(64) NOT NULL,
  `from_src_id` int NOT NULL,
  `to_src_id` int NOT NULL,
  PRIMARY KEY (`id`),
  KEY `idc_collections_data_from_src_id_1e665f79_fk_idc_colle` (`from_src_id`),
  KEY `idc_collections_data_to_src_id_aeafbe33_fk_idc_colle` (`to_src_id`),
  CONSTRAINT `idc_collections_data_from_src_id_1e665f79_fk_idc_colle` FOREIGN KEY (`from_src_id`) REFERENCES `idc_collections_datasource` (`id`),
  CONSTRAINT `idc_collections_data_to_src_id_aeafbe33_fk_idc_colle` FOREIGN KEY (`to_src_id`) REFERENCES `idc_collections_datasource` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=126 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_datasourcejoin`
--

LOCK TABLES `idc_collections_datasourcejoin` WRITE;
/*!40000 ALTER TABLE `idc_collections_datasourcejoin` DISABLE KEYS */;
INSERT INTO `idc_collections_datasourcejoin` VALUES (1,'case_barcode','case_barcode',2,3),(2,'PatientID','case_barcode',1,2),(3,'PatientID','case_barcode',1,3),(4,'SOPInstanceUID','SOPInstanceUID',4,7),(5,'SOPInstanceUID','SOPInstanceUID',4,8),(6,'SOPInstanceUID','SOPInstanceUID',4,9),(7,'SOPInstanceUID','SOPInstanceUID',7,8),(8,'SOPInstanceUID','SOPInstanceUID',7,9),(9,'SOPInstanceUID','SOPInstanceUID',8,9),(10,'case_barcode','case_barcode',6,5),(11,'PatientID','case_barcode',4,6),(12,'PatientID','case_barcode',4,5),(13,'PatientID','case_barcode',7,6),(14,'PatientID','case_barcode',7,5),(15,'PatientID','case_barcode',8,6),(16,'PatientID','case_barcode',8,5),(17,'PatientID','case_barcode',9,6),(18,'PatientID','case_barcode',9,5),(19,'PatientID','case_barcode',10,5),(20,'PatientID','case_barcode',10,6),(21,'SOPInstanceUID','SOPInstanceUID',10,7),(22,'SOPInstanceUID','SOPInstanceUID',10,8),(23,'SOPInstanceUID','SOPInstanceUID',10,9),(24,'PatientID','case_barcode',12,3),(25,'PatientID','case_barcode',12,2),(26,'PatientID','case_barcode',11,3),(27,'PatientID','case_barcode',11,2),(28,'PatientID','case_barcode',13,5),(29,'PatientID','case_barcode',13,6),(30,'PatientID','case_barcode',15,3),(31,'PatientID','case_barcode',15,2),(32,'PatientID','case_barcode',14,3),(33,'PatientID','case_barcode',14,2),(34,'PatientID','case_barcode',16,5),(35,'PatientID','case_barcode',16,6),(36,'PatientID','case_barcode',18,3),(37,'PatientID','case_barcode',18,2),(38,'PatientID','case_barcode',17,3),(39,'PatientID','case_barcode',17,2),(40,'PatientID','case_barcode',19,5),(41,'PatientID','case_barcode',19,6),(42,'PatientID','case_barcode',21,3),(43,'PatientID','case_barcode',21,2),(44,'PatientID','case_barcode',20,3),(45,'PatientID','case_barcode',20,2),(46,'PatientID','case_barcode',22,5),(47,'PatientID','case_barcode',22,6),(48,'PatientID','case_barcode',24,3),(49,'PatientID','case_barcode',24,2),(50,'PatientID','case_barcode',23,3),(51,'PatientID','case_barcode',23,2),(52,'PatientID','case_barcode',25,5),(53,'PatientID','case_barcode',25,6),(54,'PatientID','case_barcode',27,3),(55,'PatientID','case_barcode',27,2),(56,'PatientID','case_barcode',26,3),(57,'PatientID','case_barcode',26,2),(58,'PatientID','case_barcode',28,5),(59,'PatientID','case_barcode',28,6),(60,'PatientID','case_barcode',29,3),(61,'PatientID','case_barcode',29,2),(62,'PatientID','case_barcode',30,3),(63,'PatientID','case_barcode',30,2),(64,'PatientID','case_barcode',31,5),(65,'PatientID','case_barcode',31,6),(66,'PatientID','case_barcode',32,3),(67,'PatientID','case_barcode',32,2),(68,'PatientID','case_barcode',33,3),(69,'PatientID','case_barcode',33,2),(70,'PatientID','case_barcode',34,5),(71,'PatientID','case_barcode',34,6),(72,'PatientID','case_barcode',35,3),(73,'PatientID','case_barcode',35,2),(74,'PatientID','case_barcode',36,3),(75,'PatientID','case_barcode',36,2),(76,'PatientID','case_barcode',37,5),(77,'PatientID','case_barcode',37,6),(78,'PatientID','case_barcode',38,3),(79,'PatientID','case_barcode',38,2),(80,'PatientID','case_barcode',39,3),(81,'PatientID','case_barcode',39,2),(82,'PatientID','case_barcode',40,5),(83,'PatientID','case_barcode',40,6),(84,'PatientID','case_barcode',41,3),(85,'PatientID','case_barcode',41,2),(86,'PatientID','case_barcode',42,3),(87,'PatientID','case_barcode',42,2),(88,'PatientID','case_barcode',43,5),(89,'PatientID','case_barcode',43,6),(90,'PatientID','case_barcode',44,3),(91,'PatientID','case_barcode',44,2),(92,'PatientID','case_barcode',45,3),(93,'PatientID','case_barcode',45,2),(94,'PatientID','case_barcode',46,5),(95,'PatientID','case_barcode',46,6),(96,'PatientID','case_barcode',47,3),(97,'PatientID','case_barcode',47,2),(98,'PatientID','case_barcode',48,3),(99,'PatientID','case_barcode',48,2),(100,'PatientID','case_barcode',49,5),(101,'PatientID','case_barcode',49,6),(102,'PatientID','case_barcode',50,3),(103,'PatientID','case_barcode',50,2),(104,'PatientID','case_barcode',51,3),(105,'PatientID','case_barcode',51,2),(106,'PatientID','case_barcode',52,5),(107,'PatientID','case_barcode',52,6),(108,'PatientID','case_barcode',53,3),(109,'PatientID','case_barcode',53,2),(110,'PatientID','case_barcode',54,3),(111,'PatientID','case_barcode',54,2),(112,'PatientID','case_barcode',55,5),(113,'PatientID','case_barcode',55,6),(114,'PatientID','case_barcode',56,3),(115,'PatientID','case_barcode',56,2),(116,'PatientID','case_barcode',57,3),(117,'PatientID','case_barcode',57,2),(118,'PatientID','case_barcode',58,5),(119,'PatientID','case_barcode',58,6),(120,'PatientID','case_barcode',59,3),(121,'PatientID','case_barcode',59,2),(122,'PatientID','case_barcode',60,3),(123,'PatientID','case_barcode',60,2),(124,'PatientID','case_barcode',61,5),(125,'PatientID','case_barcode',61,6);
/*!40000 ALTER TABLE `idc_collections_datasourcejoin` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_dataversion`
--

DROP TABLE IF EXISTS `idc_collections_dataversion`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_dataversion` (
  `id` int NOT NULL AUTO_INCREMENT,
  `version` varchar(16) NOT NULL,
  `name` varchar(128) NOT NULL,
  `active` tinyint(1) NOT NULL,
  `current` tinyint(1) NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=40 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_dataversion`
--

LOCK TABLES `idc_collections_dataversion` WRITE;
/*!40000 ALTER TABLE `idc_collections_dataversion` DISABLE KEYS */;
INSERT INTO `idc_collections_dataversion` VALUES (1,'r9','GDC Data Release 9',1,1),(2,'0','TCIA Image Data Wave 0',0,0),(3,'0','TCIA Derived Data Wave 0',0,0),(4,'1','TCIA Image Data Wave 1',0,1),(5,'1','TCIA Derived Data Wave 1',0,1),(6,'2','TCIA Image Data Wave 2',0,1),(7,'2','TCIA Derived Data Wave 2',0,1),(8,'3','TCIA Image Data Wave 3',0,1),(9,'3','TCIA Derived Data Wave 3',0,1),(10,'4','TCIA Image Data Wave 4',0,1),(11,'4','TCIA Derived Data Wave 4',0,1),(12,'5','TCIA Image Data Version 5',0,1),(13,'5','TCIA Derived Data Version 5',0,1),(14,'6','TCIA Image Data Version 6',0,1),(15,'6','TCIA Derived Data Version 6',0,1),(16,'7','TCIA Image Data Version 7',0,1),(17,'7','TCIA Derived Data Version 7',0,1),(18,'8','TCIA Image Data Version 8',0,1),(19,'8','TCIA Derived Data Version 8',0,1),(20,'9','TCIA Image Data Version 9',0,1),(21,'9','TCIA Derived Data Version 9',0,1),(22,'10','TCIA Image Data Version 10',0,1),(23,'10','TCIA Derived Data Version 10',0,1),(24,'11','TCIA Image Data Version 11',0,1),(25,'11','TCIA Derived Data Version 11',0,1),(26,'12','TCIA Image Data Version 12',0,1),(27,'12','TCIA Derived Data Version 12',0,1),(28,'13','TCIA Image Data Version 13',0,1),(29,'13','TCIA Derived Data Version 13',0,1),(30,'14','TCIA Image Data Version 14',0,1),(31,'14','TCIA Derived Data Version 14',0,1),(32,'15','TCIA Image Data Version 15',0,1),(33,'15','TCIA Derived Data Version 15',0,1),(34,'16','TCIA Image Data Version 16',0,1),(35,'16','TCIA Derived Data Version 16',0,1),(36,'17','TCIA Image Data Version 17',0,1),(37,'17','TCIA Derived Data Version 17',0,1),(38,'18','TCIA Image Data Version 18',1,1),(39,'18','TCIA Derived Data Version 18',1,1);
/*!40000 ALTER TABLE `idc_collections_dataversion` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_dataversion_idc_versions`
--

DROP TABLE IF EXISTS `idc_collections_dataversion_idc_versions`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_dataversion_idc_versions` (
  `id` int NOT NULL AUTO_INCREMENT,
  `dataversion_id` int NOT NULL,
  `imagingdatacommonsversion_id` int NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `idc_collections_datavers_dataversion_id_imagingda_4e18a20d_uniq` (`dataversion_id`,`imagingdatacommonsversion_id`),
  KEY `idc_collections_data_imagingdatacommonsve_33b8b453_fk_idc_colle` (`imagingdatacommonsversion_id`),
  CONSTRAINT `idc_collections_data_dataversion_id_b3eb770d_fk_idc_colle` FOREIGN KEY (`dataversion_id`) REFERENCES `idc_collections_dataversion` (`id`),
  CONSTRAINT `idc_collections_data_imagingdatacommonsve_33b8b453_fk_idc_colle` FOREIGN KEY (`imagingdatacommonsversion_id`) REFERENCES `idc_collections_imagingdatacommonsversion` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=57 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_dataversion_idc_versions`
--

LOCK TABLES `idc_collections_dataversion_idc_versions` WRITE;
/*!40000 ALTER TABLE `idc_collections_dataversion_idc_versions` DISABLE KEYS */;
INSERT INTO `idc_collections_dataversion_idc_versions` VALUES (1,1,1),(8,1,2),(11,1,3),(14,1,4),(17,1,5),(20,1,6),(23,1,7),(26,1,8),(29,1,9),(32,1,10),(35,1,11),(38,1,12),(41,1,13),(44,1,14),(47,1,15),(50,1,16),(53,1,17),(56,1,18),(2,2,1),(3,3,1),(4,4,1),(5,5,1),(6,6,2),(7,7,2),(9,8,3),(10,9,3),(12,10,4),(13,11,4),(15,12,5),(16,13,5),(18,14,6),(19,15,6),(21,16,7),(22,17,7),(24,18,8),(25,19,8),(27,20,9),(28,21,9),(30,22,10),(31,23,10),(33,24,11),(34,25,11),(36,26,12),(37,27,12),(39,28,13),(40,29,13),(42,30,14),(43,31,14),(45,32,15),(46,33,15),(48,34,16),(49,35,16),(51,36,17),(52,37,17),(54,38,18),(55,39,18);
/*!40000 ALTER TABLE `idc_collections_dataversion_idc_versions` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_dataversion_programs`
--

DROP TABLE IF EXISTS `idc_collections_dataversion_programs`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_dataversion_programs` (
  `id` int NOT NULL AUTO_INCREMENT,
  `dataversion_id` int NOT NULL,
  `program_id` int NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `idc_collections_datavers_dataversion_id_program_i_5a5eb8ac_uniq` (`dataversion_id`,`program_id`),
  KEY `idc_collections_data_program_id_e16b206f_fk_idc_colle` (`program_id`),
  CONSTRAINT `idc_collections_data_dataversion_id_91f3ebcd_fk_idc_colle` FOREIGN KEY (`dataversion_id`) REFERENCES `idc_collections_dataversion` (`id`),
  CONSTRAINT `idc_collections_data_program_id_e16b206f_fk_idc_colle` FOREIGN KEY (`program_id`) REFERENCES `idc_collections_program` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_dataversion_programs`
--

LOCK TABLES `idc_collections_dataversion_programs` WRITE;
/*!40000 ALTER TABLE `idc_collections_dataversion_programs` DISABLE KEYS */;
/*!40000 ALTER TABLE `idc_collections_dataversion_programs` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_imagingdatacommonsversion`
--

DROP TABLE IF EXISTS `idc_collections_imagingdatacommonsversion`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_imagingdatacommonsversion` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` varchar(128) NOT NULL,
  `version_number` varchar(128) NOT NULL,
  `version_uid` varchar(128) DEFAULT NULL,
  `date_active` date NOT NULL,
  `active` tinyint(1) NOT NULL,
  `case_count` int NOT NULL,
  `collex_count` int NOT NULL,
  `data_volume` double NOT NULL,
  `series_count` int NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=19 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_imagingdatacommonsversion`
--

LOCK TABLES `idc_collections_imagingdatacommonsversion` WRITE;
/*!40000 ALTER TABLE `idc_collections_imagingdatacommonsversion` DISABLE KEYS */;
INSERT INTO `idc_collections_imagingdatacommonsversion` VALUES (1,'IDC Data Release','1.0',NULL,'2020-10-06',0,3652,28,1.04,51247),(2,'IDC Data Release','2.0',NULL,'2021-04-07',0,16788,112,5.66,166509),(3,'IDC Data Release','3.0',NULL,'2021-07-21',0,17174,112,6.53,168727),(4,'IDC Data Release','4.0',NULL,'2021-09-27',0,43428,113,16.7,371814),(5,'IDC Data Release','5.0',NULL,'2021-11-17',0,52042,112,21.04,420747),(6,'IDC Data Release','6.0',NULL,'2021-12-13',0,52042,112,21.04,420747),(7,'IDC Data Release','7.0',NULL,'2022-01-27',0,53544,122,20.48,426088),(8,'IDC Data Release','8.0',NULL,'2022-03-03',0,57790,107,33.6,394488),(9,'IDC Data Release','9.0',NULL,'2022-04-28',0,61134,115,36.32,412672),(10,'IDC Data Release','10.0',NULL,'2022-07-18',0,63010,121,40.25,467979),(11,'IDC Data Release','11.0',NULL,'2022-08-25',0,63271,131,40.35,470645),(12,'IDC Data Release','12.0',NULL,'2022-10-11',0,63316,128,40.37,470850),(13,'IDC Data Release','13.0',NULL,'2023-01-25',0,63316,128,44.41,478667),(14,'IDC Data Release','14.0',NULL,'2023-05-03',0,63316,128,44.41,478667),(15,'IDC Data Release','15.0',NULL,'2023-07-06',0,63788,135,45.04,482344),(16,'IDC Data Release','16.0',NULL,'2023-09-05',0,64335,137,48.42,485753),(17,'IDC Data Release','17.0',NULL,'2023-12-04',0,65066,142,51.38,511830),(18,'IDC Data Release','18.0',NULL,'2024-03-19',1,65698,143,66.38,897106);
/*!40000 ALTER TABLE `idc_collections_imagingdatacommonsversion` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_program`
--

DROP TABLE IF EXISTS `idc_collections_program`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_program` (
  `id` int NOT NULL AUTO_INCREMENT,
  `short_name` varchar(15) NOT NULL,
  `full_name` varchar(255) DEFAULT NULL,
  `description` longtext,
  `active` tinyint(1) NOT NULL,
  `is_public` tinyint(1) NOT NULL,
  `owner_id` int NOT NULL,
  `display_name` varchar(255) DEFAULT NULL,
  PRIMARY KEY (`id`),
  KEY `idc_collections_program_owner_id_03b9df99_fk_auth_user_id` (`owner_id`),
  CONSTRAINT `idc_collections_program_owner_id_03b9df99_fk_auth_user_id` FOREIGN KEY (`owner_id`) REFERENCES `auth_user` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=30 DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_program`
--

LOCK TABLES `idc_collections_program` WRITE;
/*!40000 ALTER TABLE `idc_collections_program` DISABLE KEYS */;
INSERT INTO `idc_collections_program` VALUES (1,'TCGA','The Cancer Genome Atlas',NULL,1,1,1,'TCGA'),(2,'QIN','Quantitative Imagine Network',NULL,1,1,1,'QIN'),(3,'ISPY','I-SPY Trial',NULL,0,1,1,'ISPY'),(4,'LIDC','Lung Image Database Consortium',NULL,0,1,1,'LIDC'),(5,'NSCLCR','NSCLC-Radiomics',NULL,0,1,1,'NSCLCR'),(6,'APOLLO','Applied Proteogenomics OrganizationaL Learning and Outcomes',NULL,0,1,1,'APOLLO'),(7,'RIDER','Reference Image Database to Evaluate Therapy Response',NULL,1,1,1,'RIDER'),(8,'VICTRE','Virtual Imaging Clinical Trial for Regulatory Evaluation',NULL,0,1,1,'VICTRE'),(9,'MIDRC','Medical Imaging Data Resource Center',NULL,1,1,1,'MIDRC'),(10,'ACRIN','ACR Center for Research and Innovation',NULL,0,1,1,'ACRIN'),(11,'CPTAC','Clinical Proteomic Tumor Analysis Consortium',NULL,1,1,1,'CPTAC'),(12,'PDMR','Patient-Derived Models Repository',NULL,1,1,1,'PDMR'),(13,'LCTSC','Lung CT Segmentation Challenge',NULL,0,1,1,'LCTSC'),(14,'ProstateX','SPIE-AAPM-NCI Prostate MR Classification Challenge',NULL,0,1,1,'ProstateX'),(15,'HNSCC','Head and Neck Squamous Cell Carcinoma',NULL,0,1,1,'HNSCC'),(16,'IvyGAP','Ivy Gliovlastoma Atlas Project',NULL,0,1,1,'IvyGAP'),(17,'REMBRANDT','Repository of Molecular Brain Neoplasia Data',NULL,0,1,1,'REMBRANDT'),(18,'Community','',NULL,1,1,1,'Community'),(19,'NCI_Trials','',NULL,1,1,1,'NCI Trials'),(20,'MIDI','',NULL,1,1,1,'MIDI'),(21,'FDA','',NULL,1,1,1,'FDA'),(22,'ICDC','',NULL,1,1,1,'ICDC'),(23,'NCTN','',NULL,0,1,1,'NCTN'),(24,'COVID-19','',NULL,1,1,1,'COVID-19'),(25,'QIBA','',NULL,1,1,1,'QIBA'),(26,'HTAN','Human Tumor Atlas Network',NULL,1,1,1,'HTAN'),(27,'CMB','',NULL,1,1,1,'Cancer Moonshot Biobank'),(28,'NLM','',NULL,1,1,1,'NLM'),(29,'CCDI','',NULL,1,1,1,'CCDI');
/*!40000 ALTER TABLE `idc_collections_program` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `idc_collections_project`
--

DROP TABLE IF EXISTS `idc_collections_project`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `idc_collections_project` (
  `id` int NOT NULL AUTO_INCREMENT,
  `short_name` varchar(15) NOT NULL,
  `name` varchar(255) DEFAULT NULL,
  `description` longtext,
  `active` tinyint(1) NOT NULL,
  `is_public` tinyint(1) NOT NULL,
  `owner_id` int NOT NULL,
  `program_id` int NOT NULL,
  PRIMARY KEY (`id`),
  KEY `idc_collections_project_owner_id_51390d96_fk_auth_user_id` (`owner_id`),
  KEY `idc_collections_proj_program_id_9c50d20a_fk_idc_colle` (`program_id`),
  CONSTRAINT `idc_collections_proj_program_id_9c50d20a_fk_idc_colle` FOREIGN KEY (`program_id`) REFERENCES `idc_collections_program` (`id`),
  CONSTRAINT `idc_collections_project_owner_id_51390d96_fk_auth_user_id` FOREIGN KEY (`owner_id`) REFERENCES `auth_user` (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=latin1;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `idc_collections_project`
--

LOCK TABLES `idc_collections_project` WRITE;
/*!40000 ALTER TABLE `idc_collections_project` DISABLE KEYS */;
/*!40000 ALTER TABLE `idc_collections_project` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2024-03-19 10:01:13
